export const Data = [
  {
    All: [
      {
        Tools: "sgRNAcas9",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Command-line",
        "Off‑targets": "Yes",
        "Score Oligos": "Yes",
        "Search by": "Sequence",
        Enzyme: "Custom",
        PAM: "NGG",
        Organisms: "Provided by user",
        Reference:
          "Xie S, Shen B, Zhang C, Huang X, Zhang Y sgRNAcas9: a software package for designing CRISPR sgRNA and evaluating potential off-target cleavage sites. PLoS One. 2014 Jun 23;9(6):e100448. doi: 10.1371/journal.pone.0100448. eCollection 2014. (PMID: 24956386)",
        Comments: "Cas9",
      },
      {
        Tools: "CRISPRdirect",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Web-server based",
        "Off‑targets": "Yes",
        "Score Oligos": "No",
        "Search by": "GeneID",
        Enzyme: "Custom",
        PAM: " NGG, NRG, NNGRRT, NG",
        Organisms:
          "678 entries : Mammal, vertebrate, deuterostome, insect, nematode, virus, insect, metazoa, ciliophora, plant, fungi, Bacillariophyta, Chlorobi, cyanobacteria, Other",
        Reference:
          "Naito Y, Hino K, Bono H, Ui-Tei K CRISPRdirect: software for designing CRISPR/Cas guide RNA with reduced off-target sites. Bioinformatics. 2015 Apr 1;31(7):1120-3. doi: 10.1093/bioinformatics/btu743. Epub 2014 Nov 20. (PMID: 25414360)",
        __EMPTY: "https://crispr.dbcls.jp/",
      },
      {
        "Search by": "Sequence",
      },
      {
        Tools: "CRISPR Design Tool",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Web-server based",
        "Off‑targets": "Yes",
        "Score Oligos": "No",
        "Search by": "GeneID",
        Enzyme: "Custom",
        PAM: "Custom",
        Organisms:
          "American Alligator (allMis1) Alligator mississippiensis\r\nApple (MalDomGD1) Malus domestica\r\nCat (felCat8) Felis catus\r\nChicken (galGal5) Gallus gallus\r\nChimpanzee (panTro4) Pan troglodytes\r\nChinese Hamster (criGriK1) Cricetulus griseus\r\nCotton (ASM98774v1) Gossypium hirsutum\r\nCow (bosTau8) Bos taurus\r\nDog (canFam3) Canus lupus familiaris\r\nFerret (musFur1) Mustela putorius furo\r\nFruit Fly (dm6) Drosophila melanogaster\r\nFugu (fr3) Takifugu rubripes\r\nGreen Alga (chlRei_v3) Chlamydomonas reinhardtii\r\nHorse (equCab2) Equus caballus\r\nHuman (hg38) Homo sapiens\r\nJapanese Rice (IRGSP-1.0) Oryza sativa subsp. japonica\r\nMaize (zm_B73_v4) Zea mays\r\nMarmoset (calJac3) Callithrix jacchus\r\nMedaka (oryLat2) Oryzias latipes\r\nMouse (mm10) Mus musculus\r\nNile Tilapia (oreNil2) Oreochromis niloticus\r\nOlive Baboon (papAnu2) Papio anubis\r\nOpossum (monDom5) Monodelphis domestica\r\nOrangutan (ponAbe2) Pongo abelii\r\nPig (susScr11) Sus scrofa\r\nPlatypus (ornAna2) Ornithorhynchus anatinus\r\nPurple Sea Urchin (strPur2) Strongylocentrotus purpuratus\r\nPygmy Chimpanzee (panPan1) Pan paniscus\r\nRabbit (oryCun2) Oryctolagus cuniculus\r\nRat (rn6) Rattus norvegicus\r\nRhesus (rheMac8) Macaca mulatta\r\nRoundworm (ce11) Caenorhabditis elegans\r\nSea Squirt (ci2) Ciona intestinalis\r\nSheep (oviAri3) Ovis aries\r\nSoybean (glyMax2) Glycine max\r\nStickleback (gasAcu1) Gasterosteus aculeatus\r\nTropical Clawed Frog (xenTro9) Xenopus tropicalis\r\nZebra Finch (taeGut2) Taeniopygia guttata\r\nZebrafish (danRer10) Danio rerio",
        __EMPTY:
          "https://horizondiscovery.com/en/ordering-and-calculation-tools/crispr-design-tool",
      },
      {
        "Search by": "Coordinate",
      },
      {
        "Search by": "Sequence",
      },
      {
        Tools: "CRISPRseek",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Command-line",
        "Off‑targets": "Yes",
        "Score Oligos": "Yes",
        "Search by": "Sequence",
        Enzyme: "Custom",
        PAM: "Custom",
        Organisms: "Provided by user",
        Reference:
          "Zhu LJ, Holmes BR, Aronin N, Brodsky MH CRISPRseek: a bioconductor package to identify target-specific guide RNAs for CRISPR-Cas9 genome-editing systems. PLoS One. 2014 Sep 23;9(9):e108424. doi: 10.1371/journal.pone.0108424. eCollection 2014. (PMID: 25247697)",
        Comments: "Bioconductor package. Doench (2014)-based gRNA scoring.",
        __EMPTY:
          "https://bioconductor.org/packages/release/bioc/html/CRISPRseek.html",
      },
      {
        Platform: "Other",
      },
      {
        Tools: "CCTop 1.0.1",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": "Yes",
        "Score Oligos": "Yes",
        "Search by": "Sequence",
        Enzyme:
          "NGG (Streptococcus pyogenes)\r\nNRG (Streptococcus pyogenes)\r\nNGA (variant VQR Sp.)\r\nNGCG (variant VRER Sp.)\r\nTTTN (Cpf1 Acidaminococcus or Lachnospiraceae)\r\nYTN (Cas12a Francisella novicida)\r\nTTN (Cas12a Francisella novicida)\r\nNNGRRT (Staphylococcus aureus)\r\nNNNRRT (Staphylococcus aureus)\r\nNNNNGATT (Neisseria meningitidis)\r\nNNAGAAW (Streptococcus thermophilus)\r\nNAAAAC (Treponema denticola)",
        PAM: "NGG\r\nNRG\r\nNGA\r\nNGCG\r\nTTTN\r\nNNGRRT\r\nNNNRRT\r\nNNNNGATT\r\nNNAGAAW\r\nNAAAAC\r\nNNNNRYAC",
        Organisms:
          "Anopheles gambiae (MOZ2)\r\nArabidopsis (Arabidopsisthaliana TAIR10)\r\nAspergillus flavus (JCVI-afl1-v2.0)\r\nAxolotl (Ambystoma mexicanum ASM291563v1)\r\nBacillus megaterium (Strain ATCC 14581)\r\nBacillus sp. strain RZ2MS9\r\nBarrel Medic (Medicagotruncatula Medtr4)\r\nBrassica napus (AST_PRJEB5043_v1)\r\nBroom-corn (Sorghum bicolor v3.1)\r\nCandida albicans (SC5314)\r\nCaulobacter crescentus (NA1000)\r\nC. elegans (Caenorhabditiselegans WBcel235)\r\nC. intestinalis (Cionaintestinalis KH2012)\r\nCardamine hirsuta (v1.0)\r\nCavefish (Astyanaxmexicanus AstMex102)\r\nChickpea (Cicer arietinum GCA_000331145.1)\r\nChickpea Desi (v3.0)\r\nChinese hamster (Cricetulusgriseus) ovary cell line\r\nChlamydomonas reinhardtii (v3.1)\r\nChromochloris zofingiensis (v5)\r\nClavibacter cf. michiganensis LMG 26808\r\nCow (Bos taurus UMD_3.1)\r\nEucalyptus grandis (JGI v2.0)\r\nEuropean hop (Humulus lupulus var. lupulus v1.0)\r\nFoxtail millet (Setaria italica v2.2)\r\nFruit fly (Drosophilamelanogaster BDGP6/dm6)\r\nFruit fly (Drosophilamelanogaster BDGP5/dm3)\r\nGoat (Capra aegagrus hircus ARS1)\r\nGreen foxtail (Setaria viridis v1.1)\r\nHuman (Homo sapiens GRCh38/hg38)\r\nHuman (Homo sapiens GRCh37/hg19)\r\nKluyveromyces marxianus (DMKU3-1042)\r\nLettuce (Lactuca sativa v8)\r\nMedaka(Oryzias latipes oryLat2)\r\nMouse (Mus musculus GRCm38/mm10)\r\nNannochloropsis gaditana (B-31)\r\nNannochloropsis oceanica 1779 (CCMP1779 v2.0)\r\nPaenibacillus larvae subsp. larvae DSM 25430\r\nPaenibacillus larvae subsp. larvae DSM 25719\r\nPanicum virgatum (V1.0)\r\nPantoea agglomerans (FDAARGOS_160)\r\nPapaya (Carica papaya ASGPBv0.4)\r\nPenicillium chrysogenum\r\nPetunia (Petunia axillaris)\r\nPig (Sus scrofa susScr3)\r\nPopulus alba (sPta717 v2)\r\nPopulus tremula (sPta717 v2)\r\nPopulus trichocarpa (JGI v3.1)\r\nPotato (Solanum tuberosum DM1-3 v4.04)\r\nPseudomonas aeruginosa (UCBPP-PA14)\r\nRainbow trout (Oncorhynchus mykiss Omyk_1.0)\r\nRalstonia solanacearum (GMI1000)\r\nRat (Rattus norvegicus rn6)\r\nRhesus macaque (Macacamulatta rheMac2)\r\nRice (Oryza sativa Japonica)\r\nSaccharomyces cerevisiae S288c\r\nSalmon (Salmo salar ICSASG_v2)\r\nSalt cress (Eutrema salsugineum v1.0)\r\nSoybean (Glycine max Wm82.a2.v1)\r\nSheep (Ovis aries oviAri3)\r\nSilk worm (Bombyx mori SilkDBv2.0)\r\nSpotted gar (Lepisosteus oculatus LepOcu1)\r\nStarlet sea anemone (Nematostella vectensis GCA_000209225.1)\r\nStickleback (Gasterosteusaculeatus BROADS1/gasAcu1)\r\nStreptomyces acidiscabies (NCPPB 4445)\r\nStreptomyces acidiscabies (84-104)\r\nStreptomyces mobaraensis (GCF_000342125.1)\r\nStreptomyces scabiei (87.22)\r\nStreptomyces turgidiscabies (car8)\r\nTobacco (Nicotianabenthamiana Niben101)\r\nTobacco (Nicotianatabacum Ntab-BX)\r\nTomato (Solanum lycopersicumSolyc2.5)\r\nUtricularia gibba (v4.1)\r\nX. laevis (Xenopus laevisXenla7.1)\r\nX. tropicalis (Xenopustropicalis JGI4.2/xenTro3)\r\nWater Flea (Daphnia pulex JGI 2006-09-05)\r\nWild sugarcane (Saccharum spontaneum AP85-441)\r\nYersinia enterocolitica (strain WA)\r\nZymomonas mobilis subsp. mobilis (Zm4)\r\nZebrafish (Danio rerioZv9/danRer7)\r\nZebrafish (Danio reriodanRer10)",
        Reference:
          "Stemmer M, Thumberger T, Del Sol Keyer M, Wittbrodt J, Mateo JL CCTop: An Intuitive, Flexible and Reliable CRISPR/Cas9 Target Prediction Tool. PLoS One. 2015 Apr 24;10(4):e0124633. doi: 10.1371/journal.pone.0124633. eCollection 2015. (PMID: 25909470)",
        Comments: "CCTop uses the inbuilt CRISPRater scoring algorithm",
        __EMPTY: "https://pypi.org/project/CCTop/",
      },
      {
        Platform: "Command-line",
      },
      {
        Tools: "Cas-OFFinder",
        Availability: "Yes",
        Purpose: "Finds Off-Target",
        Platform: "Web",
        "Off‑targets": "Yes",
        "Score Oligos": "No",
        "Search by": "Sequence",
        Enzyme:
          "SpCas9 from Streptococcus pyogenes: 5'-NGG-3'\r\nSpCas9 from Streptococcus pyogenes: 5'-NRG-3' (R = A or G)\r\nStCas9 from Streptococcus thermophilus: 5'-NNAGAAW-3' (W = A or T)\r\nNmCas9 from Neisseria meningitidis: 5'-NNNNGMTT-3' (M = A or C)\r\nSaCas9 from Staphylococcus aureus: 5'-NNGRRT-'3 (R=A or G)\r\nCjCas9 from Campylobacter jejuni: 5'-NNNVRYAC-3' (V = G or C or A, R = A or G, Y = C or T)\r\nCjCas9 from Campylobacter jejuni: 5'-NNNNRYAC-3' (R=A or G, Y= C or T)\r\nAsCpf1 from Acidaminococcus or LbCpf1 from Lachnospiraceae: 5'-TTTN-3'\r\nAsCpf1 from Acidaminococcus or LbCpf1 from Lachnospiraceae: 5'-TTTV-3' (V = G or C or A)\r\nSpCas9 from Streptococcus pasteurianus: 5'-NNGTGA-3'\r\nFnCpf1 from Francisella: 5'-TTN-3'\r\nSaCas9 from Staphylococcus aureus: 5'-NNNRRT-'3 (R=A or G)\r\nFnCpf1 from Francisella: 5'-KYTV-3'\r\nVRER SpCas9 from Streptococcus pyogenes: 5'-NGCG-3'\r\nVQR SpCas9 from Streptococcus pyogenes: 5'-NGA-3'\r\nXCas9 3.7 (TLIKDIV SpCas9) from Streptococcus pyogenes: 5'-NGT-3'\r\nXCas9 3.7 (TLIKDIV SpCas9) from Streptococcus pyogenes: 5'-NG-3'\r\nBhCas12b from Bacillus hisashii: 5'-TTN-3'\r\nBhCas12b v4 from Bacillus hisashii: 5'-ATTN-3'\r\nBhCas12b v4 from Bacillus hisashii: 5'-DTTN-3'\r\nSpy-macCas9 from Streptococcus pyogenes and Streptococcus macacae: 5'-NAAN-3'",
        PAM: "NGG\r\nNRG\r\nNNAGAAW\r\nNNNNGMTT\r\nNNGRRT\r\nNNNVRYAC\r\nNNNNRYAC\r\nTTTN\r\nTTTV\r\nNNGTGA\r\nTTN\r\nNNNRRT\r\nKYTV\r\nNGCG\r\nNGA\r\nNGT\r\nNG\r\nTTN\r\nATTN\r\nDTTN",
        Organisms:
          "Homo sapiens (GRCh38/hg38) - Human\r\nHomo sapiens (hg19) - Human\r\nMus musculus (mm10) - Mouse\r\nBos taurus (bosTau7) - Cow\r\nCanis familiaris (canFam3) - Dog\r\nRattus norvegicus (rn5) - Rat\r\nSus scrofa (susScr11) - Pig\r\nDanio rerio (danRer7) - Zebrafish\r\nMacaca mulatta(rheMac3) - Monkey\r\nXenopus tropicalis (JGI 4.2/xenTro3) - Western clawed frog\r\nXenopus tropicalis (JGI 7.1) - Western clawed frog\r\nXenopus laevis (JGI 7.1) - African clawed frog\r\nCricetulus griseus (v1.0) - Chinese Hamster\r\nXenopus tropicalis (JGI 8.0) - Western clawed frog\r\nXenopus tropicalis (JGI 9.0) - Western clawed frog\r\nXenopus laevis (JGI 9.1) - African clawed frog\r\nMacaca fascicularis (5.0) - Crab-eating macaque\r\nDanio rerio (GRCz10) - Zebrafish\r\nRattus norvegicus (Rnor 6.0) - Rat\r\nEquus caballus (EquCab2) - Horse\r\nOryctolagus cuniculus (OryCun2) - Rabbit\r\nMesocricetus auratus (MesAur1.0) - Golden hamster\r\nFelis catus (6.2) - Cat\r\nMochrtus ochrogaster (AHZW01) - Prairie vole\r\nBos taurus (bosTau8) - Cow\r\nCallithrix jacchus (calJac3) -marmoset\r\nPimephales promelas (SOAPdenovo v. 2) - fathead minnow\r\novis_aries (Oar_v3.1) - sheep\r\nChlorocebus_sabeus (AQIB01) - green monkey\r\nGallus gallus (Galgal4) - Chicken\r\nOreochromis niloticus (ASM185804v2)\r\nCapra hircus (ARS1) - goat\r\nPeromyscus maniculatus (Pman_1.0) - North American deer mouse\r\nMacaca mulatta (Mmul_8.0.1) - Monkey\r\nCricetulus griseus (CriGri_1.0) - Chinese hamster\r\nCavia porcellus (Cavpor3.0) -guinea pig\r\nDanio rerio (GRCz11) - zebrafish\r\nMus musculus (mm9) - Mouse\r\nAnas platyrhynchos (BGI_duck_1.0) - duck\r\nGallus gallus (Gallus_gallus-5.0) - Chicken\r\nSalmo salar (ICSASG_v2) -Atlantic salmon\r\nCyprinus carpio (common carp genome)\r\nOncorhynchus tshawytscha (Otsh_v1.0) - Chinook salmon\r\nHuman Mitochondria genome (NC_012920.1)\r\nBalbC/J mouse Mitochondria genome (AJ512208.1)\r\nBos taurus (ARS UCD1.2)\r\nParalichthys olivaceus (Flounder_ref_guided_V1.0)\r\nMyotis lucifugus (Myoluc2.0)",
        Reference:
          "Bae S, Park J, Kim JS Cas-OFFinder: a fast and versatile algorithm that searches for potential off-target sites of Cas9 RNA-guided endonucleases. Bioinformatics. 2014 May 15;30(10):1473-5. doi: 10.1093/bioinformatics/btu048. Epub 2014 Jan 24. (PMID: 24463181)",
        Comments: "Cas9, Cas12f1, Cas12e",
        __EMPTY: "http://www.rgenome.net/cas-offinder/",
      },
      {
        Platform: "Command-line",
      },
      {
        Tools: "CasFinder",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Command-line",
        "Off‑targets": "Yes",
        "Score Oligos": "No",
        "Search by": "Coordinate",
        Enzyme: "Custom",
        PAM: "Custom",
        Organisms: "Provided by user",
        Reference:
          "Aach, J., Mali, P., & Church, G. M. (2014). CasFinder: Flexible algorithm for identifying specific Cas9 targets in genomes. BioRxiv, 005074.",
        Comments:
          "SpCas9, StCas9, NmCas9 , Includes database of Cas9 target sites for human and mouse",
        __EMPTY: "https://arep.med.harvard.edu/CasFinder/",
      },
      {
        Purpose: "Database",
        "Search by": "Sequence",
      },
      {
        Tools: "CRISPR-P 2.0",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": "Yes",
        "Score Oligos": "Yes",
        "Search by": "GeneID",
        Enzyme:
          "NGG (SpCas9 from Streptococcus pyogenes: 5'-NGG-3')\r\nNRG (SpCas9 from Streptococcus pyogenes: 5'-NRG-3')\r\nNNAGAAW (StCas9 from Streptococcus thermophilus: 5'-NNAGAAW-3')\r\nNNNNGMTT (NmCas9 from Neisseria meningitidis: 5'-NNNNGMTT-3')\r\nNNGRRT (SaCas9 from Staphylococcus aureus: 5'-NNGRRT-'3)\r\nTTTV (AsCpf1 from Acidaminococcus or LbCpf1 from Lachnospiraceae: 5'-TTTV-3')\r\nTTN (FnCpf1 from Francisella: 5'-TTN-3')\r\nYCN (from Pyrococcus furiosus: 5'-YCN-3')\r\nCCW (from Clostridium difficile: 5'-CCW-3')\r\nYYC (from Bacillus halodurans: 5'-YYC-3')\r\nAWG (from Escherichia coli: 5'-AWG-3')\r\nCC (from Pseudomonas aeruginosa: 5'-CC-3')\r\nMMA (Pyrococcus furiosus: 5'-MMA-3')\r\nNG (xCas9 3.7: 5'-NG-3')",
        PAM: "NGG\r\nNRG\r\nNNAGAAW\r\nNNNNGMTT\r\nNNGRRT\r\nTTTV\r\nTTN\r\nYCN\r\nCCW\r\nYYC\r\nAWG\r\nCC\r\nMMA\r\nNG",
        Organisms:
          "Actinidia chinensis var. chinensis\r\nAnanas comosus\r\nArabidopsis lyrata (v.1.0)\r\nArabidopsis thaliana (TAIR10)\r\nArachis duranensis(v1.0)\r\nArachis ipaensis(v1.0)\r\nBetula pendula subsp. pendula(1.4c Pseudochromosomes)\r\nBrachypodium distachyon (v1.0)\r\nBrassica juncea(v1.5)\r\nBrassica napus (v4.1)\r\nBrassica oleracea (v1.0)\r\nBrassica rapa (IVFCAASv1)\r\nCapsella_rubella(v1.0 )\r\nChlamydomonas reinhardtii (v5.5)\r\nCitrullus lanatus (v1.0)\r\nCitrus sinensis (v2.0)\r\nCoffea canephora\r\nCucumis melo (v3.5)\r\nCucumis sativus\r\nCyanidioschyzon merolae (ASM9120v1)\r\nFragaria vesca(v2.0.a1)\r\nGlycine max (V1.0)\r\nGossypium hirsutum(v1.1)\r\nGossypium raimondii\r\nLactuca sativa (v8)\r\nLentinula edodes(W1-26)\r\nLentinula_edodes(B17)\r\nLotus japonicus (v3.0)\r\nMalus_x_domestica (GDDH13_v1.1)\r\nManihot esculenta (v6.1)\r\nMarchantia polymorpha(v3.1)\r\nMedicago truncatula (Mt4.0v2)\r\nMusa acuminata (MA1)\r\nNicotiana benthamiana (v0.4.4)\r\nOryza brachyantha (v1.4b)\r\nOryza glaberrima (AGI1.1)\r\nOryza indica (ASM465v1)\r\nOryza indica (MH63)\r\nOryza indica (ZS97)\r\nOryza sativa (RAP-DB)\r\nOryza sativa(MSU)\r\nPanicum virgatum(v1.1)\r\nPhyscomitrella patens (ASM242v1)\r\nPopulus trichocarpa (JGI2.0)\r\nPrunus avium (v.1.0)\r\nRice kasalath\r\nRicinus communis\r\nSalvia miltiorrhiza\r\nSelaginella moellendorffii (v1.0)\r\nSetaria italica (JGIv2.0)\r\nSetaria viridis (v1.1)\r\nSolanum lycopersicum (SL2.50) \r\nSolanum lycopersicum (SL3.0) \r\nSolanum tuberosum (3.0)\r\nSorghum bicolor (Sorbi1)\r\nUtricularia gibba\r\nVigna unguiculata (v1.1)\r\nVitis vinifera (IGGP 12x)\r\nZea mays (AGPv3.21)\r\nZea mays (AGPv4)",
        Reference:
          "Liu H, Ding Y, Zhou Y, Jin W, Xie K, Chen LL CRISPR-P 2.0: An Improved CRISPR-Cas9 Tool for Genome Editing in Plants. Mol Plant. 2017 Mar 6;10(3):530-532. doi: 10.1016/j.molp.2017.01.003. Epub 2017 Jan 13. (PMID: 28089950)",
        Comments:
          "Specific for plant genomes, SpCas9, StCas9, NmCas9, SaCas9, AsCpf1, FnCpf1, ",
        __EMPTY: "http://crispr.hzau.edu.cn/CRISPR2/",
      },
      {
        "Search by": "Coordinate",
      },
      {
        "Search by": "Sequence",
      },
      {
        Tools: "CRISPR-Local",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Command-line",
        "Off‑targets": "Yes",
        "Score Oligos": "Yes",
        "Search by": "GeneID",
        Enzyme: "Custom",
        PAM: "NGG, TTTV, TTV",
        Organisms:
          "Aegilops tauschii (ASM34733v1)\r\nAmaranthus hypochondriacus (v.PGA2_1_2212017)\r\nAmborella trichopoda (v1.0)\r\nAnanas comosus (v6)\r\nArabidopsis lyrata (v.1.0)\r\nArabidopsis thaliana (TAIR10)\r\nArachis duranensis (aradu.V14167.gnm1)\r\nArachis ipaensis (v2)\r\nAzolla filiculoides (v1.1)\r\nBeta vulgaris (RefBeet-1.2.2)\r\nBrachypodium distachyon (v1.0)\r\nBrachypodium stacei (v2)\r\nBrassica napus (AST_PRJEB5043_v1)\r\nBrassica oleracea (v2.1)\r\nBrassica rapa (IVFCAASv1)\r\nCalotropis gigantea (v1)\r\nCardamine hirsuta (v1)\r\nCatharanthus roseus (v1)\r\nChenopodium quinoa (v1.0)\r\nChlamydomonas reinhardtii (v3.1)\r\nCicer arietinum (v1.0)\r\nCitrus sinensis (v2.0)\r\nCucumis melo (v3.5)\r\nCucumis sativus\r\nCyanidioschyzon merolae (ASM9120v1)\r\nDaucus carota (v2)\r\nEucalyptus grandis (v2.0)\r\nFragaria vesca (v1.1)\r\nGlycine max (V1.0)\r\nGossypium hirsutum (v1.1)\r\nHelianthus annuus (r1.2)\r\nHordeum vulgare (Hv_IBSC_PGSB_v2)\r\nHumulus lupulus (v1.1)\r\nKalanchoe fedtschenkoi (v1.1)\r\nLactuca sativa (v8)\r\nLentinula edodes (W1-26)\r\nLotus japonicus (v3.0)\r\nMalus domestica (v1.0)\r\nManihot esculenta (v6)\r\nMarchantia polymorpha (v3.1)\r\nMedicago truncatula (MedtrA17_4.0)\r\nMusa acuminata (MA1)\r\nNicotiana benthamiana (v0.4.4)\r\nOryza brachyantha (v1.4b)\r\nOryza glaberrima (AGI1.1)\r\nOryza sativa (MSU-7.0)\r\nOryza sativa Indica (ASM465v1)\r\nOryza sativa Japonica (IRGSP-1.0)\r\nOryza sativa subsp. indica Minghui 63\r\nOryza sativa subsp. indica Zhenshan 97\r\nPanicum miliaceum (V1)\r\nPanicum virgatum (v4.1)\r\nPhaseolus vulgaris (V1)\r\nPhyscomitrella patens (ASM242v1)\r\nPopulus trichocarpa (JGI2.0)\r\nPrunus mume (v1)\r\nSalvinia cucullata (v1.2)\r\nSelaginella moellendorffii (v1.0)\r\nSetaria italica (v2.2)\r\nSetaria viridis (v2.1)\r\nSolanum lycopersicum (SL3.0)\r\nSolanum tuberosum (SolTub_3.0)\r\nSorghum bicolor (v2)\r\nTheobroma cacao (v1.1)\r\nTriticum aestivum (TGACv1)\r\nTriticum urartu (ASM34745v1)\r\nVigna angularis (vigan.Gyeongwon.gnm3)\r\nVigna unguiculata (v1.1)\r\nVitis vinifera (IGGP_12x)\r\nZea mays (AGPv3)\r\nZea mays (AGPv4)",
        Reference:
          "Sun J, Liu H, Liu J, Cheng S, Peng Y, Zhang Q, Yan J, Liu HJ, Chen LL CRISPR-Local: a local single-guide RNA (sgRNA) design tool for non-reference plant genomes. Bioinformatics. 2018 Nov 30. pii: 5221013. doi: 10.1093/bioinformatics/bty970. (PMID: 30500879)",
        Comments:
          "SpCas9, Cpf1. Useful to design candidate sgRNAs for full gene sets of plant genomes. It includes pre-calculated sgRNA databases for several genomes.",
        __EMPTY: "http://crispr.hzau.edu.cn/CRISPR-Local/",
      },
      {
        Purpose: "Database",
        Platform: "Desktop",
        "Search by": "Coordinate",
      },
      {
        "Search by": "Sequence",
      },
      {
        "Search by": "Other",
      },
      {
        Tools: "CRISPRer",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Command-line",
        "Off‑targets": "Yes",
        "Score Oligos": "No",
        "Search by": "Coordinate",
        Enzyme: "Custom",
        PAM: "Custom",
        Organisms: "Provided by user",
        Comments: "Allows installing a web server based on Galaxy",
        __EMPTY: "galaxy.informatik.uni-halle.de",
      },
      {
        Platform: "Other",
      },
      {
        Tools: "CRISPR Target",
        Availability: "Yes",
        Purpose: "Other",
        Platform: "Web",
        "Off‑targets": "No",
        "Score Oligos": "No",
        "Search by": "Sequence",
        Enzyme: "Not applicable",
        PAM: "Not Applicable",
        Organisms: "Not Applicable",
        Reference:
          "Biswas A, Gagnon JN, Brouns SJ, Fineran PC, Brown CM CRISPRTarget: bioinformatic prediction and analysis of crRNA targets. RNA Biol. 2013 May;10(5):817-27. doi: 10.4161/rna.24046. Epub 2013 Mar 14. (PMID: 23492433)",
        Comments:
          "Finds CRISPR arrays and detect Cas proteins. Tool to detect targets of CRISPR non coding RNAs. ",
        __EMPTY:
          "http://bioanalysis.otago.ac.nz/CRISPRTarget/crispr_analysis.html",
      },
      {
        Tools: "CRISPRfinder",
        Availability: "Yes",
        Purpose: "Other",
        Platform: "Web",
        "Off‑targets": "No",
        "Score Oligos": "No",
        "Search by": "Sequence",
        Enzyme: "Not applicable",
        PAM: "Not Applicable",
        Organisms: "Provided by user",
        Reference:
          "Grissa I, Vergnaud G, Pourcel C CRISPRFinder: a web tool to identify clustered regularly interspaced short palindromic repeats. Nucleic Acids Res. 2007 Jul;35(Web Server issue):W52-7. doi: 10.1093/nar/gkm360. Epub 2007 May 30. (PMID: 17537822)",
        Comments: "Finds CRISPR arrays and detect Cas proteins",
        __EMPTY: "http:// crispr.u-psud.fr/Server/CRISPRfinder.php",
      },
      {
        Tools: "CRISPRCasFinder",
        Availability: "Yes",
        Purpose: "Other",
        Platform: "Web",
        "Off‑targets": "No",
        "Score Oligos": "No",
        "Search by": "Sequence",
        Enzyme: "Not applicable",
        PAM: "Not Applicable",
        Organisms: "Provided by user",
        Reference:
          "Couvin D, Bernheim A, Toffano-Nioche C, Touchon M, Michalik J, Neron B, Rocha EPC, Vergnaud G, Gautheret D, Pourcel C CRISPRCasFinder, an update of CRISRFinder, includes a portable version, enhanced performance and integrates search for Cas proteins. Nucleic Acids Res. 2018 Jul 2;46(W1):W246-W251. doi: 10.1093/nar/gky425. (PMID: 29790974)",
        Comments:
          "An update of CRISPRFinder. Tools to detect CRISPR ad CAS genes in user-submitted sequences.",
        __EMPTY: "https://crisprcas.i2bc.paris-saclay.fr/",
      },
      {
        Platform: "Command-line",
      },
      {
        Tools: "CRISPR Optimal Target Finder",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": "Yes",
        "Score Oligos": "No",
        "Search by": "Sequence",
        Enzyme: "SpCas9",
        PAM: "NGG. NRG",
        Organisms:
          "Acyrthosiphon pisum (2)\r\nAedes aegypti (AaegL5.0 GCA_002204515.1)\r\nAnopheles gambiae (strain M) (AgamM1)\r\nAnopheles gambiae (strain S) (AgamS1)\r\nApis cerana cerana (GCA_002290385.1)\r\nApis mellifera (apiMel3)\r\nBombus impatiens (GCA_000188095.2)\r\nCaenorhabditis elegans (ce10)\r\nCeratitis capitata (1.1)\r\nDaphnia pulex (GCA_900092285.1_ASM90009228v1)\r\nDaphnia pulex (GCA_000187875.1)\r\nDrosophila ananassae (dana_r1.3_FB2011_07)\r\nDrosophila erecta (dere_r1.3_FB2011_08)\r\nDrosophila eugracilis (GCA_000236325.2)\r\nDrosophila grimshawi (dgri_r1.3_FB2010_02)\r\nDrosophila mauritiana (Dmau_MS17)\r\nDrosophila melanogaster (r_6)\r\nDrosophila melanogaster (vasa-cas9 III, BDSC_51324)\r\nDrosophila mojavensis (dmoj_r1.3_FB2011_05)\r\nDrosophila persimilis (dper_r1.3_FB2010_02)\r\nDrosophila pseudoobscura (dpse_r3.1_FB2013_03)\r\nDrosophila sechellia (droSec1)\r\nDrosophila simulans (DsimV2)\r\nDrosophila simulans (droSim1)\r\nDrosophila virilis (droVir3)\r\nDrosophila willistoni (dwil_r1.3_FB2010_02)\r\nDrosophila yakuba (droYak2)Lutzomyia longipalpis (GCA_000265325.1)\r\nManduca sexta (GCA_000262585.1)\r\nNasonia vitripennis (2.1)\r\nOncopeltus fasciatus (GCA_000696205.2)\r\nParacoccus denitrificans (GCA_000203895.1)\r\nRhodnius prolixus (GCA_000181055.3)\r\nSchistosoma mansoni (GCA_000237925.2)\r\nTribolium castaneum (Tcas 4.0 draft)",
        Reference:
          "Gratz SJ, Ukken FP, Rubinstein CD, Thiede G, Donohue LK, Cummings AM, O'Connor-Giles KM Highly specific and efficient CRISPR/Cas9-catalyzed homology-directed repair in Drosophila. Genetics. 2014 Apr;196(4):961-71. doi: 10.1534/genetics.113.160713. Epub 2014 Jan 29. (PMID: 24478335)",
        Comments: "Specific for flies, mosquitoes and worms.",
        __EMPTY: "http://targetfinder.flycrispr.neuro.brown.edu/",
      },
      {
        Tools: "CRISPR gRNA Design Tool (ATUM)",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": "Yes",
        "Score Oligos": "No",
        "Search by": "GeneID",
        Enzyme: "SpCas9 and Nickase",
        PAM: "NGG, NAG",
        Organisms:
          "Homo sapiens, Mus musculus, Arabidopsis, E.coli , Saccharomyces cerevisae",
        Comments: "Cas9 and Nickase",
        __EMPTY: "https://www.atum.bio/eCommerce/cas9/input",
      },
      {
        "Search by": "Coordinate",
      },
      {
        "Search by": "Sequence",
      },
      {
        Tools: "CRISPR Finder (HTGT WGE)",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": "Yes",
        "Score Oligos": "No",
        "Search by": "GeneID",
        Enzyme: "SpCas9",
        PAM: "NGG",
        Organisms: "Homo sapiens, Mus musculus",
        Reference:
          "Hodgkins A, Farne A, Perera S, Grego T, Parry-Smith DJ, Skarnes WC, Iyer V WGE: a CRISPR database for genome engineering. Bioinformatics. 2015 Sep 15;31(18):3078-80. doi: 10.1093/bioinformatics/btv308. Epub 2015 May 14. (PMID: 25979474)",
        __EMPTY: "https://wge.stemcell.sanger.ac.uk//find_crisprs",
      },
      {
        "Search by": "Sequence",
      },
      {
        Tools: "Off-Spotter",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": "Yes",
        "Score Oligos": "No",
        "Search by": "Sequence",
        Enzyme:
          "SpCas9 (Streptococcus pyogenes)\r\nCjCas9 (Campylobacter jejuni)\r\nSaCas9 (Staphylococcus aureus)",
        PAM: "NGG\r\nNAG\r\nNNNNACA\r\nNNGRRT",
        Organisms:
          "Homo sapiens\r\nMus musculus\r\nSaccharomyces cerevisiae (w303)",
        Reference:
          'Pliatsika V, Rigoutsos I "Off-Spotter": very fast and exhaustive enumeration of genomic lookalikes for designing CRISPR/Cas guide RNAs. Biol Direct. 2015 Jan 29;10:4. doi: 10.1186/s13062-015-0035-z. (PMID: 25630343)',
        Comments: "It provides a batch utility to remote submission",
        __EMPTY: "https://cm.jefferson.edu/Off-Spotter/",
      },
      {
        Platform: "Other",
      },
      {
        Tools: "CRISPOR",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": "Yes",
        "Score Oligos": "Yes",
        "Search by": "Sequence",
        Enzyme:
          "NGG - Sp Cas9, SpCas9-HF1, eSpCas9 1.1\r\nNNG - Cas9 S. canis\r\nNNGT - Cas9 S. canis\r\nNAA - iSpyMacCas9\r\nNNG(A/G)(A/G)T - Cas9 S. Aureus\r\nNG(G/T) - xCas9\r\nNGN or GA(A/T) - xCas9\r\nNNN(A/G)(A/G)T - KKH SaCas9\r\nNGA - Cas9 S. Pyogenes mutant VQR\r\nNNNNCC - Nme2Cas9\r\nNGCG - Cas9 S. Pyogenes mutant VRER\r\nNNAGAA - Cas9 S. Thermophilus\r\nNGGNG - Cas9 S. Thermophilus\r\nNNNNG(A/C)TT - Cas9 N. Meningitidis\r\nNNNNACA - Cas9 Campylobacter jejuni\r\nTTT(A/C/G)-23bp - Cas12a (Cpf1)  Acidaminococcus / Lachnospiraceae\r\nTTTN-23bp - Cas12a (Cpf1) Acidaminococcus / Lachnospiraceae",
        PAM: "NGG\r\nNAG\r\nNNGT\r\nNAA\r\nNNGRRT\r\nNGK\r\nNGN or GAR\r\nNNNRRT\r\nNGA\r\nNNNNCC\r\nNGCG\r\nNNAGAA\r\nNGGNG\r\nNNNNMTT\r\nNNNNACA\r\nTTTV\r\nTTN",
        Organisms:
          "Acanthamoeba castellanii str. Neff - Acanthamoeba castellanii Neff strain - NCBI GCF_000313135.1 (Acastellanii.strNEFF v1)\r\nActinobacillus pleuropneumoniae KL16 - porcine pleuropneumonia KL16 - Genbank CP022715\r\nActinobacillus pleuropneumoniae serovar 5b str. L20 - Actinobacillus pleuropneumoniae L20 - NCBI GCA_000015885.1 (ASM1588v1)\r\nAcyrthosiphon pisum - Acyrthosiphon pisum - EnsemblMetazoa 76 (GCA_000142985.2)\r\nAdineta vaga - Bdelloid Rotifers  - Genoscope V2.0\r\nAdvenella kashmirensis W13003 - Advenella kashmirensis W13003 - NCBI GCF_000506985.1 (Advkasv1.0)\r\nAedes aegypti - Yellow fever mosquito - EnsemblMetazoa 76 (AaegL3)\r\nAedes aegypti - yellow fever mosquito - NCBI GCF_002204515.2 (AaegL5.0)\r\nAedes albopictus - forest day mosquito - NCBI GCF_001876365.2 (canu_80X_arrow2.2)\r\nAgrobacterium sp. ATCC 31749 - Agrobacterium sp. ATCC 31749 - NCBI GCF_000214615.1 (Agro31749_1.0)\r\nAmphimedon queenslandica - Amphimedon queenslandica - EnsemblMetazoa 76 (Aqu1)\r\nAnaplasma phagocytophilum str. HGE1 - Anaplasma phagocytophilum str. HGE1 - NCBI GCA_000478425.1 (ASM47842v1)\r\nAnas platyrhynchos - Mallard - Ensembl 76 (BGI_duck_1.0)\r\nAnopheles arabiensis - A. arabiensis - VectorBase D1\r\nAnopheles darlingi - Anopheles darlingi - EnsemblMetazoa 76 (AdarC3)\r\nAnopheles gambiae - African malaria mosquito - EnsemblMetazoa 76 (AgamP4)\r\nApis mellifera - Apis mellifera - EnsemblMetazoa 76 (GCA_000002195.1)\r\nAquilegia caerulea - Aquilegia caerulea - Phytozome V9, Dec 2012\r\nArabidopsis lyrata - Arabidopsis lyrata - Phytozome V9, Dec 2012\r\nArabidopsis thaliana - Arabidopsis thaliana - Phytozome V9, Dec 2012\r\nArabidopsis thaliana - Thale-cress - EnsemblPlants 76 (TAIR10)\r\nArtemisia annua - sweet Annie - NCBI GCA_003112345.1 (ASM311234v1)\r\nAspergillus carbonarius ITEM 5010 - Aspergillus carbonarius ITEM 5010 - NCBI GCA_001990825.1 (Aspca3)\r\nAspergillus niger - A. niger - Phytozome 12, ATCC1015, V4.0\r\nAstyanax mexicanus - Blind cave fish - Ensembl 76 (AstMex102)\r\nAtta cephalotes - Atta cephalotes - EnsemblMetazoa 76 (GCA_000143395.2)\r\nBacillus RZ2MS9 - Bacillus sp. B14905 - Genbank NZ_AAXV01000001.1\r\nBacillus amyloliquefaciens DSM 7 - Bacillus amyloliquefaciens ATCC 23350 - NCBI GCF_000196735.1 (ASM19673v1)\r\nBacillus cereus NC7401 - Bacillus cereus NC7401 - NCBI GCF_000283675.1 (ASM28367v1)\r\nBacillus phage PfNC7401 - Bacillus phage PfNC7401 - NCBI GCA_002611025.1 (ASM261102v1)\r\nBacillus subtilis subsp. subtilis str. 168 - Bacillus subtilis 168 - NCBI GCF_000009045.1 (ASM904v1)\r\nBacillus velezensis SQR9 - Bacillus amyloliquefaciens SQR9 - NCBI GCF_000685725.1 (ASM68572v1)\r\nBactrocera tryoni - Queensland fruit fly - NCBI GCA_000695345.1 (Assembly 2.2 of Bactrocera tryoni genome)\r\nBifidobacterium longum subsp. infantis ATCC 15697 = JCM 1222 = DSM 20088 - Bifidobacterium longum bv. Infantis ATCC 15697 - NCBI GCF_000020425.1 (ASM2042v1)\r\nBiomphalaria glabrata - B. glabrata - VectorBase 1\r\nBlastomyces dermatitidis ATCC 26199 - Ajellomyces dermatitidis ATCC 26199 - NCBI GCA_000166155.1 (BD_ATCC26199_V2)\r\nBlastomyces dermatitidis ER-3 - Ajellomyces dermatitidis ER-3 - NCBI GCA_000003525.2 (BD_ER3_V1)\r\nBlastomyces gilchristii SLH14081 - Blastomyces gilchristii SLH14081 - NCBI GCA_000003855.2 (BD_SLH14081_V1)\r\nBombus terrestris - Buff-tailed bumblebee - Genbank Bter1.0\r\nBombyx mori - Bombyx mori - EnsemblMetazoa 76 (GCA_000151625.1)\r\nBos taurus - Cattle - Ensembl 76 (UMD3.1)\r\nBos taurus - domestic cow - NCBI GCF_000003205.7 (Btau_5.0.1)\r\nBos taurus - domestic cow - NCBI GCF_002263795.1 (ARS-UCD1.2)\r\nBotrylloides leachii - B. leachii - Aniseed SBv3\r\nBrachypodium distachyon - Brachypodium distachyon - Phytozome V9, Dec 2012\r\nBrachypodium distachyon - Purple false brome - Bd21-3 v1.1\r\nBrassica Napus - Rapeseed - Genoscope 4.1 Darmor-bzh\r\nBrassica oleracea - Brassica oleracea - EnsemblPlants 92 (v2.1)\r\nBrassica oleracea - wild cabbage - NCBI GCA_900416815.2 (Brassica_oleracea_HDEM)\r\nBrassica rapa - Brassica rapa - Phytozome V9, Dec 2012, Chiifu-401_v2.5\r\nBrassica rapa - field mustard - NCBI GCA_900412535.2 (Brassica_rapa_Z1)\r\nBrugia malayi - B.malayi - EnsemblMetazoa 76 (B_malayi-3.0)\r\nBurkholderia seminalis - Burkholderia seminalis - NCBI GCF_001742345.1 (ASM174234v1)\r\nC. glutamicum ATCC 13032 - Corynebacterium glutamicum ATCC 13032 - NCBI GCA_000011325.1 (ASM1132v1)\r\nCaenorhabditis brenneri - Brenneri - EnsemblMetazoa 76 (C_brenneri-6.0.1b)\r\nCaenorhabditis briggsae - Briggsae - EnsemblMetazoa 76 (CB4)\r\nCaenorhabditis elegans - C. elegans - UCSC Feb. 2013 (WBcel235/ce11)\r\nCaenorhabditis elegans - C. elegans - UCSC Oct. 2010 (WS220/ce10)\r\nCaenorhabditis japonica - Japonica - EnsemblMetazoa 76 (C_japonica-7.0.1)\r\nCaenorhabditis remanei - Remanei - EnsemblMetazoa 76 (C_remanei-15.0.1)\r\nCallithrix jacchus - white ear-tufted marmoset - NCBI GCF_000004665.1 (Callithrix jacchus-3.2)\r\nCamelina Sativa - False flax - Camelinadb.ca V2 with genes\r\nCamelina sativa - False Flax - Cs\r\nCamelina sativa - False flax - Camelinadb.ca V2\r\nCamelpox Virus - Camelpox Virus - Refseq GCF_000839105.1\r\nCandida albicans SC5314 - Candida albicans SC5314 - NCBI GCF_000182965.3 (ASM18296v3)\r\nCanis familiaris - Dog - Ensembl 79 (CanFam3.1)\r\nCannabis sativa - Marijuana, Purple Kush - canSat3, genome.ccbr.utoronto.ca\r\nCannabis sativa - marijuana - NCBI GCA_000230575.1 (canSat3)\r\nCannabis sativa - marijuana - NCBI GCA_001865755.1 (Phylos Biosc. 2016)\r\nCapitella teleta - Capitella teleta - EnsemblMetazoa 76 (GCA_000328365.1)\r\nCapra hircus - goat - RefSeq ARS1\r\nCapsella rubella - Capsella rubella - Phytozome V9, Dec 2012\r\nCapsicum annuum - Capsicum annuum - NCBI GCA_000710875.1 (Pepper Zunla 1 Ref_v1.0)\r\nCarica papaya - Carica papaya - Phytozome V9, Dec 2012\r\nCatharanthus roseus - rosy periwinkle - NCBI GCA_001292565.1 (BAC_SGD)\r\nCavia porcellus - Domestic guinea pig - Ensembl 76 (cavPor3)\r\nChlamydomonas reinhardtii - Chlamydomonas reinhardtii - NCBI v3.0\r\nChlamydomonas reinhardtii - Chlamydomonas reinhardtii - Phytozome V9, Dec 2012\r\nChlorella variabilis - Chlorella variabilis - NCBI GCA_000147415.1 (v 1.0)\r\nChlorocebus sabaeus - Green monkey - UCSC Mar. 2014 (Chlorocebus_sabeus 1.1/chlSab2)\r\nCiona Intestinalis (formerly: C. intestinalis type B) - sea vase ascidian - ANISEED MTP2012\r\nCiona robusta (formerly C. intestinalis type A) - Sea squirt ciona intestinalis - Ensembl 76 (KH2008)\r\nCiona robusta (formerly: C. intestinalis type A) - sea vase ascidian - GHOST Joined, KH2008, KH2013 Genes + SNPs: Ghost SNPs\r\nCiona robusta (formerly: C. intestinalis, type A) - sea vase ascidian - UCSC JGI 2.1, Ci2, Mar. 2005\r\nCitrus clementina - Citrus clementina - Phytozome V9, Dec 2012\r\nCitrus sinensis - Citrus sinensis - Phytozome V9, Dec 2012\r\nClytia hemisphaerica - Jellyfish C.hem. - via Tsuyoshi Momose\r\nCoccomyxa subellipsoidea C-169 - Coccomyxa subellipsoidea C-169 - Phytozome V9, Dec 2012\r\nCoprinopsis cinerea okayama7#130 - Coprinus cinereus okayama7#130 - NCBI GCA_000182895.1 (CC3)\r\nCorynebacterium glutamicum ATCC 13032 - Corynebacterium glutamicum ATCC 13032 - NCBI GCF_000011325.1 (ASM1132v1)\r\nCorynebacterium pseudotuberculosis - Bacillus pseudotuberculosis - NCBI GCF_001433475.1 (ASM143347v1)\r\nCorynebacterium pseudotuberculosis 1002 - Corynebacterium pseudotuberculosis 1002 - NCBI GCF_000144935.1 (ASM14493v2)\r\nCoturnix japonica - Japanese quail - RefSeq V2.0 GCF_001577835\r\nCrassostrea gigas - Crassostrea gigas - EnsemblMetazoa 76 (GCA_000297895.1)\r\nCricetulus griseus (CHO cells) - Chinese hamster CHO cells - NCBI GCA_900186095.1 (CHOK1S_HZDv1, Sep 2017)\r\nCricetulus griseus (CHO cells) - Chinese hamster ovary cells - UCSC Aug. 2011 (CHO K1 cell line/criGriChoV1)\r\nCricetulus griseus - Chinese hamster - UCSC Jul. 2013 (C_griseus_v1.0/criGri1)\r\nCryptosporidium parvum Iowa II - Cryptosporidium parvum strain Iowa type II - NCBI GCF_000165345.1 (ASM16534v1)\r\nCucumis melo L. - Muskmelon - Melonomics V3.5.1 Pseudo\r\nCucumis sativus - Cucumis sativus - Phytozome V9, Dec 2012\r\nCucurbita pepo subsp. pepo - field pumpkin - NCBI GCF_002806865.1 (ASM280686v2)\r\nCulex quinquefasciatus - Southern house mosquito - EnsemblMetazoa 76 (CpipJ2)\r\nDanaus plexippus - Danaus plexippus - EnsemblMetazoa 76 (DanPle_1.0)\r\nDanio rerio - Zebrafish - Ensembl 76 (Zv9)\r\nDanio rerio - Zebrafish - UCSC Jul. 2010 (Zv9/danRer7)\r\nDanio rerio - Zebrafish - UCSC May 2017 (GRCz11/danRer11)\r\nDanio rerio - Zebrafish - UCSC Sep. 2014 (GRCz10/danRer10)\r\nDanio rerio - zebra fish - NCBI GCA_000002035.4 (GRCz11)\r\nDaphnia magna - Daphnia magna - EnsemblMetazoa 95 (daphmag2.4)\r\nDaphnia magna - Daphnia magna - NCBI GCA_003990815.1 (ASM399081v1)\r\nDaphnia pulex - Daphnia pulex - EnsemblMetazoa 76 (GCA_000187875.1)\r\nDaucus carota subsp. sativus - Daucus carota var. sativus - NCBI GCF_001625215.1 (ASM162521v1)\r\nDendroctonus ponderosae - Dendroctonus ponderosae - EnsemblMetazoa 76 (GCA_000355655.1)\r\nDesulfovibrio vulgaris str. Hildenborough - Desulfovibrio vulgaris ATCC 29579 - NCBI GCA_000195755.1 (ASM19575v1)\r\nDictyostelium discoideum - Dictyostelium discoideum - Dictybase.org Apr 2017\r\nDrosophila ananassae - Drosophila ananassae - EnsemblMetazoa 76 (dana_caf1)\r\nDrosophila erecta - Drosophila erecta - EnsemblMetazoa 76 (dere_caf1)\r\nDrosophila grimshawi - Drosophila grimshawi - EnsemblMetazoa 76 (dgri_caf1)\r\nDrosophila melanogaster - D. melanogaster - UCSC Apr. 2006 (BDGP R5/dm3)\r\nDrosophila melanogaster - D. melanogaster - UCSC Aug. 2014 (BDGP Release 6 + ISO1 MT/dm6)\r\nDrosophila melanogaster - Fruit fly - EnsemblMetazoa 76 (BDGP5)\r\nDrosophila mojavensis - Drosophila mojavensis - EnsemblMetazoa 76 (dmoj_caf1)\r\nDrosophila persimilis - Drosophila persimilis - EnsemblMetazoa 76 (dper_caf1)\r\nDrosophila pseudoobscura - Drosophila pseudoobscura - EnsemblMetazoa 76 (HGSC2)\r\nDrosophila sechellia - Drosophila sechellia - EnsemblMetazoa 76 (dsec_caf1)\r\nDrosophila simulans - Drosophila simulans - EnsemblMetazoa 76 (WUGSC1)\r\nDrosophila suzukii - Spotted-Wing Drosophila - SpottedWingFlybase V1\r\nDrosophila virilis - Drosophila virilis - EnsemblMetazoa 76 (dvir_caf1)\r\nDrosophila willistoni - Drosophila willistoni - EnsemblMetazoa 76 (dwil_caf1)\r\nDrosophila yakuba - Drosophila yakuba - EnsemblMetazoa 76 (dyak_r1.3_FB2008_07)\r\nEctocarpus sp. - Ectocarpus sp. - V2 ugent.be\r\nEimeria tenella - Eimeria tenella - NCBI GCF_000499545.2 (ETH001)\r\nEquus caballus - Horse - UCSC Sep. 2007 (Broad/equCab2)\r\nEscherichia coli - E.coli - strain K-12/MG1655 (ASM584v2)\r\nEscherichia coli W - Escherichia coli ATCC 9637 - NCBI GCF_000184185.1 (ASM18418v1)\r\nEscherichia coli str. K-12 substr. DH10B - Escherichia coli str. K-12 substr. DH10B - NCBI GCF_000019425.1 (ASM1942v1)\r\nEscherichia virus Lambda - Bacteriophage lambda - NCBI GCF_000840245.1 (ViralProj14204)\r\nEucalyptus grandis - Eucalyptus grandis - Phytozome V9, Dec 2012\r\nExaiptasia pallida - Aiptasia pallida - NCBI GCA_001417965.1 (Aiptasia genome 1.1)\r\nFelis catus - Domestic cat - Ensembl 89 (Felis_catus_6.2)\r\nFonsecaea monophora - Botrytoides monophora - NCBI GCF_001642475.1 (ASM164247v1)\r\nFragaria vesca - Fragaria vesca - Phytozome V9, Dec 2012\r\nFragaria vesca subsp. vesca - wild strawberry - NCBI GCA_000184155.1 (FraVesHawaii_1.0)\r\nFragaria_vesca - wild strawberry - rosaceae.org v4.0.1a\r\nFusarium graminearum - Gibberella zeae - NCBI GCA_900044135.1 (GZPH1RResV1)\r\nGadus morhua - Atlantic cod - EnsemblVertebrates 95 (gadMor1)\r\nGallid alphaherpesvirus 1 - Infectious laryngotracheitis virus - NCBI GCA_000847005.2 (ViralProj14566)\r\nGallid herpesvirus 2 strain CVI988 - Gallid herpes 2 CVI988 - Genbank DQ530348.1\r\nGallus gallus - Chicken - UCSC Dec 2015 (Gallus_gallus-5.0/galGal5)\r\nGallus gallus - Chicken - UCSC Nov. 2011 (ICGSC Gallus_gallus-4.0/galGal4)\r\nGanoderma lucidum BCRC 37177 - Ganoderma lucidum BCRC 37177 - NCBI GCA_000338035.1 (Glu37177_1.0)\r\nGanoderma sinense ZZ0214-1 - Ganoderma sinense ZZ0214-1 - NCBI GCA_002760635.1 (GanSi1.6)\r\nGasterosteus aculeatus - Stickleback - UCSC Feb. 2006 (Broad/gasAcu1)\r\nGlossina morsitans - G. moritans - VectorBase Yale 1.6\r\nGlycine max - Glycine max - Phytozome V9, Dec 2012\r\nGoatpox virus Pellor - Goatpox virus Pellor - NCBI GCF_000840165.1 (ViralProj14197)\r\nGossypium hirsutum - American upland cotton - NCBI GCF_000987745.1 (ASM98774v1)\r\nGossypium raimondii - Gossypium raimondii - Phytozome V9, Dec 2012\r\nHBV genotype D DNA - AB554016.1 - NCBI Genbank\r\nHalocynthia aurantium - sea peach - ANISEED MTP2014\r\nHalocynthia roretzi - sea pineapple - ANISEED MTP2014\r\nHelianthus annuus - Helianthus annuus - NCBI HanXRQr1.0\r\nHeliconius melpomene - Heliconius melpomene - EnsemblMetazoa 76 (Hmel1)\r\nHelicoverpa armigera - tobacco budworm - NCBI GCA_002156985.1 (Harm_1.0)\r\nHelobdella robusta - Helobdella robusta - EnsemblMetazoa 76 (GCA_000326865.1)\r\nHeterocephalus glaber - Naked Mole Rat - NCBI HetGla_female_1.0\r\nHistoplasma capsulatum G217B - Ajellomyces capsulatus G217B - NCBI GCA_000170615.1 (ASM17061v1)\r\nHomo sapiens - Human - UCSC Dec. 2013 (GRCh38/hg38) + SNPs: dbSNP148, Kaviar\r\nHomo sapiens - Human - UCSC Feb. 2009 (GRCh37/hg19) + SNPs: 1000Genomes, ExaC\r\nHomo sapiens - Human - UCSC Mar. 2006 (NCBI36/hg18) + SNPs: HEK293\r\nHordeum vulgare - Two-rowed barley - EnsemblPlants 95 (IBSC_v2)\r\nHuman alphaherpesvirus 1 - Herpes simplex virus 1 - NCBI GCF_000859985.2 (ViralProj15217)\r\nHuman alphaherpesvirus 2 - Herpes simplex virus 2 - NCBI GCF_000858385.2 (ViralProj15218)\r\nHuman betaherpesvirus 5, Merlin strain - Human cytomegalovirus - NCBI GCF_000845245.1 (ViralProj14559)\r\nHuman betaherpesvirus 6A - Human herpesvirus 6A - NCBI GCA_000845685.2 (ViralProj14462)\r\nHuman betaherpesvirus 6B - Human herpes virus 6B - NCBI GCF_000846365.1 (ViralProj14422)\r\nHuman betaherpesvirus 7 - Herpes simplex virus 7 - NCBI GCF_000848125.1 (ViralProj14625)\r\nHuman herpesvirus-7 - HHV7 - Genbank U43400.1\r\nHydra Vulgaris - Fresh Water Poly - RP 1.0 RefSeq GCA_000004095.1\r\nHypsibius dujardini - water bear - tardigrades.org 3.15\r\nIxodes scapularis - Black-legged tick - EnsemblMetazoa 76 (IscaW1)\r\nJuglans regia - English walnut - NCBI GCF_001411555.1 (wgs.5d)\r\nKlebsiella aerogenes KCTC 2190 - Enterobacter aerogenes KCTC 2190 - NCBI GCA_000215745.1 (ASM21574v1)\r\nKlebsiella pneumoniae subsp. pneumoniae 1158 - Klebsiella pneumoniae 1158 - NCBI GCF_000814805.1 (ASM81480v1)\r\nKlebsormidium nitens - Klebsormidium nitens - NCBI GCA_000708835.1 (ASM70883v1)\r\nKluyveromyces lactis - Kluyveromyces drosophilarum - NCBI GCA_002151565.1 (ASM215156v1)\r\nKluyveromyces marxianus DMKU3-1042 - Kluyveromyces marxianus DMKU3-1042 - NCBI GCA_001417885.1 (Kmar_1.0)\r\nLactuca sativa - garden lettuce - NCBI GCF_002870075.1 (Lsat_Salinas_v7)\r\nLeishmania mexicana MHOM/GT/2001/U1103 - Leishmania mexicana MHOM/GT/2001/U1103 - NCBI GCF_000234665.1 (ASM23466v4)\r\nLeptosphaeria maculans - Blackleg of oilseed - JN3 via Colin Clairet\r\nLinum usitatissimum - Linum usitatissimum - Phytozome V9, Dec 2012\r\nLoa loa - Eye worm - EnsemblMetazoa 76 (Loa_loa_V3)\r\nLolium perenne - Perennial Rye-Grass - V1.0 L. Fuchs\r\nLottia gigantea - Lottia gigantea - EnsemblMetazoa 76 (GCA_000327385.1)\r\nLotus japonicus - Lotus japonicus - kazusa.or.jp V3\r\nLumpy skin disease KSGP 0240 - LSDV virus - Genbank KX683219.1\r\nLumpy skin disease virus NI-2490 - Lumpy skin disease virus NI-2490 - NCBI GCF_000839805.1 (ViralProj14122)\r\nLutzomyia longipalpis - Sand fly - VectorBase LionJ1\r\nLysinibacillus sphaericus C3-41 - Bacillus sphaericus C3-41 - NCBI GCA_000017965.1 (ASM1796v1)\r\nMacaca fascicularis - Crab-eating macaque - RefSeq 5.0\r\nMacaca mulatta - Rhesus - UCSC Jan. 2006 (MGSC Merged 1.0/rheMac2)\r\nMacrostomum lignano - Macrostomum lignano - Genbank GCA_001188465.1_ML2\r\nMacrostomum lignano - Macrostomum lignano - NCBI GCA_002269645.1 (Mlig_3_7)\r\nMalus domestica - Apple - INRA GDDH13 Version 1.1\r\nMalus domestica - Malus domestica - ASM211411v1\r\nMalus domestica - Malus domestica - Phytozome V9, Dec 2012\r\nManihot esculenta - Manihot esculenta - Phytozome V9, Dec 2012\r\nMarchantia polymorpha - Common liverwort - Phytozome 12, V3.1\r\nMedicago truncatula - Medicago truncatula - Phytozome V9, Dec 2012\r\nMedicago truncatula - barrel medic - NCBI GCA_002024945.1 (R108 v. 1.0)\r\nMedicago truncatula - barrel medic - medicagogenome.org V4.3\r\nMedicago truncatula - barrelclover - Medicohapmap.org R108 0.95\r\nMedicago truncatula A17 - barrelclover - INRA A17r5.0 r1.6 Pascal Gamas\r\nMegaselia scalaris - Megaselia scalaris - EnsemblMetazoa 76 (Msca1)\r\nMeleagris gallopavo - Domestic turkey - Ensembl 76 (UMD2)\r\nMesocricetus auratus - Syrian hamster - NCBI GCF_000349665.1 (MesAur1.0)\r\nMethylococcus capsulatus str. Bath - Methylococcus capsulatus ATCC 33009 - NCBI GCF_000008325.1 (ASM832v1)\r\nMicractinium conductrix - Micractinium conductrix - NCBI GCA_002245815.2 (ASM224581v2)\r\nMicrotus ochrogaster - prairie vole - NCBI GCF_000317375.1 (MicOch1.0)\r\nMimulus guttatus - Mimulus guttatus - Phytozome V9, Dec 2012\r\nMinuartia pusilla 299 - Minuartia pusilla 299 - Phytozome V9, Dec 2012\r\nMiscanthus sinensis - Chinese silver grass - Phytozome12, Msin v7.1\r\nMizuhopecten yessoensis - ezo giant scallop - NCBI GCF_002113885.1 (ASM211388v2)\r\nMoesziomyces aphidis DSM 70725 - Moesziomyces aphidis DSM 70725 - NCBI GCA_000517465.1 (PseuAph1.0)\r\nMolgula occidentalis - Sandy Sea Grape - ANISEED EL_v1.2\r\nMolgula occulta - Tail-less Sea Grape - ANISEED EL_v1.2\r\nMolgula oculata - Tailed Sea Grape - ANISEED EL_v1.2\r\nMortierella alpina ATCC 32222 - Mortierella alpina ATCC 32222 - NCBI GCA_000240685.2 (ASM24068v2)\r\nMus musculus - Mouse (reference) - UCSC Dec. 2011 (GRCm38/mm10) + SNPs: C57BL/10J, C57BR/cdJ,...\r\nMus musculus - Mouse (reference) - UCSC July 2007 (NCBI37/mm9)\r\nMus musculus - Mouse strain C57BL/6NJ - NCBI GCA_001632555.1 (C57BL_6NJ_v1)\r\nMus musculus c3hhej - Mouse - Ensembl 90 (C3H_HeJ_v1)\r\nMusa acuminata subsp. malaccensis - true apple banana - NCBI GCF_000313855.2 (ASM31385v2)\r\nMusca domestica - house fly - NCBI GCF_000371365.1 (Musca_domestica-2.0.2)\r\nMustela putorius furo - Ferret  - UCSC Apr. 2011 (MusPutFur1.0/musFur1)\r\nMycobacterium abscessus - Mycobacterium chelonae subsp. abscessus - NCBI GCF_000069185.1 (ASM6918v1)\r\nMyxoma virus - myxoma virus MYX - NCBI GCF_000843685.1 (ViralProj14396)\r\nNannochloropsis gaditana - Nannochloropsis gaditana - NCBI GCA_002838785.1 (we3730_nuc)\r\nNasonia vitripennis - Nasonia vitripennis - EnsemblMetazoa 76 (GCA_000002325.2)\r\nNeisseria meningitidis Z2491 - Neisseria meningitidis serogroup A strain Z2491 - NCBI GCA_000009105.1 (ASM910v1)\r\nNematostella vectensis - Nematostella vectensis - EnsemblMetazoa 76 (GCA_000209225.1)\r\nNeovison vison - mink - NCBI GCA_900108605.1 (NNQGG.v01)\r\nNicotiana_benthamiana - N. benthamiana - SolGenomics.net V1.01 \r\nNicotiana_tabacum - N. tabacum - SolGenomics.net V4.5 Edwards et al 2017 \r\nNocardia terpenica - Nocardia terpenica - NCBI GCF_001625105.1 (ASM162510v1)\r\nNostoc sp. PCC 7120 - Anabaena sp. PCC 7120 - NCBI GCF_000009705.1 (ASM970v1)\r\nNothobranchius furzeri - turquoise killifish - NCBI GCA_001465895.2 (Nfu_20140520)\r\nOnchocerca volvulus - O.volvulus - EnsemblMetazoa 76 (Cameroon_v3)\r\nOncorhynchus kisutch - coho salmon - NCBI GCF_002021735.1 (Okis_V1)\r\nOncorhynchus mykiss - rainbow trout - NCBI GCF_002163495.1 (Omyk_1.0)\r\nOncorhynchus tshawytscha - king salmon - NCBI GCF_002872995.1 (Otsh_v1.0)\r\nOreochromis niloticus - Nile tilapia - Ensembl 76 (Orenil1.0)\r\nOryctolagus cuniculus - Rabbit - Ensembl 76 (OryCun2.0)\r\nOryza sativa - Oryza sativa - Phytozome V9, Dec 2012\r\nOryzias latipes - Medaka - UCSC Oct. 2005 (NIG/UT MEDAKA1/oryLat2)\r\nOstreococcus lucimarinus - Ostreococcus lucimarinus - Phytozome V9, Dec 2012\r\nOstreococcus tauri - O. tauri - Genbank 140606\r\nOvis aries - Domestic sheep - Ensembl 76 (Oar_v3.1)\r\nPaenibacillus larvae subsp. larvae - Paenibacillus larvae subsp. larvae - NCBI GCF_002951875.1 (ASM295187v1)\r\nPaenibacillus larvae subsp. larvae - Paenibacillus larvae subsp. larvae - NCBI GCF_002951915.1 (ASM295191v1)\r\nPaenibacillus larvae subsp. larvae - Paenibacillus larvae subsp. larvae - NCBI GCF_002951935.1 (ASM295193v1)\r\nPaenibacillus larvae subsp. larvae DSM 25430 - Paenibacillus larvae subsp. larvae 04-309 - NCBI GCF_002951895.1 (ASM295189v1)\r\nPaenibacillus larvae subsp. larvae DSM 25719 - Paenibacillus larvae subsp. larvae 08-100 - NCBI GCF_000511115.1 (ASM51111v1)\r\nPan troglodytes - Chimp - UCSC Feb. 2011 (CSAC 2.1.4/panTro4)\r\nPanicum virgatum - Panicum virgatum - Phytozome V9, Dec 2012\r\nPantoea agglomerans - P. agg. FDAARGOS_160 - Refseq GCF_001558735.1\r\nPapio anubis - Baboon - UCSC Mar. 2012 (Baylor Panu_2.0/papAnu2)\r\nParamecium bursaria Chlorella virus 1 - Chlorella PBCV-1 virus - NCBI GCF_000847045.1 (ViralProj14564)\r\nParhyale hawaiensis - P. hawaiensis - GCA_001587735.2 but merged short contigs (PE Gallindo Torres)\r\nPassiflora edulis chloroplast - Passionfruit, chloroplast genome - Genbank KX290855.1\r\nPenaeus vannamei - white shrimp - NCBI GCF_003789085.1 (ASM378908v1)\r\nPenicillium digitatum Pd1 - Penicillium digitatum Pd1 - NCBI GCA_000315645.2 (PdigPd1_v1)\r\nPenicillium expansum - Penicillium expansum - NCBI GCA_000769755.1 (PEX1)\r\nPetromyzon marinus - Sea lamprey - Ensembl 76 (Pmarinus_7.0)\r\nPetromyzon marinus - marine lamprey - NCBI GCA_002833325.1 (Pmar_germline 1.0)\r\nPetunia axillaris - While moon petunia - Solgenomics V1.6.2\r\nPetunia inflata - Petunia inflata - Solgenomics V1\r\nPhaeodactylum tricornutum - Phaeodactylum tricornutum - EnsemblProtists 79 (ASM15095v2_bd)\r\nPhalaenopsis equestris - Phalaenopsis equestris - NCBI GCF_001263595.1 (ASM126359v1)\r\nPhallusia fumigata - P. fumigata - ANISEED - MTP 2014\r\nPhallusia mammillata - white sea squirt - ANISEED MTP 2014\r\nPhaseolus vulgaris - Phaseolus vulgaris - Phytozome V9, Dec 2012\r\nPhyscomitrella patens - A moss - EnsemblPlants 76 (ASM242v1)\r\nPhyscomitrella patens - Physcomitrella patens - Phytozome V9, Dec 2012\r\nPhyscomitrella patens - Spreading Earthmoss - Phytozome V11\r\nPimephales promelas - Fathead minnow - Genbank FHM_SOAPdenovo\r\nPinctada martensii - Akoya pearl oyster - NCBI GCA_002216045.1 (PinMar1.0)\r\nPlasmodium berghei ANKA - Plasmodium berghei ANKA - PlasmoDB 3.1\r\nPlasmodium falciparum 3D7 - Plasmodium falciparum 3D7 - PlasmoDB 3.1\r\nPlasmodium knowlesi H - Plasmodium knowlesi H - PlasmoDB 3.1\r\nPopulus trichocarpa - Populus trichocarpa - Phytozome V9, Dec 2012\r\nPristionchus pacificus - Pristionchus - EnsemblMetazoa 76 (P_pacificus-5.0)\r\nPrunus persica - Prunus persica - Phytozome V9, Dec 2012\r\nPseudocercospora fijiensis CIRAD86 - Mycosphaerella fijiensis CIRAD86 - NCBI GCF_000340215.1 (Mycfi2)\r\nPseudomonas aeruginosa PAO1 - Pseudomonas aeruginosa str. PAO1 - NCBI GCF_000006765.1 (ASM676v1)\r\nPseudomonas aeruginosa UCBPP-PA14 - P. aer. UCBPP-PA14 - MW Azam\r\nPseudomonas putida KT2440 - Pseudomonas putida (strain KT2440) - NCBI GCF_000007565.2 (ASM756v2)\r\nPseudonitzschia multistriata - P.m. diatom algae - bioinfo.szn.it, unknown version\r\nPyropia yezoensis - Susabi Nori, Marine Red Alga - NRIFS.fra.affrc..go.jp V1\r\nRamazzottius varieornatus - water bear - tardigrades.org 1.01\r\nRattus norvegicus - Norway rat - Ensembl 76 (Rnor_5.0)\r\nRattus norvegicus - Rat - UCSC Jul. 2014 (RGSC 6.0/rn6)\r\nRattus norvegicus - Rat - UCSC Mar. 2012 (RGSC 5.0/rn5)\r\nRhabditophanes sp. KR3021 - Rhabditophanes sp. KR3021 - NCBI GCA_000944355.1 (Rhabditophanes_sp_KR3021)\r\nRhodnius prolixus - Rhodnius prolixus - EnsemblMetazoa 76 (RproC1)\r\nRhodococcus opacus PD630 - Rhodococcus opacus PD630 - NCBI GCF_000599545.1 (ASM59954v1)\r\nRicinus communis - Ricinus communis - Phytozome V9, Dec 2012\r\nRosa chinensis - Rose - HapOB_Angers genome V1.0\r\nS. boulardii - Saccharomyces sp. 'boulardii' - NCBI GCA_000769245.1 (ASM76924v1)\r\nSaccharomyces cerevisiae - S. cerevisiae - UCSC Apr. 2011 (SacCer_Apr2011/sacCer3)\r\nSalmo salar - Atlantic salmon - NCBI GCF_000233375.1 (ICSASG_v2)\r\nSalmonella enterica subsp. enterica serovar Typhimurium str. 14028S - Salmonella enterica subsp. enterica serovar Typhimurium str. 14028S - NCBI GCA_000022165.1 (ASM2216v1)\r\nSchistosoma mansoni - Schistosoma mansoni - EnsemblMetazoa 76 (ASM23792v2)\r\nSchizosaccharomyces pombe - fission yeast - NCBI GCF_000002945.1 (ASM294v2)\r\nSchmidtea Mediterranea - freshwater planarian - smedgd.stowers.org 1.1\r\nSchmidtea Mediterranea - freshwater planarian - smedgd.stowers.org 4.0\r\nSchmidtea mediterranea - freshwater planarian - NCBI GCA_002600895.1 (ASM260089v1)\r\nScotinomys teguina - Alston's brown mouse - Paul Frazel NYU email\r\nSelaginella kraussiana - Selaginella kraussiana - NCBI GCA_001021135.1 (ASM102113v1)\r\nSelaginella moellendorffii - Selaginella moellendorffii - Phytozome V9, Dec 2012\r\nSerendipita indica DSM 11827 - Serendipita indica DSM 11827 - NCBI GCA_000313545.1 (ASM31354v1)\r\nSetaria italica - Setaria italica - Phytozome V9, Dec 2012\r\nSetaria viridis - Green foxtail - Phytozome 11/v1.1\r\nSheeppox virus - sheep pox virus - NCBI GCF_000840205.1 (ViralProj14196)\r\nShewanella oneidensis MR-1 - Shewanella oneidensis ATCC 700550 - NCBI GCF_000146165.2 (ASM14616v2)\r\nSolanum lycopersicum - Garden Tomato - Solgenomics 3.0\r\nSolanum lycopersicum - Solanum lycopersicum - Phytozome V9, Dec 2012\r\nSolanum lycopersicum - Tomato - Solgenomics.net SL3.0 ITAG3.0\r\nSolanum melongena - eggplant/aubergine - NCBI GCA_000787875.1 (SME_r2.5.1)\r\nSolanum tuberosum - Solanum tuberosum - Phytozome V9, Dec 2012\r\nSolenopsis invicta - Red fire ant - EnsemblMetazoa 76 (GCA_000188075.1)\r\nSorghum bicolor - Sorghum bicolor - Phytozome V9, Dec 2012\r\nSorghum bicolor - milo - NCBI GCF_000003195.3 (Sorghum_bicolor_NCBIv3)\r\nSpodoptera frugiperda - Fall Armyworm - Genouest V3.1\r\nStaphylococcus aureus - Micrococcus aureus - NCBI GCA_002085525.1 (ASM208552v1)\r\nStaphylococcus aureus subsp. aureus - Staphylococcus aureus subsp. aureus - NCBI GCF_002310395.1 (ASM231039v1)\r\nStaphylococcus aureus subsp. aureus MRSA252 - Staphylococcus aureus subsp. aureus MRSA252 - NCBI GCF_000011505.1 (ASM1150v1)\r\nStaphylococcus pseudintermedius - Staphylococcus pseudintermedius - NCBI GCF_001685665.2 (ASM168566v2)\r\nStaphylococcus pseudintermedius - Staphylococcus pseudintermedius - NCBI GCF_002847235.2 (ASM284723v2)\r\nStaphylococcus pseudintermedius ED99 - Staphylococcus pseudintermedius ED99 - NCBI GCF_000189495.1 (ASM18949v1)\r\nStreptomyces bingchenggensis BCW-1 - Streptomyces bingchenggensis BCW-1 - NCBI GCF_000092385.1 (ASM9238v1)\r\nStreptomyces bottropensis ATCC 25435 - Streptomyces bottropensis DSM 40262 - NCBI GCA_000383595.1 (ASM38359v1)\r\nStreptomyces coelicolor A3(2) - Streptomyces coelicolor A3(2) - NCBI GCF_000203835.1 (ASM20383v1)\r\nStreptomyces noursei ATCC 11455 - Streptomyces noursei ATCC 11455 - NCBI GCF_001704275.1 (ASM170427v1)\r\nStreptomyces parvulus - Streptomyces parvulus - ASM166004v1\r\nStrongylocentrotus purpuratus - Strongylocentrotus purpuratus - EnsemblMetazoa 76 (GCA_000002235.2)\r\nSus scrofa - Pig - Ensembl 76 (Sscrofa10.2)\r\nSus scrofa - Pig - NCBI GCF_000003025.6 (Sscrofa11.1)\r\nSynechococcus elongatus PCC 7942 - Anacystis nidulans R2 - NCBI GCF_000012525.1 (ASM1252v1)\r\nTaeniopygia guttata - Zebra finch - Ensembl 76 (taeGut3.2.4)\r\nTetranychus urticae - Tetranychus urticae - EnsemblMetazoa 76 (GCA_000239435.1)\r\nThellungiella halophila - Thellungiella halophila - Phytozome V9, Dec 2012\r\nTheobroma cacao - Theobroma cacao - Phytozome V9, Dec 2012\r\nThermoascus aurantiacus - soil fungus T. aurant. - JGI Theau1\r\nTribolium castaneum - Red flour beetle - EnsemblMetazoa 76 (Tcas3)\r\nTrichoderma atroviride - T. atroviride - JGI 2.0\r\nTrichoderma harzianum - Trichoderma harzianum - NCBI GCA_001990665.1 (ASM199066v1)\r\nTrichoderma harzianum - Trichoderma harzianum - NCBI GCA_002838845.1 (ASM283884v1)\r\nTrichoderma reesei - T. reesei - JGI V2 Oct 2006\r\nTrichoplusia ni - Cabbage looper - tn1 Github:weng-lab\r\nTriticum aestivum - Bread wheat - IWGSC RefSeq 1.0\r\nTriticum turgidum ssp. dicoccoides - Wild Emmer Wheat Zavitan - wheat.pw.usda.gov, Zavitan V2\r\nTrypanosoma brucei 427 - T. brucei 427 - TriTrypDb 32\r\nTrypanosoma cruzi - Chaga's disease - tritrypdb.org 4.1 CLBrenerEsmeraldo\r\nTrypanosoma cruzi - Trypanosoma cruzi - NCBI GCF_000209065.1 (ASM20906v1)\r\nUlva prolifera - Enteromorpha prolifera - NCBI GCA_004138255.1 (ASM413825v1)\r\nUrsus arctos horribilis - Grizzly bear - NCBI GCA_003584765.1 (ASM358476v1)\r\nUrsus arctos horribilis - Ursus arctos horribilis - NCBI GCF_003584765.1 (ASM358476v1)\r\nVaccinia virus - vaccinia virus VV - NCBI GCF_000860085.1 (ViralProj15241)\r\nVaccinia virus strain Lister clone VACV107 - VV - Genbank DQ121394.1\r\nVigna radiata var. radiata - green gram - NCBI GCA_000741045.2 (Vradiata_ver6)\r\nVitis vinifera - Vitis vinifera - Phytozome V9, Dec 2012\r\nVitis vinifera - wine grape - NCBI GCA_002922885.1 (Chkhaveri_assembly01)\r\nVolvox carteri - Volvox carteri - Phytozome V9, Dec 2012\r\nXanthomonas oryzae pv. oryzae PXO99A - Xanthomonas oryzae pv. oryzae PXO99A - NCBI GCF_000019585.2 (ASM1958v2)\r\nXanthophyllomyces dendrorhous - Xanthophyllomyces dendrorhous - NCBI GCA_001007165.2 (Xden1)\r\nXenopus laevis - African Clawed Frog - Xenbase 9.1\r\nXenopus laevis - X. laevis - Xenbase V7.1\r\nXenopus tropicalis - Western Clawed Frog - Xenbase 9.0\r\nXenopus tropicalis - X. tropicalis - UCSC Nov. 2009 (JGI 4.2/xenTro3)\r\nYarrowia lipolytica - Y. lipolytica - CLIB122 CR382127 Genolevures 2008\r\nZea mays - Maize - Gramene AGPv4\r\nZea mays - Zea mays - Phytozome V9, Dec 2012\r\nZymoseptoria tritici IPO323 - Mycosphaerella graminicola IPO323 - NCBI GCA_000219625.1 (MYCGR v2.0)\r\n[Nectria] haematococca mpVI 77-13-4 - [Nectria] haematococca mpVI 77-13-4 - NCBI GCF_000151355.1 (v2.0)",
        Reference:
          "Haeussler M, Schonig K, Eckert H, Eschstruth A, Mianne J, Renaud JB, Schneider-Maunoury S, Shkumatava A, Teboul L, Kent J, Joly JS, Concordet JP Evaluation of off-target and on-target scoring algorithms and integration into the guide RNA selection tool CRISPOR. Genome Biol. 2016 Jul 5;17(1):148. doi: 10.1186/s13059-016-1012-2. (PMID: 27380939)",
        __EMPTY: "http://crispor.tefor.net/",
      },
      {
        Platform: "Command-line",
        Reference:
          "Canver MC, Haeussler M, Bauer DE, Orkin SH, Sanjana NE, Shalem O, Yuan GC, Zhang F, Concordet JP, Pinello L Integrated design, execution, and analysis of arrayed and pooled CRISPR genome-editing experiments. Nat Protoc. 2018 May;13(5):946-986. doi: 10.1038/nprot.2018.005. Epub 2018 Apr 12. (PMID: 29762716)",
      },
      {
        Reference:
          "Concordet JP, Haeussler M CRISPOR: intuitive guide selection for CRISPR/Cas9 genome editing experiments and screens. Nucleic Acids Res. 2018 Jul 2;46(W1):W242-W245. doi: 10.1093/nar/gky354. (PMID: 29651054)",
      },
      {
        Tools: "E-CRISP",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": "Yes",
        "Score Oligos": "No",
        "Search by": "GeneID",
        Enzyme: "SpCas9",
        PAM: "NGG, NAG",
        Organisms:
          "Aedes aegypti AaegL3.31\r\nAnopheles darlingi AdarC3.31\r\nAnopheles gambiae AgamP4.31\r\nArabidopsis thaliana TAIR10.31\r\nAspergillus nidulans ASM1142v1.31\r\nAspergillus niger CADRE.31\r\nBos taurus UMD3.1\r\nBrachypodium distachyon v1.0\r\nCaenorhabditis elegans WBcel235\r\nCanis familiaris CanFam3.1\r\nChlamydomonas reinhardtii v3.1\r\nCiona intestinalis KH\r\nCricetulus griseus CriGri_1.0\r\nDanio rerio GRCz10\r\nDictyostelium discoideum dictybase.01\r\nDrosophila melanogaster BDGP6\r\nDrosophila pseudoobscura GCA_000001765.2\r\nEmiliania huxleyi GCA_000372725.1\r\nGallus gallus Galgal4\r\nGasterosteus aculeatus BROADS1\r\nHomo sapiens GRCh38\r\nHordeum vulgare ASM32608v1.31\r\nKomagataella pastoris GCA_000027005.1\r\nLeishmania major ASM272v2.31\r\nMacaca fascicularis\r\nMacaca mulatta MMUL_1\r\nMagnaporthe oryzae MG8.31\r\nMus musculus GRCm38\r\nNematostella vectensis GCA_000209225.1\r\nNeurospora crassa ASM18292v1.31\r\nOryctolagus cuniculus OryCun2.0\r\nOryza indica ASM465v1.31\r\nOryza nivara AWHD00000000.31\r\nOryza sativa IRGSP-1.0.31\r\nOryzias latipes MEDAKA1\r\nOvis aries Oar_v3.1\r\nPhyscomitrella patens ASM242v1.31\r\nPlasmodium falciparum ASM276v1.31\r\nPlasmodium vivax ASM241v1.31\r\nPopulus trichocarpa JGI2.0\r\nPuccinia graminis ASM14992v1.31\r\nRattus norvegicus Rnor_6.0\r\nSaccharomyces cerevisiae R64-1-1\r\nSchizosaccharomyces pombe ASM294v2.31\r\nSus scrofa Sscrofa10.2\r\nTaeniopygia guttata taeGut3.2\r\nToxoplasma gondii ToxoDB-7.1.31\r\nTrichoderma reesei GCA_000167675.2\r\nTriticum aestivum IWGSC1+popseq.31\r\nUstilago maydis UM1.31\r\nVitis vinifera IGGP_12x.31\r\nXenopus tropicalis JGI_4.2\r\nZea mays AGPv3.31\r\nZymoseptoria tritici MG2.31\r\nZymomonas mobilis ASM27775v1",
        Reference:
          "Heigwer F, Kerr G, Boutros M E-CRISP: fast CRISPR target site identification. Nat Methods. 2014 Feb;11(2):122-3. doi: 10.1038/nmeth.2812. (PMID: 24481216)",
        __EMPTY: "http://www.e-crisp.org/E-CRISP/index.html",
      },
      {
        Platform: "Other",
        "Search by": "Sequence",
      },
      {
        Tools: "CRISPR sgRNA Design Tool",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": "Yes",
        "Score Oligos": "No",
        "Search by": "Sequence",
        Enzyme: "SpCas9",
        PAM: "NGG",
        Organisms:
          "Human (hg19)\r\nchicken (galGal4)\r\nc. elegans (ce10)\r\ndog (canFam3)\r\nfly (dm3)\r\nMouse (mm9)\r\npig (susScr3)\r\npossum (monDom5)\r\nrabbit (oryCun2)\r\nrat (rn5)\r\nstickleback (gasAcu1)\r\nzebrafish (danRer7)",
        __EMPTY: "https://www.genscript.com/gRNA-design-tool.html",
      },
      {
        Tools: "CT-Finder",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": "Yes",
        "Score Oligos": "No",
        "Search by": "Sequence",
        Enzyme: "Custom",
        PAM: "Custom",
        Organisms:
          "Thale cress (Arabidopsis thaliana) genome TAIR10.27 Ensembl\r\nRice (Oryza sativa) genome IRGSP-1.0.27 Ensembl\r\nHuman (Homo sapiens) genome GRCh38.81 Ensembl\r\nMouse (Mus musculus) genome GRCm38.81 Ensembl\r\nZebrafish (Danio rerio) genome GRCz10.84 Ensembl\r\nHuman immunodeficiency virus 1 (HIV-1) genome NCBI\r\nChicken (Gallus gallus) genome Galgal4.84 Ensembl\r\nFruit fly (Drosophila melanogaster) genome BDGP6.84 Ensembl\r\nBrewer's yeast (Saccharomyces cerevisiae) genome R64-1-1.84 Ensembl\r\nDomestic cat (Felis catus) genome Felis_catus_6.2.84 Ensembl\r\nEscherichia coli (Escherichia coli) genome HUSEC2011CHR1.31 Ensembl\r\nCaenorhabditis elegans (Caenorhabditis elegans) genome WBcel235.84 Ensembl\r\nMacaque (Macaca mulatta) genome MMUL_1.84 Ensembl\r\nZea mays (Zea mays) genome AGPv3.31 Ensembl\r\nBordetella avium 197N genome ASM7046v1.38 Ensembl\r\nLactococcus lactis AI06 genome ASM76111v1.38 Ensembl\r\nToxoplasma gondii ME49 genome TGA4.39 Ensembl",
        Reference:
          "Zhu H, Misel L, Graham M, Robinson ML, Liang C CT-Finder: A Web Service for CRISPR Optimal Target Prediction and Visualization. Sci Rep. 2016 May 23;6:25516. doi: 10.1038/srep25516. (PMID: 27210050)",
        __EMPTY: "http://bioinfolab.miamioh.edu/ct-finder/",
      },
      {
        Platform: "Webapp",
      },
      {
        Tools: "CRISPR-PN",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": "Yes",
        "Score Oligos": "No",
        "Search by": "Sequence",
        Enzyme: "Custom",
        PAM: "NGG",
        Organisms:
          "Heterorhabditis bacteriophora WS236\r\nSteinernema carpocapsae PRJNA202318.WBPS11\r\nStrongyloides stercoralis PRJEB528\r\nStrongyloides ratti PRJEA62033.WS237\r\nNippostrongylus brasiliensis PRJEB511.WBPS11\r\nTrichuris muris PRJEB126.WS264\r\nNecator americanus PRJNA72135.WS242\r\nAncylostoma ceylanicum PRJNA231479.WS248\r\nAncylostoma caninum PRJNA72585.WBPS11\r\nPristionchus pacificus PRJNA12644.WS260\r\nBrugia malayi PRJNA10729.WS260\r\nLoa loa PRJNA60051.WS237\r\nHaemonchus contortus PRJEB506.WS248\r\nMeloidogyne incognita PRJEA28837.WS248",
        Reference:
          "O'Halloran DM Genome aware CRISPR gRNA target prediction for parasitic nematodes. Mol Biochem Parasitol. 2019 Jan;227:25-28. doi: 10.1016/j.molbiopara.2018.12.001. Epub 2018 Dec 7. (PMID: 30529475)",
        Comments: "CRISPR gRNA target prediction for parasitic nematodes",
        __EMPTY: "http://ohalloranlab.net/crispr-pn2/",
      },
      {
        Tools: "CRISPy-web",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": "Yes",
        "Score Oligos": "Yes",
        "Search by": "Coordinate",
        Enzyme: "SpCas9, Cas13a",
        PAM: "NGG",
        Organisms: "Provided by user",
        Reference:
          "Blin K, Pedersen LE, Weber T, Lee SY CRISPy-web: An online resource to design sgRNAs for CRISPR applications. Synth Syst Biotechnol. 2016 Feb 12;1(2):118-121. doi: 10.1016/j.synbio.2016.01.003. eCollection 2016 Jun. (PMID: 29062934)",
        __EMPTY: "https://crispy.secondarymetabolites.org/#/input",
      },
      {
        "Search by": "Other",
      },
      {
        Tools: "CRISPR-ERA",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": "Yes",
        "Score Oligos": "No",
        "Search by": "GeneID",
        Enzyme: "SpCas9 and nickase",
        PAM: "NGG\r\nNAG\r\nNRG",
        Organisms:
          "Human(GRCh37/hg19)\r\nMouse(GRCm38/mm10)\r\nRat(RGSC 5.0/rn5)\r\nZebrafish(Zv9/danRer7)\r\nD.melanogaster(BDGP R5/dm3)\r\nC.elegans(WS220/ce6)\r\nS.cerevisiae(SacCer_Apr2011/sacCer3)\r\nEscherichia coli(NC_000913)\r\nBacillus subtilis(NC_000964)",
        Reference:
          "Liu H, Wei Z, Dominguez A, Li Y, Wang X, Qi LS CRISPR-ERA: a comprehensive design tool for CRISPR-mediated gene editing, repression and activation. Bioinformatics. 2015 Nov 15;31(22):3676-8. doi: 10.1093/bioinformatics/btv423. Epub 2015 Jul 23. (PMID: 26209430)",
        Comments:
          "It provides source code for the generation of whole-genome sgRNA libraries in other species",
        __EMPTY: "http://crispr-era.stanford.edu/",
      },
      {
        Purpose: "Database",
        Platform: "Command-line",
        "Search by": "Coordinate",
      },
      {
        "Search by": "Sequence",
      },
      {
        Tools: "EuPaGDT",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": "Yes",
        "Score Oligos": "Yes",
        "Search by": "Sequence",
        Enzyme:
          "SpCas9: gRNA length 20; 3'PAM: NGG; off-target PAM: NAG, NGA\r\nSaCas9: gRNA length 21; 3'PAM: NNGRRT; off-target PAM: NNGRRA,NNGRRG,NNGRRC\r\nAsCpf1: gRNA length 20; 5'PAM: TTTN;\r\nCjCas9: gRNA length 23; 3'PAM: NNNNACA; off-target PAM: NNNNACAC,NNNNRYAC\r\nFnCas9: gRNA length 21; 3'PAM: NGG; off-target PAM: NGA,NAG,NAA\r\nFnCpf1: gRNA length 23; 5'PAM: TTN,YG; off-target PAM: CTN\r\nLbCpf1: gRNA length 20; 5'PAM: TTTN; off-target PAM: None\r\nNmCas9: gRNA length 20; 3'PAM: NNNNGANN; off-target PAM: NNNNGTTN,NNNNGNNT,NNNNGTNN,NNNNGNTN\r\nSt1Cas9: gRNA length 20; 3'PAM: NNAGAA,NNGGAA; off-target PAM: NNAGGA,NNANAA,NNGGGA\r\nTdCas9: gRNA length 20; 3'PAM: NAAAAC; off-target PAM: None",
        PAM: "NGG\r\nNNGRRT\r\nTTTN\r\nNNNNACA\r\nNNNNACAC\r\nNGG\r\nYG\r\nTTTN\r\nNNNNGANN\r\nNNAGAA\r\nNNGGAA\r\nNNAAAAC",
        Organisms:
          "A. castellanii Neff AmoebaDB-26\r\nE. dispar SAW760 AmoebaDB-26\r\nE. histolytica (Amoeba) HM1IMSS AmoebaDB v26\r\nE. histolytica (Amoeba) HM1IMSS AmoebaDB v28\r\nE. histolytica HM1IMSS-A AmoebaDB-26\r\nE. histolytica HM1IMSS-B AmoebaDB-26\r\nE. histolytica HM3IMSS AmoebaDB-26\r\nE. histolytica KU27 AmoebaDB-26\r\nE. invadens IP1 AmoebaDB-26\r\nE. moshkovskii Laredo AmoebaDB-26\r\nE. nuttalli P19 AmoebaDB-26\r\nN. fowleri ATCC30863 AmoebaDB-26\r\nC. hominis TU502 CryptoDB-26\r\nC. muris RN66 CryptoDB-26\r\nC. parvum IowaII CryptoDB-26\r\nC. parvum IowaII CryptoDB-28\r\nC. velia CCMP2878 CryptoDB-26\r\nG. niphandrodes Unknown CryptoDB-26\r\nV. brassicaformis CCMP3155 CryptoDB-26\r\nA. aculeatus ATCC16872 FungiDB-26\r\nA. capsulatus G186AR FungiDB-26\r\nA. capsulatus NAm1 FungiDB-26\r\nA. carbonarius ITEM5010 FungiDB-26\r\nA. clavatus NRRL1 FungiDB-26\r\nA. flavus NRRL3357 FungiDB-26\r\nA. fumigatus Af293 FungiDB-26\r\nA. macrogynus ATCC38327 FungiDB-26\r\nA. nidulans FGSCA4 FungiDB-26\r\nA. niger ATCC1015 FungiDB-26\r\nA. niger CBS513-88 FungiDB-26\r\nA. oryzae RIB40 FungiDB-26\r\nA. terreus NIH2624 FungiDB-26\r\nB. dendrobatidis JEL423 FungiDB-26\r\nB. fuckeliana B05-10 FungiDB-26\r\nC. albicans SC5314 FungiDB-26\r\nC. cinerea okay7-130 FungiDB-26\r\nC. gattii R265 FungiDB-26\r\nC. gattii WM276 FungiDB-26\r\nC. glabrata CBS138 FungiDB-26\r\nC. immitis H538-4 FungiDB-26\r\nC. immitis RS FungiDB-26\r\nC. neoformans B-3501A FungiDB-26\r\nC. neoformans H99 FungiDB-26\r\nC. neoformans JEC21 FungiDB-26\r\nC. parapsilosis CDC317 candidagenome.org\r\nC. posadasii C735deltSOWgp FungiDB-26\r\nC. posadasii RMSCC3488 FungiDB-26\r\nC. posadasii Silveira FungiDB-26\r\nF. graminearum PH-1 FungiDB-26\r\nF. oxysporum 4287 FungiDB-26\r\nF. verticillioides 7600 FungiDB-26\r\nH.capsulatum G217B Washington University\r\nM. circinelloides CBS277-49 FungiDB-26\r\nM. globosa CBS7966 FungiDB-26\r\nM. larici-populina 98AG31 FungiDB-26\r\nM. oryzae 70-15 FungiDB-26\r\nN.crassa OR74A FungiDB-26\r\nN. discreta FGSC8579 FungiDB-26\r\nN. fischeri NRRL181 FungiDB-26\r\nN. tetrasperma FGSC2508 FungiDB-26\r\nP. blakesleeanus NRRL1555- FungiDB-26\r\nP. chrysosporium RP-78 FungiDB-26\r\nP. graminis CRL75-36-700-3 FungiDB-26\r\nP. jirovecii SE8 FungiDB-26\r\nR. delemar RA99-880 FungiDB-26\r\nS. cerevisiae S288c FungiDB-26\r\nS. japonicus yFS275 FungiDB-26\r\nS. macrospora k-hell FungiDB-26\r\nS. octosporus yFS286 FungiDB-26\r\nS. pombe 972h- FungiDB-26\r\nS. punctatus DAOMBR117 FungiDB-26\r\nS. reilianum SRZ2 FungiDB-26\r\nS. sclerotiorum 1980UF-70 FungiDB-26\r\nT. marneffei ATCC18224 FungiDB-26\r\nT. mesenterica DSM1558 FungiDB-26\r\nT. reesei QM6a FungiDB-26\r\nT. harzianum ASM98886v1 EnsemblFungi\r\nT. harzianum GCA_002838845.1_ASM283884v1 NCBI\r\nT. stipitatus ATCC10500 FungiDB-26\r\nU. maydis 521 FungiDB-26\r\nY. lipolytica CLIB122 FungiDB-26\r\nG. intestinalis Assemblage_A GiardiaDB-26\r\nG. intestinalis Assemblage_A GiardiaDB-28\r\nG. intestinalis AssemblageADH GiardiaDB-26\r\nG. intestinalis Assemblage_B GiardiaDB-26\r\nG. intestinalisAssemblageBGS B GiardiaDB-26\r\nG. intestinalis AssemblageE GiardiaDB-26\r\nS. salmonicida ATCC50377 GiardiaDB-26\r\nA. algerae PRA109 MicrosporidiaDB-26\r\nA. algerae PRA339 MicrosporidiaDB-26\r\nE. aedis USNM41457 MicrosporidiaDB-26\r\nE. bieneusi H348 MicrosporidiaDB-26\r\nE. cuniculi1 EC1 MicrosporidiaDB-26\r\nE. cuniculi EC2 MicrosporidiaDB-26\r\nE. cuniculi EC3 MicrosporidiaDB-26\r\nE. cuniculi GBM1 MicrosporidiaDB-26\r\nE. hellem ATCC50504 MicrosporidiaDB-26\r\nE. hellem Swiss MicrosporidiaDB-26\r\nE. intestinalis ATCC50506 MicrosporidiaDB-26\r\nE. romaleae SJ2008 MicrosporidiaDB-26\r\nM. daphniae UGP3 MicrosporidiaDB-26\r\nN. bombycis CQ1 MicrosporidiaDB-26\r\nN. ceranae BRL01 MicrosporidiaDB-26\r\nN. ematocida Sp1ERTm2 MicrosporidiaDB-26\r\nN. parisii ERTm1 MicrosporidiaDB-26\r\nN. parisii ERTm3 MicrosporidiaDB-26\r\nN. sp 1ERTm6 MicrosporidiaDB-26\r\nO. colligata OC4 MicrosporidiaDB-26\r\nS. lophii42 110 MicrosporidiaDB-26\r\nT. hominis Unknown MicrosporidiaDB-26\r\nV. corneae ATCC50505 MicrosporidiaDB-26\r\nV. culicisfloridensis MicrosporidiaDB-26 \r\nA. astaci APO3 FungiDB-26\r\nA. candida 2VRR FungiDB-26\r\nA. invadans NJM9701 FungiDB-26\r\nA. laibachii Nc14 FungiDB-26\r\nH. arabidopsidis Emoy2 FungiDB-26\r\nP. aphanidermatum DAOMBR444 FungiDB-26\r\nP. arrhenomanes ATCC12531 FungiDB-26\r\nP. capsici LT1534 FungiDB-26\r\nP. cinnamomi CBS144-22 FungiDB-26\r\nP. infestans T30-4 FungiDB-26\r\nP. irregulare DAOMBR486 FungiDB-26\r\nP. iwayamai DAOMBR242034 FungiDB-26\r\nP. parasitica INRA-310 FungiDB-26\r\nP. ramorum Pr-102 FungiDB-26\r\nP. sojae P6497 FungiDB-26\r\nP. ultimum BR650 FungiDB-26\r\nP. ultimum DAOMBR144 FungiDB-26\r\nP. vexans DAOMBR484 FungiDB-26\r\nS. diclina VS20 FungiDB-26\r\nS. parasitica CBS223-65 FungiDB-26\r\nB. bigemina BOND PiroplasmaDB-26\r\nB. bovis T2Bo PiroplasmaDB-26\r\nB. microti RI PiroplasmaDB-26\r\nC. felis Winnie PiroplasmaDB-26\r\nT. annulata Ankara PiroplasmaDB-26\r\nT. equi WA PiroplasmaDB-26\r\nT. orientalis Shintoku PiroplasmaDB-26\r\nT. parva Muguga PiroplasmaDB-26\r\nP. berghei ANKA PlasmoDB-26\r\nP. berghei ANKA PlasmoDB-28\r\nP. chabaudichabaudi PlasmoDB-26 \r\nP. cynomolgi B PlasmoDB-26\r\nP. falciparum 3D7 PlasmoDB-26\r\nP. falciparum 3D7 PlasmoDB-28\r\nP. falciparum 3D7 PlasmoDB-32\r\nP. falciparum IT PlasmoDB-26\r\nP. gallinaceum 8A PlasmoDB-26\r\nP. knowlesi H PlasmoDB-26\r\nP. reichenowi CDC PlasmoDB-26\r\nP. vivax SaI1 PlasmoDB-26\r\nP. yoeliiyoelii 17X PlasmoDB-26\r\nP. yoeliiyoelii 17X PlasmoDB-28\r\nP. yoeliiyoelii 17XNL PlasmoDB-26\r\nP. yoeliiyoelii YM PlasmoDB-26\r\nP. yoeliiyoelii YM PlasmoDB-28\r\nBundle of P. falciparum P. vivax P. Malariae P. ovale P.knowlesi PlasmoDB-28\r\nE. acervulina Houghton ToxoDB-26\r\nE. brunetti Houghton ToxoDB-26\r\nE. falciformis BayerHaberkorn1970 ToxoDB-26\r\nE. maxima Weybridge ToxoDB-26\r\nE. mitis Houghton ToxoDB-26\r\nE. necatrix Houghton ToxoDB-26\r\nE. praecox Houghton ToxoDB-26\r\nE. tenella Houghton ToxoDB-26\r\nE. tenella Houghton ToxoDB-40\r\nH. hammondi HH34 ToxoDB-26\r\nN.caninum LIV ToxoDB-26\r\nS. neurona SN1 ToxoDB-26\r\nS. neurona SN3 ToxoDB-26\r\nT. gondii GT1 ToxoDB-26\r\nT. gondii GT1 ToxoDB-28\r\nT. gondii GT1 ToxoDB-32\r\nT. gondii ME49 ToxoDB-26\r\nT. gondii ME49 ToxoDB-28\r\nT. gondii ME49 ToxoDB-32\r\nT. gondii RH ToxoDB-26(limited annotation)\r\nT. gondii RH ToxoDB-28\r\nT. gondii VEG ToxoDB-26\r\nT. gondii VEG ToxoDB-28\r\nT. vaginalis TrichDB v1.3\r\nT. vaginalis G3 TrichDB-26\r\nA. deanei European Nucleotide Archive GCA_000442575.2 Dec 2014 v83.1\r\nC. fasciculata CfCl TriTrypDB-26\r\nCrithidia mellificae (L. passim) Lotmaria passim whole genome shotgun (WGS) project, accession number AHIJ01000000, last update 2015-03-11\r\nCrithidia bombi ENA acc# OESO01000000\r\nE. monterogeii LV88 TriTrypDB-26\r\nL. aethiopica L147 TriTrypDB-26\r\nL. arabica LEM1108 TriTrypDB-26\r\nL. braziliensis MHOMBR75M2903 TritrypDB-26\r\nL. braziliensis MHOMBR75M2903 TritrypDB-28\r\nL. braziliensis MHOMBR75M2904 TriTrypDB-26\r\nL. donovani BPK282A1 TritrypDB-26\r\nL. donovani BPK282A1 TritrypDB-28\r\nL. enriettii LEM3045 TriTrypDB-26\r\nL. gerbilli LEM452 TriTrypDB-26\r\nL. infantum JPCM5 TritrypDB-26\r\nL. infantum JPCM5 TritrypDB-28\r\nL. major Friedlin TritrypDB-26\r\nL. major Friedlin TritrypDB-28\r\nL. major LV39c5 TriTrypDB-26\r\nL. major LV39c5 TriTrypDB-28\r\nL. major SD75.1 TriTrypDB-26\r\nL. major SD75.1 TriTrypDB-28\r\nL. mexicana MHOMGT2001U1103 TritrypDB-26\r\nL. mexicana MHOMGT2001U1103 TritrypDB-28\r\nL. panamensis MHOMCOL81L13 TriTrypDB-26\r\nL. pyrrhocoris H10 TriTrypDB-26\r\nL. seymouri ATCC30220 TriTrypDB-26\r\nL. seymouri ATCC30220 TriTrypDB-28\r\nL. sp MARLEM2494 TriTrypDB-26\r\nL. tarentolae ParrotTarII TriTrypDB-26\r\nL. tropica L590 TriTrypDB-26\r\nL. turanica LEM423 TriTrypDB-26\r\nT. brucei Lister427 TriTrypDB-26\r\nT. brucei Lister427 TriTrypDB-28\r\nT. brucei TREU927 TritrypDB-28\r\nT. bruceigambiense DAL972 TriTrypDB-26\r\nT. congolense IL3000 TriTrypDB-26\r\nT. cruzi CLBrener Non-Esmeraldo-like TriTrypDB-28\r\nT. cruzi CLBrener Esmeraldo-like TriTrypDB-28\r\nT. cruzi CLBrener TritrypDB-28 (hybrid diploid)\r\nT. cruzi CLBrener TritrypDB-29 (hybrid diploid, including unassigned contigs)\r\nT. cruzi CLBrener TritrypDB-32 (hybrid diploid)\r\nT. cruzi Brazil pacbio CANU assembly\r\nT. cruzi Brazil pacbio SMRT assembly\r\nT. cruzi Brazil pacbio assembly\r\nT. cruzi Dm28c TritrypDB-26\r\nT. cruzi Dm28c TritrypDB-28\r\nT. cruzi SylvioX10-1 TritrypDB-26\r\nT. cruzi SylvioX10-1 TritrypDB-28\r\nT. cruzi marinkellei B7 TriTrypDB-26\r\nT. cruzi Tulauhein Biomedical Research Institute, AIST\r\nT. evansi STIB805 TriTrypDB-26\r\nT. grayi ANR4 TriTrypDB-26\r\nT. rangeli SC58 TriTrypDB-26\r\nT. vivax Y486 TriTrypDB-26\r\nParamecium tetraurelia Stock d4-2 strain (repeat masked)\r\nBiomphalaria glabrata BB02 strain released 2013.4 (repeat masked)\r\nCanis_familiaris v3.1 (repeat masked)\r\nD. melanogaster r6.04 FB2015_01\r\nD. melanogaster r6.19 FB2017_06\r\nA.gambiae AgamP4\r\nEnterobacteria phage T4\r\nMus musculus BALB/c \r\nMus musculus C57BL/6J \r\nHomo Sapiens hg38, GRCh38 Genome Reference Consortium Human Reference 38 (GCA_000001405.2) \r\nTetradesmus obliquus DOE0152z \r\nMycobacterium bovis BCG 1",
        Reference:
          "Peng D, Tarleton R EuPaGDT: a web tool tailored to design CRISPR guide RNAs for eukaryotic pathogens. Microb Genom. 2015 Oct 30;1(4):e000033. doi: 10.1099/mgen.0.000033. eCollection 2015 Oct. (PMID: 28348817)",
        Comments:
          "Mostly for eukaryotic pathogens. User can provide its own file with a non-listed organism (in FASTA format)",
        __EMPTY: "http://grna.ctegd.uga.edu/",
      },
      {
        Tools: "CROP-IT",
        Availability: "No",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": "Yes",
        "Score Oligos": "Yes",
        "Search by": "Coordinate",
        Enzyme: "SpCas9",
        PAM: "NGG\r\nNNG",
        Organisms: "Homo sapiens, Mus musculus",
        Reference:
          "Singh R, Kuscu C, Quinlan A, Qi Y, Adli M Cas9-chromatin binding information enables more accurate CRISPR off-target prediction. Nucleic Acids Res. 2015 Oct 15;43(18):e118. doi: 10.1093/nar/gkv575. Epub 2015 Jun 1. (PMID: 26032770)",
        Comments:
          "sgRNA design and off-target prediction are splitted in two separated tools. All results are provided only via e-mail.",
        __EMPTY:
          "http://cheetah.bioch.virginia.edu/AdliLab/CROP-IT/homepage.html",
      },
      {
        Purpose: "Other",
        "Search by": "Sequence",
      },
      {
        Tools: "Cas-Designer",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": "Yes",
        "Score Oligos": "Yes",
        "Search by": "Sequence",
        Enzyme:
          "SpCas9 from Streptococcus pyogenes: 5'-NGG-3'\r\nSpCas9 from Streptococcus pyogenes: 5'-NGG-3' for target, 5'-NRG-3' for off-target (R = A or G)\r\nStCas9 from Streptococcus thermophilus: 5'-NNAGAAW-3' (W = A or T)\r\nNmCas9 from Neisseria meningitidis: 5'-NNNNGMTT-3' (M = A or C)\r\nSaCas9 from Staphylococcus aureus: 5'-NNGRRT-'3 (R=A or G)\r\nCjCas9 from Campylobacter jejuni: 5'-NNNVRYAC-3' (V = G or C or A, R = A or G, Y = C or T)\r\nCjCas9 from Campylobacter jejuni: 5'-NNNNRYAC-3' (R = A or G, Y = C or T)\r\nAsCpf1 from Acidaminococcus or LbCpf1 from Lachnospiraceae: 5'-TTTN-3'\r\nAsCpf1 from Acidaminococcus or LbCpf1 from Lachnospiraceae: 5'-TTTV-3' (V = G or C or A)\r\nSpCas9 from Streptococcus pasteurianus: 5'-NNGTGA-3'\r\nFnCpf1 from Francisella: 5'-TTN-3'\r\nSaCas9 from Staphylococcus aureus: 5'-NNNRRT-'3 (R=A or G)\r\nFnCpf1 from Francisella: 5'-KYTV-3'\r\nVRER SpCas9 from Streptococcus pyogenes: 5'-NGCG-3'\r\nVQR SpCas9 from Streptococcus pyogenes: 5'-NGA-3'\r\nxCas9 3.7 (TLIKDIV SpCas9) from Streptococcus pyogenes: 5'-NGT-3'\r\nBhCas12b from Bacillus hisashii: 5'-TTN-3'\r\nBhCas12b v4 from Bacillus hisashii: 5'-ATTN-3'\r\nBhCas12b v4 from Bacillus hisashii: 5'-DTTN-3'\r\nSpy-macCas9 from Streptococcus pyogenes and Streptococcus macacae: 5'-NAAN-3'",
        PAM: "NGG\r\nNRG\r\nNNAGAAW\r\nNNNNGMTT\r\nNNGRRT\r\nNNNVRYAC\r\nNNNNRYAC\r\nTTTN\r\nTTTV\r\nNNGTGA\r\nTTN\r\nNNNRRT\r\nKYTV\r\nNGCG\r\nNGA\r\nNGT\r\nNG",
        Organisms:
          "Anas platyrhynchos\r\nBos taurus\r\nCallithrix jacchus\r\nCanis familiaris\r\nCapra hircus\r\nCavia porcellus\r\nChlorocebus sabeus\r\nCricetulus griseus\r\nCyprinus carpio\r\nDanio rerio\r\nEquus caballus\r\nFelis catus\r\nGallus gallus\r\nHomo sapiens\r\nMacaca fascicularis\r\nMacaca mulatta\r\nMesocricetus auratus\r\nMochrtus ochrogaster\r\nMus musculus\r\nOncorhynchus tshawytscha\r\nOreochromis niloticus\r\nOryctolagus cuniculus\r\novis_aries\r\nPeromyscus maniculatus\r\nPimephales promelas\r\nRattus norvegicus\r\nSalmo salar\r\nSus scrofa\r\nXenopus laevis\r\nXenopus tropicalis",
        Reference:
          "Park J, Bae S, Kim JS Cas-Designer: a web-based tool for choice of CRISPR-Cas9 target sites. Bioinformatics. 2015 Dec 15;31(24):4014-6. doi: 10.1093/bioinformatics/btv537. Epub 2015 Sep 10. (PMID: 26358729)",
        Comments: "It forms part of the CRISPR RGEN Tools",
        __EMPTY: "http://www.rgenome.net/cas-designer/",
      },
      {
        Platform: "Desktop",
      },
      {
        Tools: "CRISPick",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": "Yes",
        "Score Oligos": "Yes",
        "Search by": "GeneID",
        Enzyme:
          "SpyoCas9 (NGG)\r\nSaurCas9(NNGRR)\r\nAsCas12a(TTTV)\r\nenAsCas12a",
        PAM: "NGG\r\nNNGRR\r\nTTTV",
        Organisms: "Homo sapiens\r\nMus musculus\r\nRattus norvegicus",
        Reference:
          "Doench JG, Fusi N, Sullender M, Hegde M, Vaimberg EW, Donovan KF, Smith I, Tothova Z, Wilen C, Orchard R, Virgin HW, Listgarten J, Root DE Optimized sgRNA design to maximize activity and minimize off-target effects of CRISPR-Cas9. Nat Biotechnol. 2016 Feb;34(2):184-191. doi: 10.1038/nbt.3437. Epub 2016 Jan 18. (PMID: 29251726)",
        Comments:
          "Off-targets not shown. The scoring programs they use (Azimuth, Elevation) can be downloaded from their server.",
        __EMPTY: "https://portals.broadinstitute.org/gppx/crispick/public",
      },
      {
        Purpose: "Other",
        "Search by": "Sequence",
        Reference:
          "Najm FJ, Strand C, Donovan KF, Hegde M, Sanson KR, Vaimberg EW, Sullender ME, Hartenian E, Kalani Z, Fusi N, Listgarten J, Younger ST, Bernstein BE, Root DE, Doench JG Orthologous CRISPR-Cas9 enzymes for combinatorial genetic screens. Nat Biotechnol. 2018 Feb;36(2):179-189. doi: 10.1038/nbt.4048. Epub 2017 Dec 18. (PMID: 26780180)",
      },
      {
        Tools: "CRISPR-DO",
        Availability: "No",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": "Yes",
        "Score Oligos": "Yes",
        "Search by": "GeneID",
        Enzyme: "SpCas9 (S. pyogenes)",
        PAM: "NGG",
        Organisms:
          "Homo sapiens\r\nMus musculus\r\nDrosophila melanogaster\r\nCaenorhabditis elegans\r\nDanio rerio",
        Reference:
          "Ma J, Koster J, Qin Q, Hu S, Li W, Chen C, Cao Q, Wang J, Mei S, Liu Q, Xu H, Liu XS CRISPR-DO for genome-wide CRISPR design and optimization. Bioinformatics. 2016 Nov 1;32(21):3336-3338. doi: 10.1093/bioinformatics/btw476. Epub 2016 Jul 10. (PMID: 27402906)",
        __EMPTY: "http://cistrome.org/crispr/",
      },
      {
        Purpose: "Other",
        Platform: "Command-line",
        "Search by": "Sequence",
      },
      {
        Tools: "SSC",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": "No",
        "Score Oligos": "Yes",
        "Search by": "Sequence",
        Enzyme: "SpCas9 (Streptococcus pyogenes)",
        PAM: "NGG",
        Organisms: "Homo sapiens\r\nMus musculus",
        Reference:
          "Xu H, Xiao T, Chen CH, Li W, Meyer CA, Wu Q, Wu D, Cong L, Zhang F, Liu JS, Brown M, Liu XS Sequence determinants of improved CRISPR sgRNA design. Genome Res. 2015 Aug;25(8):1147-57. doi: 10.1101/gr.191452.115. Epub 2015 Jun 10. (PMID: 26063738)",
        Comments:
          "It only analyzes the target input sequence using spacer matrices optimized for human and mouse, not genomes.",
        __EMPTY: "http://crispr.dfci.harvard.edu/SSC/",
      },
      {
        Platform: "Command-line",
      },
      {
        Tools: "CRISPRscan",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": "Yes",
        "Score Oligos": "Yes",
        "Search by": "GeneID",
        Enzyme:
          "SpCas9 (Streptococcus pyogenes)\r\nAsCpf1 (Acidaminococcus)\r\nLbCpf1 (Lachnospiraceae)",
        PAM: "NGG\r\nTTTV\r\nTTTN",
        Organisms:
          "Homo sapiens\r\nMus musculus\r\nDrosophila melanogaster\r\nCaenorhabditis elegans\r\nDanio rerio\r\nRattus norvegicus\r\nSaccahromyces cerevisiae\r\nXenopus trop챙calis\r\nGallus gallus\r\nPan troglodytes\r\nCiona intestinalis\r\nBos taurus\r\nXenopus laevis\r\nXenopus tropicalis\r\nNothobranchius furzeri\r\nCallithrix jacchus\r\nOryzias latipes\r\nNemanosctella vectensis\r\nStrongylocentrotus purpuratus\r\nMnemiopsis leidyi",
        Reference:
          "Moreno-Mateos MA, Vejnar CE, Beaudoin JD, Fernandez JP, Mis EK, Khokha MK, Giraldez AJ CRISPRscan: designing highly efficient sgRNAs for CRISPR-Cas9 targeting in vivo. Nat Methods. 2015 Oct;12(10):982-8. doi: 10.1038/nmeth.3543. Epub 2015 Aug 31. (PMID: 26322839)",
        Comments:
          "gRNA designer for in-vitro CRISPR/Cas9 using T7/sp6 promoter. Returns gRNAs restricted to 5' GG or GA sequences, as required by T7 and Sp6 promoters. Scoring algorithm is based on in-vitro T7-based data only. Use for U6-based in vivo CRISPR/Cas9 limited.",
        __EMPTY: "https://www.crisprscan.org/",
      },
      {
        Purpose: "Other",
        "Search by": "Sequence",
      },
      {
        Tools: "WU-CRISPR",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": "No",
        "Score Oligos": "Yes",
        "Search by": "GeneID",
        Enzyme: "SpCas9 (Streptococcus pyogenes)",
        PAM: "NGG",
        Organisms: "Homo sapiens\r\nMus musculus",
        Reference:
          "Wong N, Liu W, Wang X WU-CRISPR: characteristics of functional guide RNAs for the CRISPR/Cas9 system. Genome Biol. 2015 Nov 2;16:218. doi: 10.1186/s13059-015-0784-0. (PMID: 26521937)",
        Comments: "Standalone gRNA design tool is available",
        __EMPTY: "https://crisprdb.org/wu-crispr/",
      },
      {
        Purpose: "Other",
        Platform: "Command-line",
        "Search by": "Sequence",
      },
      {
        Tools: "CRISPR Library Designer (CLD)",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Command-line",
        "Off‑targets": "Yes",
        "Score Oligos": "Yes",
        "Search by": "GeneID",
        Enzyme: "Custom",
        PAM: "Custom",
        Organisms: "Provided by user",
        Reference:
          "Heigwer F, Zhan T, Breinig M, Winter J, Brugemann D, Leible S, Boutros M CRISPR library designer (CLD): software for multispecies design of single guide RNA libraries. Genome Biol. 2016 Mar 24;17:55. doi: 10.1186/s13059-016-0915-2. (PMID: 27013184)",
        Comments: "It uses Docker",
        __EMPTY: "https://github.com/boutroslab/cld_docker",
      },
      {
        Purpose: "Other",
        "Search by": "Coordinate",
      },
      {
        Tools: "CRISPR-GA",
        Availability: "Yes",
        Purpose: "Post-analysis",
        Platform: "Web",
        "Off‑targets": "No",
        "Score Oligos": "No",
        "Search by": "Other",
        Enzyme: "Not applicable",
        PAM: "Not Applicable",
        Organisms: "Not Applicable",
        Reference:
          "Guell M, Yang L, Church GM Genome editing assessment using CRISPR Genome Analyzer (CRISPR-GA). Bioinformatics. 2014 Oct 15;30(20):2968-70. doi: 10.1093/bioinformatics/btu427. Epub 2014 Jul 1. (PMID: 24990609)",
        Comments:
          "Assess the quality of gene editing using next gen sequencing data",
        __EMPTY: "crispr-ga.net",
      },
      {
        Tools: "Microhomology Predictor",
        Availability: "Yes",
        Purpose: "Other",
        Platform: "Web",
        "Off‑targets": "Yes",
        "Score Oligos": "Yes",
        "Search by": "Sequence",
        Enzyme: "Not applicable",
        PAM: "Not Applicable",
        Organisms: "Not Applicable",
        Reference:
          "Bae S, Kweon J, Kim HS, Kim JS Microhomology-based choice of Cas9 nuclease target sites. Nat Methods. 2014 Jul;11(7):705-6. doi: 10.1038/nmeth.3015. (PMID: 24972169)",
        Comments:
          "For microhomology-based choice of Cas9 nuclease target sites. It predicts mutation patterns caused by MMEJ in an input sequence and estimate frequency of in frame deletions. It forms part of the RGEN suite of tools.",
        __EMPTY: "http://www.rgenome.net/mich-calculator/",
      },
      {
        Tools: "caRpools",
        Availability: "Yes",
        Purpose: "Post-analysis",
        Platform: "Command-line",
        "Off‑targets": "No",
        "Score Oligos": "No",
        "Search by": "Other",
        Enzyme: "Not applicable",
        PAM: "Not Applicable",
        Organisms: "Not Applicable",
        Reference:
          "Winter J, Breinig M, Heigwer F, Brugemann D, Leible S, Pelz O, Zhan T, Boutros M caRpools: an R package for exploratory data analysis and documentation of pooled CRISPR/Cas9 screens. Bioinformatics. 2016 Feb 15;32(4):632-4. doi: 10.1093/bioinformatics/btv617. Epub 2015 Oct 27. (PMID: 26508755)",
        Comments:
          "For exploratory data analysis of pooled CRISPR/Cas9 screens (screening quality, candidate hit calling). Author recommends using an update tool, CRISPRAnalyzeR, instead of this one.",
        __EMPTY: "https://github.com/boutroslab/caRpools",
      },
      {
        Tools: "MAGeCK",
        Availability: "Yes",
        Purpose: "Post-analysis",
        Platform: "Command-line",
        "Off‑targets": "No",
        "Score Oligos": "No",
        "Search by": "Other",
        Enzyme: "Not applicable",
        PAM: "Not Applicable",
        Organisms: "Provided by user",
        Reference:
          "Li W, Xu H, Xiao T, Cong L, Love MI, Zhang F, Irizarry RA, Liu JS, Brown M, Liu XS MAGeCK enables robust identification of essential genes from genome-scale CRISPR/Cas9 knockout screens. Genome Biol. 2014;15(12):554. doi: 10.1186/s13059-014-0554-4. (PMID: 25476604)",
        Comments:
          "For identifying  positively or negatively selected sgRNAs, genes or pathways from two CRISPR/Cas9 knockout screens in two different conditions.",
        __EMPTY: "http://liulab.dfci.harvard.edu/Mageck",
      },
      {
        Tools: "CRISPResso",
        Availability: "No",
        Purpose: "Post-analysis",
        Platform: "Web",
        "Off‑targets": "No",
        "Score Oligos": "No",
        "Search by": "Other",
        Enzyme: "Custom",
        PAM: "Custom",
        Organisms: "Not Applicable",
        Reference:
          "Pinello L, Canver MC, Hoban MD, Orkin SH, Kohn DB, Bauer DE, Yuan GC Analyzing CRISPR genome-editing experiments with CRISPResso. Nat Biotechnol. 2016 Jul 12;34(7):695-7. doi: 10.1038/nbt.3583. (PMID: 27404874)",
        Comments:
          "(Deprecated) Analyze the output of an CRISPR-Cas9 single or paired ends NGS experiment from their FASTQ files: NEHJ events like insertions, deletions or substitutions and their effects",
        __EMPTY: "http://crispresso.rocks/",
      },
      {
        Platform: "Command-line",
      },
      {
        Tools: "CRISPResso2",
        Availability: "Yes",
        Purpose: "Post-analysis",
        Platform: "Web",
        "Off‑targets": "No",
        "Score Oligos": "No",
        "Search by": "Other",
        Enzyme: "Custom",
        PAM: "Custom",
        Organisms: "Not Applicable",
        Reference:
          "Clement K, Rees H, Canver MC, Gehrke JM, Farouni R, Hsu JY, Cole MA, Liu DR, Joung JK, Bauer DE, Pinello L CRISPResso2 provides accurate and rapid genome editing sequence analysis. Nat Biotechnol. 2019 Feb 26. pii: 10.1038/s41587-019-0032-3. doi: 10.1038/s41587-019-0032-3. (PMID: 30809026)",
        Comments:
          "Analysis and visualization of next-generation sequencing data, including analysis of sequencing data from Cas9 and variants, base editors, and prime editors. Several modes allow for the analysis and comparison of different samples (batch mode) different sites (pooled mode) or different sequencing strategies (e.g. WGS mode). Other features include allele-specific quantification of heterozygous or polymorphic references and a biologically informed alignment algorithm.",
        __EMPTY: "http://crispresso.pinellolab.org/submission",
      },
      {
        Platform: "Command-line",
      },
      {
        Tools: "CrispRVariants",
        Availability: "Yes",
        Purpose: "Post-analysis",
        Platform: "Command-line",
        "Off‑targets": "No",
        "Score Oligos": "No",
        "Search by": "Other",
        Enzyme: "Custom",
        PAM: "Custom",
        Organisms: "Provided by user",
        Reference:
          "Lindsay H, Burger A, Biyong B, Felker A, Hess C, Zaugg J, Chiavacci E, Anders C, Jinek M, Mosimann C, Robinson MD CrispRVariants charts the mutation spectrum of genome engineering experiments. Nat Biotechnol. 2016 Jul 12;34(7):701-2. doi: 10.1038/nbt.3628. (PMID: 27404876)",
        Comments:
          "Extract the variations produced in an experiment of CRISPR-Cas9 NGS sequencing. It also calculates the mutation efficiency, get consensus alleles and plot chimeric alignments.",
        __EMPTY: "https://github.com/markrobinsonuzh/CrispRVariants",
      },
      {
        Tools: "CAGE",
        Availability: "Yes",
        Purpose: "Post-analysis",
        Platform: "Command-line",
        "Off‑targets": "No",
        "Score Oligos": "No",
        "Search by": "Other",
        Enzyme: "Custom",
        PAM: "Custom",
        Organisms: "Provided by user",
        Reference:
          "Chuai G, Yang F, Yan J, Chen Y, Ma Q, Zhou C, Zhu C, Gu F, Liu Q Deciphering relationship between microhomology and in-frame mutation occurrence in human CRISPR-based gene knockout. Mol Ther Nucleic Acids. 2016 Jun 14;5(6):e323. doi: 10.1038/mtna.2016.35. (PMID: 27301063)",
        Comments:
          "Analysis of indels and microhomology patterns and enrichment analyis and evaluation of the sgRNA KO efficiency and identification of features based on the CRISPR-Cas9 KO NGS data.",
        __EMPTY: "https://github.com/bm2-lab/cage",
      },
      {
        Tools: "ScreenBEAM",
        Availability: "Yes",
        Purpose: "Post-analysis",
        Platform: "Command-line",
        "Off‑targets": "No",
        "Score Oligos": "No",
        "Search by": "Other",
        Enzyme: "Custom",
        PAM: "Not Applicable",
        Organisms: "Not Applicable",
        Reference:
          "Yu J, Silva J, Califano A ScreenBEAM: a novel meta-analysis algorithm for functional genomics screens via Bayesian hierarchical modeling. Bioinformatics. 2016 Jan 15;32(2):260-7. doi: 10.1093/bioinformatics/btv556. Epub 2015 Sep 28. (PMID: 26415723)",
        Comments:
          "Analysis of differential efficiency of CRISPR-CAs9 NGS experiments between samples in different conditions. Inputs are count tables.",
        __EMPTY: "https://github.com/jyyu/ScreenBEAM",
      },
      {
        Purpose: "Other",
      },
      {
        Tools: "HitSelect",
        Availability: "Yes",
        Purpose: "Post-analysis",
        Platform: "Desktop",
        "Off‑targets": "No",
        "Score Oligos": "No",
        "Search by": "Other",
        Enzyme: "Custom",
        PAM: "Not Applicable",
        Organisms: "Not Applicable",
        Reference:
          "Diaz AA, Qin H, Ramalho-Santos M, Song JS HiTSelect: a comprehensive tool for high-complexity-pooled screen analysis. Nucleic Acids Res. 2015 Feb 18;43(3):e16. doi: 10.1093/nar/gku1197. Epub 2014 Nov 26. (PMID: 25428347)",
        Comments:
          "Performs functional enrichment (using DAVID), pathways identification and gene interaction study from CRISPR-CAs9 NGS experiments of samples in different conditions. Inputs are count tables and differential expression results (p-value and fold change).",
        __EMPTY: "https://sourceforge.net/projects/hitselect/",
      },
      {
        Tools: "CRISPR-DT",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": "Yes",
        "Score Oligos": "Yes",
        "Search by": "Sequence",
        Enzyme: "Custom",
        PAM: "Custom",
        Organisms:
          "Homo sapiens\r\nMus musculus\r\nArabidopsis thaliana\r\nOryza sativa\r\nGallus gallus\r\nDrosophila melanogaster\r\nCaenorhabditis elegans\r\nGlycine max",
        Reference:
          "Zhu H, Liang C CRISPR-DT: designing gRNAs for the CRISPR-Cpf1 system with improved target efficiency and specificity. Bioinformatics. 2019 Jan 7. pii: 5273481. doi: 10.1093/bioinformatics/bty1061. (PMID: 30615056)",
        Comments:
          "It provides a tool for calculate the efficiency of the target for several caspases",
        __EMPTY: "http://bioinfolab.miamioh.edu/CRISPR-DT",
      },
      {
        Purpose: "Other",
      },
      {
        Tools: "Breaking Cas",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": "Yes",
        "Score Oligos": "No",
        "Search by": "Sequence",
        Enzyme:
          "SpCas9\r\nSaCas9\r\nAsCpf1-RVR\r\nAsCpf1-WT\r\nAsCpf1-RR\r\nxCas9\r\nMAD7",
        PAM: "NGG, NAG, NNGRRN, TVTV. TTTV, TYCV, YTTN",
        Organisms:
          "324 genomes : Vertebrate, Protist, Fungi, plant, Invertebrate metazoa",
        Reference:
          "Oliveros JC, Franch M, Tabas-Madrid D, San-Leon D, Montoliu L, Cubas P, Pazos F Breaking-Cas-interactive design of guide RNAs for CRISPR-Cas experiments for ENSEMBL genomes. Nucleic Acids Res. 2016 Jul 8;44(W1):W267-71. doi: 10.1093/nar/gkw407. Epub 2016 May 10. (PMID: 27166368)",
        Comments:
          "List of on-targets and off-targets sorted also by colour. It allows to change the positional weights for scoring",
        __EMPTY: "https://bioinfogp.cnb.csic.es/tools/breakingcas/",
      },
      {
        Tools: "Benchling (CRISPR guides)",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": "Yes",
        "Score Oligos": "Yes",
        "Search by": "GeneID",
        Enzyme:
          "SpCas9 (Streptococcus pyogenes)\r\nNmCas9 (Neisseria meningitidis)\r\nSt1Cas9 (Streptococcus thermophilus)\r\nTdCas9 (Treponema denticola)\r\nSaCas9 (Streptococcus aureus)\r\nAsCpf1 (Acidaminococcus sp.)\r\nLbCpf1 (Lachnospiraceae bacterium)",
        PAM: "NGG\r\nNAG\r\nNNNNGATT\r\nNNAGAAW\r\nNAAAAC\r\nNNGRR\r\nNNGRRT\r\nTTTN",
        Organisms:
          "Acanthamoeba castellanii\r\nAcetobacterium woodii\r\nAedes aegypti\r\nAfrican swine\r\nAmbystoma mexicanum\r\nAnabaena variabilis\r\nAnolis carolinensis\r\nAnopheles gambiae\r\nAnthracocystis flocculosa\r\nApis mellifera\r\nArabidopsis thaliana\r\nAspergillus fumigatus\r\nAspergillus nidulans\r\nBacillus methanolicus\r\nBacillus mycoides\r\nBacillus subtilis\r\nBos taurus\r\nBrachypodium distachyon\r\nBradyrhizobium diazoefficiens\r\nBrassica napus\r\nBrassica oleracea\r\nBrassica rapa\r\nCaenorhabditis elegans\r\nCallithrix jacchus\r\nCandida albicans\r\nCanis lupus familiaris\r\nCapra hircus\r\nChlamydia trachomatis\r\nChlamydomonas reinhardtii\r\nCitrus clementina\r\nCitrus sinensis\r\nClaviceps purpurea\r\nClostridium acetobutylicum\r\nClostridium autoethanogenum\r\nClostridium botulinum\r\nClostridium ljungdahlii\r\nClostridium pasteurianum\r\nClostridium saccharobutylicum\r\nClostridium saccharoperbutylacetonicum\r\nClostridium sporogenes\r\nCricetulus griseus\r\nCryptosporidium parvum\r\nCucumis melo\r\nCulex quinquefasciatus\r\nDanio rerio\r\nDrosophila eugracilis\r\nDrosophila melanogaster\r\nDrosophila Yakuba\r\nEquus caballus\r\nEragrostis tef\r\nEscherichia coli\r\nEubacterium limosum\r\nFelis catus\r\nFusarium graminearum\r\nGallus gallus\r\nGekko japonicus\r\nGeobacillus thermoglucosidasius\r\nGiardia intestinalis\r\nGiardia lamblia\r\nGlycine max\r\nGossypium hirsutum\r\nHepatitis C\r\nHomo sapiens\r\nHordeum vulgare\r\nKomagataella pastoris\r\nLeishmania infantum\r\nLeishmania major\r\nLolium perenne\r\nLutzomyia longipalpis\r\nMacaca fascicularis\r\nMacaca mulatta\r\nMalus domestica\r\nManihot esculenta\r\nMarchantia polymorpha\r\nMesocricetus auratus\r\nMethylobacterium extorquens\r\nMethylocystis parvus\r\nMicrotus ochrogaster\r\nMoorella thermoacetica\r\nMusa acuminata\r\nMus musculus\r\nMustela putorius\r\nNematostella vectensis\r\nNicotiana benthamiana\r\nNicotiana tabacum\r\nOryctolagus cuniculus\r\nOryza sativa\r\nOryzias latipes\r\nOvis aries\r\nPaenibacillus polymyxa\r\nPatiria miniata\r\nPeptoclostridium difficile\r\nPhaeodactylum tricornutum\r\nPhlebotomus papatasi\r\nPiscirickettsia salmonis\r\nPlasmodium berghei\r\nPlasmodium falciparum\r\nPopulus trichocarpa\r\nProteus mirabilis\r\nPseudomonas aeruginosa\r\nRalstonia eutropha\r\nRattus norvegicus\r\nRhodobacter sphaeroides\r\nRhodococcus jostii\r\nSaccharomyces cerevisiae\r\nSaccharopolyspora spinosa\r\nSaccoglossus kowalevskii\r\nSalmo salar\r\nSaprolegnia parasitica\r\nSchistosoma japonicum\r\nSchizosaccharomyces pombe\r\nSolanum lycopersicum\r\nSorghum bicolor\r\nSpodoptera frugiperda\r\nStreptomyces coelicolor\r\nStreptomyces rapamycinicus\r\nStreptomyces sp\r\nStrongylocentrotus purpuratus\r\nSus scrofa\r\nSynechococcus elongatus\r\nSynechococcus sp\r\nSynechocystis sp\r\nTaeniopygia guttata\r\nToxoplasma gondii\r\nTribolium castaneum\r\nTrichoderma virens\r\nTrypanosoma brucei\r\nVerrucosispora maris\r\nVitis vinifera\r\nXanthomonas campestris\r\nXenopus tropicalis\r\nYarrowia lipolytica\r\nZea mays\r\nZymomonas mobilis",
        Comments: "Commercial",
        __EMPTY: "https://benchling.com/",
      },
      {
        "Search by": "Coordinate",
      },
      {
        "Search by": "Sequence",
      },
      {
        Tools: "CRISPy CHO",
        Availability: "Yes",
        Purpose: "Database",
        Platform: "Web",
        "Off‑targets": "Yes",
        "Score Oligos": "Yes",
        "Search by": "GeneID",
        Enzyme: "SpCas9 (S. pyogenes)",
        PAM: "NGG",
        Organisms: "Cricetulus griseus",
        Reference:
          "Ronda C, Pedersen LE, Hansen HG, Kallehauge TB, Betenbaugh MJ, Nielsen AT, Kildegaard HF Accelerating genome editing in CHO cells using CRISPR Cas9 and CRISPy, a web-based target finding tool. Biotechnol Bioeng. 2014 Aug;111(8):1604-16. doi: 10.1002/bit.25233. Epub 2014 May 22. (PMID: 24827782)",
        Comments: "Only for Chinese Hamsters Ovary (CHO) cells",
        __EMPTY: "http://staff.biosustain.dtu.dk/laeb/crispy/",
      },
      {
        Tools: "CRISPRmap",
        Availability: "Yes",
        Purpose: "Database",
        Platform: "Web",
        "Off‑targets": "No",
        "Score Oligos": "No",
        "Search by": "Sequence",
        Enzyme: "not applicable",
        PAM: "Not Applicable",
        Organisms: "Not Applicable",
        Reference:
          "Lange SJ, Alkhnbashi OS, Rose D, Will S, Backofen R CRISPRmap: an automated classification of repeat conservation in prokaryotic adaptive immune systems. Nucleic Acids Res. 2013 Sep;41(17):8034-44. doi: 10.1093/nar/gkt606. Epub 2013 Jul 17. (PMID: 23863837)",
        Comments:
          "Intended for the analysis and classification of a CRISPR system from a list of CRISPR sequences. Includes a database of CRISPR consensus repeats.",
        __EMPTY: "http://rna.informatik.uni-freiburg.de/CRISPRmap/Input.jsp",
      },
      {
        Purpose: "Other",
      },
      {
        Tools: "Yeastriction",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": "Yes",
        "Score Oligos": "Yes",
        "Search by": "GeneID",
        Enzyme: "SpCas9 (S. pyogenes)",
        PAM: "NGG",
        Organisms:
          "Saccharomyces cerevisiae AWRI1631\r\nSaccharomyces cerevisiae AWRI796\r\nSaccharomyces cerevisiae BY4741\r\nSaccharomyces cerevisiae BY4742\r\nSaccharomyces cerevisiae CBS7960\r\nSaccharomyces cerevisiae CEN.PK113-7D\r\nSaccharomyces cerevisiae CLIB215\r\nSaccharomyces cerevisiae CLIB324\r\nSaccharomyces cerevisiae CLIB382\r\nSaccharomyces cerevisiae EC1118\r\nSaccharomyces cerevisiae EC9-8\r\nSaccharomyces cerevisiae FL100\r\nSaccharomyces cerevisiae FostersB\r\nSaccharomyces cerevisiae FostersO\r\nSaccharomyces cerevisiae JAY291\r\nSaccharomyces cerevisiae Kyokai7\r\nSaccharomyces cerevisiae LalvinQA23\r\nSaccharomyces cerevisiae M22\r\nSaccharomyces cerevisiae PW5\r\nSaccharomyces cerevisiae RM11-1a\r\nSaccharomyces cerevisiae S288C\r\nSaccharomyces cerevisiae Sigma1278b\r\nSaccharomyces cerevisiae T7\r\nSaccharomyces cerevisiae T73\r\nSaccharomyces cerevisiae UC5\r\nSaccharomyces cerevisiae Vin13\r\nSaccharomyces cerevisiae VL3\r\nSaccharomyces cerevisiae ZTW1\r\nSaccharomyces cerevisiae W303\r\nSaccharomyces cerevisiae Y10\r\nSaccharomyces cerevisiae YJM269\r\nSaccharomyces cerevisiae YJM789\r\nSaccharomyces cerevisiae YPS163",
        Reference:
          "Mans R, van Rossum HM, Wijsman M, Backx A, Kuijpers NG, van den Broek M, Daran-Lapujade P, Pronk JT, van Maris AJ, Daran JM CRISPR/Cas9: a molecular Swiss army knife for simultaneous introduction of multiple genetic modifications in Saccharomyces cerevisiae. FEMS Yeast Res. 2015 Mar;15(2). pii: fov004. doi: 10.1093/femsyr/fov004. Epub 2015 Mar 4. (PMID: 25743786)",
        Comments:
          "For yeast only. Provides information of the RNA secondary structure. Source code available to be installed in your own web server.",
        __EMPTY: "http://yeastriction.tnw.tudelft.nl/#!/",
      },
      {
        Platform: "Other",
      },
      {
        Tools: "DESKGEN Cloud",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": "Yes",
        "Score Oligos": "Yes",
        "Search by": "GeneID",
        Enzyme:
          "AsCpf1\r\nFnCpf1\r\nLbCPf1\r\nNmCas9\r\nSaCas9\r\nSpcas9\r\nSpCas9D10A-pairs\r\nSpCas9D10A-singles\r\nSpdCas9-FokI\r\nSt1Cas9",
        PAM: "NGG\r\nNGRRT\r\nNGRRN\r\nNNNNGATT\r\nNNNNRYAC\r\nNNAGAAW\r\nTTTV",
        Organisms:
          "Anopheles gambiae\r\nArabidopsis thaliana\r\nBos taurus\r\nCaenorhabditis elegans\r\nCandida albicans\r\nCandida glabrata\r\nDanio rerio\r\nDrosophila melanogaster\r\nEscherichia coli\r\nGallus gallus\r\nHomo sapiens\r\nHordeum vulgare\r\nMus musculus\r\nOryza sativa\r\nPenicillium roqueforti\r\nPhyscomitrella patens\r\nPlasmodium falciparum\r\nRattus norvegicus\r\nRhodococcus jostii\r\nSaccharomyces cerevisiae\r\nSchizosaccharomyces pombe\r\nSynechocystis sp\r\nTriticum aestivum\r\nZea mays",
        Comments: "Commercial, free, registration is required",
        __EMPTY: "https://www.deskgen.com/landing/documentation.html",
      },
      {
        Purpose: "Database",
      },
      {
        Purpose: "Other",
      },
      {
        Tools: "GenomeCRISPR",
        Availability: "Yes",
        Purpose: "Database",
        Platform: "Web",
        "Off‑targets": "?",
        "Score Oligos": "?",
        "Search by": "GeneID",
        Enzyme: "SpCas9 (S. pyogenes)",
        PAM: "NGG",
        Organisms: "Homo sapiens",
        Reference:
          "Rauscher B, Heigwer F, Breinig M, Winter J, Boutros M GenomeCRISPR - a database for high-throughput CRISPR/Cas9 screens. Nucleic Acids Res. 2017 Jan 4;45(D1):D679-D686. doi: 10.1093/nar/gkw997. Epub 2016 Oct 26. (PMID: 27789686)",
        Comments:
          "Database for genome-scale CRISPR/Cas9 screens in human. Provides the experimentally measured sgRNA effect in the context each screen.",
        __EMPTY: "http://genomecrispr.dkfz.de/#!/",
      },
      {
        Platform: "Webapp",
        "Search by": "Coordinate",
      },
      {
        Tools: "COSMID",
        Availability: "Yes",
        Purpose: "Post-analysis",
        Platform: "Web",
        "Off‑targets": "Yes",
        "Score Oligos": "No",
        "Search by": "Sequence",
        Enzyme: "Custom",
        PAM: "Custom",
        Organisms:
          "Homo sapiens GRCh38 (hg38)\r\nHomo sapiens GRCh37 (hg19)\r\nHomo sapiens NCBI36 (hg18)\r\nC elegans (ce10/WS220)\r\nMacaca mulatta Mmul_051212 (rheMac2)\r\nRattus norvegicus (rn6)\r\nDanio rerio Zebrafish (danRer10)\r\nSus scrofa Pig (susScr3)\r\nMus musculus GRCm38 (mm10)\r\nMus musculus NCBI37 (mm9)",
        Reference:
          "Cradick TJ, Qiu P, Lee CM, Fine EJ, Bao G COSMID: A Web-based Tool for Identifying and Validating CRISPR/Cas Off-target Sites. Mol Ther Nucleic Acids. 2014 Dec 2;3:e214. doi: 10.1038/mtna.2014.64. (PMID: 25462530)",
        Comments: "Tool for identifying and validating Off-target sites.",
        __EMPTY: "https://crispr.bme.gatech.edu/",
      },
      {
        Tools: "AGEseq",
        Availability: "Yes",
        Purpose: "Post-analysis",
        Platform: "Command-line",
        "Off‑targets": "No",
        "Score Oligos": "No",
        "Search by": "Sequence",
        Enzyme: "Custom",
        PAM: "Custom",
        Organisms: "Provided by user",
        Reference:
          "Xue LJ, Tsai CJ AGEseq: Analysis of Genome Editing by Sequencing. Mol Plant. 2015 Sep;8(9):1428-30. doi: 10.1016/j.molp.2015.06.001. Epub 2015 Jun 6. (PMID: 26057235)",
        Comments:
          "PERL script available at https://github.com/liangjiaoxue/AGEseq. Galaxy server plugin (it uses port 8085)",
        __EMPTY: "https://www.aspendb.org/databases",
      },
      {
        Platform: "Other",
        "Search by": "Other",
      },
      {
        Tools: "CRISPR ML",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": "Yes",
        "Score Oligos": "Yes",
        "Search by": "GeneID",
        Enzyme: "SpCas9 (S. pyogenes)",
        PAM: "NGG",
        Organisms: "Homo sapiens",
        Reference:
          "Listgarten J, Weinstein M, Kleinstiver BP, Sousa AA, Joung JK, Crawford J, Gao K, Hoang L, Elibol M, Doench JG, Fusi N Prediction of off-target activities for the end-to-end design of CRISPR guide RNAs. Nat Biomed Eng. 2018 Jan;2(1):38-47. doi: 10.1038/s41551-017-0178-6. Epub 2018 Jan 10. (PMID: 29998038)",
        Comments:
          "Off-target detection for GeneID search only. Source code of the algorithms available at https://github.com/michael-weinstein/dsNickFury3PlusOrchid and https://github.com/microsoftResearch//azimuth",
        __EMPTY: "https://crispr.ml/",
      },
      {
        "Search by": "Sequence",
      },
      {
        Tools: "CRISPETa",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": "Yes",
        "Score Oligos": "Yes",
        "Search by": "Coordinate",
        Enzyme: "SpCas9 (S. pyogenes)",
        PAM: "NGG",
        Organisms:
          "Human (hg19)\r\nMouse (mm10)\r\nZebrafish (danRer10)\r\nDrosophila (dm6)\r\nC. elegans (ce11)",
        Reference:
          "Pulido-Quetglas C, Aparicio-Prat E, Arnan C, Polidori T, Hermoso T, Palumbo E, Ponomarenko J, Guigo R, Johnson R Scalable Design of Paired CRISPR Guide RNAs for Genomic Deletion. PLoS Comput Biol. 2017 Mar 2;13(3):e1005341. doi: 10.1371/journal.pcbi.1005341. eCollection 2017 Mar. (PMID: 28253259)",
        Comments:
          "Intended for CRISPR deletion projects with sgRNA pairs. Allows specifying favored and/or disfavored genome regions. Pre-computed knockout libraries for a variety of genomic element classes and intergenic regions available for download.",
        __EMPTY: "http://crispeta.crg.eu/",
      },
      {
        Platform: "Command-line",
        "Search by": "Sequence",
      },
      {
        Tools: "PhytoCRISP-Ex",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": "Yes",
        "Score Oligos": "No",
        "Search by": "Sequence",
        Enzyme: "SpCas9 (S. pyogenes)",
        PAM: "NGG\r\nNAG",
        Organisms:
          "Phaeodactylum tricornutum \r\nChlamydomonas reinhardtii \r\nChlorella vulgaris \r\nFistulifera \r\nFragilariopsis cylindrus \r\nNannochloropsis gaditana \r\nOstreococcus lucimarinus \r\nOstreococcus tauri \r\nThalassiosira oceanica \r\nThalassiosira pseudonana \r\nEmiliania huxleyi \r\nPsuedo-nitzschia multiseries \r\nVolvox carteri",
        Reference:
          "Rastogi A, Murik O, Bowler C, Tirichine L PhytoCRISP-Ex: a web-based and stand-alone application to find specific target sequences for CRISPR/CAS editing. BMC Bioinformatics. 2016 Jul 1;17(1):261. doi: 10.1186/s12859-016-1143-1. (PMID: 27363443)",
        Comments: "Exclusively for phytoplankton genomes.",
        __EMPTY: "https://www.phytocrispex.bio.ens.psl.eu/CRISP-Ex/",
      },
      {
        Platform: "Command-line",
      },
      {
        Tools: "IDT Design custom gRNA",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": "Yes",
        "Score Oligos": "Yes",
        "Search by": "GeneID",
        Enzyme: "SpCas9 (S. pyogenes)",
        PAM: "NGG",
        Organisms:
          "Homo sapiens\r\nMus musculus\r\nRattus norvegicus\r\nDanio rerio\r\nCaenorhabditis elegans",
        Comments:
          "Requires registration. Commercial kits with the designed sgRNAs can be purchased in the web site. Allows other species without off-target detection.",
        __EMPTY:
          "https://sg.idtdna.com/site/order/designtool/index/CRISPR_CUSTOM",
      },
      {
        "Search by": "Sequence",
      },
      {
        Tools: "CRISPR LifePipe",
        Availability: "No",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": "Yes",
        "Score Oligos": "Yes",
        "Search by": "GeneID",
        Enzyme:
          "Cas9sp, eCas9sp, Cas9sp-HF1, 20bp [NGG]\r\nCas9sa, 21bp [NNG(A/G)(A/G)T]\r\nCas9sp VQR, 20bp [NGA]\r\nCas9sp VRER, 20bp [NGCG]\r\nCas9sp nickase, 20bp [NGG]",
        PAM: "NGG\r\nNNGRRT\r\nNAG\r\nNGCG\r\nNGG\r\nTTTN",
        Organisms:
          "Human (Homo sapiens GRCh38 p11)\r\nRat (Rattus norvegicus Rnor_6.0)\r\nMouse (Mus musculus GRCm38.p6)\r\nZebrafish (Danio rerio  GRCz11)\r\nFruit fly (Drosophila melanogaster 6 plus ISO1 MT)",
        Comments:
          "Commercial (free version limits submissions per month). Allows design for targeting an specific amino acid.",
        __EMPTY: "https://crispr.lifeandsoft.com/",
      },
      {
        Purpose: "Other",
        "Search by": "Sequence",
      },
      {
        "Search by": "Other",
      },
      {
        Tools: "Synthego CRISPR Guide Design",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": "Yes",
        "Score Oligos": "Yes",
        "Search by": "GeneID",
        Enzyme: "SpCas9 (S. pyogenes)",
        PAM: "NGG",
        Organisms: "Provided by user",
        Comments:
          "Commercial kit for the designed sgRNA can be purchased in the web site.",
        __EMPTY:
          "https://www.synthego.com/products/bioinformatics/crispr-design-tool",
      },
      {
        Purpose: "Post-analysis",
      },
      {
        Tools: "GuideScan",
        Availability: "Yes",
        Purpose: "Post-analysis",
        Platform: "Web",
        "Off‑targets": "Yes",
        "Score Oligos": "Yes",
        "Search by": "GeneID",
        Enzyme: "SpCas9\r\nAsCpf1/LbCpf1",
        PAM: "NGG\r\nNAG\r\nTTTN",
        Organisms:
          "Homo sapiens (hg38)\r\nMus musculus (mm10)\r\nDrosphila melanogaster (dm6)\r\nSaccharomyces cerevisiae (SacCerv)\r\nDanio rerio (danRer10)\r\nCaenorhabditis elegans (ce11)",
        Reference:
          "Perez AR, Pritykin Y, Vidigal JA, Chhangawala S, Zamparo L, Leslie CS, Ventura A GuideScan software for improved single and paired CRISPR guide RNA design. Nat Biotechnol. 2017 Apr;35(4):347-349. doi: 10.1038/nbt.3804. Epub 2017 Mar 6. (PMID: 28263296)",
        Comments: "Allows mixed queries of coordinates and gene symbols.",
        __EMPTY: "https://guidescan.com/",
      },
      {
        Platform: "Command-line",
        "Search by": "Coordinate",
      },
      {
        Tools: "GUIDES",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": "Yes",
        "Score Oligos": "Yes",
        "Search by": "GeneID",
        Enzyme: "SpCas9 (S. pyogenes)",
        PAM: "NGG",
        Organisms: "Human (Homo sapiens GRCh37)\r\nMouse (Mus musculus GRCm38)",
        Reference:
          "Meier JA, Zhang F, Sanjana NE GUIDES: sgRNA design for loss-of-function screens. Nat Methods. 2017 Aug 31;14(9):831-832. doi: 10.1038/nmeth.4423. (PMID: 28858339)",
        Comments:
          "Retrieves external information for targeting specific tissues, highly expressed exons or known functional domains. Can include non-targeting sgRNAs (controls) and synthesis ready constructs in the results. Source code available to install the web server locally.",
        __EMPTY: "http://guides.sanjanalab.org/#/",
      },
      {
        Platform: "Other",
      },
      {
        Tools: "Geneious",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Desktop",
        "Off‑targets": "Yes",
        "Score Oligos": "Yes",
        "Search by": "Sequence",
        Enzyme: "SpCas9 (S. pyogenes)\r\nCas9 D10 Nickase\r\nCpf1",
        PAM: "NGG\r\nTTTN",
        Organisms: "Provided by user",
        Comments: "Commercial (30 day trial)",
        __EMPTY: "https://www.geneious.com/tutorials/finding-crispr-sites-r9/",
      },
      {
        Tools: "Protospacer Workbench",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Desktop",
        "Off‑targets": "Yes",
        "Score Oligos": "Yes",
        "Search by": "GeneID",
        Enzyme: "SpCas9 (S. pyogenes)",
        PAM: "NGG\r\nNAG",
        Organisms: "Provided by user",
        Reference:
          "MacPherson CR, Scherf A Flexible guide-RNA design for CRISPR applications using Protospacer Workbench. Nat Biotechnol. 2015 Aug;33(8):805-6. doi: 10.1038/nbt.3291. Epub 2015 Jun 29. (PMID: 26121414)",
        Comments: "Only for Mac OS",
        __EMPTY: "https://sourceforge.net/projects/protospacerwb/",
      },
      {
        "Search by": "Coordinate",
      },
      {
        "Search by": "Sequence",
      },
      {
        Tools: "CRISPR-FOCUS",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": "Yes",
        "Score Oligos": "Yes",
        "Search by": "GeneID",
        Enzyme: "SpCas9 (S. pyogenes)",
        PAM: "NGG",
        Organisms: "Homo sapiens\r\nMus musculus",
        Reference:
          "Cao Q, Ma J, Chen CH, Xu H, Chen Z, Li W, Liu XS CRISPR-FOCUS: A web server for designing focused CRISPR screening experiments. PLoS One. 2017 Sep 5;12(9):e0184281. doi: 10.1371/journal.pone.0184281. eCollection 2017. (PMID: 28873439)",
        __EMPTY: "http://cistrome.org/crispr-focus/",
      },
      {
        Tools: "MAGeCK-VISPR",
        Availability: "Yes",
        Purpose: "Post-analysis",
        Platform: "Command-line",
        "Off‑targets": "?",
        "Score Oligos": "?",
        "Search by": "Other",
        Enzyme: "not applicable",
        PAM: "Not Applicable",
        Organisms: "Not Applicable",
        Reference:
          "Li W, Koster J, Xu H, Chen CH, Xiao T, Liu JS, Brown M, Liu XS Quality control, modeling, and visualization of CRISPR screens with MAGeCK-VISPR. Genome Biol. 2015 Dec 16;16:281. doi: 10.1186/s13059-015-0843-6. (PMID: 26673418)",
        Comments:
          "Tool for quality control analysis and visualization of CRISPR/Cas9 screens.",
        __EMPTY: "https://bitbucket.org/liulab/mageck-vispr/src/master/",
      },
      {
        Tools: "CAS-Analyzer",
        Availability: "Yes",
        Purpose: "Post-analysis",
        Platform: "Web",
        "Off‑targets": "No",
        "Score Oligos": "No",
        "Search by": "Sequence",
        Enzyme:
          "SpCas9 from Streptococcus pyogenes: 5'-NGG-3'\r\nStCas9 from Streptococcus thermophilus: 5'-NNAGAAW-3' (W = A or T)\r\nNmCas9 from Neisseria meningitidis: 5'-NNNNGMTT-3' (M = A or C)\r\nSaCas9 from Staphylococcus aureus: 5'-NNGRRT-'3 (R=A or G)\r\nCjCas9 from Campylobacter jejuni: 5'-NNNNACAC-3'\r\nAsCpf1 from Acidaminococcus or LbCpf1 from Lachnospiraceae: 5'-TTTN-3'\r\nxCas9 from Streptococcus pyogene 5'-NGT-3'\r\nFnCpf1 from francisella novicida: 5'-TTN-3'\r\nSpy-macCas9 from Streptococcus pyogenes and Streptococcus macacae: 5'-NAAN-3'\r\nZinc finger nucleases (ZFNs)\r\nTale effector nucleases (TALENs)\r\nCas9 Nickases\r\nCRISPR RNA guided FokI nucleases (dCas9-FokI)",
        PAM: "NGG\r\nNNAGAAW\r\nNNNNGMTT\r\nNNGRRT\r\nNNNNACAC\r\nTTTN\r\nNGT",
        Organisms: "Provided by user",
        Reference:
          "Park J, Lim K, Kim JS, Bae S Cas-analyzer: an online tool for assessing genome editing results using NGS data. Bioinformatics. 2017 Jan 15;33(2):286-288. doi: 10.1093/bioinformatics/btw561. Epub 2016 Aug 24. (PMID: 27559154)",
        Comments:
          "Alignment and analysis of NGS reads against a region reference sequence",
        __EMPTY: "http://www.rgenome.net/cas-analyzer/#!",
      },
      {
        Tools: "DeepCRISPR",
        Availability: "No",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": "Yes",
        "Score Oligos": "Yes",
        "Search by": "Sequence",
        Enzyme: "SpCas9 (S. pyogenes)",
        PAM: "NGG",
        Organisms: "Homo sapiens",
        Reference:
          "Chuai G, Ma H, Yan J, Chen M, Hong N, Xue D, Zhou C, Zhu C, Chen K, Duan B, Gu F, Qu S, Huang D, Wei J, Liu Q DeepCRISPR: optimized CRISPR guide RNA design by deep learning. Genome Biol. 2018 Jun 26;19(1):80. doi: 10.1186/s13059-018-1459-4. (PMID: 29945655)",
        __EMPTY: "https://www.deepcrispr.net/",
      },
      {
        Tools: "CRISPR Recognition Tool (CRT)",
        Availability: "Yes",
        Purpose: "Other",
        Platform: "Command-line",
        "Off‑targets": "No",
        "Score Oligos": "No",
        "Search by": "Sequence",
        Enzyme: "not applicable",
        PAM: "Not Applicable",
        Organisms: "Provided by user",
        Reference:
          "Bland C, Ramsey TL, Sabree F, Lowe M, Brown K, Kyrpides NC, Hugenholtz P CRISPR recognition tool (CRT): a tool for automatic detection of clustered regularly interspaced palindromic repeats. BMC Bioinformatics. 2007 Jun 18;8:209. doi: 10.1186/1471-2105-8-209. (PMID: 17577412)",
        Comments: "Detect CRISPR patterns in genomic sequences",
        __EMPTY: "http://www.room220.com/crt/",
      },
      {
        Platform: "Desktop",
      },
      {
        Tools: "CGAT",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": "Yes",
        "Score Oligos": "Yes",
        "Search by": "GeneID",
        Enzyme: "SpCas9 (S. pyogenes)",
        PAM: "NGG",
        Organisms:
          "Barley: Hordeum vulgare L. 082214v1.29\r\nMaize: Zea mays ssp. mays B73 RefGen_v3 5b+\r\nPeanut: Arachis duranensis V1\r\nPeanut: Arachis ipaensis V1\r\nRice: O. sativa L. cv. Nipponbare (assembly v. 204)\r\nSoybean: Glycine max 275 Williams 82 a2 v1",
        Reference:
          "Brazelton VA Jr, Zarecor S, Wright DA, Wang Y, Liu J, Chen K, Yang B, Lawrence-Dill CJ A quick guide to CRISPR sgRNA design tools. GM Crops Food. 2015;6(4):266-76. doi: 10.1080/21645698.2015.1137690. (PMID: 26745836)",
        __EMPTY: "http://cbc.gdcb.iastate.edu/cgat/",
      },
      {
        "Search by": "Sequence",
      },
      {
        Tools: "CRISPR/Cas9 sgRNA Design",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Command-line",
        "Off‑targets": "Yes",
        "Score Oligos": "No",
        "Search by": "GeneID",
        Enzyme: "custom",
        PAM: "Custom",
        Organisms: "Provided by user",
        Reference:
          "Brocal I, White RJ, Dooley CM, Carruthers SN, Clark R, Hall A, Busch-Nentwich EM, Stemple DL, Kettleborough RN Efficient identification of CRISPR/Cas9-induced insertions/deletions by direct germline screening in zebrafish. BMC Genomics. 2016 Mar 24;17:259. doi: 10.1186/s12864-016-2563-z. (PMID: 27009152)",
        Comments:
          "Sites can be filtered based on known SNP data. Allows designing PCR primers for screening and post-analysis of MiSeq sequencing data. Oligos are scored for genomic location: exon-oligos score highest, intron oligos lower, intergenic lowest. Oligo scoring for targeting efficiency is not performed.",
        __EMPTY: "https://github.com/richysix/Crispr",
      },
      {
        Purpose: "Post-analysis",
        "Search by": "Coordinate",
      },
      {
        Tools: "DRSC Find CRISPRs",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": "Yes",
        "Score Oligos": "Yes",
        "Search by": "GeneID",
        Enzyme: "SpCas9 (S. pyogenes)",
        PAM: "NGG",
        Organisms: "Drosophila melanogaster",
        Reference:
          "Ren X, Sun J, Housden BE, Hu Y, Roesel C, Lin S, Liu LP, Yang Z, Mao D, Sun L, Wu Q, Ji JY, Xi J, Mohr SE, Xu J, Perrimon N, Ni JQ Optimized gene editing technology for Drosophila melanogaster using germ line-specific Cas9. Proc Natl Acad Sci U S A. 2013 Nov 19;110(47):19012-7. doi: 10.1073/pnas.1318481110. Epub 2013 Nov 4. (PMID: 24191015)",
        __EMPTY: "https://www.flyrnai.org/crispr/",
      },
      {
        "Search by": "Coordinate",
        Reference:
          "Housden BE, Valvezan AJ, Kelley C, Sopko R, Hu Y, Roesel C, Lin S, Buckner M, Tao R, Yilmazel B, Mohr SE, Manning BD, Perrimon N Identification of potential drug targets for tuberous sclerosis complex by synthetic screens combining CRISPR-based knockouts with RNAi. Sci Signal. 2015 Sep 8;8(393):rs9. doi: 10.1126/scisignal.aab3729. (PMID: 26350902)",
      },
      {
        Tools: "getCRISPRY",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Command-line",
        "Off‑targets": "Yes",
        "Score Oligos": "Yes",
        "Search by": "GeneID",
        Enzyme: "SpCas9 (S. pyogenes)",
        PAM: "NGG",
        Organisms: "Provided by user",
        __EMPTY: "https://github.com/asistradition/getCRISPRY",
      },
      {
        "Search by": "Coordinate",
      },
      {
        "Search by": "Sequence",
      },
      {
        Tools: "CHOMP",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Command-line",
        "Off‑targets": "Yes",
        "Score Oligos": "No",
        "Search by": "Sequence",
        Enzyme: "SpCas9 (S. pyogenes)",
        PAM: "NGG",
        Organisms: "Provided by user",
        __EMPTY: "https://github.com/bretonics/CHOMP",
      },
      {
        Purpose: "Other",
      },
      {
        Tools: "GT-Scan",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": "Yes",
        "Score Oligos": "Yes",
        "Search by": "Sequence",
        Enzyme: "SpCas9\r\nSaCas9\r\nCas12a\r\n",
        PAM: "NGG\r\nNNGRRT\r\nTTTN\r\n",
        Organisms:
          "Anole lizard\r\nArabidopsis thaliana\r\nCaenorhabditis elegans \r\nCanis familiaris\r\nCapsicum annunm\r\nCavia porcellus\r\nCiona intestinalis \r\nCiona savignyi \r\nColumba livia\r\nDanio rerio\r\nDrosophila melanogaster\r\nEquus caballus\r\nEscherichia coli K-12\r\nFelis catus\r\nGadus morhua\r\nGallus gallus\r\nGlycine max\r\nHomo sapiens\r\nHordeum vulgare\r\nLamprey\r\nLancelet\r\nMedaka\r\nMegabat\r\nMicrobat\r\nMicrocebus  murinus \r\nMus musculus\r\nOrnithorhynchus anatinus\r\nOryctolagus cuniculus\r\nOryza sativa Indica\r\nOryza sativa Japonica\r\nPan troglodytes\r\nPenicillium brevicompactum\r\nPenicillium oxalicumi\r\nPhyscomitrella patens\r\nPika pika\r\nRattus norvegicus\r\nSaccharomyces cerevisiae\r\nSarcophilus harrisii\r\nSetaria italica\r\nSetaria viridis\r\nSolanum  lycopersicum \r\nStickleback\r\nStreptomyces ambofaciens\r\nSus scrofa\r\nTalaromyces aculeatus\r\nTrichoderma reesei\r\nWallaby\r\nXenopus tropicalis\r\nZea mays",
        Reference:
          "O'Brien A, Bailey TL GT-Scan: identifying unique genomic targets. Bioinformatics. 2014 Sep 15;30(18):2673-5. doi: 10.1093/bioinformatics/btu354. Epub 2014 May 23. (PMID: 24860161)",
        __EMPTY: "https://gt-scan.csiro.au/gt-scan",
      },
      {
        Tools: "CHOPCHOP",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": "Yes",
        "Score Oligos": "Yes",
        "Search by": "GeneID",
        Enzyme: "Cas9\r\nCas9 nickase\r\nCpf1\r\nTALEN",
        PAM: "Custom",
        Organisms:
          "Acanthamoeba castellanii (GCF_000313135.1)\r\nAcyrthosiphon pisum (Acyr_2.0)\r\nAedes aegypti (AaegL3)\r\nAedes aegypti (AaegL5)\r\nAedes albopictus (GCA_001876365.2)\r\nAfrican Swine Fever Virus Kenya (IX 1033)\r\nAmphimedon queenslandica (GCF_000090795.1)\r\nAnabaena (PCC 7120)\r\nAnopheles albimanus (AalbS2)\r\nAnopheles funestus (AfunF1)\r\nAnopheles gambiae (AgamP4)\r\nAnopheles gambiae (anoGam1)\r\nAnopheles stephensi (Indian, AsteI2)\r\nAnopheles stephensi (SDA-500, AsteS1)\r\nApis mellifera (HAv3.1)\r\nAquilegia coerulea (v3.1)\r\nArabidopsis thaliana (TAIR10)\r\nArachis duranensis (Aradu 1.0)\r\nArachis ipaensis (Araip 1.0)\r\nArthrobotrys oligospora (GCF_000225545.1)\r\nAspergillus fumigatus (Af293)\r\nAspergillus niger (ATCC 1015)\r\nAstatotilapia burtoni (AstBur1.0)\r\nAstyanax mexicanus (AstMex102)\r\nAutographa californica nucleopolyhedrovirus (AcMNPV) (NC_001623.1)\r\nAzotobacter vinelandii DJ (NC_012560.1)\r\nBacillus RZ2MS9 (MJBF01)\r\nBacillus subtilis 168 (NC_000964.3)\r\nBacteroides fragilis 638R (FQ312004.1)\r\nBacteroides fragilis 9343 (NC_003228.3)\r\nBacteroides thetaiotaomicron (FR901300.1)\r\nBactrocera cucurbitae (ASM80634v1)\r\nBactrocera dorsalis (ASM78921v2)\r\nBactrocera oleae (GCA_001188975.2)\r\nBdellovibrio bacteriovorus (ASM19617v1)\r\nBetta splendens (v1.1)\r\nBiomphalaria glabrata (BglaB1)\r\nBombyx mori (release 31)\r\nBos taurus (bosTau9)\r\nBos taurus (Btau 5.0)\r\nBotrytis cinerea (ASM83294v1)\r\nBovine diarrhea virus (NC_001461.1)\r\nCaenorhabditis elegans (ce10/WS220)\r\nCallithrix jacchus (C_jacchus3.2.1)\r\nCamponotus floridanus (GCF_003227725.1)\r\nCandida albicans (A22)\r\nCandida auris (GCA_002759435.2)\r\nCandida glabrata (CBS138)\r\nCandida krusei (GCA_003054445.1)\r\nCandida lusitaniae (GCA_000003835.1)\r\nCandida parapsilosis (GCA_000182765.2)\r\nCandida tropicalis (MYA-3404)\r\nCandidatus Carsonella ruddii (NC_018417.1)\r\nCanis familiaris (CanFam3)\r\nCannabis sativa (v2)\r\nCardamine hirsuta (v1.0)\r\nCaulobacter crescentus (NA1000)\r\nCeratitis capitata (Ccap1.0)\r\nChinese Hamster Ovary (GCF_000223135.1/CriGri_1.0)\r\nChlamydomonas reinhardtii (v5.6)\r\nChlorella sorokiniana (CSI2_1230)\r\nChrysanthemum seticuspe (CSE_r1.0)\r\nCicer arietinum (ASM33114v1)\r\nCiona intestinalis (KH2012)\r\nClostridium acetobutylicum ATCC 824 (GCF_000008765.1)\r\nClostridium beijerinckii 6423 (GCF_900010805.1)\r\nClostridium botulinum ATCC 19397 (NC_009697.1)\r\nCochliobolus heterostrophus C5 (v2.0)\r\nCoffea canephora (v1.0)\r\nCollectotrichum graminicola M1.001 (Colgr1)\r\nCorynebacterium glutamicum ATCC 13032 (GCF_000011325.1)\r\nCorynebacterium pseudotuberculosis 1002 (GCF_000144935.1)\r\nCoturnix japonica (GCF_001577835.1)\r\nCoxiella burnetii RSA 493 (GCF_000007765.2)\r\nCulex quinquefasciatus (CpipJ2)\r\nCupriavidus malaysiensis USMAA1020 chr1 (CP017754.1)\r\nCupriavidus malaysiensis USMAA1020 chr2 (CP017754.1)\r\nCyprinodon variegatus (GCF_000732505.1)\r\nDalotia coriaria (Dcor_v1)\r\nDanio rerio (danRer10/GRCz10)\r\nDanio rerio (danRer11/GRCz11)\r\nDendrobium catenatum (ASM160598v2)\r\nDiaphorina Citri (GCF_000475195.1)\r\nDrosophila melanogaster (dm3)\r\nDrosophila melanogaster (dm6)\r\nDrosophila suzukii (Dsuz 1.0)\r\nEmiliania huxleyi (CCMP1516)\r\nEnterobacteria phage lambda (J02459.1)\r\nEnterococcus faecalis (GCF_000007785.1)\r\nEnterococcus faecalis OG1RF (GCA_000172575.2)\r\nEquus caballus (EquCab2)\r\nEragrostis tef (1.1.2)\r\nEscherichia coli (C321.ΔA/GCA_000474035.1)\r\nEscherichia coli (Nissle 1917/CP007799.1)\r\nEscherichia coli (str. CFT073/ASM744v1)\r\nEscherichia coli (str. K-12/MG1655)\r\nEucalyptus grandis (v1.0)\r\nEutrema salsugineum (v1.0)\r\nGallid herpesvirus 1 (NC_006623.1)\r\nGallid herpesvirus 2 (str. CVI988, DQ530348.1)\r\nGallid herpesvirus 3 (str. SB-1, HQ840738.1)\r\nGallus gallus (galGal4)\r\nGallus gallus (galGal5)\r\nGasterosteus aculeatus (gasAcu1)\r\nGlycine max (GCA_000004515.3)\r\nHarpegnathos saltator (GCF_003227715.1)\r\nHIV-1 (AF033819.3)\r\nHomo sapiens (hg19/GRCh37)\r\nHomo sapiens (hg38/GRCh38)\r\nHuman herpesvirus 1 (str. 17, JN555585.1)\r\nHuman herpesvirus 2 (str. 333)\r\nHuman herpesvirus 2 (str. HG52)\r\nHuman herpesvirus 2 (str. SD90e)\r\nHuman herpesvirus 4 (str. Akata)\r\nHuman herpesvirus 6A (U1102_X83413)\r\nHuman herpesvirus 6B (Z29_NC000898)\r\nIxodes scapularis (IscaW1)\r\nLactobacillus delbrueckii (ATCC 11842)\r\nLarimichthys crocea (GCA_900246015.1)\r\nLegionella pneumophila (GCF_000008485.1)\r\nLeishmania braziliensis (MHOM/BR/75/M2904)\r\nLeishmania major Friedlin (LmjF.01)\r\nLepisosteus oculatus (LepOcu1)\r\nLeptospira biflexa (GCF_000017685.1)\r\nLutzomyia longipalpis (LlonJ1.5)\r\nLysobacter enzymogenes (NZ_CP013140.1)\r\nLytechinus variegatus (Lvar_2.2)\r\nManduca sexta (Mansex1.0)\r\nMayetiola destructor (v1.0)\r\nMeleagrid herpesvirus 1 strain FC126 (AF291866.1)\r\nMetarhizium anisopliae (MAN 1.0)\r\nMnemiopsis leidyi (GCA_000226015.1)\r\nMortierella alpine ATCC 32222 (GCA_000240685.2)\r\nMus musculus (mm10/GRCm38)\r\nMustela putorius furo (MusPutFur1.0/musFur1)\r\nMycobacterium marinum M (ASM1834v1)\r\nMycobacterium smegmatis MC2 155 (GCF_000015005.1)\r\nMycobacterium tuberculosis H37Rv (ASM19595v2)\r\nNaegleria gruberi (GCF_000004985.1)\r\nNannochloropsis oceanica (IMET1.v2)\r\nNasonia vitripennis (Nvit2.1)\r\nNematostella vectensis (nemVec1)\r\nNeurospora crassa (GCA_000182925.2)\r\nNothobranchius furzeri (NotFur1)\r\nNothobranchius furzeri v2 (GCF_001465895.1)\r\nN. vitripennis (Nvit1.0)\r\nOctopus bimaculoides (PRJNA270931)\r\nOgataea polymorpha NCYC495 leu1.1 (Hanpo2)\r\nOikopleura dioica (Odioica)\r\nOncorhynchus mykiss (Omyk_1.0)\r\nOreochromis niloticus (ASM185804v2)\r\nOrientia tsutsugamushi (GCF_900327275.1)\r\nOryctolagus cuniculus (GCA_000003625.1)\r\nOryza sativa (MSU-version_7.0)\r\nOryzias latipes (ASM223467v1)\r\nOryzias latipes (oryLat2)\r\nOvis aries (Oar_v3.1)\r\nPaecilomyces variotii (CBS101075)\r\nPanicum virgatum (Panicum virgatum-v4)\r\nPantoea agglomerans (NZ_CP014129.1)\r\nPapilio polytes (Ppolytes v1)\r\nPenicillium roqueforti (FM164)\r\nPhaeodactylum tricornutum (ASM15095v2)\r\nPhalaenopsis equestris (ASM126359v1)\r\nPhlebotomus papatasi (Ppapl1.5)\r\nPhyscomitrella patens (Ppatens_318_v3)\r\nPichia pastoris (v2 CBS 7435)\r\nPlasmodium berghei (ANKA v2.0)\r\nPlasmodium cynomolgi (GCF_000321355.1)\r\nPlasmodium falciparum (3D7 v3.0)\r\nPochonia chlamydosporia (GCF_001653235.2)\r\nPopulus deltoides (WV94 v2.1)\r\nPopulus trichocarpa (v3.1)\r\nPristionchus pacificus (TAU2011)\r\nPseudomonas aeruginosa (UCBPP-PA14)\r\nPseudomonas cichorii JBC1 (PRJNA232591)\r\nPseudonitzschia multistriata (PsnmuV1.4)\r\nPurpureocillium lilacinum (GCA_001653205.1)\r\nRattus norvegicus (rn6)\r\nRhodnius prolixus (RproC3)\r\nRhodobacter capsulatus SB 1003 (CP001312.1)\r\nRhodobacter sphaeroides (KD131)\r\nRhodopseudomonas palustris CGA009 (NC_005296.1)\r\nSaccharomyces cerevisiae (sacCer3/S288c)\r\nSalmonella enterica subsp. (GCF_000022165.1)\r\nSalmo salar (GCF_000233375.1)\r\nSchistosoma mansoni (Smansoni_v7)\r\nSchizosaccharomyces pombe (ASM294v2.30)\r\nSchmidtea mediterranea (schMed4)\r\nShigella flexneri 1c (CP020753)\r\nSolanum lycopersicum (SL3.0)\r\nSorghum bicolor (Phytozome v3.1.1)\r\nStaphylococcus aureus (MRSA-ATCC 43300, v1.0)\r\nStaphylococcus aureus (MSSA-GP01, v1.0)\r\nStarmerella bombicola (JCM_9596_assembly_v001)\r\nStreptococcus pyogenes M1 GAS (GCA_000321355.1)\r\nStreptomyces platensis (DSM-40041)\r\nStrongylocentrotus purpuratus (EchinoBase v3.1)\r\nSus scrofa (Sscrofa10.2)\r\nSus scrofa (Sscrofa11.1)\r\nTetrahymena thermophila (MAC)\r\nTetrahymena thermophila (MIC)\r\nTetranychus urticae (Sept 2009)\r\nThermoanaerobacter ethanolicus JW 200 (GCF_003722315.1)\r\nToxoplasma gondii GT1 (ToxoDB-28)\r\nToxoplasma gondii ME49 (ToxoDB-28)\r\nTreponema caldarium (GCF_000219725.1)\r\nTribolium castaneum (Tcas5)\r\nTrichoderma harzianum T22 (TriharT22)\r\nTrichoderma reesei (GCA_000167675.2)\r\nTriticum aestivum (IWGSC v1.1)\r\nTrypanosoma brucei (TREU927)\r\nTrypanosoma cruzi (TcruziSylvioX10-1-2012)\r\nVaccinia virus (str. WR/AY243312.1)\r\nXenopus laevis (v9.2)\r\nXenopus laevis (Xenla9.1/JGI v9.1)\r\nXenopus tropicalis (v9.1)\r\nXenopus tropicalis (xenTro9.0/JGI v9.0)\r\nYarrowia lipolytica (GCF_000002525.2)\r\nYarrowia lipolytica (W29)\r\nZea mays (B73 AGPv4)",
        Reference:
          "Montague TG, Cruz JM, Gagnon JA, Church GM, Valen E CHOPCHOP: a CRISPR/Cas9 and TALEN web tool for genome editing. Nucleic Acids Res. 2014 Jul;42(Web Server issue):W401-7. doi: 10.1093/nar/gku410. Epub 2014 May 26. (PMID: 24861617)",
        __EMPTY: "https://chopchop.cbu.uib.no/",
      },
      {
        Platform: "Command-line",
        "Search by": "Coordinate",
        Reference:
          "Labun K, Montague TG, Gagnon JA, Thyme SB, Valen E CHOPCHOP v2: a web tool for the next generation of CRISPR genome engineering. Nucleic Acids Res. 2016 Jul 8;44(W1):W272-6. doi: 10.1093/nar/gkw398. Epub 2016 May 16. (PMID: 27185894)",
      },
      {
        "Search by": "Sequence",
        Reference:
          "Labun K, Montague TG, Krause M, Torres Cleuren YN, Tjeldnes H, Valen E CHOPCHOP v3: expanding the CRISPR web toolbox beyond genome editing. Nucleic Acids Res. 2019 Jul 2;47(W1):W171-W174. doi: 10.1093/nar/gkz365. (PMID: 31106371)",
      },
      {
        Tools: "CRISPR MultiTargeter",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": "Yes",
        "Score Oligos": "Yes",
        "Search by": "GeneID",
        Enzyme: "Custom",
        PAM: "Custom",
        Organisms:
          "Homo sapiens\r\nMus musculus\r\nRattus norvegicus\r\nGallus gallus\r\nXenopus tropicalis\r\nDanio rerio\r\nOryzias latipes\r\nDrosophila melanogaster\r\nCaenorhabditis elegans\r\nArabidopsis thaliana\r\nOryza sativa japonica\r\nZea mays",
        Reference:
          "Prykhozhij SV, Rajan V, Gaston D, Berman JN CRISPR multitargeter: a web tool to find common and unique CRISPR single guide RNA targets in a set of similar sequences. PLoS One. 2015 Mar 5;10(3):e0119372. doi: 10.1371/journal.pone.0119372. eCollection 2015. (PMID: 25742428)",
        __EMPTY: "http://www.multicrispr.net/",
      },
      {
        "Search by": "Sequence",
      },
      {
        Tools: "Genedata Selector",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Other",
        "Off‑targets": "Yes",
        "Score Oligos": "No",
        "Search by": "Sequence",
        Enzyme: "Custom",
        PAM: "Custom",
        Organisms: "Provided by user",
        Comments: "Commercial",
        __EMPTY:
          "https://www.genedata.com/products-services/selector/applications",
      },
      {
        Tools: "CUNE",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": "No",
        "Score Oligos": "Yes",
        "Search by": "Coordinate",
        Enzyme: "Custom",
        PAM: "Custom",
        Organisms: "Mus musculus",
        Reference:
          "O'Brien AR, Wilson LOW, Burgio G, Bauer DC Unlocking HDR-mediated nucleotide editing by identifying high-efficiency target sites using machine learning. Sci Rep. 2019 Feb 26;9(1):2788. doi: 10.1038/s41598-019-39142-0. (PMID: 30808944)",
        Comments: "Api available, base editor, nucleotide editor, HDR",
        __EMPTY: "https://gt-scan.csiro.au/cune/Mus_musculus",
      },
      {
        Purpose: "Other",
      },
      {
        Tools: "GUIDEseq",
        Availability: "Yes",
        Purpose: "Post-analysis",
        Platform: "Command-line",
        "Off‑targets": "Yes",
        "Score Oligos": "Yes",
        "Search by": "Other",
        Enzyme: "Custom",
        PAM: "Custom",
        Organisms: "Provided by user",
        Reference:
          "Zhu LJ, Lawrence M, Gupta A, Pages H, Kucukural A, Garber M, Wolfe SA GUIDEseq: a bioconductor package to analyze GUIDE-Seq datasets for CRISPR-Cas nucleases. BMC Genomics. 2017 May 15;18(1):379. doi: 10.1186/s12864-017-3746-y. (PMID: 28506212)",
        Comments:
          "R library for processing GUIDE-seq data. Organism availability depends on BSgenome R package.",
        __EMPTY:
          "https://bioconductor.org/packages/release/bioc/html/GUIDEseq.html",
      },
      {
        Tools: "guideseq",
        Availability: "Yes",
        Purpose: "Post-analysis",
        Platform: "Command-line",
        "Off‑targets": "Yes",
        "Score Oligos": "?",
        "Search by": "Sequence",
        PAM: "Custom",
        Organisms: "Provided by user",
        Reference:
          "Tsai SQ, Topkar VV, Joung JK, Aryee MJ Open-source guideseq software for analysis of GUIDE-seq data. Nat Biotechnol. 2016 May 6;34(5):483. doi: 10.1038/nbt.3534. (PMID: 27153277)",
        Comments: "Python library for processing GUIDE-seq data",
        __EMPTY: "https://github.com/aryeelab/guideseq",
      },
      {
        Tools: "CrisprZ",
        Availability: "Yes",
        Purpose: "Database",
        Platform: "Web",
        "Off‑targets": "No",
        "Score Oligos": "Yes",
        "Search by": "GeneID",
        Enzyme: "not applicable",
        PAM: "Not Applicable",
        Organisms: "Danio rerio",
        Reference:
          "Varshney GK, Zhang S, Pei W, Adomako-Ankomah A, Fohtung J, Schaffer K, Carrington B, Maskeri A, Slevin C, Wolfsberg T, Ledin J, Sood R, Burgess SM CRISPRz: a database of zebrafish validated sgRNAs. Nucleic Acids Res. 2016 Jan 4;44(D1):D822-6. doi: 10.1093/nar/gkv998. Epub 2015 Oct 4. (PMID: 26438539)",
        Comments:
          "It provides links to the literature related to each CRISPR sequence. It allows users to add new sequences (moderated).",
        __EMPTY: "https://research.nhgri.nih.gov/CRISPRz/",
      },
      {
        "Search by": "Other",
      },
      {
        Tools: "CrisprGE",
        Availability: "Yes",
        Purpose: "Database",
        Platform: "Web",
        "Off‑targets": "No",
        "Score Oligos": "Yes",
        "Search by": "GeneID",
        Enzyme: "not applicable",
        PAM: "Not Applicable",
        Organisms:
          "Arabidopsis thaliana\r\nBombyx mori\r\nCaenorhabditis elegans\r\nCapra aegagrus hircus\r\nCiona intestinalis\r\nCitrus sinensis\r\nCricetulus griseus\r\nDanio rerio\r\nDaphnia\r\nDrosophila melanogaster\r\nFundulus heteroclitus\r\nGlycine max\r\nHerpes simplex virus\r\nHomo sapiens\r\nMus musculus\r\nNematostella vectensis\r\nNicotiana benthamiana\r\nNicotiana tabacum\r\nOreochromis niloticus\r\nOryza sativa\r\nOryzias latipes\r\nPlasmodium falciparum\r\nPorcine\r\nRattus norvegicus\r\nSaccharomyces cerevisiae\r\nSolanum lycopersicum\r\nStreptomyces coelicolor\r\nSurrogate reporter\r\nToxoplasma gondii\r\nTriticum aestivum\r\nXenopus tropicalis\r\nZea mays",
        Reference:
          "Kaur K, Tandon H, Gupta AK, Kumar M CrisprGE: a central hub of CRISPR/Cas-based genome editing. Database (Oxford). 2015 Jun 27;2015:bav055. doi: 10.1093/database/bav055. Print 2015. (PMID: 26120138)",
        Comments:
          "Contains 4680 sequence entries for 223 unique target genes. It provides links to the literature for each CRISPR sequence.",
        __EMPTY: "http://crdd.osdd.net/servers/crisprge/",
      },
      {
        "Search by": "Sequence",
      },
      {
        "Search by": "Other",
      },
      {
        Tools: "Toronto Knockout Library",
        Availability: "Yes",
        Purpose: "Database",
        Platform: "Web",
        "Off‑targets": "No",
        "Score Oligos": "Yes",
        "Search by": "GeneID",
        Enzyme: "not applicable",
        PAM: "Not Applicable",
        Organisms: "Homo sapiens",
        Reference:
          "Hart T, Tong AHY, Chan K, Van Leeuwen J, Seetharaman A, Aregger M, Chandrashekhar M, Hustedt N, Seth S, Noonan A, Habsid A, Sizova O, Nedyalkova L, Climie R, Tworzyanski L, Lawson K, Sartori MA, Alibeh S, Tieu D, Masud S, Mero P, Weiss A, Brown KR, Usaj M, Billmann M, Rahman M, Constanzo M, Myers CL, Andrews BJ, Boone C, Durocher D, Moffat J Evaluation and Design of Genome-Wide CRISPR/SpCas9 Knockout Screens. G3 (Bethesda). 2017 Aug 7;7(8):2719-2727. doi: 10.1534/g3.117.041277. (PMID: 28655737)",
        Comments:
          "Current version (TKOv3) is accessible through a genomic viewer (https://crispr.ccbr.utoronto.ca/public/library/TKOv3/) . A file containing a previous version (TKOv1) of the database can be downloaded.",
        __EMPTY: "http://tko.ccbr.utoronto.ca/",
      },
      {
        Tools: "CRISPY-yeast",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": "Yes",
        "Score Oligos": "No",
        "Search by": "GeneID",
        Enzyme: "SpCas9 (S. pyogenes)",
        PAM: "NGG",
        Organisms: "Saccharomyces cerevisiae S288C",
        Reference:
          "Jakociunas T, Bonde I, Herrgard M, Harrison SJ, Kristensen M, Pedersen LE, Jensen MK, Keasling JD Multiplex metabolic pathway engineering using CRISPR/Cas9 in Saccharomyces cerevisiae. Metab Eng. 2015 Mar;28:213-222. doi: 10.1016/j.ymben.2015.01.008. Epub 2015 Jan 28. (PMID: 25638686)",
        __EMPTY: "http://staff.biosustain.dtu.dk/laeb/crispy_yeast/",
      },
      {
        Tools: "GUIDE Picker",
        Availability: "Yes",
        Purpose: "Database",
        Platform: "Web",
        "Off‑targets": "No",
        "Score Oligos": "Yes",
        "Search by": "GeneID",
        Enzyme: "SpCas9",
        PAM: "NRG",
        Organisms: "Homo sapiens, Mus musculus",
        Reference:
          "Hough SH, Kancleris K, Brody L, Humphryes-Kirilov N, Wolanski J, Dunaway K, Ajetunmobi A, Dillard V Guide Picker is a comprehensive design tool for visualizing and selecting guides for CRISPR experiments. BMC Bioinformatics. 2017 Mar 14;18(1):167. doi: 10.1186/s12859-017-1581-4. (PMID: 28288556)",
      },
      {
        Tools: "CRISPR Surf",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": "No",
        "Score Oligos": "No",
        "Search by": "Coordinate",
        Enzyme: "Custom",
        PAM: "Custom",
        Organisms: "Homo Sapiens",
        Reference:
          "Hsu JY, Fulco CP, Cole MA, Canver MC, Pellin D, Sher F, Farouni R, Clement K, Guo JA, Biasco L, Orkin SH, Engreitz JM, Lander ES, Joung JK, Bauer DE, Pinello L CRISPR-SURF: discovering regulatory elements by deconvolution of CRISPR tiling screen data. Nat Methods. 2018 Dec;15(12):992-993. doi: 10.1038/s41592-018-0225-6. (PMID: 30504875)",
        Comments:
          "CRISPR-SURF can be run as a stand-alone\r\ncommand line utility (https://github.com/\r\npinellolab/CRISPR-SURF) or as a web\r\napplication (http://crisprsurf.pinellolab.org/)",
      },
      {
        Purpose: "Post-analysis",
        Platform: "Command-line",
      },
      {
        Tools: "CINDEL",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": "No",
        "Score Oligos": "Yes",
        "Search by": "Sequence",
        Enzyme: "Not applicable",
        PAM: "Not Applicable",
        Organisms: "Not Applicable",
        Reference:
          "Kim HK, Song M, Lee J, Menon AV, Jung S, Kang YM, Choi JW, Woo E, Koh HC, Nam JW, Kim H In vivo high-throughput profiling of CRISPR-Cpf1 activity. Nat Methods. 2017 Feb;14(2):153-159. doi: 10.1038/nmeth.4104. Epub 2016 Dec 19. (PMID: 27992409)",
        Comments:
          "A tool to predict and score indel frequencies for a given target sequence of the AsCpf1 caspase",
        __EMPTY: "http://big.hanyang.ac.kr/cindel/",
      },
      {
        Tools:
          "GeneArt CRISPR Search and Design Tool (from Thermo Fisher Scientific)",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": "Yes",
        "Score Oligos": "Yes",
        "Search by": "GeneID",
        Enzyme: "SpCas9",
        PAM: "NGG",
        Organisms: "Human, mouse, rat, zebrafish, roundworm",
        Comments:
          "Design CRISPRs de novo or search CRISPR targets from pre-designed database of > 600,000 gRNAs. Output include PAM sites, potential binding sites, exon maps with gRNA locations, and recommendations based on potential off-target effects for each CRISPR sequence. The tool will design the correct gRNA format for your preferred CRISPR-Cas9 product format. ",
      },
      {
        Purpose: "Database",
        "Search by": "Sequence",
      },
      {
        Tools: "CRISPR-RT",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": "No",
        "Score Oligos": "No",
        "Search by": "Sequence",
        Enzyme: "Cas13a (C2c2)",
        PAM: "Custom",
        Organisms: "Provided by user",
        Reference:
          "Zhu H, Richmond E, Liang C CRISPR-RT: a web application for designing CRISPR-C2c2 crRNA with improved target specificity. Bioinformatics. 2018 Jan 1;34(1):117-119. doi: 10.1093/bioinformatics/btx580. (PMID: 28968770)",
        Comments:
          "For RNA targeting. Desktop version downloadable from https://github.com/bioinfolabmu/CRISPR-RT.",
        __EMPTY: "http://bioinfolab.miamioh.edu/CRISPR-RT/",
      },
      {
        Platform: "Desktop",
      },
      {
        Tools: "OutKnocker",
        Availability: "Yes",
        Purpose: "Post-analysis",
        Platform: "Web",
        "Off‑targets": "No",
        "Score Oligos": "No",
        "Search by": "Other",
        Enzyme: "Not applicable",
        PAM: "Not Applicable",
        Organisms: "Homo sapiens , Mus musculus",
        Reference:
          "Schmid-Burgk JL, Schmidt T, Gaidt MM, Pelka K, Latz E, Ebert TS, Hornung V OutKnocker: a web tool for rapid and simple genotyping of designer nuclease edited cell lines. Genome Res. 2014 Oct;24(10):1719-23. doi: 10.1101/gr.176701.114. Epub 2014 Sep 3. (PMID: 25186908)",
        Comments:
          "Easy-to-follow deep sequencing workflow which allows convenient, reliable and cost-effective identification of knock-out cell lines",
      },
      {
        Tools: "MaGeCKFlute",
        Availability: "Yes",
        Purpose: "Post-analysis",
        Platform: "Command-line",
        "Off‑targets": "No",
        "Score Oligos": "No",
        "Search by": "Other",
        Enzyme: "Not applicable",
        PAM: "Not Applicable",
        Organisms: "Not Applicable",
        Reference:
          "Wang B, Wang M, Zhang W, Xiao T, Chen CH, Wu A, Wu F, Traugh N, Wang X, Li Z, Mei S, Cui Y, Shi S, Lipp JJ, Hinterndorfer M, Zuber J, Brown M, Li W, Liu XS Integrative analysis of pooled CRISPR genetic screens using MAGeCKFlute. Nat Protoc. 2019 Mar;14(3):756-780. doi: 10.1038/s41596-018-0113-7. Epub 2019 Feb 1. (PMID: 30710114)",
        Comments:
          "R package to perform quality control (QC), normalization, batch effect removal, gene hit identification and downstream functional enrichment analysis for CRISPR screens.",
        __EMPTY: "https://github.com/WubingZhang/MAGeCKFlute",
      },
      {
        Tools: "AsCRISPR",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": "Yes",
        "Score Oligos": "Yes",
        "Search by": "GeneID",
        Enzyme:
          "SpCas9\r\nSpCas9-VRER\r\nSpCas9-EQR\r\nSpCas9-V(R)QR\r\nSaCas9-KKH\r\nNmCas9\r\nStCas9\r\nTdCas9\r\nCjCas9\r\nAs(Lb)Cpf1\r\nFnCpf1\r\nAsCpf1-RR\r\nAsCpf1-RVR\r\nAaCas12b\r\nAkCas12b\r\nSpCas9\r\nSpCas9-VRER\r\nSpCas9-EQR\r\nSpCas9-V(R)QR\r\nSaCas9-KKH\r\nNmCas9\r\nStCas9\r\nTdCas9\r\nCjCas9\r\nAs(Lb)Cpf1\r\nFnCpf1\r\nAsCpf1-RR\r\nAsCpf1-RVR\r\nAaCas12b\r\nAkCas12b\r\nBhCas12b\r\nDpb(Plm)CasX",
        PAM: "NGG\r\nNGCG\r\nNGAG\r\nNGA\r\nNNGRRT (R = A or G)\r\nNNNRRT (R = A or G)\r\nNNNNGATT\r\nNNAGAAW (W = A or T)\r\nNAAAAC\r\nNNNNRYAC (R = A or G; Y = C or T )\r\nTTTV (V = A or C or G)\r\nTTV (V = A or C or G)\r\nKYTV (K = G or T; Y = C or T; V = A or C or G)\r\nTYCV (Y = C or T; V = A or C or G)\r\nTATV (V = A or C or G)\r\nTTN\r\nTTTN\r\nATTN\r\nTTCN",
        Organisms:
          "Homo sapiens (hg19/GRCh37)\r\nHomo sapiens (hg38/GRCh38)\r\nMus musculus (mm10/GRCm38)",
        Reference: "https://www.biorxiv.org/content/10.1101/672634v2",
      },
      {
        "Search by": "Sequence",
      },
      {
        "Search by": "Other",
      },
      {
        Tools: "predictSGRNA",
        Availability: "Yes",
        Purpose: "analysis of sgRNA",
        Platform: "Command-line",
        "Off‑targets": "No",
        "Score Oligos": "Yes",
        "Search by": "Other",
        Enzyme: "Not Applicable",
        PAM: "Not Applicable",
        Organisms: "Not Applicable",
        Reference:
          "Kuan PF, Powers S, He S, Li K, Zhao X, Huang B A systematic evaluation of nucleotide properties for CRISPR sgRNA design. BMC Bioinformatics. 2017 Jun 6;18(1):297. doi: 10.1186/s12859-017-1697-6. (PMID: 28587596)",
        Comments:
          "Machine learning approach for the prediction of sgRNA design efficiency",
      },
      {
        Tools: "multicrispr",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Command-line",
        "Off‑targets": "Yes",
        "Score Oligos": "Yes",
        "Search by": "GeneID",
        Enzyme: "Custom",
        PAM: "Custom",
        Organisms: "Provided by user",
        Reference:
          "Bhagwat AM, Graumann J, Wiegandt R, Bentsen M, Welker J, Kuenne C, Preussner J, Braun T, Looso M multicrispr: gRNA design for prime editing and parallel targeting of thousands of targets. Life Sci Alliance. 2020 Sep 9;3(11). pii: 3/11/e202000757. doi: 10.26508/lsa.202000757. Print 2020 Nov. (PMID: 32907859)",
        Comments: "R/Bioconductor tool",
        __EMPTY: "https://multicrispr.net/",
      },
      {
        Platform: "Other",
        "Search by": "Coordinate",
      },
      {
        Tools: "CRISP-view",
        Availability: "Yes",
        Purpose: "Database",
        Platform: "Web",
        "Off‑targets": "No",
        "Score Oligos": "Yes",
        "Search by": "GeneID",
        Enzyme: "Cas9",
        PAM: "Cas9",
        Organisms: "Human, mouse, monkey",
        Reference:
          "Cui Y, Cheng X, Chen Q, Song B, Chiu A, Gao Y, Dawson T, Chao L, Zhang W, Li D, Zeng Z, Yu J, Li Z, Fei T, Peng S, Li W CRISP-view: a database of functional genetic screens spanning multiple phenotypes. Nucleic Acids Res. 2021 Jan 8;49(D1):D848-D854. doi: 10.1093/nar/gkaa809. (PMID: 33010154)",
        Comments:
          "A comprehensive database of CRISPR/Cas9 and RNAi screening datasets that span multiple phenotypes, including in vitro and in vivo cell proliferation and viability, response to cancer immunotherapy, virus response, protein expression, etc.",
        __EMPTY: "http://crispview.weililab.org/",
      },
      {
        Tools: "OffScan",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Command-line",
        "Off‑targets": "Yes",
        "Score Oligos": "Yes",
        "Search by": "Other",
        Enzyme: "Not Applicable",
        PAM: "Not Applicable",
        Organisms: "Not Applicable",
        Reference:
          "Cui Y, Liao X, Peng S, Tang T, Huang C, Yang C OffScan: a universal and fast CRISPR off-target sites detection tool. BMC Genomics. 2020 Mar 5;21(Suppl 1):872. doi: 10.1186/s12864-019-6241-9. (PMID: 32138651)",
        Comments:
          "OffScan is not limited by the number of mismatches and allows custom protospacer-adjacent motif (PAM), which is the target site by Cas protein. Besides, OffScan adopts the FM-index, which efficiently improves query speed and reduce memory consumption.",
      },
      {
        Tools: "MHcut",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": "No",
        "Score Oligos": "Yes",
        "Search by": "GeneID",
        Enzyme: "Custom",
        PAM: "Custom",
        Organisms: "Homo sapiens",
        Reference:
          "Grajcarek J, Monlong J, Nishinaka-Arai Y, Nakamura M, Nagai M, Matsuo S, Lougheed D, Sakurai H, Saito MK, Bourque G, Woltjen K Genome-wide microhomologies enable precise template-free editing of biologically relevant deletion mutations. Nat Commun. 2019 Oct 24;10(1):4856. doi: 10.1038/s41467-019-12829-8. (PMID: 31649251)",
        Comments:
          "MHcut Browser contains deletion data from dbSNP and ClinVar, and may be searched by gene, location, or rsID. Deletions are scored by inDelphi. Export unique gRNAs to re-create deletion variants. Python script and Docker image available at: https://github.com/WoltjenLab/MHcut",
      },
      {
        Purpose: "Database",
        Platform: "Command-line",
        "Search by": "Coordinate",
      },
      {
        "Search by": "Other",
      },
    ],
    "For Plants": [
      {
        Tools: "CRISPR-P 2.0",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": "Yes",
        "Score Oligos": "Yes",
        "Search by": "GeneID",
        Enzyme:
          "NGG (SpCas9 from Streptococcus pyogenes: 5'-NGG-3')\r\nNRG (SpCas9 from Streptococcus pyogenes: 5'-NRG-3')\r\nNNAGAAW (StCas9 from Streptococcus thermophilus: 5'-NNAGAAW-3')\r\nNNNNGMTT (NmCas9 from Neisseria meningitidis: 5'-NNNNGMTT-3')\r\nNNGRRT (SaCas9 from Staphylococcus aureus: 5'-NNGRRT-'3)\r\nTTTV (AsCpf1 from Acidaminococcus or LbCpf1 from Lachnospiraceae: 5'-TTTV-3')\r\nTTN (FnCpf1 from Francisella: 5'-TTN-3')\r\nYCN (from Pyrococcus furiosus: 5'-YCN-3')\r\nCCW (from Clostridium difficile: 5'-CCW-3')\r\nYYC (from Bacillus halodurans: 5'-YYC-3')\r\nAWG (from Escherichia coli: 5'-AWG-3')\r\nCC (from Pseudomonas aeruginosa: 5'-CC-3')\r\nMMA (Pyrococcus furiosus: 5'-MMA-3')\r\nNG (xCas9 3.7: 5'-NG-3')",
        PAM: "NGG\r\nNRG\r\nNNAGAAW\r\nNNNNGMTT\r\nNNGRRT\r\nTTTV\r\nTTN\r\nYCN\r\nCCW\r\nYYC\r\nAWG\r\nCC\r\nMMA\r\nNG",
        Reference:
          "Actinidia chinensis var. chinensis\r\nAnanas comosus\r\nArabidopsis lyrata (v.1.0)\r\nArabidopsis thaliana (TAIR10)\r\nArachis duranensis(v1.0)\r\nArachis ipaensis(v1.0)\r\nBetula pendula subsp. pendula(1.4c Pseudochromosomes)\r\nBrachypodium distachyon (v1.0)\r\nBrassica juncea(v1.5)\r\nBrassica napus (v4.1)\r\nBrassica oleracea (v1.0)\r\nBrassica rapa (IVFCAASv1)\r\nCapsella_rubella(v1.0 )\r\nChlamydomonas reinhardtii (v5.5)\r\nCitrullus lanatus (v1.0)\r\nCitrus sinensis (v2.0)\r\nCoffea canephora\r\nCucumis melo (v3.5)\r\nCucumis sativus\r\nCyanidioschyzon merolae (ASM9120v1)\r\nFragaria vesca(v2.0.a1)\r\nGlycine max (V1.0)\r\nGossypium hirsutum(v1.1)\r\nGossypium raimondii\r\nLactuca sativa (v8)\r\nLentinula edodes(W1-26)\r\nLentinula_edodes(B17)\r\nLotus japonicus (v3.0)\r\nMalus_x_domestica (GDDH13_v1.1)\r\nManihot esculenta (v6.1)\r\nMarchantia polymorpha(v3.1)\r\nMedicago truncatula (Mt4.0v2)\r\nMusa acuminata (MA1)\r\nNicotiana benthamiana (v0.4.4)\r\nOryza brachyantha (v1.4b)\r\nOryza glaberrima (AGI1.1)\r\nOryza indica (ASM465v1)\r\nOryza indica (MH63)\r\nOryza indica (ZS97)\r\nOryza sativa (RAP-DB)\r\nOryza sativa(MSU)\r\nPanicum virgatum(v1.1)\r\nPhyscomitrella patens (ASM242v1)\r\nPopulus trichocarpa (JGI2.0)\r\nPrunus avium (v.1.0)\r\nRice kasalath\r\nRicinus communis\r\nSalvia miltiorrhiza\r\nSelaginella moellendorffii (v1.0)\r\nSetaria italica (JGIv2.0)\r\nSetaria viridis (v1.1)\r\nSolanum lycopersicum (SL2.50) \r\nSolanum lycopersicum (SL3.0) \r\nSolanum tuberosum (3.0)\r\nSorghum bicolor (Sorbi1)\r\nUtricularia gibba\r\nVigna unguiculata (v1.1)\r\nVitis vinifera (IGGP 12x)\r\nZea mays (AGPv3.21)\r\nZea mays (AGPv4)",
        __EMPTY:
          "Liu H, Ding Y, Zhou Y, Jin W, Xie K, Chen LL CRISPR-P 2.0: An Improved CRISPR-Cas9 Tool for Genome Editing in Plants. Mol Plant. 2017 Mar 6;10(3):530-532. doi: 10.1016/j.molp.2017.01.003. Epub 2017 Jan 13. (PMID: 28089950)",
        __EMPTY_2:
          "Specific for plant genomes, SpCas9, StCas9, NmCas9, SaCas9, AsCpf1, FnCpf1, ",
        __EMPTY_4: "http://crispr.hzau.edu.cn/CRISPR2/",
      },
      {
        "Search by": "Coordinate",
      },
      {
        "Search by": "Sequence",
      },
      {
        Tools: "CRISPR-Local",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Command-line",
        "Off‑targets": "Yes",
        "Score Oligos": "Yes",
        "Search by": "GeneID",
        Enzyme: "Custom",
        PAM: "NGG, TTTV, TTV",
        Reference:
          "Aegilops tauschii (ASM34733v1)\r\nAmaranthus hypochondriacus (v.PGA2_1_2212017)\r\nAmborella trichopoda (v1.0)\r\nAnanas comosus (v6)\r\nArabidopsis lyrata (v.1.0)\r\nArabidopsis thaliana (TAIR10)\r\nArachis duranensis (aradu.V14167.gnm1)\r\nArachis ipaensis (v2)\r\nAzolla filiculoides (v1.1)\r\nBeta vulgaris (RefBeet-1.2.2)\r\nBrachypodium distachyon (v1.0)\r\nBrachypodium stacei (v2)\r\nBrassica napus (AST_PRJEB5043_v1)\r\nBrassica oleracea (v2.1)\r\nBrassica rapa (IVFCAASv1)\r\nCalotropis gigantea (v1)\r\nCardamine hirsuta (v1)\r\nCatharanthus roseus (v1)\r\nChenopodium quinoa (v1.0)\r\nChlamydomonas reinhardtii (v3.1)\r\nCicer arietinum (v1.0)\r\nCitrus sinensis (v2.0)\r\nCucumis melo (v3.5)\r\nCucumis sativus\r\nCyanidioschyzon merolae (ASM9120v1)\r\nDaucus carota (v2)\r\nEucalyptus grandis (v2.0)\r\nFragaria vesca (v1.1)\r\nGlycine max (V1.0)\r\nGossypium hirsutum (v1.1)\r\nHelianthus annuus (r1.2)\r\nHordeum vulgare (Hv_IBSC_PGSB_v2)\r\nHumulus lupulus (v1.1)\r\nKalanchoe fedtschenkoi (v1.1)\r\nLactuca sativa (v8)\r\nLentinula edodes (W1-26)\r\nLotus japonicus (v3.0)\r\nMalus domestica (v1.0)\r\nManihot esculenta (v6)\r\nMarchantia polymorpha (v3.1)\r\nMedicago truncatula (MedtrA17_4.0)\r\nMusa acuminata (MA1)\r\nNicotiana benthamiana (v0.4.4)\r\nOryza brachyantha (v1.4b)\r\nOryza glaberrima (AGI1.1)\r\nOryza sativa (MSU-7.0)\r\nOryza sativa Indica (ASM465v1)\r\nOryza sativa Japonica (IRGSP-1.0)\r\nOryza sativa subsp. indica Minghui 63\r\nOryza sativa subsp. indica Zhenshan 97\r\nPanicum miliaceum (V1)\r\nPanicum virgatum (v4.1)\r\nPhaseolus vulgaris (V1)\r\nPhyscomitrella patens (ASM242v1)\r\nPopulus trichocarpa (JGI2.0)\r\nPrunus mume (v1)\r\nSalvinia cucullata (v1.2)\r\nSelaginella moellendorffii (v1.0)\r\nSetaria italica (v2.2)\r\nSetaria viridis (v2.1)\r\nSolanum lycopersicum (SL3.0)\r\nSolanum tuberosum (SolTub_3.0)\r\nSorghum bicolor (v2)\r\nTheobroma cacao (v1.1)\r\nTriticum aestivum (TGACv1)\r\nTriticum urartu (ASM34745v1)\r\nVigna angularis (vigan.Gyeongwon.gnm3)\r\nVigna unguiculata (v1.1)\r\nVitis vinifera (IGGP_12x)\r\nZea mays (AGPv3)\r\nZea mays (AGPv4)",
        __EMPTY:
          "Sun J, Liu H, Liu J, Cheng S, Peng Y, Zhang Q, Yan J, Liu HJ, Chen LL CRISPR-Local: a local single-guide RNA (sgRNA) design tool for non-reference plant genomes. Bioinformatics. 2018 Nov 30. pii: 5221013. doi: 10.1093/bioinformatics/bty970. (PMID: 30500879)",
        __EMPTY_2:
          "SpCas9, Cpf1. Useful to design candidate sgRNAs for full gene sets of plant genomes. It includes pre-calculated sgRNA databases for several genomes.",
        __EMPTY_4: "http://crispr.hzau.edu.cn/CRISPR-Local/",
      },
      {
        Purpose: "Database",
        Platform: "Desktop",
        "Search by": "Coordinate",
      },
      {
        "Search by": "Sequence",
      },
      {
        "Search by": "Other",
      },
      {
        Tools: "PhytoCRISP-Ex",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": "Yes",
        "Score Oligos": "No",
        "Search by": "Sequence",
        Enzyme: "SpCas9 (S. pyogenes)",
        PAM: "NGG\r\nNAG",
        Reference:
          "Phaeodactylum tricornutum \r\nChlamydomonas reinhardtii \r\nChlorella vulgaris \r\nFistulifera \r\nFragilariopsis cylindrus \r\nNannochloropsis gaditana \r\nOstreococcus lucimarinus \r\nOstreococcus tauri \r\nThalassiosira oceanica \r\nThalassiosira pseudonana \r\nEmiliania huxleyi \r\nPsuedo-nitzschia multiseries \r\nVolvox carteri",
        __EMPTY:
          "Rastogi A, Murik O, Bowler C, Tirichine L PhytoCRISP-Ex: a web-based and stand-alone application to find specific target sequences for CRISPR/CAS editing. BMC Bioinformatics. 2016 Jul 1;17(1):261. doi: 10.1186/s12859-016-1143-1. (PMID: 27363443)",
        __EMPTY_2: "Exclusively for phytoplankton genomes.",
        __EMPTY_4: "https://www.phytocrispex.bio.ens.psl.eu/CRISP-Ex/",
      },
      {
        Platform: "Command-line",
      },
      {
        Tools: "CGAT",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": "Yes",
        "Score Oligos": "Yes",
        "Search by": "GeneID",
        Enzyme: "SpCas9 (S. pyogenes)",
        PAM: "NGG",
        Reference:
          "Barley: Hordeum vulgare L. 082214v1.29\r\nMaize: Zea mays ssp. mays B73 RefGen_v3 5b+\r\nPeanut: Arachis duranensis V1\r\nPeanut: Arachis ipaensis V1\r\nRice: O. sativa L. cv. Nipponbare (assembly v. 204)\r\nSoybean: Glycine max 275 Williams 82 a2 v1",
        __EMPTY:
          "Brazelton VA Jr, Zarecor S, Wright DA, Wang Y, Liu J, Chen K, Yang B, Lawrence-Dill CJ A quick guide to CRISPR sgRNA design tools. GM Crops Food. 2015;6(4):266-76. doi: 10.1080/21645698.2015.1137690. (PMID: 26745836)",
        __EMPTY_4: "http://cbc.gdcb.iastate.edu/cgat/",
      },
      {
        "Search by": "Sequence",
      },
    ],
    "For Animals": [
      {
        Tools: "sgRNAcas9",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Command-line",
        "Off‑targets": "Yes",
        "Score Oligos": "Yes",
        "Search by": "Sequence",
        Enzyme: "Custom",
        PAM: "NGG",
        Organisms: "Provided by user",
        Reference:
          "Xie S, Shen B, Zhang C, Huang X, Zhang Y sgRNAcas9: a software package for designing CRISPR sgRNA and evaluating potential off-target cleavage sites. PLoS One. 2014 Jun 23;9(6):e100448. doi: 10.1371/journal.pone.0100448. eCollection 2014. (PMID: 24956386)",
        Comments: "Cas9",
      },
      {
        Tools: "CRISPR Design Tool",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Web-server based",
        "Off‑targets": "Yes",
        "Score Oligos": "No",
        "Search by": "GeneID",
        Enzyme: "Custom",
        PAM: "Custom",
        Organisms:
          "American Alligator (allMis1) Alligator mississippiensis\r\nApple (MalDomGD1) Malus domestica\r\nCat (felCat8) Felis catus\r\nChicken (galGal5) Gallus gallus\r\nChimpanzee (panTro4) Pan troglodytes\r\nChinese Hamster (criGriK1) Cricetulus griseus\r\nCotton (ASM98774v1) Gossypium hirsutum\r\nCow (bosTau8) Bos taurus\r\nDog (canFam3) Canus lupus familiaris\r\nFerret (musFur1) Mustela putorius furo\r\nFruit Fly (dm6) Drosophila melaNogaster\r\nFugu (fr3) Takifugu rubripes\r\nGreen Alga (chlRei_v3) Chlamydomonas reinhardtii\r\nHorse (equCab2) Equus caballus\r\nHuman (hg38) Homo sapiens\r\nJapanese Rice (IRGSP-1.0) Oryza sativa subsp. japonica\r\nMaize (zm_B73_v4) Zea mays\r\nMarmoset (calJac3) Callithrix jacchus\r\nMedaka (oryLat2) Oryzias latipes\r\nMouse (mm10) Mus musculus\r\nNile Tilapia (oreNil2) Oreochromis niloticus\r\nOlive Baboon (papAnu2) Papio anubis\r\nOpossum (monDom5) MoNodelphis domestica\r\nOrangutan (ponAbe2) Pongo abelii\r\nPig (susScr11) Sus scrofa\r\nPlatypus (ornAna2) Ornithorhynchus anatinus\r\nPurple Sea Urchin (strPur2) Strongylocentrotus purpuratus\r\nPygmy Chimpanzee (panPan1) Pan paniscus\r\nRabbit (oryCun2) Oryctolagus cuniculus\r\nRat (rn6) Rattus Norvegicus\r\nRhesus (rheMac8) Macaca mulatta\r\nRoundworm (ce11) CaeNorhabditis elegans\r\nSea Squirt (ci2) Ciona intestinalis\r\nSheep (oviAri3) Ovis aries\r\nSoybean (glyMax2) Glycine max\r\nStickleback (gasAcu1) Gasterosteus aculeatus\r\nTropical Clawed Frog (xenTro9) XeNopus tropicalis\r\nZebra Finch (taeGut2) Taeniopygia guttata\r\nZebrafish (danRer10) Danio rerio",
        __EMPTY:
          "https://horizondiscovery.com/en/ordering-and-calculation-tools/crispr-design-tool",
      },
      {
        "Search by": "Coordinate",
      },
      {
        "Search by": "Sequence",
      },
      {
        Tools: "CRISPRseek",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Command-line",
        "Off‑targets": "Yes",
        "Score Oligos": "Yes",
        "Search by": "Sequence",
        Enzyme: "Custom",
        PAM: "Custom",
        Organisms: "Provided by user",
        Reference:
          "Zhu LJ, Holmes BR, Aronin N, Brodsky MH CRISPRseek: a bioconductor package to identify target-specific guide RNAs for CRISPR-Cas9 geNome-editing systems. PLoS One. 2014 Sep 23;9(9):e108424. doi: 10.1371/journal.pone.0108424. eCollection 2014. (PMID: 25247697)",
        Comments: "Bioconductor package. Doench (2014)-based gRNA scoring.",
        __EMPTY:
          "https://bioconductor.org/packages/release/bioc/html/CRISPRseek.html",
      },
      {
        Platform: "Other",
      },
      {
        Tools: "CCTop 1.0.1",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": "Yes",
        "Score Oligos": "Yes",
        "Search by": "Sequence",
        Enzyme:
          "NGG (Streptococcus pyogenes)\r\nNRG (Streptococcus pyogenes)\r\nNGA (variant VQR Sp.)\r\nNGCG (variant VRER Sp.)\r\nTTTN (Cpf1 AcidamiNococcus or LachNospiraceae)\r\nYTN (Cas12a Francisella Novicida)\r\nTTN (Cas12a Francisella Novicida)\r\nNNGRRT (Staphylococcus aureus)\r\nNNNRRT (Staphylococcus aureus)\r\nNNNNGATT (Neisseria meningitidis)\r\nNNAGAAW (Streptococcus thermophilus)\r\nNAAAAC (Treponema denticola)",
        PAM: "NGG\r\nNRG\r\nNGA\r\nNGCG\r\nTTTN\r\nNNGRRT\r\nNNNRRT\r\nNNNNGATT\r\nNNAGAAW\r\nNAAAAC\r\nNNNNRYAC",
        Organisms:
          "ANopheles gambiae (MOZ2)\r\nArabidopsis (Arabidopsisthaliana TAIR10)\r\nAspergillus flavus (JCVI-afl1-v2.0)\r\nAxolotl (Ambystoma mexicanum ASM291563v1)\r\nBacillus megaterium (Strain ATCC 14581)\r\nBacillus sp. strain RZ2MS9\r\nBarrel Medic (Medicagotruncatula Medtr4)\r\nBrassica napus (AST_PRJEB5043_v1)\r\nBroom-corn (Sorghum bicolor v3.1)\r\nCandida albicans (SC5314)\r\nCaulobacter crescentus (NA1000)\r\nC. elegans (CaeNorhabditiselegans WBcel235)\r\nC. intestinalis (Cionaintestinalis KH2012)\r\nCardamine hirsuta (v1.0)\r\nCavefish (Astyanaxmexicanus AstMex102)\r\nChickpea (Cicer arietinum GCA_000331145.1)\r\nChickpea Desi (v3.0)\r\nChinese hamster (Cricetulusgriseus) ovary cell line\r\nChlamydomonas reinhardtii (v3.1)\r\nChromochloris zofingiensis (v5)\r\nClavibacter cf. michiganensis LMG 26808\r\nCow (Bos taurus UMD_3.1)\r\nEucalyptus grandis (JGI v2.0)\r\nEuropean hop (Humulus lupulus var. lupulus v1.0)\r\nFoxtail millet (Setaria italica v2.2)\r\nFruit fly (DrosophilamelaNogaster BDGP6/dm6)\r\nFruit fly (DrosophilamelaNogaster BDGP5/dm3)\r\nGoat (Capra aegagrus hircus ARS1)\r\nGreen foxtail (Setaria viridis v1.1)\r\nHuman (Homo sapiens GRCh38/hg38)\r\nHuman (Homo sapiens GRCh37/hg19)\r\nKluyveromyces marxianus (DMKU3-1042)\r\nLettuce (Lactuca sativa v8)\r\nMedaka(Oryzias latipes oryLat2)\r\nMouse (Mus musculus GRCm38/mm10)\r\nNanNochloropsis gaditana (B-31)\r\nNanNochloropsis oceanica 1779 (CCMP1779 v2.0)\r\nPaenibacillus larvae subsp. larvae DSM 25430\r\nPaenibacillus larvae subsp. larvae DSM 25719\r\nPanicum virgatum (V1.0)\r\nPantoea agglomerans (FDAARGOS_160)\r\nPapaya (Carica papaya ASGPBv0.4)\r\nPenicillium chrysogenum\r\nPetunia (Petunia axillaris)\r\nPig (Sus scrofa susScr3)\r\nPopulus alba (sPta717 v2)\r\nPopulus tremula (sPta717 v2)\r\nPopulus trichocarpa (JGI v3.1)\r\nPotato (Solanum tuberosum DM1-3 v4.04)\r\nPseudomonas aerugiNosa (UCBPP-PA14)\r\nRainbow trout (Oncorhynchus mykiss Omyk_1.0)\r\nRalstonia solanacearum (GMI1000)\r\nRat (Rattus Norvegicus rn6)\r\nRhesus macaque (Macacamulatta rheMac2)\r\nRice (Oryza sativa Japonica)\r\nSaccharomyces cerevisiae S288c\r\nSalmon (Salmo salar ICSASG_v2)\r\nSalt cress (Eutrema salsugineum v1.0)\r\nSoybean (Glycine max Wm82.a2.v1)\r\nSheep (Ovis aries oviAri3)\r\nSilk worm (Bombyx mori SilkDBv2.0)\r\nSpotted gar (Lepisosteus oculatus LepOcu1)\r\nStarlet sea anemone (Nematostella vectensis GCA_000209225.1)\r\nStickleback (Gasterosteusaculeatus BROADS1/gasAcu1)\r\nStreptomyces acidiscabies (NCPPB 4445)\r\nStreptomyces acidiscabies (84-104)\r\nStreptomyces mobaraensis (GCF_000342125.1)\r\nStreptomyces scabiei (87.22)\r\nStreptomyces turgidiscabies (car8)\r\nTobacco (Nicotianabenthamiana Niben101)\r\nTobacco (Nicotianatabacum Ntab-BX)\r\nTomato (Solanum lycopersicumSolyc2.5)\r\nUtricularia gibba (v4.1)\r\nX. laevis (XeNopus laevisXenla7.1)\r\nX. tropicalis (XeNopustropicalis JGI4.2/xenTro3)\r\nWater Flea (Daphnia pulex JGI 2006-09-05)\r\nWild sugarcane (Saccharum spontaneum AP85-441)\r\nYersinia enterocolitica (strain WA)\r\nZymomonas mobilis subsp. mobilis (Zm4)\r\nZebrafish (Danio rerioZv9/danRer7)\r\nZebrafish (Danio reriodanRer10)",
        Reference:
          "Stemmer M, Thumberger T, Del Sol Keyer M, Wittbrodt J, Mateo JL CCTop: An Intuitive, Flexible and Reliable CRISPR/Cas9 Target Prediction Tool. PLoS One. 2015 Apr 24;10(4):e0124633. doi: 10.1371/journal.pone.0124633. eCollection 2015. (PMID: 25909470)",
        Comments: "CCTop uses the inbuilt CRISPRater scoring algorithm",
        __EMPTY: "https://pypi.org/project/CCTop/",
      },
      {
        Platform: "Command-line",
      },
      {
        Tools: "CasFinder",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Command-line",
        "Off‑targets": "Yes",
        "Score Oligos": "No",
        "Search by": "Coordinate",
        Enzyme: "Custom",
        PAM: "Custom",
        Organisms: "Provided by user",
        Reference:
          "Aach, J., Mali, P., & Church, G. M. (2014). CasFinder: Flexible algorithm for identifying specific Cas9 targets in geNomes. BioRxiv, 005074.",
        Comments:
          "SpCas9, StCas9, NmCas9 , Includes database of Cas9 target sites for human and mouse",
        __EMPTY: "https://arep.med.harvard.edu/CasFinder/",
      },
      {
        Purpose: "Database",
        "Search by": "Sequence",
      },
      {
        Tools: "CRISPRer",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Command-line",
        "Off‑targets": "Yes",
        "Score Oligos": "No",
        "Search by": "Coordinate",
        Enzyme: "Custom",
        PAM: "Custom",
        Organisms: "Provided by user",
        Comments: "Allows installing a web server based on Galaxy",
        __EMPTY: "galaxy.informatik.uni-halle.de",
      },
      {
        Platform: "Other",
      },
      {
        Tools: "CRISPR gRNA Design Tool (ATUM)",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": "Yes",
        "Score Oligos": "No",
        "Search by": "GeneID",
        Enzyme: "SpCas9 and Nickase",
        PAM: "NGG, NAG",
        Organisms:
          "Homo sapiens, Mus musculus, Arabidopsis, E.coli , Saccharomyces cerevisae",
        Comments: "Cas9 and Nickase",
        __EMPTY: "https://www.atum.bio/eCommerce/cas9/input",
      },
      {
        "Search by": "Coordinate",
      },
      {
        "Search by": "Sequence",
      },
      {
        Tools: "CRISPOR",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": "Yes",
        "Score Oligos": "Yes",
        "Search by": "Sequence",
        Enzyme:
          "NGG - Sp Cas9, SpCas9-HF1, eSpCas9 1.1\r\nNNG - Cas9 S. canis\r\nNNGT - Cas9 S. canis\r\nNAA - iSpyMacCas9\r\nNNG(A/G)(A/G)T - Cas9 S. Aureus\r\nNG(G/T) - xCas9\r\nNGN or GA(A/T) - xCas9\r\nNNN(A/G)(A/G)T - KKH SaCas9\r\nNGA - Cas9 S. Pyogenes mutant VQR\r\nNNNNCC - Nme2Cas9\r\nNGCG - Cas9 S. Pyogenes mutant VRER\r\nNNAGAA - Cas9 S. Thermophilus\r\nNGGNG - Cas9 S. Thermophilus\r\nNNNNG(A/C)TT - Cas9 N. Meningitidis\r\nNNNNACA - Cas9 Campylobacter jejuni\r\nTTT(A/C/G)-23bp - Cas12a (Cpf1)  AcidamiNococcus / LachNospiraceae\r\nTTTN-23bp - Cas12a (Cpf1) AcidamiNococcus / LachNospiraceae",
        PAM: "NGG\r\nNAG\r\nNNGT\r\nNAA\r\nNNGRRT\r\nNGK\r\nNGN or GAR\r\nNNNRRT\r\nNGA\r\nNNNNCC\r\nNGCG\r\nNNAGAA\r\nNGGNG\r\nNNNNMTT\r\nNNNNACA\r\nTTTV\r\nTTN",
        Organisms:
          "Acanthamoeba castellanii str. Neff - Acanthamoeba castellanii Neff strain - NCBI GCF_000313135.1 (Acastellanii.strNEFF v1)\r\nActiNobacillus pleuropneumoniae KL16 - porcine pleuropneumonia KL16 - Genbank CP022715\r\nActiNobacillus pleuropneumoniae serovar 5b str. L20 - ActiNobacillus pleuropneumoniae L20 - NCBI GCA_000015885.1 (ASM1588v1)\r\nAcyrthosiphon pisum - Acyrthosiphon pisum - EnsemblMetazoa 76 (GCA_000142985.2)\r\nAdineta vaga - Bdelloid Rotifers  - GeNoscope V2.0\r\nAdvenella kashmirensis W13003 - Advenella kashmirensis W13003 - NCBI GCF_000506985.1 (Advkasv1.0)\r\nAedes aegypti - Yellow fever mosquito - EnsemblMetazoa 76 (AaegL3)\r\nAedes aegypti - yellow fever mosquito - NCBI GCF_002204515.2 (AaegL5.0)\r\nAedes albopictus - forest day mosquito - NCBI GCF_001876365.2 (canu_80X_arrow2.2)\r\nAgrobacterium sp. ATCC 31749 - Agrobacterium sp. ATCC 31749 - NCBI GCF_000214615.1 (Agro31749_1.0)\r\nAmphimedon queenslandica - Amphimedon queenslandica - EnsemblMetazoa 76 (Aqu1)\r\nAnaplasma phagocytophilum str. HGE1 - Anaplasma phagocytophilum str. HGE1 - NCBI GCA_000478425.1 (ASM47842v1)\r\nAnas platyrhynchos - Mallard - Ensembl 76 (BGI_duck_1.0)\r\nANopheles arabiensis - A. arabiensis - VectorBase D1\r\nANopheles darlingi - ANopheles darlingi - EnsemblMetazoa 76 (AdarC3)\r\nANopheles gambiae - African malaria mosquito - EnsemblMetazoa 76 (AgamP4)\r\nApis mellifera - Apis mellifera - EnsemblMetazoa 76 (GCA_000002195.1)\r\nAquilegia caerulea - Aquilegia caerulea - Phytozome V9, Dec 2012\r\nArabidopsis lyrata - Arabidopsis lyrata - Phytozome V9, Dec 2012\r\nArabidopsis thaliana - Arabidopsis thaliana - Phytozome V9, Dec 2012\r\nArabidopsis thaliana - Thale-cress - EnsemblPlants 76 (TAIR10)\r\nArtemisia annua - sweet Annie - NCBI GCA_003112345.1 (ASM311234v1)\r\nAspergillus carbonarius ITEM 5010 - Aspergillus carbonarius ITEM 5010 - NCBI GCA_001990825.1 (Aspca3)\r\nAspergillus niger - A. niger - Phytozome 12, ATCC1015, V4.0\r\nAstyanax mexicanus - Blind cave fish - Ensembl 76 (AstMex102)\r\nAtta cephalotes - Atta cephalotes - EnsemblMetazoa 76 (GCA_000143395.2)\r\nBacillus RZ2MS9 - Bacillus sp. B14905 - Genbank NZ_AAXV01000001.1\r\nBacillus amyloliquefaciens DSM 7 - Bacillus amyloliquefaciens ATCC 23350 - NCBI GCF_000196735.1 (ASM19673v1)\r\nBacillus cereus NC7401 - Bacillus cereus NC7401 - NCBI GCF_000283675.1 (ASM28367v1)\r\nBacillus phage PfNC7401 - Bacillus phage PfNC7401 - NCBI GCA_002611025.1 (ASM261102v1)\r\nBacillus subtilis subsp. subtilis str. 168 - Bacillus subtilis 168 - NCBI GCF_000009045.1 (ASM904v1)\r\nBacillus velezensis SQR9 - Bacillus amyloliquefaciens SQR9 - NCBI GCF_000685725.1 (ASM68572v1)\r\nBactrocera tryoni - Queensland fruit fly - NCBI GCA_000695345.1 (Assembly 2.2 of Bactrocera tryoni geNome)\r\nBifidobacterium longum subsp. infantis ATCC 15697 = JCM 1222 = DSM 20088 - Bifidobacterium longum bv. Infantis ATCC 15697 - NCBI GCF_000020425.1 (ASM2042v1)\r\nBiomphalaria glabrata - B. glabrata - VectorBase 1\r\nBlastomyces dermatitidis ATCC 26199 - Ajellomyces dermatitidis ATCC 26199 - NCBI GCA_000166155.1 (BD_ATCC26199_V2)\r\nBlastomyces dermatitidis ER-3 - Ajellomyces dermatitidis ER-3 - NCBI GCA_000003525.2 (BD_ER3_V1)\r\nBlastomyces gilchristii SLH14081 - Blastomyces gilchristii SLH14081 - NCBI GCA_000003855.2 (BD_SLH14081_V1)\r\nBombus terrestris - Buff-tailed bumblebee - Genbank Bter1.0\r\nBombyx mori - Bombyx mori - EnsemblMetazoa 76 (GCA_000151625.1)\r\nBos taurus - Cattle - Ensembl 76 (UMD3.1)\r\nBos taurus - domestic cow - NCBI GCF_000003205.7 (Btau_5.0.1)\r\nBos taurus - domestic cow - NCBI GCF_002263795.1 (ARS-UCD1.2)\r\nBotrylloides leachii - B. leachii - Aniseed SBv3\r\nBrachypodium distachyon - Brachypodium distachyon - Phytozome V9, Dec 2012\r\nBrachypodium distachyon - Purple false brome - Bd21-3 v1.1\r\nBrassica Napus - Rapeseed - GeNoscope 4.1 Darmor-bzh\r\nBrassica oleracea - Brassica oleracea - EnsemblPlants 92 (v2.1)\r\nBrassica oleracea - wild cabbage - NCBI GCA_900416815.2 (Brassica_oleracea_HDEM)\r\nBrassica rapa - Brassica rapa - Phytozome V9, Dec 2012, Chiifu-401_v2.5\r\nBrassica rapa - field mustard - NCBI GCA_900412535.2 (Brassica_rapa_Z1)\r\nBrugia malayi - B.malayi - EnsemblMetazoa 76 (B_malayi-3.0)\r\nBurkholderia seminalis - Burkholderia seminalis - NCBI GCF_001742345.1 (ASM174234v1)\r\nC. glutamicum ATCC 13032 - Corynebacterium glutamicum ATCC 13032 - NCBI GCA_000011325.1 (ASM1132v1)\r\nCaeNorhabditis brenneri - Brenneri - EnsemblMetazoa 76 (C_brenneri-6.0.1b)\r\nCaeNorhabditis briggsae - Briggsae - EnsemblMetazoa 76 (CB4)\r\nCaeNorhabditis elegans - C. elegans - UCSC Feb. 2013 (WBcel235/ce11)\r\nCaeNorhabditis elegans - C. elegans - UCSC Oct. 2010 (WS220/ce10)\r\nCaeNorhabditis japonica - Japonica - EnsemblMetazoa 76 (C_japonica-7.0.1)\r\nCaeNorhabditis remanei - Remanei - EnsemblMetazoa 76 (C_remanei-15.0.1)\r\nCallithrix jacchus - white ear-tufted marmoset - NCBI GCF_000004665.1 (Callithrix jacchus-3.2)\r\nCamelina Sativa - False flax - Camelinadb.ca V2 with genes\r\nCamelina sativa - False Flax - Cs\r\nCamelina sativa - False flax - Camelinadb.ca V2\r\nCamelpox Virus - Camelpox Virus - Refseq GCF_000839105.1\r\nCandida albicans SC5314 - Candida albicans SC5314 - NCBI GCF_000182965.3 (ASM18296v3)\r\nCanis familiaris - Dog - Ensembl 79 (CanFam3.1)\r\nCannabis sativa - Marijuana, Purple Kush - canSat3, geNome.ccbr.utoronto.ca\r\nCannabis sativa - marijuana - NCBI GCA_000230575.1 (canSat3)\r\nCannabis sativa - marijuana - NCBI GCA_001865755.1 (Phylos Biosc. 2016)\r\nCapitella teleta - Capitella teleta - EnsemblMetazoa 76 (GCA_000328365.1)\r\nCapra hircus - goat - RefSeq ARS1\r\nCapsella rubella - Capsella rubella - Phytozome V9, Dec 2012\r\nCapsicum annuum - Capsicum annuum - NCBI GCA_000710875.1 (Pepper Zunla 1 Ref_v1.0)\r\nCarica papaya - Carica papaya - Phytozome V9, Dec 2012\r\nCatharanthus roseus - rosy periwinkle - NCBI GCA_001292565.1 (BAC_SGD)\r\nCavia porcellus - Domestic guinea pig - Ensembl 76 (cavPor3)\r\nChlamydomonas reinhardtii - Chlamydomonas reinhardtii - NCBI v3.0\r\nChlamydomonas reinhardtii - Chlamydomonas reinhardtii - Phytozome V9, Dec 2012\r\nChlorella variabilis - Chlorella variabilis - NCBI GCA_000147415.1 (v 1.0)\r\nChlorocebus sabaeus - Green monkey - UCSC Mar. 2014 (Chlorocebus_sabeus 1.1/chlSab2)\r\nCiona Intestinalis (formerly: C. intestinalis type B) - sea vase ascidian - ANISEED MTP2012\r\nCiona robusta (formerly C. intestinalis type A) - Sea squirt ciona intestinalis - Ensembl 76 (KH2008)\r\nCiona robusta (formerly: C. intestinalis type A) - sea vase ascidian - GHOST Joined, KH2008, KH2013 Genes + SNPs: Ghost SNPs\r\nCiona robusta (formerly: C. intestinalis, type A) - sea vase ascidian - UCSC JGI 2.1, Ci2, Mar. 2005\r\nCitrus clementina - Citrus clementina - Phytozome V9, Dec 2012\r\nCitrus sinensis - Citrus sinensis - Phytozome V9, Dec 2012\r\nClytia hemisphaerica - Jellyfish C.hem. - via Tsuyoshi Momose\r\nCoccomyxa subellipsoidea C-169 - Coccomyxa subellipsoidea C-169 - Phytozome V9, Dec 2012\r\nCopriNopsis cinerea okayama7#130 - Coprinus cinereus okayama7#130 - NCBI GCA_000182895.1 (CC3)\r\nCorynebacterium glutamicum ATCC 13032 - Corynebacterium glutamicum ATCC 13032 - NCBI GCF_000011325.1 (ASM1132v1)\r\nCorynebacterium pseudotuberculosis - Bacillus pseudotuberculosis - NCBI GCF_001433475.1 (ASM143347v1)\r\nCorynebacterium pseudotuberculosis 1002 - Corynebacterium pseudotuberculosis 1002 - NCBI GCF_000144935.1 (ASM14493v2)\r\nCoturnix japonica - Japanese quail - RefSeq V2.0 GCF_001577835\r\nCrassostrea gigas - Crassostrea gigas - EnsemblMetazoa 76 (GCA_000297895.1)\r\nCricetulus griseus (CHO cells) - Chinese hamster CHO cells - NCBI GCA_900186095.1 (CHOK1S_HZDv1, Sep 2017)\r\nCricetulus griseus (CHO cells) - Chinese hamster ovary cells - UCSC Aug. 2011 (CHO K1 cell line/criGriChoV1)\r\nCricetulus griseus - Chinese hamster - UCSC Jul. 2013 (C_griseus_v1.0/criGri1)\r\nCryptosporidium parvum Iowa II - Cryptosporidium parvum strain Iowa type II - NCBI GCF_000165345.1 (ASM16534v1)\r\nCucumis melo L. - Muskmelon - MeloNomics V3.5.1 Pseudo\r\nCucumis sativus - Cucumis sativus - Phytozome V9, Dec 2012\r\nCucurbita pepo subsp. pepo - field pumpkin - NCBI GCF_002806865.1 (ASM280686v2)\r\nCulex quinquefasciatus - Southern house mosquito - EnsemblMetazoa 76 (CpipJ2)\r\nDanaus plexippus - Danaus plexippus - EnsemblMetazoa 76 (DanPle_1.0)\r\nDanio rerio - Zebrafish - Ensembl 76 (Zv9)\r\nDanio rerio - Zebrafish - UCSC Jul. 2010 (Zv9/danRer7)\r\nDanio rerio - Zebrafish - UCSC May 2017 (GRCz11/danRer11)\r\nDanio rerio - Zebrafish - UCSC Sep. 2014 (GRCz10/danRer10)\r\nDanio rerio - zebra fish - NCBI GCA_000002035.4 (GRCz11)\r\nDaphnia magna - Daphnia magna - EnsemblMetazoa 95 (daphmag2.4)\r\nDaphnia magna - Daphnia magna - NCBI GCA_003990815.1 (ASM399081v1)\r\nDaphnia pulex - Daphnia pulex - EnsemblMetazoa 76 (GCA_000187875.1)\r\nDaucus carota subsp. sativus - Daucus carota var. sativus - NCBI GCF_001625215.1 (ASM162521v1)\r\nDendroctonus ponderosae - Dendroctonus ponderosae - EnsemblMetazoa 76 (GCA_000355655.1)\r\nDesulfovibrio vulgaris str. Hildenborough - Desulfovibrio vulgaris ATCC 29579 - NCBI GCA_000195755.1 (ASM19575v1)\r\nDictyostelium discoideum - Dictyostelium discoideum - Dictybase.org Apr 2017\r\nDrosophila ananassae - Drosophila ananassae - EnsemblMetazoa 76 (dana_caf1)\r\nDrosophila erecta - Drosophila erecta - EnsemblMetazoa 76 (dere_caf1)\r\nDrosophila grimshawi - Drosophila grimshawi - EnsemblMetazoa 76 (dgri_caf1)\r\nDrosophila melaNogaster - D. melaNogaster - UCSC Apr. 2006 (BDGP R5/dm3)\r\nDrosophila melaNogaster - D. melaNogaster - UCSC Aug. 2014 (BDGP Release 6 + ISO1 MT/dm6)\r\nDrosophila melaNogaster - Fruit fly - EnsemblMetazoa 76 (BDGP5)\r\nDrosophila mojavensis - Drosophila mojavensis - EnsemblMetazoa 76 (dmoj_caf1)\r\nDrosophila persimilis - Drosophila persimilis - EnsemblMetazoa 76 (dper_caf1)\r\nDrosophila pseudoobscura - Drosophila pseudoobscura - EnsemblMetazoa 76 (HGSC2)\r\nDrosophila sechellia - Drosophila sechellia - EnsemblMetazoa 76 (dsec_caf1)\r\nDrosophila simulans - Drosophila simulans - EnsemblMetazoa 76 (WUGSC1)\r\nDrosophila suzukii - Spotted-Wing Drosophila - SpottedWingFlybase V1\r\nDrosophila virilis - Drosophila virilis - EnsemblMetazoa 76 (dvir_caf1)\r\nDrosophila willistoni - Drosophila willistoni - EnsemblMetazoa 76 (dwil_caf1)\r\nDrosophila yakuba - Drosophila yakuba - EnsemblMetazoa 76 (dyak_r1.3_FB2008_07)\r\nEctocarpus sp. - Ectocarpus sp. - V2 ugent.be\r\nEimeria tenella - Eimeria tenella - NCBI GCF_000499545.2 (ETH001)\r\nEquus caballus - Horse - UCSC Sep. 2007 (Broad/equCab2)\r\nEscherichia coli - E.coli - strain K-12/MG1655 (ASM584v2)\r\nEscherichia coli W - Escherichia coli ATCC 9637 - NCBI GCF_000184185.1 (ASM18418v1)\r\nEscherichia coli str. K-12 substr. DH10B - Escherichia coli str. K-12 substr. DH10B - NCBI GCF_000019425.1 (ASM1942v1)\r\nEscherichia virus Lambda - Bacteriophage lambda - NCBI GCF_000840245.1 (ViralProj14204)\r\nEucalyptus grandis - Eucalyptus grandis - Phytozome V9, Dec 2012\r\nExaiptasia pallida - Aiptasia pallida - NCBI GCA_001417965.1 (Aiptasia geNome 1.1)\r\nFelis catus - Domestic cat - Ensembl 89 (Felis_catus_6.2)\r\nFonsecaea moNophora - Botrytoides moNophora - NCBI GCF_001642475.1 (ASM164247v1)\r\nFragaria vesca - Fragaria vesca - Phytozome V9, Dec 2012\r\nFragaria vesca subsp. vesca - wild strawberry - NCBI GCA_000184155.1 (FraVesHawaii_1.0)\r\nFragaria_vesca - wild strawberry - rosaceae.org v4.0.1a\r\nFusarium graminearum - Gibberella zeae - NCBI GCA_900044135.1 (GZPH1RResV1)\r\nGadus morhua - Atlantic cod - EnsemblVertebrates 95 (gadMor1)\r\nGallid alphaherpesvirus 1 - Infectious laryngotracheitis virus - NCBI GCA_000847005.2 (ViralProj14566)\r\nGallid herpesvirus 2 strain CVI988 - Gallid herpes 2 CVI988 - Genbank DQ530348.1\r\nGallus gallus - Chicken - UCSC Dec 2015 (Gallus_gallus-5.0/galGal5)\r\nGallus gallus - Chicken - UCSC Nov. 2011 (ICGSC Gallus_gallus-4.0/galGal4)\r\nGaNoderma lucidum BCRC 37177 - GaNoderma lucidum BCRC 37177 - NCBI GCA_000338035.1 (Glu37177_1.0)\r\nGaNoderma sinense ZZ0214-1 - GaNoderma sinense ZZ0214-1 - NCBI GCA_002760635.1 (GanSi1.6)\r\nGasterosteus aculeatus - Stickleback - UCSC Feb. 2006 (Broad/gasAcu1)\r\nGlossina morsitans - G. moritans - VectorBase Yale 1.6\r\nGlycine max - Glycine max - Phytozome V9, Dec 2012\r\nGoatpox virus Pellor - Goatpox virus Pellor - NCBI GCF_000840165.1 (ViralProj14197)\r\nGossypium hirsutum - American upland cotton - NCBI GCF_000987745.1 (ASM98774v1)\r\nGossypium raimondii - Gossypium raimondii - Phytozome V9, Dec 2012\r\nHBV geNotype D DNA - AB554016.1 - NCBI Genbank\r\nHalocynthia aurantium - sea peach - ANISEED MTP2014\r\nHalocynthia roretzi - sea pineapple - ANISEED MTP2014\r\nHelianthus annuus - Helianthus annuus - NCBI HanXRQr1.0\r\nHeliconius melpomene - Heliconius melpomene - EnsemblMetazoa 76 (Hmel1)\r\nHelicoverpa armigera - tobacco budworm - NCBI GCA_002156985.1 (Harm_1.0)\r\nHelobdella robusta - Helobdella robusta - EnsemblMetazoa 76 (GCA_000326865.1)\r\nHeterocephalus glaber - Naked Mole Rat - NCBI HetGla_female_1.0\r\nHistoplasma capsulatum G217B - Ajellomyces capsulatus G217B - NCBI GCA_000170615.1 (ASM17061v1)\r\nHomo sapiens - Human - UCSC Dec. 2013 (GRCh38/hg38) + SNPs: dbSNP148, Kaviar\r\nHomo sapiens - Human - UCSC Feb. 2009 (GRCh37/hg19) + SNPs: 1000GeNomes, ExaC\r\nHomo sapiens - Human - UCSC Mar. 2006 (NCBI36/hg18) + SNPs: HEK293\r\nHordeum vulgare - Two-rowed barley - EnsemblPlants 95 (IBSC_v2)\r\nHuman alphaherpesvirus 1 - Herpes simplex virus 1 - NCBI GCF_000859985.2 (ViralProj15217)\r\nHuman alphaherpesvirus 2 - Herpes simplex virus 2 - NCBI GCF_000858385.2 (ViralProj15218)\r\nHuman betaherpesvirus 5, Merlin strain - Human cytomegalovirus - NCBI GCF_000845245.1 (ViralProj14559)\r\nHuman betaherpesvirus 6A - Human herpesvirus 6A - NCBI GCA_000845685.2 (ViralProj14462)\r\nHuman betaherpesvirus 6B - Human herpes virus 6B - NCBI GCF_000846365.1 (ViralProj14422)\r\nHuman betaherpesvirus 7 - Herpes simplex virus 7 - NCBI GCF_000848125.1 (ViralProj14625)\r\nHuman herpesvirus-7 - HHV7 - Genbank U43400.1\r\nHydra Vulgaris - Fresh Water Poly - RP 1.0 RefSeq GCA_000004095.1\r\nHypsibius dujardini - water bear - tardigrades.org 3.15\r\nIxodes scapularis - Black-legged tick - EnsemblMetazoa 76 (IscaW1)\r\nJuglans regia - English walnut - NCBI GCF_001411555.1 (wgs.5d)\r\nKlebsiella aerogenes KCTC 2190 - Enterobacter aerogenes KCTC 2190 - NCBI GCA_000215745.1 (ASM21574v1)\r\nKlebsiella pneumoniae subsp. pneumoniae 1158 - Klebsiella pneumoniae 1158 - NCBI GCF_000814805.1 (ASM81480v1)\r\nKlebsormidium nitens - Klebsormidium nitens - NCBI GCA_000708835.1 (ASM70883v1)\r\nKluyveromyces lactis - Kluyveromyces drosophilarum - NCBI GCA_002151565.1 (ASM215156v1)\r\nKluyveromyces marxianus DMKU3-1042 - Kluyveromyces marxianus DMKU3-1042 - NCBI GCA_001417885.1 (Kmar_1.0)\r\nLactuca sativa - garden lettuce - NCBI GCF_002870075.1 (Lsat_Salinas_v7)\r\nLeishmania mexicana MHOM/GT/2001/U1103 - Leishmania mexicana MHOM/GT/2001/U1103 - NCBI GCF_000234665.1 (ASM23466v4)\r\nLeptosphaeria maculans - Blackleg of oilseed - JN3 via Colin Clairet\r\nLinum usitatissimum - Linum usitatissimum - Phytozome V9, Dec 2012\r\nLoa loa - Eye worm - EnsemblMetazoa 76 (Loa_loa_V3)\r\nLolium perenne - Perennial Rye-Grass - V1.0 L. Fuchs\r\nLottia gigantea - Lottia gigantea - EnsemblMetazoa 76 (GCA_000327385.1)\r\nLotus japonicus - Lotus japonicus - kazusa.or.jp V3\r\nLumpy skin disease KSGP 0240 - LSDV virus - Genbank KX683219.1\r\nLumpy skin disease virus NI-2490 - Lumpy skin disease virus NI-2490 - NCBI GCF_000839805.1 (ViralProj14122)\r\nLutzomyia longipalpis - Sand fly - VectorBase LionJ1\r\nLysinibacillus sphaericus C3-41 - Bacillus sphaericus C3-41 - NCBI GCA_000017965.1 (ASM1796v1)\r\nMacaca fascicularis - Crab-eating macaque - RefSeq 5.0\r\nMacaca mulatta - Rhesus - UCSC Jan. 2006 (MGSC Merged 1.0/rheMac2)\r\nMacrostomum lignaNo - Macrostomum lignaNo - Genbank GCA_001188465.1_ML2\r\nMacrostomum lignaNo - Macrostomum lignaNo - NCBI GCA_002269645.1 (Mlig_3_7)\r\nMalus domestica - Apple - INRA GDDH13 Version 1.1\r\nMalus domestica - Malus domestica - ASM211411v1\r\nMalus domestica - Malus domestica - Phytozome V9, Dec 2012\r\nManihot esculenta - Manihot esculenta - Phytozome V9, Dec 2012\r\nMarchantia polymorpha - Common liverwort - Phytozome 12, V3.1\r\nMedicago truncatula - Medicago truncatula - Phytozome V9, Dec 2012\r\nMedicago truncatula - barrel medic - NCBI GCA_002024945.1 (R108 v. 1.0)\r\nMedicago truncatula - barrel medic - medicagogeNome.org V4.3\r\nMedicago truncatula - barrelclover - Medicohapmap.org R108 0.95\r\nMedicago truncatula A17 - barrelclover - INRA A17r5.0 r1.6 Pascal Gamas\r\nMegaselia scalaris - Megaselia scalaris - EnsemblMetazoa 76 (Msca1)\r\nMeleagris gallopavo - Domestic turkey - Ensembl 76 (UMD2)\r\nMesocricetus auratus - Syrian hamster - NCBI GCF_000349665.1 (MesAur1.0)\r\nMethylococcus capsulatus str. Bath - Methylococcus capsulatus ATCC 33009 - NCBI GCF_000008325.1 (ASM832v1)\r\nMicractinium conductrix - Micractinium conductrix - NCBI GCA_002245815.2 (ASM224581v2)\r\nMicrotus ochrogaster - prairie vole - NCBI GCF_000317375.1 (MicOch1.0)\r\nMimulus guttatus - Mimulus guttatus - Phytozome V9, Dec 2012\r\nMinuartia pusilla 299 - Minuartia pusilla 299 - Phytozome V9, Dec 2012\r\nMiscanthus sinensis - Chinese silver grass - Phytozome12, Msin v7.1\r\nMizuhopecten yessoensis - ezo giant scallop - NCBI GCF_002113885.1 (ASM211388v2)\r\nMoesziomyces aphidis DSM 70725 - Moesziomyces aphidis DSM 70725 - NCBI GCA_000517465.1 (PseuAph1.0)\r\nMolgula occidentalis - Sandy Sea Grape - ANISEED EL_v1.2\r\nMolgula occulta - Tail-less Sea Grape - ANISEED EL_v1.2\r\nMolgula oculata - Tailed Sea Grape - ANISEED EL_v1.2\r\nMortierella alpina ATCC 32222 - Mortierella alpina ATCC 32222 - NCBI GCA_000240685.2 (ASM24068v2)\r\nMus musculus - Mouse (reference) - UCSC Dec. 2011 (GRCm38/mm10) + SNPs: C57BL/10J, C57BR/cdJ,...\r\nMus musculus - Mouse (reference) - UCSC July 2007 (NCBI37/mm9)\r\nMus musculus - Mouse strain C57BL/6NJ - NCBI GCA_001632555.1 (C57BL_6NJ_v1)\r\nMus musculus c3hhej - Mouse - Ensembl 90 (C3H_HeJ_v1)\r\nMusa acuminata subsp. malaccensis - true apple banana - NCBI GCF_000313855.2 (ASM31385v2)\r\nMusca domestica - house fly - NCBI GCF_000371365.1 (Musca_domestica-2.0.2)\r\nMustela putorius furo - Ferret  - UCSC Apr. 2011 (MusPutFur1.0/musFur1)\r\nMycobacterium abscessus - Mycobacterium chelonae subsp. abscessus - NCBI GCF_000069185.1 (ASM6918v1)\r\nMyxoma virus - myxoma virus MYX - NCBI GCF_000843685.1 (ViralProj14396)\r\nNanNochloropsis gaditana - NanNochloropsis gaditana - NCBI GCA_002838785.1 (we3730_nuc)\r\nNasonia vitripennis - Nasonia vitripennis - EnsemblMetazoa 76 (GCA_000002325.2)\r\nNeisseria meningitidis Z2491 - Neisseria meningitidis serogroup A strain Z2491 - NCBI GCA_000009105.1 (ASM910v1)\r\nNematostella vectensis - Nematostella vectensis - EnsemblMetazoa 76 (GCA_000209225.1)\r\nNeovison vison - mink - NCBI GCA_900108605.1 (NNQGG.v01)\r\nNicotiana_benthamiana - N. benthamiana - SolGeNomics.net V1.01 \r\nNicotiana_tabacum - N. tabacum - SolGeNomics.net V4.5 Edwards et al 2017 \r\nNocardia terpenica - Nocardia terpenica - NCBI GCF_001625105.1 (ASM162510v1)\r\nNostoc sp. PCC 7120 - Anabaena sp. PCC 7120 - NCBI GCF_000009705.1 (ASM970v1)\r\nNothobranchius furzeri - turquoise killifish - NCBI GCA_001465895.2 (Nfu_20140520)\r\nOnchocerca volvulus - O.volvulus - EnsemblMetazoa 76 (Cameroon_v3)\r\nOncorhynchus kisutch - coho salmon - NCBI GCF_002021735.1 (Okis_V1)\r\nOncorhynchus mykiss - rainbow trout - NCBI GCF_002163495.1 (Omyk_1.0)\r\nOncorhynchus tshawytscha - king salmon - NCBI GCF_002872995.1 (Otsh_v1.0)\r\nOreochromis niloticus - Nile tilapia - Ensembl 76 (Orenil1.0)\r\nOryctolagus cuniculus - Rabbit - Ensembl 76 (OryCun2.0)\r\nOryza sativa - Oryza sativa - Phytozome V9, Dec 2012\r\nOryzias latipes - Medaka - UCSC Oct. 2005 (NIG/UT MEDAKA1/oryLat2)\r\nOstreococcus lucimarinus - Ostreococcus lucimarinus - Phytozome V9, Dec 2012\r\nOstreococcus tauri - O. tauri - Genbank 140606\r\nOvis aries - Domestic sheep - Ensembl 76 (Oar_v3.1)\r\nPaenibacillus larvae subsp. larvae - Paenibacillus larvae subsp. larvae - NCBI GCF_002951875.1 (ASM295187v1)\r\nPaenibacillus larvae subsp. larvae - Paenibacillus larvae subsp. larvae - NCBI GCF_002951915.1 (ASM295191v1)\r\nPaenibacillus larvae subsp. larvae - Paenibacillus larvae subsp. larvae - NCBI GCF_002951935.1 (ASM295193v1)\r\nPaenibacillus larvae subsp. larvae DSM 25430 - Paenibacillus larvae subsp. larvae 04-309 - NCBI GCF_002951895.1 (ASM295189v1)\r\nPaenibacillus larvae subsp. larvae DSM 25719 - Paenibacillus larvae subsp. larvae 08-100 - NCBI GCF_000511115.1 (ASM51111v1)\r\nPan troglodytes - Chimp - UCSC Feb. 2011 (CSAC 2.1.4/panTro4)\r\nPanicum virgatum - Panicum virgatum - Phytozome V9, Dec 2012\r\nPantoea agglomerans - P. agg. FDAARGOS_160 - Refseq GCF_001558735.1\r\nPapio anubis - Baboon - UCSC Mar. 2012 (Baylor Panu_2.0/papAnu2)\r\nParamecium bursaria Chlorella virus 1 - Chlorella PBCV-1 virus - NCBI GCF_000847045.1 (ViralProj14564)\r\nParhyale hawaiensis - P. hawaiensis - GCA_001587735.2 but merged short contigs (PE Gallindo Torres)\r\nPassiflora edulis chloroplast - Passionfruit, chloroplast geNome - Genbank KX290855.1\r\nPenaeus vannamei - white shrimp - NCBI GCF_003789085.1 (ASM378908v1)\r\nPenicillium digitatum Pd1 - Penicillium digitatum Pd1 - NCBI GCA_000315645.2 (PdigPd1_v1)\r\nPenicillium expansum - Penicillium expansum - NCBI GCA_000769755.1 (PEX1)\r\nPetromyzon marinus - Sea lamprey - Ensembl 76 (Pmarinus_7.0)\r\nPetromyzon marinus - marine lamprey - NCBI GCA_002833325.1 (Pmar_germline 1.0)\r\nPetunia axillaris - While moon petunia - SolgeNomics V1.6.2\r\nPetunia inflata - Petunia inflata - SolgeNomics V1\r\nPhaeodactylum tricornutum - Phaeodactylum tricornutum - EnsemblProtists 79 (ASM15095v2_bd)\r\nPhalaeNopsis equestris - PhalaeNopsis equestris - NCBI GCF_001263595.1 (ASM126359v1)\r\nPhallusia fumigata - P. fumigata - ANISEED - MTP 2014\r\nPhallusia mammillata - white sea squirt - ANISEED MTP 2014\r\nPhaseolus vulgaris - Phaseolus vulgaris - Phytozome V9, Dec 2012\r\nPhyscomitrella patens - A moss - EnsemblPlants 76 (ASM242v1)\r\nPhyscomitrella patens - Physcomitrella patens - Phytozome V9, Dec 2012\r\nPhyscomitrella patens - Spreading Earthmoss - Phytozome V11\r\nPimephales promelas - Fathead minNow - Genbank FHM_SOAPdeNovo\r\nPinctada martensii - Akoya pearl oyster - NCBI GCA_002216045.1 (PinMar1.0)\r\nPlasmodium berghei ANKA - Plasmodium berghei ANKA - PlasmoDB 3.1\r\nPlasmodium falciparum 3D7 - Plasmodium falciparum 3D7 - PlasmoDB 3.1\r\nPlasmodium kNowlesi H - Plasmodium kNowlesi H - PlasmoDB 3.1\r\nPopulus trichocarpa - Populus trichocarpa - Phytozome V9, Dec 2012\r\nPristionchus pacificus - Pristionchus - EnsemblMetazoa 76 (P_pacificus-5.0)\r\nPrunus persica - Prunus persica - Phytozome V9, Dec 2012\r\nPseudocercospora fijiensis CIRAD86 - Mycosphaerella fijiensis CIRAD86 - NCBI GCF_000340215.1 (Mycfi2)\r\nPseudomonas aerugiNosa PAO1 - Pseudomonas aerugiNosa str. PAO1 - NCBI GCF_000006765.1 (ASM676v1)\r\nPseudomonas aerugiNosa UCBPP-PA14 - P. aer. UCBPP-PA14 - MW Azam\r\nPseudomonas putida KT2440 - Pseudomonas putida (strain KT2440) - NCBI GCF_000007565.2 (ASM756v2)\r\nPseudonitzschia multistriata - P.m. diatom algae - bioinfo.szn.it, unkNown version\r\nPyropia yezoensis - Susabi Nori, Marine Red Alga - NRIFS.fra.affrc..go.jp V1\r\nRamazzottius varieornatus - water bear - tardigrades.org 1.01\r\nRattus Norvegicus - Norway rat - Ensembl 76 (RNor_5.0)\r\nRattus Norvegicus - Rat - UCSC Jul. 2014 (RGSC 6.0/rn6)\r\nRattus Norvegicus - Rat - UCSC Mar. 2012 (RGSC 5.0/rn5)\r\nRhabditophanes sp. KR3021 - Rhabditophanes sp. KR3021 - NCBI GCA_000944355.1 (Rhabditophanes_sp_KR3021)\r\nRhodnius prolixus - Rhodnius prolixus - EnsemblMetazoa 76 (RproC1)\r\nRhodococcus opacus PD630 - Rhodococcus opacus PD630 - NCBI GCF_000599545.1 (ASM59954v1)\r\nRicinus communis - Ricinus communis - Phytozome V9, Dec 2012\r\nRosa chinensis - Rose - HapOB_Angers geNome V1.0\r\nS. boulardii - Saccharomyces sp. 'boulardii' - NCBI GCA_000769245.1 (ASM76924v1)\r\nSaccharomyces cerevisiae - S. cerevisiae - UCSC Apr. 2011 (SacCer_Apr2011/sacCer3)\r\nSalmo salar - Atlantic salmon - NCBI GCF_000233375.1 (ICSASG_v2)\r\nSalmonella enterica subsp. enterica serovar Typhimurium str. 14028S - Salmonella enterica subsp. enterica serovar Typhimurium str. 14028S - NCBI GCA_000022165.1 (ASM2216v1)\r\nSchistosoma mansoni - Schistosoma mansoni - EnsemblMetazoa 76 (ASM23792v2)\r\nSchizosaccharomyces pombe - fission yeast - NCBI GCF_000002945.1 (ASM294v2)\r\nSchmidtea Mediterranea - freshwater planarian - smedgd.stowers.org 1.1\r\nSchmidtea Mediterranea - freshwater planarian - smedgd.stowers.org 4.0\r\nSchmidtea mediterranea - freshwater planarian - NCBI GCA_002600895.1 (ASM260089v1)\r\nScotiNomys teguina - Alston's brown mouse - Paul Frazel NYU email\r\nSelaginella kraussiana - Selaginella kraussiana - NCBI GCA_001021135.1 (ASM102113v1)\r\nSelaginella moellendorffii - Selaginella moellendorffii - Phytozome V9, Dec 2012\r\nSerendipita indica DSM 11827 - Serendipita indica DSM 11827 - NCBI GCA_000313545.1 (ASM31354v1)\r\nSetaria italica - Setaria italica - Phytozome V9, Dec 2012\r\nSetaria viridis - Green foxtail - Phytozome 11/v1.1\r\nSheeppox virus - sheep pox virus - NCBI GCF_000840205.1 (ViralProj14196)\r\nShewanella oneidensis MR-1 - Shewanella oneidensis ATCC 700550 - NCBI GCF_000146165.2 (ASM14616v2)\r\nSolanum lycopersicum - Garden Tomato - SolgeNomics 3.0\r\nSolanum lycopersicum - Solanum lycopersicum - Phytozome V9, Dec 2012\r\nSolanum lycopersicum - Tomato - SolgeNomics.net SL3.0 ITAG3.0\r\nSolanum melongena - eggplant/aubergine - NCBI GCA_000787875.1 (SME_r2.5.1)\r\nSolanum tuberosum - Solanum tuberosum - Phytozome V9, Dec 2012\r\nSoleNopsis invicta - Red fire ant - EnsemblMetazoa 76 (GCA_000188075.1)\r\nSorghum bicolor - Sorghum bicolor - Phytozome V9, Dec 2012\r\nSorghum bicolor - milo - NCBI GCF_000003195.3 (Sorghum_bicolor_NCBIv3)\r\nSpodoptera frugiperda - Fall Armyworm - GeNouest V3.1\r\nStaphylococcus aureus - Micrococcus aureus - NCBI GCA_002085525.1 (ASM208552v1)\r\nStaphylococcus aureus subsp. aureus - Staphylococcus aureus subsp. aureus - NCBI GCF_002310395.1 (ASM231039v1)\r\nStaphylococcus aureus subsp. aureus MRSA252 - Staphylococcus aureus subsp. aureus MRSA252 - NCBI GCF_000011505.1 (ASM1150v1)\r\nStaphylococcus pseudintermedius - Staphylococcus pseudintermedius - NCBI GCF_001685665.2 (ASM168566v2)\r\nStaphylococcus pseudintermedius - Staphylococcus pseudintermedius - NCBI GCF_002847235.2 (ASM284723v2)\r\nStaphylococcus pseudintermedius ED99 - Staphylococcus pseudintermedius ED99 - NCBI GCF_000189495.1 (ASM18949v1)\r\nStreptomyces bingchenggensis BCW-1 - Streptomyces bingchenggensis BCW-1 - NCBI GCF_000092385.1 (ASM9238v1)\r\nStreptomyces bottropensis ATCC 25435 - Streptomyces bottropensis DSM 40262 - NCBI GCA_000383595.1 (ASM38359v1)\r\nStreptomyces coelicolor A3(2) - Streptomyces coelicolor A3(2) - NCBI GCF_000203835.1 (ASM20383v1)\r\nStreptomyces Noursei ATCC 11455 - Streptomyces Noursei ATCC 11455 - NCBI GCF_001704275.1 (ASM170427v1)\r\nStreptomyces parvulus - Streptomyces parvulus - ASM166004v1\r\nStrongylocentrotus purpuratus - Strongylocentrotus purpuratus - EnsemblMetazoa 76 (GCA_000002235.2)\r\nSus scrofa - Pig - Ensembl 76 (Sscrofa10.2)\r\nSus scrofa - Pig - NCBI GCF_000003025.6 (Sscrofa11.1)\r\nSynechococcus elongatus PCC 7942 - Anacystis nidulans R2 - NCBI GCF_000012525.1 (ASM1252v1)\r\nTaeniopygia guttata - Zebra finch - Ensembl 76 (taeGut3.2.4)\r\nTetranychus urticae - Tetranychus urticae - EnsemblMetazoa 76 (GCA_000239435.1)\r\nThellungiella halophila - Thellungiella halophila - Phytozome V9, Dec 2012\r\nTheobroma cacao - Theobroma cacao - Phytozome V9, Dec 2012\r\nThermoascus aurantiacus - soil fungus T. aurant. - JGI Theau1\r\nTribolium castaneum - Red flour beetle - EnsemblMetazoa 76 (Tcas3)\r\nTrichoderma atroviride - T. atroviride - JGI 2.0\r\nTrichoderma harzianum - Trichoderma harzianum - NCBI GCA_001990665.1 (ASM199066v1)\r\nTrichoderma harzianum - Trichoderma harzianum - NCBI GCA_002838845.1 (ASM283884v1)\r\nTrichoderma reesei - T. reesei - JGI V2 Oct 2006\r\nTrichoplusia ni - Cabbage looper - tn1 Github:weng-lab\r\nTriticum aestivum - Bread wheat - IWGSC RefSeq 1.0\r\nTriticum turgidum ssp. dicoccoides - Wild Emmer Wheat Zavitan - wheat.pw.usda.gov, Zavitan V2\r\nTrypaNosoma brucei 427 - T. brucei 427 - TriTrypDb 32\r\nTrypaNosoma cruzi - Chaga's disease - tritrypdb.org 4.1 CLBrenerEsmeraldo\r\nTrypaNosoma cruzi - TrypaNosoma cruzi - NCBI GCF_000209065.1 (ASM20906v1)\r\nUlva prolifera - Enteromorpha prolifera - NCBI GCA_004138255.1 (ASM413825v1)\r\nUrsus arctos horribilis - Grizzly bear - NCBI GCA_003584765.1 (ASM358476v1)\r\nUrsus arctos horribilis - Ursus arctos horribilis - NCBI GCF_003584765.1 (ASM358476v1)\r\nVaccinia virus - vaccinia virus VV - NCBI GCF_000860085.1 (ViralProj15241)\r\nVaccinia virus strain Lister clone VACV107 - VV - Genbank DQ121394.1\r\nVigna radiata var. radiata - green gram - NCBI GCA_000741045.2 (Vradiata_ver6)\r\nVitis vinifera - Vitis vinifera - Phytozome V9, Dec 2012\r\nVitis vinifera - wine grape - NCBI GCA_002922885.1 (Chkhaveri_assembly01)\r\nVolvox carteri - Volvox carteri - Phytozome V9, Dec 2012\r\nXanthomonas oryzae pv. oryzae PXO99A - Xanthomonas oryzae pv. oryzae PXO99A - NCBI GCF_000019585.2 (ASM1958v2)\r\nXanthophyllomyces dendrorhous - Xanthophyllomyces dendrorhous - NCBI GCA_001007165.2 (Xden1)\r\nXeNopus laevis - African Clawed Frog - Xenbase 9.1\r\nXeNopus laevis - X. laevis - Xenbase V7.1\r\nXeNopus tropicalis - Western Clawed Frog - Xenbase 9.0\r\nXeNopus tropicalis - X. tropicalis - UCSC Nov. 2009 (JGI 4.2/xenTro3)\r\nYarrowia lipolytica - Y. lipolytica - CLIB122 CR382127 GeNolevures 2008\r\nZea mays - Maize - Gramene AGPv4\r\nZea mays - Zea mays - Phytozome V9, Dec 2012\r\nZymoseptoria tritici IPO323 - Mycosphaerella graminicola IPO323 - NCBI GCA_000219625.1 (MYCGR v2.0)\r\n[Nectria] haematococca mpVI 77-13-4 - [Nectria] haematococca mpVI 77-13-4 - NCBI GCF_000151355.1 (v2.0)",
        Reference:
          "Haeussler M, Schonig K, Eckert H, Eschstruth A, Mianne J, Renaud JB, Schneider-MauNoury S, Shkumatava A, Teboul L, Kent J, Joly JS, Concordet JP Evaluation of off-target and on-target scoring algorithms and integration into the guide RNA selection tool CRISPOR. GeNome Biol. 2016 Jul 5;17(1):148. doi: 10.1186/s13059-016-1012-2. (PMID: 27380939)",
        __EMPTY: "http://crispor.tefor.net/",
      },
      {
        Platform: "Command-line",
        Reference:
          "Canver MC, Haeussler M, Bauer DE, Orkin SH, Sanjana NE, Shalem O, Yuan GC, Zhang F, Concordet JP, Pinello L Integrated design, execution, and analysis of arrayed and pooled CRISPR geNome-editing experiments. Nat Protoc. 2018 May;13(5):946-986. doi: 10.1038/nprot.2018.005. Epub 2018 Apr 12. (PMID: 29762716)",
      },
      {
        Reference:
          "Concordet JP, Haeussler M CRISPOR: intuitive guide selection for CRISPR/Cas9 geNome editing experiments and screens. Nucleic Acids Res. 2018 Jul 2;46(W1):W242-W245. doi: 10.1093/nar/gky354. (PMID: 29651054)",
      },
      {
        Tools: "E-CRISP",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": "Yes",
        "Score Oligos": "No",
        "Search by": "GeneID",
        Enzyme: "SpCas9",
        PAM: "NGG, NAG",
        Organisms:
          "Aedes aegypti AaegL3.31\r\nANopheles darlingi AdarC3.31\r\nANopheles gambiae AgamP4.31\r\nArabidopsis thaliana TAIR10.31\r\nAspergillus nidulans ASM1142v1.31\r\nAspergillus niger CADRE.31\r\nBos taurus UMD3.1\r\nBrachypodium distachyon v1.0\r\nCaeNorhabditis elegans WBcel235\r\nCanis familiaris CanFam3.1\r\nChlamydomonas reinhardtii v3.1\r\nCiona intestinalis KH\r\nCricetulus griseus CriGri_1.0\r\nDanio rerio GRCz10\r\nDictyostelium discoideum dictybase.01\r\nDrosophila melaNogaster BDGP6\r\nDrosophila pseudoobscura GCA_000001765.2\r\nEmiliania huxleyi GCA_000372725.1\r\nGallus gallus Galgal4\r\nGasterosteus aculeatus BROADS1\r\nHomo sapiens GRCh38\r\nHordeum vulgare ASM32608v1.31\r\nKomagataella pastoris GCA_000027005.1\r\nLeishmania major ASM272v2.31\r\nMacaca fascicularis\r\nMacaca mulatta MMUL_1\r\nMagnaporthe oryzae MG8.31\r\nMus musculus GRCm38\r\nNematostella vectensis GCA_000209225.1\r\nNeurospora crassa ASM18292v1.31\r\nOryctolagus cuniculus OryCun2.0\r\nOryza indica ASM465v1.31\r\nOryza nivara AWHD00000000.31\r\nOryza sativa IRGSP-1.0.31\r\nOryzias latipes MEDAKA1\r\nOvis aries Oar_v3.1\r\nPhyscomitrella patens ASM242v1.31\r\nPlasmodium falciparum ASM276v1.31\r\nPlasmodium vivax ASM241v1.31\r\nPopulus trichocarpa JGI2.0\r\nPuccinia graminis ASM14992v1.31\r\nRattus Norvegicus RNor_6.0\r\nSaccharomyces cerevisiae R64-1-1\r\nSchizosaccharomyces pombe ASM294v2.31\r\nSus scrofa Sscrofa10.2\r\nTaeniopygia guttata taeGut3.2\r\nToxoplasma gondii ToxoDB-7.1.31\r\nTrichoderma reesei GCA_000167675.2\r\nTriticum aestivum IWGSC1+popseq.31\r\nUstilago maydis UM1.31\r\nVitis vinifera IGGP_12x.31\r\nXeNopus tropicalis JGI_4.2\r\nZea mays AGPv3.31\r\nZymoseptoria tritici MG2.31\r\nZymomonas mobilis ASM27775v1",
        Reference:
          "Heigwer F, Kerr G, Boutros M E-CRISP: fast CRISPR target site identification. Nat Methods. 2014 Feb;11(2):122-3. doi: 10.1038/nmeth.2812. (PMID: 24481216)",
        __EMPTY: "http://www.e-crisp.org/E-CRISP/index.html",
      },
      {
        Platform: "Other",
        "Search by": "Sequence",
      },
      {
        Tools: "CT-Finder",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": "Yes",
        "Score Oligos": "No",
        "Search by": "Sequence",
        Enzyme: "Custom",
        PAM: "Custom",
        Organisms:
          "Thale cress (Arabidopsis thaliana) geNome TAIR10.27 Ensembl\r\nRice (Oryza sativa) geNome IRGSP-1.0.27 Ensembl\r\nHuman (Homo sapiens) geNome GRCh38.81 Ensembl\r\nMouse (Mus musculus) geNome GRCm38.81 Ensembl\r\nZebrafish (Danio rerio) geNome GRCz10.84 Ensembl\r\nHuman immuNodeficiency virus 1 (HIV-1) geNome NCBI\r\nChicken (Gallus gallus) geNome Galgal4.84 Ensembl\r\nFruit fly (Drosophila melaNogaster) geNome BDGP6.84 Ensembl\r\nBrewer's yeast (Saccharomyces cerevisiae) geNome R64-1-1.84 Ensembl\r\nDomestic cat (Felis catus) geNome Felis_catus_6.2.84 Ensembl\r\nEscherichia coli (Escherichia coli) geNome HUSEC2011CHR1.31 Ensembl\r\nCaeNorhabditis elegans (CaeNorhabditis elegans) geNome WBcel235.84 Ensembl\r\nMacaque (Macaca mulatta) geNome MMUL_1.84 Ensembl\r\nZea mays (Zea mays) geNome AGPv3.31 Ensembl\r\nBordetella avium 197N geNome ASM7046v1.38 Ensembl\r\nLactococcus lactis AI06 geNome ASM76111v1.38 Ensembl\r\nToxoplasma gondii ME49 geNome TGA4.39 Ensembl",
        Reference:
          "Zhu H, Misel L, Graham M, Robinson ML, Liang C CT-Finder: A Web Service for CRISPR Optimal Target Prediction and Visualization. Sci Rep. 2016 May 23;6:25516. doi: 10.1038/srep25516. (PMID: 27210050)",
        __EMPTY: "http://bioinfolab.miamioh.edu/ct-finder/",
      },
      {
        Platform: "Webapp",
      },
      {
        Tools: "CRISPy-web",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": "Yes",
        "Score Oligos": "Yes",
        "Search by": "Coordinate",
        Enzyme: "SpCas9, Cas13a",
        PAM: "NGG",
        Organisms: "Provided by user",
        Reference:
          "Blin K, Pedersen LE, Weber T, Lee SY CRISPy-web: An online resource to design sgRNAs for CRISPR applications. Synth Syst BiotechNol. 2016 Feb 12;1(2):118-121. doi: 10.1016/j.synbio.2016.01.003. eCollection 2016 Jun. (PMID: 29062934)",
        __EMPTY: "https://crispy.secondarymetabolites.org/#/input",
      },
      {
        "Search by": "Other",
      },
      {
        Tools: "CRISPR Library Designer (CLD)",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Command-line",
        "Off‑targets": "Yes",
        "Score Oligos": "Yes",
        "Search by": "GeneID",
        Enzyme: "Custom",
        PAM: "Custom",
        Organisms: "Provided by user",
        Reference:
          "Heigwer F, Zhan T, Breinig M, Winter J, Brugemann D, Leible S, Boutros M CRISPR library designer (CLD): software for multispecies design of single guide RNA libraries. GeNome Biol. 2016 Mar 24;17:55. doi: 10.1186/s13059-016-0915-2. (PMID: 27013184)",
        Comments: "It uses Docker",
        __EMPTY: "https://github.com/boutroslab/cld_docker",
      },
      {
        Purpose: "Other",
        "Search by": "Coordinate",
      },
      {
        Tools: "CRISPR-DT",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": "Yes",
        "Score Oligos": "Yes",
        "Search by": "Sequence",
        Enzyme: "Custom",
        PAM: "Custom",
        Organisms:
          "Homo sapiens\r\nMus musculus\r\nArabidopsis thaliana\r\nOryza sativa\r\nGallus gallus\r\nDrosophila melaNogaster\r\nCaeNorhabditis elegans\r\nGlycine max",
        Reference:
          "Zhu H, Liang C CRISPR-DT: designing gRNAs for the CRISPR-Cpf1 system with improved target efficiency and specificity. Bioinformatics. 2019 Jan 7. pii: 5273481. doi: 10.1093/bioinformatics/bty1061. (PMID: 30615056)",
        Comments:
          "It provides a tool for calculate the efficiency of the target for several caspases",
        __EMPTY: "http://bioinfolab.miamioh.edu/CRISPR-DT",
      },
      {
        Purpose: "Other",
      },
      {
        Tools: "Breaking Cas",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": "Yes",
        "Score Oligos": "No",
        "Search by": "Sequence",
        Enzyme:
          "SpCas9\r\nSaCas9\r\nAsCpf1-RVR\r\nAsCpf1-WT\r\nAsCpf1-RR\r\nxCas9\r\nMAD7",
        PAM: "NGG, NAG, NNGRRN, TVTV. TTTV, TYCV, YTTN",
        Organisms:
          "324 geNomes : Vertebrate, Protist, Fungi, plant, Invertebrate metazoa",
        Reference:
          "Oliveros JC, Franch M, Tabas-Madrid D, San-Leon D, Montoliu L, Cubas P, Pazos F Breaking-Cas-interactive design of guide RNAs for CRISPR-Cas experiments for ENSEMBL geNomes. Nucleic Acids Res. 2016 Jul 8;44(W1):W267-71. doi: 10.1093/nar/gkw407. Epub 2016 May 10. (PMID: 27166368)",
        Comments:
          "List of on-targets and off-targets sorted also by colour. It allows to change the positional weights for scoring",
        __EMPTY: "https://bioinfogp.cnb.csic.es/tools/breakingcas/",
      },
      {
        Tools: "Benchling (CRISPR guides)",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": "Yes",
        "Score Oligos": "Yes",
        "Search by": "GeneID",
        Enzyme:
          "SpCas9 (Streptococcus pyogenes)\r\nNmCas9 (Neisseria meningitidis)\r\nSt1Cas9 (Streptococcus thermophilus)\r\nTdCas9 (Treponema denticola)\r\nSaCas9 (Streptococcus aureus)\r\nAsCpf1 (AcidamiNococcus sp.)\r\nLbCpf1 (LachNospiraceae bacterium)",
        PAM: "NGG\r\nNAG\r\nNNNNGATT\r\nNNAGAAW\r\nNAAAAC\r\nNNGRR\r\nNNGRRT\r\nTTTN",
        Organisms:
          "Acanthamoeba castellanii\r\nAcetobacterium woodii\r\nAedes aegypti\r\nAfrican swine\r\nAmbystoma mexicanum\r\nAnabaena variabilis\r\nANolis carolinensis\r\nANopheles gambiae\r\nAnthracocystis flocculosa\r\nApis mellifera\r\nArabidopsis thaliana\r\nAspergillus fumigatus\r\nAspergillus nidulans\r\nBacillus methaNolicus\r\nBacillus mycoides\r\nBacillus subtilis\r\nBos taurus\r\nBrachypodium distachyon\r\nBradyrhizobium diazoefficiens\r\nBrassica napus\r\nBrassica oleracea\r\nBrassica rapa\r\nCaeNorhabditis elegans\r\nCallithrix jacchus\r\nCandida albicans\r\nCanis lupus familiaris\r\nCapra hircus\r\nChlamydia trachomatis\r\nChlamydomonas reinhardtii\r\nCitrus clementina\r\nCitrus sinensis\r\nClaviceps purpurea\r\nClostridium acetobutylicum\r\nClostridium autoethaNogenum\r\nClostridium botulinum\r\nClostridium ljungdahlii\r\nClostridium pasteurianum\r\nClostridium saccharobutylicum\r\nClostridium saccharoperbutylacetonicum\r\nClostridium sporogenes\r\nCricetulus griseus\r\nCryptosporidium parvum\r\nCucumis melo\r\nCulex quinquefasciatus\r\nDanio rerio\r\nDrosophila eugracilis\r\nDrosophila melaNogaster\r\nDrosophila Yakuba\r\nEquus caballus\r\nEragrostis tef\r\nEscherichia coli\r\nEubacterium limosum\r\nFelis catus\r\nFusarium graminearum\r\nGallus gallus\r\nGekko japonicus\r\nGeobacillus thermoglucosidasius\r\nGiardia intestinalis\r\nGiardia lamblia\r\nGlycine max\r\nGossypium hirsutum\r\nHepatitis C\r\nHomo sapiens\r\nHordeum vulgare\r\nKomagataella pastoris\r\nLeishmania infantum\r\nLeishmania major\r\nLolium perenne\r\nLutzomyia longipalpis\r\nMacaca fascicularis\r\nMacaca mulatta\r\nMalus domestica\r\nManihot esculenta\r\nMarchantia polymorpha\r\nMesocricetus auratus\r\nMethylobacterium extorquens\r\nMethylocystis parvus\r\nMicrotus ochrogaster\r\nMoorella thermoacetica\r\nMusa acuminata\r\nMus musculus\r\nMustela putorius\r\nNematostella vectensis\r\nNicotiana benthamiana\r\nNicotiana tabacum\r\nOryctolagus cuniculus\r\nOryza sativa\r\nOryzias latipes\r\nOvis aries\r\nPaenibacillus polymyxa\r\nPatiria miniata\r\nPeptoclostridium difficile\r\nPhaeodactylum tricornutum\r\nPhlebotomus papatasi\r\nPiscirickettsia salmonis\r\nPlasmodium berghei\r\nPlasmodium falciparum\r\nPopulus trichocarpa\r\nProteus mirabilis\r\nPseudomonas aerugiNosa\r\nRalstonia eutropha\r\nRattus Norvegicus\r\nRhodobacter sphaeroides\r\nRhodococcus jostii\r\nSaccharomyces cerevisiae\r\nSaccharopolyspora spiNosa\r\nSaccoglossus kowalevskii\r\nSalmo salar\r\nSaprolegnia parasitica\r\nSchistosoma japonicum\r\nSchizosaccharomyces pombe\r\nSolanum lycopersicum\r\nSorghum bicolor\r\nSpodoptera frugiperda\r\nStreptomyces coelicolor\r\nStreptomyces rapamycinicus\r\nStreptomyces sp\r\nStrongylocentrotus purpuratus\r\nSus scrofa\r\nSynechococcus elongatus\r\nSynechococcus sp\r\nSynechocystis sp\r\nTaeniopygia guttata\r\nToxoplasma gondii\r\nTribolium castaneum\r\nTrichoderma virens\r\nTrypaNosoma brucei\r\nVerrucosispora maris\r\nVitis vinifera\r\nXanthomonas campestris\r\nXeNopus tropicalis\r\nYarrowia lipolytica\r\nZea mays\r\nZymomonas mobilis",
        Comments: "Commercial",
        __EMPTY: "https://benchling.com/",
      },
      {
        "Search by": "Coordinate",
      },
      {
        "Search by": "Sequence",
      },
      {
        Tools: "CRISPRmap",
        Availability: "Yes",
        Purpose: "Database",
        Platform: "Web",
        "Off‑targets": "No",
        "Score Oligos": "No",
        "Search by": "Sequence",
        Enzyme: "Not applicable",
        PAM: "Not Applicable",
        Organisms: "Not Applicable",
        Reference:
          "Lange SJ, Alkhnbashi OS, Rose D, Will S, Backofen R CRISPRmap: an automated classification of repeat conservation in prokaryotic adaptive immune systems. Nucleic Acids Res. 2013 Sep;41(17):8034-44. doi: 10.1093/nar/gkt606. Epub 2013 Jul 17. (PMID: 23863837)",
        Comments:
          "Intended for the analysis and classification of a CRISPR system from a list of CRISPR sequences. Includes a database of CRISPR consensus repeats.",
        __EMPTY: "http://rna.informatik.uni-freiburg.de/CRISPRmap/Input.jsp",
      },
      {
        Purpose: "Other",
      },
      {
        Tools: "DESKGEN Cloud",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": "Yes",
        "Score Oligos": "Yes",
        "Search by": "GeneID",
        Enzyme:
          "AsCpf1\r\nFnCpf1\r\nLbCPf1\r\nNmCas9\r\nSaCas9\r\nSpcas9\r\nSpCas9D10A-pairs\r\nSpCas9D10A-singles\r\nSpdCas9-FokI\r\nSt1Cas9",
        PAM: "NGG\r\nNGRRT\r\nNGRRN\r\nNNNNGATT\r\nNNNNRYAC\r\nNNAGAAW\r\nTTTV",
        Organisms:
          "ANopheles gambiae\r\nArabidopsis thaliana\r\nBos taurus\r\nCaeNorhabditis elegans\r\nCandida albicans\r\nCandida glabrata\r\nDanio rerio\r\nDrosophila melaNogaster\r\nEscherichia coli\r\nGallus gallus\r\nHomo sapiens\r\nHordeum vulgare\r\nMus musculus\r\nOryza sativa\r\nPenicillium roqueforti\r\nPhyscomitrella patens\r\nPlasmodium falciparum\r\nRattus Norvegicus\r\nRhodococcus jostii\r\nSaccharomyces cerevisiae\r\nSchizosaccharomyces pombe\r\nSynechocystis sp\r\nTriticum aestivum\r\nZea mays",
        Comments: "Commercial, free, registration is required",
        __EMPTY: "https://www.deskgen.com/landing/documentation.html",
      },
      {
        Purpose: "Database",
      },
      {
        Purpose: "Other",
      },
      {
        Tools: "Geneious",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Desktop",
        "Off‑targets": "Yes",
        "Score Oligos": "Yes",
        "Search by": "Sequence",
        Enzyme: "SpCas9 (S. pyogenes)\r\nCas9 D10 Nickase\r\nCpf1",
        PAM: "NGG\r\nTTTN",
        Organisms: "Provided by user",
        Comments: "Commercial (30 day trial)",
        __EMPTY: "https://www.geneious.com/tutorials/finding-crispr-sites-r9/",
      },
      {
        Tools: "Protospacer Workbench",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Desktop",
        "Off‑targets": "Yes",
        "Score Oligos": "Yes",
        "Search by": "GeneID",
        Enzyme: "SpCas9 (S. pyogenes)",
        PAM: "NGG\r\nNAG",
        Organisms: "Provided by user",
        Reference:
          "MacPherson CR, Scherf A Flexible guide-RNA design for CRISPR applications using Protospacer Workbench. Nat BiotechNol. 2015 Aug;33(8):805-6. doi: 10.1038/nbt.3291. Epub 2015 Jun 29. (PMID: 26121414)",
        Comments: "Only for Mac OS",
        __EMPTY: "https://sourceforge.net/projects/protospacerwb/",
      },
      {
        "Search by": "Coordinate",
      },
      {
        "Search by": "Sequence",
      },
      {
        Tools: "CRISPR Recognition Tool (CRT)",
        Availability: "Yes",
        Purpose: "Other",
        Platform: "Command-line",
        "Off‑targets": "No",
        "Score Oligos": "No",
        "Search by": "Sequence",
        Enzyme: "Not applicable",
        PAM: "Not Applicable",
        Organisms: "Provided by user",
        Reference:
          "Bland C, Ramsey TL, Sabree F, Lowe M, Brown K, Kyrpides NC, Hugenholtz P CRISPR recognition tool (CRT): a tool for automatic detection of clustered regularly interspaced palindromic repeats. BMC Bioinformatics. 2007 Jun 18;8:209. doi: 10.1186/1471-2105-8-209. (PMID: 17577412)",
        Comments: "Detect CRISPR patterns in geNomic sequences",
        __EMPTY: "http://www.room220.com/crt/",
      },
      {
        Platform: "Desktop",
      },
      {
        Tools: "getCRISPRY",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Command-line",
        "Off‑targets": "Yes",
        "Score Oligos": "Yes",
        "Search by": "GeneID",
        Enzyme: "SpCas9 (S. pyogenes)",
        PAM: "NGG",
        Organisms: "Provided by user",
        __EMPTY: "https://github.com/asistradition/getCRISPRY",
      },
      {
        "Search by": "Coordinate",
      },
      {
        "Search by": "Sequence",
      },
      {
        Tools: "CHOMP",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Command-line",
        "Off‑targets": "Yes",
        "Score Oligos": "No",
        "Search by": "Sequence",
        Enzyme: "SpCas9 (S. pyogenes)",
        PAM: "NGG",
        Organisms: "Provided by user",
        __EMPTY: "https://github.com/bretonics/CHOMP",
      },
      {
        Purpose: "Other",
      },
      {
        Tools: "GT-Scan",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": "Yes",
        "Score Oligos": "Yes",
        "Search by": "Sequence",
        Enzyme: "SpCas9\r\nSaCas9\r\nCas12a\r\n",
        PAM: "NGG\r\nNNGRRT\r\nTTTN\r\n",
        Organisms:
          "ANole lizard\r\nArabidopsis thaliana\r\nCaeNorhabditis elegans \r\nCanis familiaris\r\nCapsicum annunm\r\nCavia porcellus\r\nCiona intestinalis \r\nCiona savignyi \r\nColumba livia\r\nDanio rerio\r\nDrosophila melaNogaster\r\nEquus caballus\r\nEscherichia coli K-12\r\nFelis catus\r\nGadus morhua\r\nGallus gallus\r\nGlycine max\r\nHomo sapiens\r\nHordeum vulgare\r\nLamprey\r\nLancelet\r\nMedaka\r\nMegabat\r\nMicrobat\r\nMicrocebus  murinus \r\nMus musculus\r\nOrnithorhynchus anatinus\r\nOryctolagus cuniculus\r\nOryza sativa Indica\r\nOryza sativa Japonica\r\nPan troglodytes\r\nPenicillium brevicompactum\r\nPenicillium oxalicumi\r\nPhyscomitrella patens\r\nPika pika\r\nRattus Norvegicus\r\nSaccharomyces cerevisiae\r\nSarcophilus harrisii\r\nSetaria italica\r\nSetaria viridis\r\nSolanum  lycopersicum \r\nStickleback\r\nStreptomyces ambofaciens\r\nSus scrofa\r\nTalaromyces aculeatus\r\nTrichoderma reesei\r\nWallaby\r\nXeNopus tropicalis\r\nZea mays",
        Reference:
          "O'Brien A, Bailey TL GT-Scan: identifying unique geNomic targets. Bioinformatics. 2014 Sep 15;30(18):2673-5. doi: 10.1093/bioinformatics/btu354. Epub 2014 May 23. (PMID: 24860161)",
        __EMPTY: "https://gt-scan.csiro.au/gt-scan",
      },
      {
        Tools: "CHOPCHOP",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": "Yes",
        "Score Oligos": "Yes",
        "Search by": "GeneID",
        Enzyme: "Cas9\r\nCas9 nickase\r\nCpf1\r\nTALEN",
        PAM: "Custom",
        Organisms:
          "Acanthamoeba castellanii (GCF_000313135.1)\r\nAcyrthosiphon pisum (Acyr_2.0)\r\nAedes aegypti (AaegL3)\r\nAedes aegypti (AaegL5)\r\nAedes albopictus (GCA_001876365.2)\r\nAfrican Swine Fever Virus Kenya (IX 1033)\r\nAmphimedon queenslandica (GCF_000090795.1)\r\nAnabaena (PCC 7120)\r\nANopheles albimanus (AalbS2)\r\nANopheles funestus (AfunF1)\r\nANopheles gambiae (AgamP4)\r\nANopheles gambiae (aNoGam1)\r\nANopheles stephensi (Indian, AsteI2)\r\nANopheles stephensi (SDA-500, AsteS1)\r\nApis mellifera (HAv3.1)\r\nAquilegia coerulea (v3.1)\r\nArabidopsis thaliana (TAIR10)\r\nArachis duranensis (Aradu 1.0)\r\nArachis ipaensis (Araip 1.0)\r\nArthrobotrys oligospora (GCF_000225545.1)\r\nAspergillus fumigatus (Af293)\r\nAspergillus niger (ATCC 1015)\r\nAstatotilapia burtoni (AstBur1.0)\r\nAstyanax mexicanus (AstMex102)\r\nAutographa californica nucleopolyhedrovirus (AcMNPV) (NC_001623.1)\r\nAzotobacter vinelandii DJ (NC_012560.1)\r\nBacillus RZ2MS9 (MJBF01)\r\nBacillus subtilis 168 (NC_000964.3)\r\nBacteroides fragilis 638R (FQ312004.1)\r\nBacteroides fragilis 9343 (NC_003228.3)\r\nBacteroides thetaiotaomicron (FR901300.1)\r\nBactrocera cucurbitae (ASM80634v1)\r\nBactrocera dorsalis (ASM78921v2)\r\nBactrocera oleae (GCA_001188975.2)\r\nBdellovibrio bacteriovorus (ASM19617v1)\r\nBetta splendens (v1.1)\r\nBiomphalaria glabrata (BglaB1)\r\nBombyx mori (release 31)\r\nBos taurus (bosTau9)\r\nBos taurus (Btau 5.0)\r\nBotrytis cinerea (ASM83294v1)\r\nBovine diarrhea virus (NC_001461.1)\r\nCaeNorhabditis elegans (ce10/WS220)\r\nCallithrix jacchus (C_jacchus3.2.1)\r\nCampoNotus floridanus (GCF_003227725.1)\r\nCandida albicans (A22)\r\nCandida auris (GCA_002759435.2)\r\nCandida glabrata (CBS138)\r\nCandida krusei (GCA_003054445.1)\r\nCandida lusitaniae (GCA_000003835.1)\r\nCandida parapsilosis (GCA_000182765.2)\r\nCandida tropicalis (MYA-3404)\r\nCandidatus Carsonella ruddii (NC_018417.1)\r\nCanis familiaris (CanFam3)\r\nCannabis sativa (v2)\r\nCardamine hirsuta (v1.0)\r\nCaulobacter crescentus (NA1000)\r\nCeratitis capitata (Ccap1.0)\r\nChinese Hamster Ovary (GCF_000223135.1/CriGri_1.0)\r\nChlamydomonas reinhardtii (v5.6)\r\nChlorella sorokiniana (CSI2_1230)\r\nChrysanthemum seticuspe (CSE_r1.0)\r\nCicer arietinum (ASM33114v1)\r\nCiona intestinalis (KH2012)\r\nClostridium acetobutylicum ATCC 824 (GCF_000008765.1)\r\nClostridium beijerinckii 6423 (GCF_900010805.1)\r\nClostridium botulinum ATCC 19397 (NC_009697.1)\r\nCochliobolus heterostrophus C5 (v2.0)\r\nCoffea canephora (v1.0)\r\nCollectotrichum graminicola M1.001 (Colgr1)\r\nCorynebacterium glutamicum ATCC 13032 (GCF_000011325.1)\r\nCorynebacterium pseudotuberculosis 1002 (GCF_000144935.1)\r\nCoturnix japonica (GCF_001577835.1)\r\nCoxiella burnetii RSA 493 (GCF_000007765.2)\r\nCulex quinquefasciatus (CpipJ2)\r\nCupriavidus malaysiensis USMAA1020 chr1 (CP017754.1)\r\nCupriavidus malaysiensis USMAA1020 chr2 (CP017754.1)\r\nCypriNodon variegatus (GCF_000732505.1)\r\nDalotia coriaria (Dcor_v1)\r\nDanio rerio (danRer10/GRCz10)\r\nDanio rerio (danRer11/GRCz11)\r\nDendrobium catenatum (ASM160598v2)\r\nDiaphorina Citri (GCF_000475195.1)\r\nDrosophila melaNogaster (dm3)\r\nDrosophila melaNogaster (dm6)\r\nDrosophila suzukii (Dsuz 1.0)\r\nEmiliania huxleyi (CCMP1516)\r\nEnterobacteria phage lambda (J02459.1)\r\nEnterococcus faecalis (GCF_000007785.1)\r\nEnterococcus faecalis OG1RF (GCA_000172575.2)\r\nEquus caballus (EquCab2)\r\nEragrostis tef (1.1.2)\r\nEscherichia coli (C321.ΔA/GCA_000474035.1)\r\nEscherichia coli (Nissle 1917/CP007799.1)\r\nEscherichia coli (str. CFT073/ASM744v1)\r\nEscherichia coli (str. K-12/MG1655)\r\nEucalyptus grandis (v1.0)\r\nEutrema salsugineum (v1.0)\r\nGallid herpesvirus 1 (NC_006623.1)\r\nGallid herpesvirus 2 (str. CVI988, DQ530348.1)\r\nGallid herpesvirus 3 (str. SB-1, HQ840738.1)\r\nGallus gallus (galGal4)\r\nGallus gallus (galGal5)\r\nGasterosteus aculeatus (gasAcu1)\r\nGlycine max (GCA_000004515.3)\r\nHarpegnathos saltator (GCF_003227715.1)\r\nHIV-1 (AF033819.3)\r\nHomo sapiens (hg19/GRCh37)\r\nHomo sapiens (hg38/GRCh38)\r\nHuman herpesvirus 1 (str. 17, JN555585.1)\r\nHuman herpesvirus 2 (str. 333)\r\nHuman herpesvirus 2 (str. HG52)\r\nHuman herpesvirus 2 (str. SD90e)\r\nHuman herpesvirus 4 (str. Akata)\r\nHuman herpesvirus 6A (U1102_X83413)\r\nHuman herpesvirus 6B (Z29_NC000898)\r\nIxodes scapularis (IscaW1)\r\nLactobacillus delbrueckii (ATCC 11842)\r\nLarimichthys crocea (GCA_900246015.1)\r\nLegionella pneumophila (GCF_000008485.1)\r\nLeishmania braziliensis (MHOM/BR/75/M2904)\r\nLeishmania major Friedlin (LmjF.01)\r\nLepisosteus oculatus (LepOcu1)\r\nLeptospira biflexa (GCF_000017685.1)\r\nLutzomyia longipalpis (LlonJ1.5)\r\nLysobacter enzymogenes (NZ_CP013140.1)\r\nLytechinus variegatus (Lvar_2.2)\r\nManduca sexta (Mansex1.0)\r\nMayetiola destructor (v1.0)\r\nMeleagrid herpesvirus 1 strain FC126 (AF291866.1)\r\nMetarhizium anisopliae (MAN 1.0)\r\nMnemiopsis leidyi (GCA_000226015.1)\r\nMortierella alpine ATCC 32222 (GCA_000240685.2)\r\nMus musculus (mm10/GRCm38)\r\nMustela putorius furo (MusPutFur1.0/musFur1)\r\nMycobacterium marinum M (ASM1834v1)\r\nMycobacterium smegmatis MC2 155 (GCF_000015005.1)\r\nMycobacterium tuberculosis H37Rv (ASM19595v2)\r\nNaegleria gruberi (GCF_000004985.1)\r\nNanNochloropsis oceanica (IMET1.v2)\r\nNasonia vitripennis (Nvit2.1)\r\nNematostella vectensis (nemVec1)\r\nNeurospora crassa (GCA_000182925.2)\r\nNothobranchius furzeri (NotFur1)\r\nNothobranchius furzeri v2 (GCF_001465895.1)\r\nN. vitripennis (Nvit1.0)\r\nOctopus bimaculoides (PRJNA270931)\r\nOgataea polymorpha NCYC495 leu1.1 (Hanpo2)\r\nOikopleura dioica (Odioica)\r\nOncorhynchus mykiss (Omyk_1.0)\r\nOreochromis niloticus (ASM185804v2)\r\nOrientia tsutsugamushi (GCF_900327275.1)\r\nOryctolagus cuniculus (GCA_000003625.1)\r\nOryza sativa (MSU-version_7.0)\r\nOryzias latipes (ASM223467v1)\r\nOryzias latipes (oryLat2)\r\nOvis aries (Oar_v3.1)\r\nPaecilomyces variotii (CBS101075)\r\nPanicum virgatum (Panicum virgatum-v4)\r\nPantoea agglomerans (NZ_CP014129.1)\r\nPapilio polytes (Ppolytes v1)\r\nPenicillium roqueforti (FM164)\r\nPhaeodactylum tricornutum (ASM15095v2)\r\nPhalaeNopsis equestris (ASM126359v1)\r\nPhlebotomus papatasi (Ppapl1.5)\r\nPhyscomitrella patens (Ppatens_318_v3)\r\nPichia pastoris (v2 CBS 7435)\r\nPlasmodium berghei (ANKA v2.0)\r\nPlasmodium cyNomolgi (GCF_000321355.1)\r\nPlasmodium falciparum (3D7 v3.0)\r\nPochonia chlamydosporia (GCF_001653235.2)\r\nPopulus deltoides (WV94 v2.1)\r\nPopulus trichocarpa (v3.1)\r\nPristionchus pacificus (TAU2011)\r\nPseudomonas aerugiNosa (UCBPP-PA14)\r\nPseudomonas cichorii JBC1 (PRJNA232591)\r\nPseudonitzschia multistriata (PsnmuV1.4)\r\nPurpureocillium lilacinum (GCA_001653205.1)\r\nRattus Norvegicus (rn6)\r\nRhodnius prolixus (RproC3)\r\nRhodobacter capsulatus SB 1003 (CP001312.1)\r\nRhodobacter sphaeroides (KD131)\r\nRhodopseudomonas palustris CGA009 (NC_005296.1)\r\nSaccharomyces cerevisiae (sacCer3/S288c)\r\nSalmonella enterica subsp. (GCF_000022165.1)\r\nSalmo salar (GCF_000233375.1)\r\nSchistosoma mansoni (Smansoni_v7)\r\nSchizosaccharomyces pombe (ASM294v2.30)\r\nSchmidtea mediterranea (schMed4)\r\nShigella flexneri 1c (CP020753)\r\nSolanum lycopersicum (SL3.0)\r\nSorghum bicolor (Phytozome v3.1.1)\r\nStaphylococcus aureus (MRSA-ATCC 43300, v1.0)\r\nStaphylococcus aureus (MSSA-GP01, v1.0)\r\nStarmerella bombicola (JCM_9596_assembly_v001)\r\nStreptococcus pyogenes M1 GAS (GCA_000321355.1)\r\nStreptomyces platensis (DSM-40041)\r\nStrongylocentrotus purpuratus (EchiNoBase v3.1)\r\nSus scrofa (Sscrofa10.2)\r\nSus scrofa (Sscrofa11.1)\r\nTetrahymena thermophila (MAC)\r\nTetrahymena thermophila (MIC)\r\nTetranychus urticae (Sept 2009)\r\nThermoanaerobacter ethaNolicus JW 200 (GCF_003722315.1)\r\nToxoplasma gondii GT1 (ToxoDB-28)\r\nToxoplasma gondii ME49 (ToxoDB-28)\r\nTreponema caldarium (GCF_000219725.1)\r\nTribolium castaneum (Tcas5)\r\nTrichoderma harzianum T22 (TriharT22)\r\nTrichoderma reesei (GCA_000167675.2)\r\nTriticum aestivum (IWGSC v1.1)\r\nTrypaNosoma brucei (TREU927)\r\nTrypaNosoma cruzi (TcruziSylvioX10-1-2012)\r\nVaccinia virus (str. WR/AY243312.1)\r\nXeNopus laevis (v9.2)\r\nXeNopus laevis (Xenla9.1/JGI v9.1)\r\nXeNopus tropicalis (v9.1)\r\nXeNopus tropicalis (xenTro9.0/JGI v9.0)\r\nYarrowia lipolytica (GCF_000002525.2)\r\nYarrowia lipolytica (W29)\r\nZea mays (B73 AGPv4)",
        Reference:
          "Montague TG, Cruz JM, GagNon JA, Church GM, Valen E CHOPCHOP: a CRISPR/Cas9 and TALEN web tool for geNome editing. Nucleic Acids Res. 2014 Jul;42(Web Server issue):W401-7. doi: 10.1093/nar/gku410. Epub 2014 May 26. (PMID: 24861617)",
        __EMPTY: "https://chopchop.cbu.uib.No/",
      },
      {
        Platform: "Command-line",
        "Search by": "Coordinate",
        Reference:
          "Labun K, Montague TG, GagNon JA, Thyme SB, Valen E CHOPCHOP v2: a web tool for the next generation of CRISPR geNome engineering. Nucleic Acids Res. 2016 Jul 8;44(W1):W272-6. doi: 10.1093/nar/gkw398. Epub 2016 May 16. (PMID: 27185894)",
      },
      {
        "Search by": "Sequence",
        Reference:
          "Labun K, Montague TG, Krause M, Torres Cleuren YN, Tjeldnes H, Valen E CHOPCHOP v3: expanding the CRISPR web toolbox beyond geNome editing. Nucleic Acids Res. 2019 Jul 2;47(W1):W171-W174. doi: 10.1093/nar/gkz365. (PMID: 31106371)",
      },
      {
        Tools: "CRISPR MultiTargeter",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": "Yes",
        "Score Oligos": "Yes",
        "Search by": "GeneID",
        Enzyme: "Custom",
        PAM: "Custom",
        Organisms:
          "Homo sapiens\r\nMus musculus\r\nRattus Norvegicus\r\nGallus gallus\r\nXeNopus tropicalis\r\nDanio rerio\r\nOryzias latipes\r\nDrosophila melaNogaster\r\nCaeNorhabditis elegans\r\nArabidopsis thaliana\r\nOryza sativa japonica\r\nZea mays",
        Reference:
          "Prykhozhij SV, Rajan V, Gaston D, Berman JN CRISPR multitargeter: a web tool to find common and unique CRISPR single guide RNA targets in a set of similar sequences. PLoS One. 2015 Mar 5;10(3):e0119372. doi: 10.1371/journal.pone.0119372. eCollection 2015. (PMID: 25742428)",
        __EMPTY: "http://www.multicrispr.net/",
      },
      {
        "Search by": "Sequence",
      },
      {
        Tools: "Genedata Selector",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Other",
        "Off‑targets": "Yes",
        "Score Oligos": "No",
        "Search by": "Sequence",
        Enzyme: "Custom",
        PAM: "Custom",
        Organisms: "Provided by user",
        Comments: "Commercial",
        __EMPTY:
          "https://www.genedata.com/products-services/selector/applications",
      },
      {
        Tools: "CrisprGE",
        Availability: "Yes",
        Purpose: "Database",
        Platform: "Web",
        "Off‑targets": "No",
        "Score Oligos": "Yes",
        "Search by": "GeneID",
        Enzyme: "Not applicable",
        PAM: "Not Applicable",
        Organisms:
          "Arabidopsis thaliana\r\nBombyx mori\r\nCaeNorhabditis elegans\r\nCapra aegagrus hircus\r\nCiona intestinalis\r\nCitrus sinensis\r\nCricetulus griseus\r\nDanio rerio\r\nDaphnia\r\nDrosophila melaNogaster\r\nFundulus heteroclitus\r\nGlycine max\r\nHerpes simplex virus\r\nHomo sapiens\r\nMus musculus\r\nNematostella vectensis\r\nNicotiana benthamiana\r\nNicotiana tabacum\r\nOreochromis niloticus\r\nOryza sativa\r\nOryzias latipes\r\nPlasmodium falciparum\r\nPorcine\r\nRattus Norvegicus\r\nSaccharomyces cerevisiae\r\nSolanum lycopersicum\r\nStreptomyces coelicolor\r\nSurrogate reporter\r\nToxoplasma gondii\r\nTriticum aestivum\r\nXeNopus tropicalis\r\nZea mays",
        Reference:
          "Kaur K, Tandon H, Gupta AK, Kumar M CrisprGE: a central hub of CRISPR/Cas-based geNome editing. Database (Oxford). 2015 Jun 27;2015:bav055. doi: 10.1093/database/bav055. Print 2015. (PMID: 26120138)",
        Comments:
          "Contains 4680 sequence entries for 223 unique target genes. It provides links to the literature for each CRISPR sequence.",
        __EMPTY: "http://crdd.osdd.net/servers/crisprge/",
      },
      {
        "Search by": "Sequence",
      },
      {
        "Search by": "Other",
      },
      {
        Tools: "CINDEL",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": "No",
        "Score Oligos": "Yes",
        "Search by": "Sequence",
        Enzyme: "Not applicable",
        PAM: "Not Applicable",
        Organisms: "Not Applicable",
        Reference:
          "Kim HK, Song M, Lee J, MeNon AV, Jung S, Kang YM, Choi JW, Woo E, Koh HC, Nam JW, Kim H In vivo high-throughput profiling of CRISPR-Cpf1 activity. Nat Methods. 2017 Feb;14(2):153-159. doi: 10.1038/nmeth.4104. Epub 2016 Dec 19. (PMID: 27992409)",
        Comments:
          "A tool to predict and score indel frequencies for a given target sequence of the AsCpf1 caspase",
        __EMPTY: "http://big.hanyang.ac.kr/cindel/",
      },
      {
        Tools: "CRISPR-RT",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": "No",
        "Score Oligos": "No",
        "Search by": "Sequence",
        Enzyme: "Cas13a (C2c2)",
        PAM: "Custom",
        Organisms: "Provided by user",
        Reference:
          "Zhu H, Richmond E, Liang C CRISPR-RT: a web application for designing CRISPR-C2c2 crRNA with improved target specificity. Bioinformatics. 2018 Jan 1;34(1):117-119. doi: 10.1093/bioinformatics/btx580. (PMID: 28968770)",
        Comments:
          "For RNA targeting. Desktop version downloadable from https://github.com/bioinfolabmu/CRISPR-RT.",
        __EMPTY: "http://bioinfolab.miamioh.edu/CRISPR-RT/",
      },
      {
        Platform: "Desktop",
      },
      {
        Tools: "multicrispr",
        Availability: "Yes",
        Purpose: "Oligo Designer",
        Platform: "Command-line",
        "Off‑targets": "Yes",
        "Score Oligos": "Yes",
        "Search by": "GeneID",
        Enzyme: "Custom",
        PAM: "Custom",
        Organisms: "Provided by user",
        Reference:
          "Bhagwat AM, Graumann J, Wiegandt R, Bentsen M, Welker J, Kuenne C, Preussner J, Braun T, Looso M multicrispr: gRNA design for prime editing and parallel targeting of thousands of targets. Life Sci Alliance. 2020 Sep 9;3(11). pii: 3/11/e202000757. doi: 10.26508/lsa.202000757. Print 2020 Nov. (PMID: 32907859)",
        Comments: "R/Bioconductor tool",
        __EMPTY: "https://multicrispr.net/",
      },
      {
        Platform: "Other",
        "Search by": "Coordinate",
      },
    ],
    "Microbes and Flies": [
      {
        Tools: "CRISPRdirect",
        Availability: " Yes",
        Purpose: "Oligo Designer",
        Platform: "Web-server based",
        "Off‑targets": " Yes",
        "Score Oligos": "No",
        "Search by": "GeneID",
        Enzyme: "Custom",
        PAM: " NGG, NRG, NNGRRT, NG",
        Organisms:
          "678 entries : Mammal, vertebrate, deuterostome, insect, nematode, virus, insect, metazoa, ciliophora, plant, fungi, Bacillariophyta, Chlorobi, cyanobacteria, Other",
        Reference:
          "Naito Y, Hino K, Bono H, Ui-Tei K CRISPRdirect: software for designing CRISPR/Cas guide RNA with reduced off-target sites. Bioinformatics. 2015 Apr 1;31(7):1120-3. doi: 10.1093/bioinformatics/btu743. Epub 2014 Nov 20. (PMID: 25414360)",
        __EMPTY: "https://crispr.dbcls.jp/",
      },
      {
        Availability: " ",
        "Search by": "Coordinate",
      },
      {
        "Search by": "Sequence",
      },
      {
        Tools: "Cas-OFFinder",
        Availability: " Yes",
        Purpose: "Finds Off-Target",
        Platform: "Web",
        "Off‑targets": " Yes",
        "Score Oligos": "No",
        "Search by": "Sequence",
        Enzyme:
          "SpCas9 from Streptococcus pyogenes: 5'-NGG-3'\r\nSpCas9 from Streptococcus pyogenes: 5'-NRG-3' (R = A or G)\r\nStCas9 from Streptococcus thermophilus: 5'-NNAGAAW-3' (W = A or T)\r\nNmCas9 from Neisseria meningitidis: 5'-NNNNGMTT-3' (M = A or C)\r\nSaCas9 from Staphylococcus aureus: 5'-NNGRRT-'3 (R=A or G)\r\nCjCas9 from Campylobacter jejuni: 5'-NNNVRYAC-3' (V = G or C or A, R = A or G, Y = C or T)\r\nCjCas9 from Campylobacter jejuni: 5'-NNNNRYAC-3' (R=A or G, Y= C or T)\r\nAsCpf1 from Acidaminococcus or LbCpf1 from Lachnospiraceae: 5'-TTTN-3'\r\nAsCpf1 from Acidaminococcus or LbCpf1 from Lachnospiraceae: 5'-TTTV-3' (V = G or C or A)\r\nSpCas9 from Streptococcus pasteurianus: 5'-NNGTGA-3'\r\nFnCpf1 from Francisella: 5'-TTN-3'\r\nSaCas9 from Staphylococcus aureus: 5'-NNNRRT-'3 (R=A or G)\r\nFnCpf1 from Francisella: 5'-KYTV-3'\r\nVRER SpCas9 from Streptococcus pyogenes: 5'-NGCG-3'\r\nVQR SpCas9 from Streptococcus pyogenes: 5'-NGA-3'\r\nXCas9 3.7 (TLIKDIV SpCas9) from Streptococcus pyogenes: 5'-NGT-3'\r\nXCas9 3.7 (TLIKDIV SpCas9) from Streptococcus pyogenes: 5'-NG-3'\r\nBhCas12b from Bacillus hisashii: 5'-TTN-3'\r\nBhCas12b v4 from Bacillus hisashii: 5'-ATTN-3'\r\nBhCas12b v4 from Bacillus hisashii: 5'-DTTN-3'\r\nSpy-macCas9 from Streptococcus pyogenes and Streptococcus macacae: 5'-NAAN-3'",
        PAM: "NGG\r\nNRG\r\nNNAGAAW\r\nNNNNGMTT\r\nNNGRRT\r\nNNNVRYAC\r\nNNNNRYAC\r\nTTTN\r\nTTTV\r\nNNGTGA\r\nTTN\r\nNNNRRT\r\nKYTV\r\nNGCG\r\nNGA\r\nNGT\r\nNG\r\nTTN\r\nATTN\r\nDTTN",
        Organisms:
          "Homo sapiens (GRCh38/hg38) - Human\r\nHomo sapiens (hg19) - Human\r\nMus musculus (mm10) - Mouse\r\nBos taurus (bosTau7) - Cow\r\nCanis familiaris (canFam3) - Dog\r\nRattus norvegicus (rn5) - Rat\r\nSus scrofa (susScr11) - Pig\r\nDanio rerio (danRer7) - Zebrafish\r\nMacaca mulatta(rheMac3) - Monkey\r\nXenopus tropicalis (JGI 4.2/xenTro3) - Western clawed frog\r\nXenopus tropicalis (JGI 7.1) - Western clawed frog\r\nXenopus laevis (JGI 7.1) - African clawed frog\r\nCricetulus griseus (v1.0) - Chinese Hamster\r\nXenopus tropicalis (JGI 8.0) - Western clawed frog\r\nXenopus tropicalis (JGI 9.0) - Western clawed frog\r\nXenopus laevis (JGI 9.1) - African clawed frog\r\nMacaca fascicularis (5.0) - Crab-eating macaque\r\nDanio rerio (GRCz10) - Zebrafish\r\nRattus norvegicus (Rnor 6.0) - Rat\r\nEquus caballus (EquCab2) - Horse\r\nOryctolagus cuniculus (OryCun2) - Rabbit\r\nMesocricetus auratus (MesAur1.0) - Golden hamster\r\nFelis catus (6.2) - Cat\r\nMochrtus ochrogaster (AHZW01) - Prairie vole\r\nBos taurus (bosTau8) - Cow\r\nCallithrix jacchus (calJac3) -marmoset\r\nPimephales promelas (SOAPdenovo v. 2) - fathead minnow\r\novis_aries (Oar_v3.1) - sheep\r\nChlorocebus_sabeus (AQIB01) - green monkey\r\nGallus gallus (Galgal4) - Chicken\r\nOreochromis niloticus (ASM185804v2)\r\nCapra hircus (ARS1) - goat\r\nPeromyscus maniculatus (Pman_1.0) - North American deer mouse\r\nMacaca mulatta (Mmul_8.0.1) - Monkey\r\nCricetulus griseus (CriGri_1.0) - Chinese hamster\r\nCavia porcellus (Cavpor3.0) -guinea pig\r\nDanio rerio (GRCz11) - zebrafish\r\nMus musculus (mm9) - Mouse\r\nAnas platyrhynchos (BGI_duck_1.0) - duck\r\nGallus gallus (Gallus_gallus-5.0) - Chicken\r\nSalmo salar (ICSASG_v2) -Atlantic salmon\r\nCyprinus carpio (common carp genome)\r\nOncorhynchus tshawytscha (Otsh_v1.0) - Chinook salmon\r\nHuman Mitochondria genome (NC_012920.1)\r\nBalbC/J mouse Mitochondria genome (AJ512208.1)\r\nBos taurus (ARS UCD1.2)\r\nParalichthys olivaceus (Flounder_ref_guided_V1.0)\r\nMyotis lucifugus (Myoluc2.0)",
        Reference:
          "Bae S, Park J, Kim JS Cas-OFFinder: a fast and versatile algorithm that searches for potential off-target sites of Cas9 RNA-guided endonucleases. Bioinformatics. 2014 May 15;30(10):1473-5. doi: 10.1093/bioinformatics/btu048. Epub 2014 Jan 24. (PMID: 24463181)",
        Comments: "Cas9, Cas12f1, Cas12e",
        __EMPTY: "http://www.rgenome.net/cas-offinder/",
      },
      {
        Availability: " ",
        Platform: "Command-line",
      },
      {
        Tools: "CRISPR Optimal Target Finder",
        Availability: " Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": " Yes",
        "Score Oligos": "No",
        "Search by": "Sequence",
        Enzyme: "SpCas9",
        PAM: "NGG. NRG",
        Organisms:
          "Acyrthosiphon pisum (2)\r\nAedes aegypti (AaegL5.0 GCA_002204515.1)\r\nAnopheles gambiae (strain M) (AgamM1)\r\nAnopheles gambiae (strain S) (AgamS1)\r\nApis cerana cerana (GCA_002290385.1)\r\nApis mellifera (apiMel3)\r\nBombus impatiens (GCA_000188095.2)\r\nCaenorhabditis elegans (ce10)\r\nCeratitis capitata (1.1)\r\nDaphnia pulex (GCA_900092285.1_ASM90009228v1)\r\nDaphnia pulex (GCA_000187875.1)\r\nDrosophila ananassae (dana_r1.3_FB2011_07)\r\nDrosophila erecta (dere_r1.3_FB2011_08)\r\nDrosophila eugracilis (GCA_000236325.2)\r\nDrosophila grimshawi (dgri_r1.3_FB2010_02)\r\nDrosophila mauritiana (Dmau_MS17)\r\nDrosophila melanogaster (r_6)\r\nDrosophila melanogaster (vasa-cas9 III, BDSC_51324)\r\nDrosophila mojavensis (dmoj_r1.3_FB2011_05)\r\nDrosophila persimilis (dper_r1.3_FB2010_02)\r\nDrosophila pseudoobscura (dpse_r3.1_FB2013_03)\r\nDrosophila sechellia (droSec1)\r\nDrosophila simulans (DsimV2)\r\nDrosophila simulans (droSim1)\r\nDrosophila virilis (droVir3)\r\nDrosophila willistoni (dwil_r1.3_FB2010_02)\r\nDrosophila yakuba (droYak2)Lutzomyia longipalpis (GCA_000265325.1)\r\nManduca sexta (GCA_000262585.1)\r\nNasonia vitripennis (2.1)\r\nOncopeltus fasciatus (GCA_000696205.2)\r\nParacoccus denitrificans (GCA_000203895.1)\r\nRhodnius prolixus (GCA_000181055.3)\r\nSchistosoma mansoni (GCA_000237925.2)\r\nTribolium castaneum (Tcas 4.0 draft)",
        Reference:
          "Gratz SJ, Ukken FP, Rubinstein CD, Thiede G, Donohue LK, Cummings AM, O'Connor-Giles KM Highly specific and efficient CRISPR/Cas9-catalyzed homology-directed repair in Drosophila. Genetics. 2014 Apr;196(4):961-71. doi: 10.1534/genetics.113.160713. Epub 2014 Jan 29. (PMID: 24478335)",
        Comments: "Specific for flies, mosquitoes and worms.",
        __EMPTY: "http://targetfinder.flycrispr.neuro.brown.edu/",
      },
      {
        Availability: " ",
      },
      {
        Tools: "CRISPR Finder (HTGT WGE)",
        Availability: " Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": " Yes",
        "Score Oligos": "No",
        "Search by": "GeneID",
        Enzyme: "SpCas9",
        PAM: "NGG",
        Organisms: "Homo sapiens, Mus musculus",
        Reference:
          "Hodgkins A, Farne A, Perera S, Grego T, Parry-Smith DJ, Skarnes WC, Iyer V WGE: a CRISPR database for genome engineering. Bioinformatics. 2015 Sep 15;31(18):3078-80. doi: 10.1093/bioinformatics/btv308. Epub 2015 May 14. (PMID: 25979474)",
        __EMPTY: "https://wge.stemcell.sanger.ac.uk//find_crisprs",
      },
      {
        Availability: " ",
        "Search by": "Sequence",
      },
      {
        Tools: "Off-Spotter",
        Availability: " Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": " Yes",
        "Score Oligos": "No",
        "Search by": "Sequence",
        Enzyme:
          "SpCas9 (Streptococcus pyogenes)\r\nCjCas9 (Campylobacter jejuni)\r\nSaCas9 (Staphylococcus aureus)",
        PAM: "NGG\r\nNAG\r\nNNNNACA\r\nNNGRRT",
        Organisms:
          "Homo sapiens\r\nMus musculus\r\nSaccharomyces cerevisiae (w303)",
        Reference:
          'Pliatsika V, Rigoutsos I "Off-Spotter": very fast and exhaustive enumeration of genomic lookalikes for designing CRISPR/Cas guide RNAs. Biol Direct. 2015 Jan 29;10:4. doi: 10.1186/s13062-015-0035-z. (PMID: 25630343)',
        Comments: "It provides a batch utility to remote submission",
        __EMPTY: "https://cm.jefferson.edu/Off-Spotter/",
      },
      {
        Availability: " ",
        Platform: "Other",
      },
      {
        Tools: "CRISPR sgRNA Design Tool",
        Availability: " Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": " Yes",
        "Score Oligos": "No",
        "Search by": "Sequence",
        Enzyme: "SpCas9",
        PAM: "NGG",
        Organisms:
          "Human (hg19)\r\nchicken (galGal4)\r\nc. elegans (ce10)\r\ndog (canFam3)\r\nfly (dm3)\r\nMouse (mm9)\r\npig (susScr3)\r\npossum (monDom5)\r\nrabbit (oryCun2)\r\nrat (rn5)\r\nstickleback (gasAcu1)\r\nzebrafish (danRer7)",
        __EMPTY: "https://www.genscript.com/gRNA-design-tool.html",
      },
      {
        Availability: " ",
      },
      {
        Tools: "CRISPR-PN",
        Availability: " Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": " Yes",
        "Score Oligos": "No",
        "Search by": "Sequence",
        Enzyme: "Custom",
        PAM: "NGG",
        Organisms:
          "Heterorhabditis bacteriophora WS236\r\nSteinernema carpocapsae PRJNA202318.WBPS11\r\nStrongyloides stercoralis PRJEB528\r\nStrongyloides ratti PRJEA62033.WS237\r\nNippostrongylus brasiliensis PRJEB511.WBPS11\r\nTrichuris muris PRJEB126.WS264\r\nNecator americanus PRJNA72135.WS242\r\nAncylostoma ceylanicum PRJNA231479.WS248\r\nAncylostoma caninum PRJNA72585.WBPS11\r\nPristionchus pacificus PRJNA12644.WS260\r\nBrugia malayi PRJNA10729.WS260\r\nLoa loa PRJNA60051.WS237\r\nHaemonchus contortus PRJEB506.WS248\r\nMeloidogyne incognita PRJEA28837.WS248",
        Reference:
          "O'Halloran DM Genome aware CRISPR gRNA target prediction for parasitic nematodes. Mol Biochem Parasitol. 2019 Jan;227:25-28. doi: 10.1016/j.molbiopara.2018.12.001. Epub 2018 Dec 7. (PMID: 30529475)",
        Comments: "CRISPR gRNA target prediction for parasitic nematodes",
        __EMPTY: "http://ohalloranlab.net/crispr-pn2/",
      },
      {
        Availability: " ",
      },
      {
        Tools: "CRISPR-ERA",
        Availability: " Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": " Yes",
        "Score Oligos": "No",
        "Search by": "GeneID",
        Enzyme: "SpCas9 and nickase",
        PAM: "NGG\r\nNAG\r\nNRG",
        Organisms:
          "Human(GRCh37/hg19)\r\nMouse(GRCm38/mm10)\r\nRat(RGSC 5.0/rn5)\r\nZebrafish(Zv9/danRer7)\r\nD.melanogaster(BDGP R5/dm3)\r\nC.elegans(WS220/ce6)\r\nS.cerevisiae(SacCer_Apr2011/sacCer3)\r\nEscherichia coli(NC_000913)\r\nBacillus subtilis(NC_000964)",
        Reference:
          "Liu H, Wei Z, Dominguez A, Li Y, Wang X, Qi LS CRISPR-ERA: a comprehensive design tool for CRISPR-mediated gene editing, repression and activation. Bioinformatics. 2015 Nov 15;31(22):3676-8. doi: 10.1093/bioinformatics/btv423. Epub 2015 Jul 23. (PMID: 26209430)",
        Comments:
          "It provides source code for the generation of whole-genome sgRNA libraries in other species",
        __EMPTY: "http://crispr-era.stanford.edu/",
      },
      {
        Availability: " ",
        Purpose: "Database",
        Platform: "Command-line",
        "Search by": "Coordinate",
      },
      {
        "Search by": "Sequence",
      },
      {
        Tools: "EuPaGDT",
        Availability: " Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": " Yes",
        "Score Oligos": " Yes",
        "Search by": "Sequence",
        Enzyme:
          "SpCas9: gRNA length 20; 3'PAM: NGG; off-target PAM: NAG, NGA\r\nSaCas9: gRNA length 21; 3'PAM: NNGRRT; off-target PAM: NNGRRA,NNGRRG,NNGRRC\r\nAsCpf1: gRNA length 20; 5'PAM: TTTN;\r\nCjCas9: gRNA length 23; 3'PAM: NNNNACA; off-target PAM: NNNNACAC,NNNNRYAC\r\nFnCas9: gRNA length 21; 3'PAM: NGG; off-target PAM: NGA,NAG,NAA\r\nFnCpf1: gRNA length 23; 5'PAM: TTN,YG; off-target PAM: CTN\r\nLbCpf1: gRNA length 20; 5'PAM: TTTN; off-target PAM: None\r\nNmCas9: gRNA length 20; 3'PAM: NNNNGANN; off-target PAM: NNNNGTTN,NNNNGNNT,NNNNGTNN,NNNNGNTN\r\nSt1Cas9: gRNA length 20; 3'PAM: NNAGAA,NNGGAA; off-target PAM: NNAGGA,NNANAA,NNGGGA\r\nTdCas9: gRNA length 20; 3'PAM: NAAAAC; off-target PAM: None",
        PAM: "NGG\r\nNNGRRT\r\nTTTN\r\nNNNNACA\r\nNNNNACAC\r\nNGG\r\nYG\r\nTTTN\r\nNNNNGANN\r\nNNAGAA\r\nNNGGAA\r\nNNAAAAC",
        Organisms:
          "A. castellanii Neff AmoebaDB-26\r\nE. dispar SAW760 AmoebaDB-26\r\nE. histolytica (Amoeba) HM1IMSS AmoebaDB v26\r\nE. histolytica (Amoeba) HM1IMSS AmoebaDB v28\r\nE. histolytica HM1IMSS-A AmoebaDB-26\r\nE. histolytica HM1IMSS-B AmoebaDB-26\r\nE. histolytica HM3IMSS AmoebaDB-26\r\nE. histolytica KU27 AmoebaDB-26\r\nE. invadens IP1 AmoebaDB-26\r\nE. moshkovskii Laredo AmoebaDB-26\r\nE. nuttalli P19 AmoebaDB-26\r\nN. fowleri ATCC30863 AmoebaDB-26\r\nC. hominis TU502 CryptoDB-26\r\nC. muris RN66 CryptoDB-26\r\nC. parvum IowaII CryptoDB-26\r\nC. parvum IowaII CryptoDB-28\r\nC. velia CCMP2878 CryptoDB-26\r\nG. niphandrodes Unknown CryptoDB-26\r\nV. brassicaformis CCMP3155 CryptoDB-26\r\nA. aculeatus ATCC16872 FungiDB-26\r\nA. capsulatus G186AR FungiDB-26\r\nA. capsulatus NAm1 FungiDB-26\r\nA. carbonarius ITEM5010 FungiDB-26\r\nA. clavatus NRRL1 FungiDB-26\r\nA. flavus NRRL3357 FungiDB-26\r\nA. fumigatus Af293 FungiDB-26\r\nA. macrogynus ATCC38327 FungiDB-26\r\nA. nidulans FGSCA4 FungiDB-26\r\nA. niger ATCC1015 FungiDB-26\r\nA. niger CBS513-88 FungiDB-26\r\nA. oryzae RIB40 FungiDB-26\r\nA. terreus NIH2624 FungiDB-26\r\nB. dendrobatidis JEL423 FungiDB-26\r\nB. fuckeliana B05-10 FungiDB-26\r\nC. albicans SC5314 FungiDB-26\r\nC. cinerea okay7-130 FungiDB-26\r\nC. gattii R265 FungiDB-26\r\nC. gattii WM276 FungiDB-26\r\nC. glabrata CBS138 FungiDB-26\r\nC. immitis H538-4 FungiDB-26\r\nC. immitis RS FungiDB-26\r\nC. neoformans B-3501A FungiDB-26\r\nC. neoformans H99 FungiDB-26\r\nC. neoformans JEC21 FungiDB-26\r\nC. parapsilosis CDC317 candidagenome.org\r\nC. posadasii C735deltSOWgp FungiDB-26\r\nC. posadasii RMSCC3488 FungiDB-26\r\nC. posadasii Silveira FungiDB-26\r\nF. graminearum PH-1 FungiDB-26\r\nF. oxysporum 4287 FungiDB-26\r\nF. verticillioides 7600 FungiDB-26\r\nH.capsulatum G217B Washington University\r\nM. circinelloides CBS277-49 FungiDB-26\r\nM. globosa CBS7966 FungiDB-26\r\nM. larici-populina 98AG31 FungiDB-26\r\nM. oryzae 70-15 FungiDB-26\r\nN.crassa OR74A FungiDB-26\r\nN. discreta FGSC8579 FungiDB-26\r\nN. fischeri NRRL181 FungiDB-26\r\nN. tetrasperma FGSC2508 FungiDB-26\r\nP. blakesleeanus NRRL1555- FungiDB-26\r\nP. chrysosporium RP-78 FungiDB-26\r\nP. graminis CRL75-36-700-3 FungiDB-26\r\nP. jirovecii SE8 FungiDB-26\r\nR. delemar RA99-880 FungiDB-26\r\nS. cerevisiae S288c FungiDB-26\r\nS. japonicus yFS275 FungiDB-26\r\nS. macrospora k-hell FungiDB-26\r\nS. octosporus yFS286 FungiDB-26\r\nS. pombe 972h- FungiDB-26\r\nS. punctatus DAOMBR117 FungiDB-26\r\nS. reilianum SRZ2 FungiDB-26\r\nS. sclerotiorum 1980UF-70 FungiDB-26\r\nT. marneffei ATCC18224 FungiDB-26\r\nT. mesenterica DSM1558 FungiDB-26\r\nT. reesei QM6a FungiDB-26\r\nT. harzianum ASM98886v1 EnsemblFungi\r\nT. harzianum GCA_002838845.1_ASM283884v1 NCBI\r\nT. stipitatus ATCC10500 FungiDB-26\r\nU. maydis 521 FungiDB-26\r\nY. lipolytica CLIB122 FungiDB-26\r\nG. intestinalis Assemblage_A GiardiaDB-26\r\nG. intestinalis Assemblage_A GiardiaDB-28\r\nG. intestinalis AssemblageADH GiardiaDB-26\r\nG. intestinalis Assemblage_B GiardiaDB-26\r\nG. intestinalisAssemblageBGS B GiardiaDB-26\r\nG. intestinalis AssemblageE GiardiaDB-26\r\nS. salmonicida ATCC50377 GiardiaDB-26\r\nA. algerae PRA109 MicrosporidiaDB-26\r\nA. algerae PRA339 MicrosporidiaDB-26\r\nE. aedis USNM41457 MicrosporidiaDB-26\r\nE. bieneusi H348 MicrosporidiaDB-26\r\nE. cuniculi1 EC1 MicrosporidiaDB-26\r\nE. cuniculi EC2 MicrosporidiaDB-26\r\nE. cuniculi EC3 MicrosporidiaDB-26\r\nE. cuniculi GBM1 MicrosporidiaDB-26\r\nE. hellem ATCC50504 MicrosporidiaDB-26\r\nE. hellem Swiss MicrosporidiaDB-26\r\nE. intestinalis ATCC50506 MicrosporidiaDB-26\r\nE. romaleae SJ2008 MicrosporidiaDB-26\r\nM. daphniae UGP3 MicrosporidiaDB-26\r\nN. bombycis CQ1 MicrosporidiaDB-26\r\nN. ceranae BRL01 MicrosporidiaDB-26\r\nN. ematocida Sp1ERTm2 MicrosporidiaDB-26\r\nN. parisii ERTm1 MicrosporidiaDB-26\r\nN. parisii ERTm3 MicrosporidiaDB-26\r\nN. sp 1ERTm6 MicrosporidiaDB-26\r\nO. colligata OC4 MicrosporidiaDB-26\r\nS. lophii42 110 MicrosporidiaDB-26\r\nT. hominis Unknown MicrosporidiaDB-26\r\nV. corneae ATCC50505 MicrosporidiaDB-26\r\nV. culicisfloridensis MicrosporidiaDB-26 \r\nA. astaci APO3 FungiDB-26\r\nA. candida 2VRR FungiDB-26\r\nA. invadans NJM9701 FungiDB-26\r\nA. laibachii Nc14 FungiDB-26\r\nH. arabidopsidis Emoy2 FungiDB-26\r\nP. aphanidermatum DAOMBR444 FungiDB-26\r\nP. arrhenomanes ATCC12531 FungiDB-26\r\nP. capsici LT1534 FungiDB-26\r\nP. cinnamomi CBS144-22 FungiDB-26\r\nP. infestans T30-4 FungiDB-26\r\nP. irregulare DAOMBR486 FungiDB-26\r\nP. iwayamai DAOMBR242034 FungiDB-26\r\nP. parasitica INRA-310 FungiDB-26\r\nP. ramorum Pr-102 FungiDB-26\r\nP. sojae P6497 FungiDB-26\r\nP. ultimum BR650 FungiDB-26\r\nP. ultimum DAOMBR144 FungiDB-26\r\nP. vexans DAOMBR484 FungiDB-26\r\nS. diclina VS20 FungiDB-26\r\nS. parasitica CBS223-65 FungiDB-26\r\nB. bigemina BOND PiroplasmaDB-26\r\nB. bovis T2Bo PiroplasmaDB-26\r\nB. microti RI PiroplasmaDB-26\r\nC. felis Winnie PiroplasmaDB-26\r\nT. annulata Ankara PiroplasmaDB-26\r\nT. equi WA PiroplasmaDB-26\r\nT. orientalis Shintoku PiroplasmaDB-26\r\nT. parva Muguga PiroplasmaDB-26\r\nP. berghei ANKA PlasmoDB-26\r\nP. berghei ANKA PlasmoDB-28\r\nP. chabaudichabaudi PlasmoDB-26 \r\nP. cynomolgi B PlasmoDB-26\r\nP. falciparum 3D7 PlasmoDB-26\r\nP. falciparum 3D7 PlasmoDB-28\r\nP. falciparum 3D7 PlasmoDB-32\r\nP. falciparum IT PlasmoDB-26\r\nP. gallinaceum 8A PlasmoDB-26\r\nP. knowlesi H PlasmoDB-26\r\nP. reichenowi CDC PlasmoDB-26\r\nP. vivax SaI1 PlasmoDB-26\r\nP. yoeliiyoelii 17X PlasmoDB-26\r\nP. yoeliiyoelii 17X PlasmoDB-28\r\nP. yoeliiyoelii 17XNL PlasmoDB-26\r\nP. yoeliiyoelii YM PlasmoDB-26\r\nP. yoeliiyoelii YM PlasmoDB-28\r\nBundle of P. falciparum P. vivax P. Malariae P. ovale P.knowlesi PlasmoDB-28\r\nE. acervulina Houghton ToxoDB-26\r\nE. brunetti Houghton ToxoDB-26\r\nE. falciformis BayerHaberkorn1970 ToxoDB-26\r\nE. maxima Weybridge ToxoDB-26\r\nE. mitis Houghton ToxoDB-26\r\nE. necatrix Houghton ToxoDB-26\r\nE. praecox Houghton ToxoDB-26\r\nE. tenella Houghton ToxoDB-26\r\nE. tenella Houghton ToxoDB-40\r\nH. hammondi HH34 ToxoDB-26\r\nN.caninum LIV ToxoDB-26\r\nS. neurona SN1 ToxoDB-26\r\nS. neurona SN3 ToxoDB-26\r\nT. gondii GT1 ToxoDB-26\r\nT. gondii GT1 ToxoDB-28\r\nT. gondii GT1 ToxoDB-32\r\nT. gondii ME49 ToxoDB-26\r\nT. gondii ME49 ToxoDB-28\r\nT. gondii ME49 ToxoDB-32\r\nT. gondii RH ToxoDB-26(limited annotation)\r\nT. gondii RH ToxoDB-28\r\nT. gondii VEG ToxoDB-26\r\nT. gondii VEG ToxoDB-28\r\nT. vaginalis TrichDB v1.3\r\nT. vaginalis G3 TrichDB-26\r\nA. deanei European Nucleotide Archive GCA_000442575.2 Dec 2014 v83.1\r\nC. fasciculata CfCl TriTrypDB-26\r\nCrithidia mellificae (L. passim) Lotmaria passim whole genome shotgun (WGS) project, accession number AHIJ01000000, last update 2015-03-11\r\nCrithidia bombi ENA acc# OESO01000000\r\nE. monterogeii LV88 TriTrypDB-26\r\nL. aethiopica L147 TriTrypDB-26\r\nL. arabica LEM1108 TriTrypDB-26\r\nL. braziliensis MHOMBR75M2903 TritrypDB-26\r\nL. braziliensis MHOMBR75M2903 TritrypDB-28\r\nL. braziliensis MHOMBR75M2904 TriTrypDB-26\r\nL. donovani BPK282A1 TritrypDB-26\r\nL. donovani BPK282A1 TritrypDB-28\r\nL. enriettii LEM3045 TriTrypDB-26\r\nL. gerbilli LEM452 TriTrypDB-26\r\nL. infantum JPCM5 TritrypDB-26\r\nL. infantum JPCM5 TritrypDB-28\r\nL. major Friedlin TritrypDB-26\r\nL. major Friedlin TritrypDB-28\r\nL. major LV39c5 TriTrypDB-26\r\nL. major LV39c5 TriTrypDB-28\r\nL. major SD75.1 TriTrypDB-26\r\nL. major SD75.1 TriTrypDB-28\r\nL. mexicana MHOMGT2001U1103 TritrypDB-26\r\nL. mexicana MHOMGT2001U1103 TritrypDB-28\r\nL. panamensis MHOMCOL81L13 TriTrypDB-26\r\nL. pyrrhocoris H10 TriTrypDB-26\r\nL. seymouri ATCC30220 TriTrypDB-26\r\nL. seymouri ATCC30220 TriTrypDB-28\r\nL. sp MARLEM2494 TriTrypDB-26\r\nL. tarentolae ParrotTarII TriTrypDB-26\r\nL. tropica L590 TriTrypDB-26\r\nL. turanica LEM423 TriTrypDB-26\r\nT. brucei Lister427 TriTrypDB-26\r\nT. brucei Lister427 TriTrypDB-28\r\nT. brucei TREU927 TritrypDB-28\r\nT. bruceigambiense DAL972 TriTrypDB-26\r\nT. congolense IL3000 TriTrypDB-26\r\nT. cruzi CLBrener Non-Esmeraldo-like TriTrypDB-28\r\nT. cruzi CLBrener Esmeraldo-like TriTrypDB-28\r\nT. cruzi CLBrener TritrypDB-28 (hybrid diploid)\r\nT. cruzi CLBrener TritrypDB-29 (hybrid diploid, including unassigned contigs)\r\nT. cruzi CLBrener TritrypDB-32 (hybrid diploid)\r\nT. cruzi Brazil pacbio CANU assembly\r\nT. cruzi Brazil pacbio SMRT assembly\r\nT. cruzi Brazil pacbio assembly\r\nT. cruzi Dm28c TritrypDB-26\r\nT. cruzi Dm28c TritrypDB-28\r\nT. cruzi SylvioX10-1 TritrypDB-26\r\nT. cruzi SylvioX10-1 TritrypDB-28\r\nT. cruzi marinkellei B7 TriTrypDB-26\r\nT. cruzi Tulauhein Biomedical Research Institute, AIST\r\nT. evansi STIB805 TriTrypDB-26\r\nT. grayi ANR4 TriTrypDB-26\r\nT. rangeli SC58 TriTrypDB-26\r\nT. vivax Y486 TriTrypDB-26\r\nParamecium tetraurelia Stock d4-2 strain (repeat masked)\r\nBiomphalaria glabrata BB02 strain released 2013.4 (repeat masked)\r\nCanis_familiaris v3.1 (repeat masked)\r\nD. melanogaster r6.04 FB2015_01\r\nD. melanogaster r6.19 FB2017_06\r\nA.gambiae AgamP4\r\nEnterobacteria phage T4\r\nMus musculus BALB/c \r\nMus musculus C57BL/6J \r\nHomo Sapiens hg38, GRCh38 Genome Reference Consortium Human Reference 38 (GCA_000001405.2) \r\nTetradesmus obliquus DOE0152z \r\nMycobacterium bovis BCG 1",
        Reference:
          "Peng D, Tarleton R EuPaGDT: a web tool tailored to design CRISPR guide RNAs for eukaryotic pathogens. Microb Genom. 2015 Oct 30;1(4):e000033. doi: 10.1099/mgen.0.000033. eCollection 2015 Oct. (PMID: 28348817)",
        Comments:
          "Mostly for eukaryotic pathogens. User can provide its own file with a non-listed organism (in FASTA format)",
        __EMPTY: "http://grna.ctegd.uga.edu/",
      },
      {
        Availability: " ",
      },
      {
        Tools: "CROP-IT",
        Availability: "No",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": " Yes",
        "Score Oligos": " Yes",
        "Search by": "Coordinate",
        Enzyme: "SpCas9",
        PAM: "NGG\r\nNNG",
        Organisms: "Homo sapiens, Mus musculus",
        Reference:
          "Singh R, Kuscu C, Quinlan A, Qi Y, Adli M Cas9-chromatin binding information enables more accurate CRISPR off-target prediction. Nucleic Acids Res. 2015 Oct 15;43(18):e118. doi: 10.1093/nar/gkv575. Epub 2015 Jun 1. (PMID: 26032770)",
        Comments:
          "sgRNA design and off-target prediction are splitted in two separated tools. All results are provided only via e-mail.",
        __EMPTY:
          "http://cheetah.bioch.virginia.edu/AdliLab/CROP-IT/homepage.html",
      },
      {
        Availability: " ",
        Purpose: "Other",
        "Search by": "Sequence",
      },
      {
        Tools: "Cas-Designer",
        Availability: " Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": " Yes",
        "Score Oligos": " Yes",
        "Search by": "Sequence",
        Enzyme:
          "SpCas9 from Streptococcus pyogenes: 5'-NGG-3'\r\nSpCas9 from Streptococcus pyogenes: 5'-NGG-3' for target, 5'-NRG-3' for off-target (R = A or G)\r\nStCas9 from Streptococcus thermophilus: 5'-NNAGAAW-3' (W = A or T)\r\nNmCas9 from Neisseria meningitidis: 5'-NNNNGMTT-3' (M = A or C)\r\nSaCas9 from Staphylococcus aureus: 5'-NNGRRT-'3 (R=A or G)\r\nCjCas9 from Campylobacter jejuni: 5'-NNNVRYAC-3' (V = G or C or A, R = A or G, Y = C or T)\r\nCjCas9 from Campylobacter jejuni: 5'-NNNNRYAC-3' (R = A or G, Y = C or T)\r\nAsCpf1 from Acidaminococcus or LbCpf1 from Lachnospiraceae: 5'-TTTN-3'\r\nAsCpf1 from Acidaminococcus or LbCpf1 from Lachnospiraceae: 5'-TTTV-3' (V = G or C or A)\r\nSpCas9 from Streptococcus pasteurianus: 5'-NNGTGA-3'\r\nFnCpf1 from Francisella: 5'-TTN-3'\r\nSaCas9 from Staphylococcus aureus: 5'-NNNRRT-'3 (R=A or G)\r\nFnCpf1 from Francisella: 5'-KYTV-3'\r\nVRER SpCas9 from Streptococcus pyogenes: 5'-NGCG-3'\r\nVQR SpCas9 from Streptococcus pyogenes: 5'-NGA-3'\r\nxCas9 3.7 (TLIKDIV SpCas9) from Streptococcus pyogenes: 5'-NGT-3'\r\nBhCas12b from Bacillus hisashii: 5'-TTN-3'\r\nBhCas12b v4 from Bacillus hisashii: 5'-ATTN-3'\r\nBhCas12b v4 from Bacillus hisashii: 5'-DTTN-3'\r\nSpy-macCas9 from Streptococcus pyogenes and Streptococcus macacae: 5'-NAAN-3'",
        PAM: "NGG\r\nNRG\r\nNNAGAAW\r\nNNNNGMTT\r\nNNGRRT\r\nNNNVRYAC\r\nNNNNRYAC\r\nTTTN\r\nTTTV\r\nNNGTGA\r\nTTN\r\nNNNRRT\r\nKYTV\r\nNGCG\r\nNGA\r\nNGT\r\nNG",
        Organisms:
          "Anas platyrhynchos\r\nBos taurus\r\nCallithrix jacchus\r\nCanis familiaris\r\nCapra hircus\r\nCavia porcellus\r\nChlorocebus sabeus\r\nCricetulus griseus\r\nCyprinus carpio\r\nDanio rerio\r\nEquus caballus\r\nFelis catus\r\nGallus gallus\r\nHomo sapiens\r\nMacaca fascicularis\r\nMacaca mulatta\r\nMesocricetus auratus\r\nMochrtus ochrogaster\r\nMus musculus\r\nOncorhynchus tshawytscha\r\nOreochromis niloticus\r\nOryctolagus cuniculus\r\novis_aries\r\nPeromyscus maniculatus\r\nPimephales promelas\r\nRattus norvegicus\r\nSalmo salar\r\nSus scrofa\r\nXenopus laevis\r\nXenopus tropicalis",
        Reference:
          "Park J, Bae S, Kim JS Cas-Designer: a web-based tool for choice of CRISPR-Cas9 target sites. Bioinformatics. 2015 Dec 15;31(24):4014-6. doi: 10.1093/bioinformatics/btv537. Epub 2015 Sep 10. (PMID: 26358729)",
        Comments: "It forms part of the CRISPR RGEN Tools",
        __EMPTY: "http://www.rgenome.net/cas-designer/",
      },
      {
        Availability: " ",
        Platform: "Desktop",
      },
      {
        Tools: "CRISPick",
        Availability: " Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": " Yes",
        "Score Oligos": " Yes",
        "Search by": "GeneID",
        Enzyme:
          "SpyoCas9 (NGG)\r\nSaurCas9(NNGRR)\r\nAsCas12a(TTTV)\r\nenAsCas12a",
        PAM: "NGG\r\nNNGRR\r\nTTTV",
        Organisms: "Homo sapiens\r\nMus musculus\r\nRattus norvegicus",
        Reference:
          "Doench JG, Fusi N, Sullender M, Hegde M, Vaimberg EW, Donovan KF, Smith I, Tothova Z, Wilen C, Orchard R, Virgin HW, Listgarten J, Root DE Optimized sgRNA design to maximize activity and minimize off-target effects of CRISPR-Cas9. Nat Biotechnol. 2016 Feb;34(2):184-191. doi: 10.1038/nbt.3437. Epub 2016 Jan 18. (PMID: 29251726)",
        Comments:
          "Off-targets not shown. The scoring programs they use (Azimuth, Elevation) can be downloaded from their server.",
        __EMPTY: "https://portals.broadinstitute.org/gppx/crispick/public",
      },
      {
        Availability: " ",
        Purpose: "Other",
        "Search by": "Sequence",
        Reference:
          "Najm FJ, Strand C, Donovan KF, Hegde M, Sanson KR, Vaimberg EW, Sullender ME, Hartenian E, Kalani Z, Fusi N, Listgarten J, Younger ST, Bernstein BE, Root DE, Doench JG Orthologous CRISPR-Cas9 enzymes for combinatorial genetic screens. Nat Biotechnol. 2018 Feb;36(2):179-189. doi: 10.1038/nbt.4048. Epub 2017 Dec 18. (PMID: 26780180)",
      },
      {
        Tools: "CRISPR-DO",
        Availability: "No",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": " Yes",
        "Score Oligos": " Yes",
        "Search by": "GeneID",
        Enzyme: "SpCas9 (S. pyogenes)",
        PAM: "NGG",
        Organisms:
          "Homo sapiens\r\nMus musculus\r\nDrosophila melanogaster\r\nCaenorhabditis elegans\r\nDanio rerio",
        Reference:
          "Ma J, Koster J, Qin Q, Hu S, Li W, Chen C, Cao Q, Wang J, Mei S, Liu Q, Xu H, Liu XS CRISPR-DO for genome-wide CRISPR design and optimization. Bioinformatics. 2016 Nov 1;32(21):3336-3338. doi: 10.1093/bioinformatics/btw476. Epub 2016 Jul 10. (PMID: 27402906)",
        __EMPTY: "http://cistrome.org/crispr/",
      },
      {
        Availability: " ",
        Purpose: "Other",
        Platform: "Command-line",
        "Search by": "Sequence",
      },
      {
        Tools: "SSC",
        Availability: " Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": "No",
        "Score Oligos": " Yes",
        "Search by": "Sequence",
        Enzyme: "SpCas9 (Streptococcus pyogenes)",
        PAM: "NGG",
        Organisms: "Homo sapiens\r\nMus musculus",
        Reference:
          "Xu H, Xiao T, Chen CH, Li W, Meyer CA, Wu Q, Wu D, Cong L, Zhang F, Liu JS, Brown M, Liu XS Sequence determinants of improved CRISPR sgRNA design. Genome Res. 2015 Aug;25(8):1147-57. doi: 10.1101/gr.191452.115. Epub 2015 Jun 10. (PMID: 26063738)",
        Comments:
          "It only analyzes the target input sequence using spacer matrices optimized for human and mouse, not genomes.",
        __EMPTY: "http://crispr.dfci.harvard.edu/SSC/",
      },
      {
        Availability: " ",
        Platform: "Command-line",
      },
      {
        Tools: "CRISPRscan",
        Availability: " Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": " Yes",
        "Score Oligos": " Yes",
        "Search by": "GeneID",
        Enzyme:
          "SpCas9 (Streptococcus pyogenes)\r\nAsCpf1 (Acidaminococcus)\r\nLbCpf1 (Lachnospiraceae)",
        PAM: "NGG\r\nTTTV\r\nTTTN",
        Organisms:
          "Homo sapiens\r\nMus musculus\r\nDrosophila melanogaster\r\nCaenorhabditis elegans\r\nDanio rerio\r\nRattus norvegicus\r\nSaccahromyces cerevisiae\r\nXenopus trop챙calis\r\nGallus gallus\r\nPan troglodytes\r\nCiona intestinalis\r\nBos taurus\r\nXenopus laevis\r\nXenopus tropicalis\r\nNothobranchius furzeri\r\nCallithrix jacchus\r\nOryzias latipes\r\nNemanosctella vectensis\r\nStrongylocentrotus purpuratus\r\nMnemiopsis leidyi",
        Reference:
          "Moreno-Mateos MA, Vejnar CE, Beaudoin JD, Fernandez JP, Mis EK, Khokha MK, Giraldez AJ CRISPRscan: designing highly efficient sgRNAs for CRISPR-Cas9 targeting in vivo. Nat Methods. 2015 Oct;12(10):982-8. doi: 10.1038/nmeth.3543. Epub 2015 Aug 31. (PMID: 26322839)",
        Comments:
          "gRNA designer for in-vitro CRISPR/Cas9 using T7/sp6 promoter. Returns gRNAs restricted to 5' GG or GA sequences, as required by T7 and Sp6 promoters. Scoring algorithm is based on in-vitro T7-based data only. Use for U6-based in vivo CRISPR/Cas9 limited.",
        __EMPTY: "https://www.crisprscan.org/",
      },
      {
        Availability: " ",
        Purpose: "Other",
        "Search by": "Sequence",
      },
      {
        Tools: "WU-CRISPR",
        Availability: " Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": "No",
        "Score Oligos": " Yes",
        "Search by": "GeneID",
        Enzyme: "SpCas9 (Streptococcus pyogenes)",
        PAM: "NGG",
        Organisms: "Homo sapiens\r\nMus musculus",
        Reference:
          "Wong N, Liu W, Wang X WU-CRISPR: characteristics of functional guide RNAs for the CRISPR/Cas9 system. Genome Biol. 2015 Nov 2;16:218. doi: 10.1186/s13059-015-0784-0. (PMID: 26521937)",
        Comments: "Standalone gRNA design tool is available",
        __EMPTY: "https://crisprdb.org/wu-crispr/",
      },
      {
        Availability: " ",
        Purpose: "Other",
        Platform: "Command-line",
        "Search by": "Sequence",
      },
      {
        Tools: "CRISPy CHO",
        Availability: " Yes",
        Purpose: "Database",
        Platform: "Web",
        "Off‑targets": " Yes",
        "Score Oligos": " Yes",
        "Search by": "GeneID",
        Enzyme: "SpCas9 (S. pyogenes)",
        PAM: "NGG",
        Organisms: "Cricetulus griseus",
        Reference:
          "Ronda C, Pedersen LE, Hansen HG, Kallehauge TB, Betenbaugh MJ, Nielsen AT, Kildegaard HF Accelerating genome editing in CHO cells using CRISPR Cas9 and CRISPy, a web-based target finding tool. Biotechnol Bioeng. 2014 Aug;111(8):1604-16. doi: 10.1002/bit.25233. Epub 2014 May 22. (PMID: 24827782)",
        Comments: "Only for Chinese Hamsters Ovary (CHO) cells",
        __EMPTY: "http://staff.biosustain.dtu.dk/laeb/crispy/",
      },
      {
        Availability: " ",
      },
      {
        Tools: "Yeastriction",
        Availability: " Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": " Yes",
        "Score Oligos": " Yes",
        "Search by": "GeneID",
        Enzyme: "SpCas9 (S. pyogenes)",
        PAM: "NGG",
        Organisms:
          "Saccharomyces cerevisiae AWRI1631\r\nSaccharomyces cerevisiae AWRI796\r\nSaccharomyces cerevisiae BY4741\r\nSaccharomyces cerevisiae BY4742\r\nSaccharomyces cerevisiae CBS7960\r\nSaccharomyces cerevisiae CEN.PK113-7D\r\nSaccharomyces cerevisiae CLIB215\r\nSaccharomyces cerevisiae CLIB324\r\nSaccharomyces cerevisiae CLIB382\r\nSaccharomyces cerevisiae EC1118\r\nSaccharomyces cerevisiae EC9-8\r\nSaccharomyces cerevisiae FL100\r\nSaccharomyces cerevisiae FostersB\r\nSaccharomyces cerevisiae FostersO\r\nSaccharomyces cerevisiae JAY291\r\nSaccharomyces cerevisiae Kyokai7\r\nSaccharomyces cerevisiae LalvinQA23\r\nSaccharomyces cerevisiae M22\r\nSaccharomyces cerevisiae PW5\r\nSaccharomyces cerevisiae RM11-1a\r\nSaccharomyces cerevisiae S288C\r\nSaccharomyces cerevisiae Sigma1278b\r\nSaccharomyces cerevisiae T7\r\nSaccharomyces cerevisiae T73\r\nSaccharomyces cerevisiae UC5\r\nSaccharomyces cerevisiae Vin13\r\nSaccharomyces cerevisiae VL3\r\nSaccharomyces cerevisiae ZTW1\r\nSaccharomyces cerevisiae W303\r\nSaccharomyces cerevisiae Y10\r\nSaccharomyces cerevisiae YJM269\r\nSaccharomyces cerevisiae YJM789\r\nSaccharomyces cerevisiae YPS163",
        Reference:
          "Mans R, van Rossum HM, Wijsman M, Backx A, Kuijpers NG, van den Broek M, Daran-Lapujade P, Pronk JT, van Maris AJ, Daran JM CRISPR/Cas9: a molecular Swiss army knife for simultaneous introduction of multiple genetic modifications in Saccharomyces cerevisiae. FEMS Yeast Res. 2015 Mar;15(2). pii: fov004. doi: 10.1093/femsyr/fov004. Epub 2015 Mar 4. (PMID: 25743786)",
        Comments:
          "For yeast only. Provides information of the RNA secondary structure. Source code available to be installed in your own web server.",
        __EMPTY: "http://yeastriction.tnw.tudelft.nl/#!/",
      },
      {
        Availability: " ",
        Platform: "Other",
      },
      {
        Tools: "GenomeCRISPR",
        Availability: " Yes",
        Purpose: "Database",
        Platform: "Web",
        "Off‑targets": "?",
        "Score Oligos": "?",
        "Search by": "GeneID",
        Enzyme: "SpCas9 (S. pyogenes)",
        PAM: "NGG",
        Organisms: "Homo sapiens",
        Reference:
          "Rauscher B, Heigwer F, Breinig M, Winter J, Boutros M GenomeCRISPR - a database for high-throughput CRISPR/Cas9 screens. Nucleic Acids Res. 2017 Jan 4;45(D1):D679-D686. doi: 10.1093/nar/gkw997. Epub 2016 Oct 26. (PMID: 27789686)",
        Comments:
          "Database for genome-scale CRISPR/Cas9 screens in human. Provides the experimentally measured sgRNA effect in the context each screen.",
        __EMPTY: "http://genomecrispr.dkfz.de/#!/",
      },
      {
        Availability: " ",
        Platform: "Webapp",
        "Search by": "Coordinate",
      },
      {
        Tools: "CRISPR ML",
        Availability: " Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": " Yes",
        "Score Oligos": " Yes",
        "Search by": "GeneID",
        Enzyme: "SpCas9 (S. pyogenes)",
        PAM: "NGG",
        Organisms: "Homo sapiens",
        Reference:
          "Listgarten J, Weinstein M, Kleinstiver BP, Sousa AA, Joung JK, Crawford J, Gao K, Hoang L, Elibol M, Doench JG, Fusi N Prediction of off-target activities for the end-to-end design of CRISPR guide RNAs. Nat Biomed Eng. 2018 Jan;2(1):38-47. doi: 10.1038/s41551-017-0178-6. Epub 2018 Jan 10. (PMID: 29998038)",
        Comments:
          "Off-target detection for GeneID search only. Source code of the algorithms available at https://github.com/michael-weinstein/dsNickFury3PlusOrchid and https://github.com/microsoftResearch//azimuth",
        __EMPTY: "https://crispr.ml/",
      },
      {
        Availability: " ",
        "Search by": "Sequence",
      },
      {
        Tools: "CRISPETa",
        Availability: " Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": " Yes",
        "Score Oligos": " Yes",
        "Search by": "Coordinate",
        Enzyme: "SpCas9 (S. pyogenes)",
        PAM: "NGG",
        Organisms:
          "Human (hg19)\r\nMouse (mm10)\r\nZebrafish (danRer10)\r\nDrosophila (dm6)\r\nC. elegans (ce11)",
        Reference:
          "Pulido-Quetglas C, Aparicio-Prat E, Arnan C, Polidori T, Hermoso T, Palumbo E, Ponomarenko J, Guigo R, Johnson R Scalable Design of Paired CRISPR Guide RNAs for Genomic Deletion. PLoS Comput Biol. 2017 Mar 2;13(3):e1005341. doi: 10.1371/journal.pcbi.1005341. eCollection 2017 Mar. (PMID: 28253259)",
        Comments:
          "Intended for CRISPR deletion projects with sgRNA pairs. Allows specifying favored and/or disfavored genome regions. Pre-computed knockout libraries for a variety of genomic element classes and intergenic regions available for download.",
        __EMPTY: "http://crispeta.crg.eu/",
      },
      {
        Availability: " ",
        Platform: "Command-line",
        "Search by": "Sequence",
      },
      {
        Tools: "IDT Design custom gRNA",
        Availability: " Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": " Yes",
        "Score Oligos": " Yes",
        "Search by": "GeneID",
        Enzyme: "SpCas9 (S. pyogenes)",
        PAM: "NGG",
        Organisms:
          "Homo sapiens\r\nMus musculus\r\nRattus norvegicus\r\nDanio rerio\r\nCaenorhabditis elegans",
        Comments:
          "Requires registration. Commercial kits with the designed sgRNAs can be purchased in the web site. Allows other species without off-target detection.",
        __EMPTY:
          "https://sg.idtdna.com/site/order/designtool/index/CRISPR_CUSTOM",
      },
      {
        Availability: " ",
        "Search by": "Sequence",
      },
      {
        Tools: "CRISPR LifePipe",
        Availability: "No",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": " Yes",
        "Score Oligos": " Yes",
        "Search by": "GeneID",
        Enzyme:
          "Cas9sp, eCas9sp, Cas9sp-HF1, 20bp [NGG]\r\nCas9sa, 21bp [NNG(A/G)(A/G)T]\r\nCas9sp VQR, 20bp [NGA]\r\nCas9sp VRER, 20bp [NGCG]\r\nCas9sp nickase, 20bp [NGG]",
        PAM: "NGG\r\nNNGRRT\r\nNAG\r\nNGCG\r\nNGG\r\nTTTN",
        Organisms:
          "Human (Homo sapiens GRCh38 p11)\r\nRat (Rattus norvegicus Rnor_6.0)\r\nMouse (Mus musculus GRCm38.p6)\r\nZebrafish (Danio rerio  GRCz11)\r\nFruit fly (Drosophila melanogaster 6 plus ISO1 MT)",
        Comments:
          "Commercial (free version limits submissions per month). Allows design for targeting an specific amino acid.",
        __EMPTY: "https://crispr.lifeandsoft.com/",
      },
      {
        Availability: " ",
        Purpose: "Other",
        "Search by": "Sequence",
      },
      {
        "Search by": "Other",
      },
      {
        Tools: "GUIDES",
        Availability: " Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": " Yes",
        "Score Oligos": " Yes",
        "Search by": "GeneID",
        Enzyme: "SpCas9 (S. pyogenes)",
        PAM: "NGG",
        Organisms: "Human (Homo sapiens GRCh37)\r\nMouse (Mus musculus GRCm38)",
        Reference:
          "Meier JA, Zhang F, Sanjana NE GUIDES: sgRNA design for loss-of-function screens. Nat Methods. 2017 Aug 31;14(9):831-832. doi: 10.1038/nmeth.4423. (PMID: 28858339)",
        Comments:
          "Retrieves external information for targeting specific tissues, highly expressed exons or known functional domains. Can include non-targeting sgRNAs (controls) and synthesis ready constructs in the results. Source code available to install the web server locally.",
        __EMPTY: "http://guides.sanjanalab.org/#/",
      },
      {
        Availability: " ",
        Platform: "Other",
      },
      {
        Tools: "CRISPR-FOCUS",
        Availability: " Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": " Yes",
        "Score Oligos": " Yes",
        "Search by": "GeneID",
        Enzyme: "SpCas9 (S. pyogenes)",
        PAM: "NGG",
        Organisms: "Homo sapiens\r\nMus musculus",
        Reference:
          "Cao Q, Ma J, Chen CH, Xu H, Chen Z, Li W, Liu XS CRISPR-FOCUS: A web server for designing focused CRISPR screening experiments. PLoS One. 2017 Sep 5;12(9):e0184281. doi: 10.1371/journal.pone.0184281. eCollection 2017. (PMID: 28873439)",
        __EMPTY: "http://cistrome.org/crispr-focus/",
      },
      {
        Availability: " ",
      },
      {
        Tools: "DeepCRISPR",
        Availability: "No",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": " Yes",
        "Score Oligos": " Yes",
        "Search by": "Sequence",
        Enzyme: "SpCas9 (S. pyogenes)",
        PAM: "NGG",
        Organisms: "Homo sapiens",
        Reference:
          "Chuai G, Ma H, Yan J, Chen M, Hong N, Xue D, Zhou C, Zhu C, Chen K, Duan B, Gu F, Qu S, Huang D, Wei J, Liu Q DeepCRISPR: optimized CRISPR guide RNA design by deep learning. Genome Biol. 2018 Jun 26;19(1):80. doi: 10.1186/s13059-018-1459-4. (PMID: 29945655)",
        __EMPTY: "https://www.deepcrispr.net/",
      },
      {
        Availability: " ",
      },
      {
        Tools: "DRSC Find CRISPRs",
        Availability: " Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": " Yes",
        "Score Oligos": " Yes",
        "Search by": "GeneID",
        Enzyme: "SpCas9 (S. pyogenes)",
        PAM: "NGG",
        Organisms: "Drosophila melanogaster",
        Reference:
          "Ren X, Sun J, Housden BE, Hu Y, Roesel C, Lin S, Liu LP, Yang Z, Mao D, Sun L, Wu Q, Ji JY, Xi J, Mohr SE, Xu J, Perrimon N, Ni JQ Optimized gene editing technology for Drosophila melanogaster using germ line-specific Cas9. Proc Natl Acad Sci U S A. 2013 Nov 19;110(47):19012-7. doi: 10.1073/pnas.1318481110. Epub 2013 Nov 4. (PMID: 24191015)",
        __EMPTY: "https://www.flyrnai.org/crispr/",
      },
      {
        Availability: " ",
        "Search by": "Coordinate",
        Reference:
          "Housden BE, Valvezan AJ, Kelley C, Sopko R, Hu Y, Roesel C, Lin S, Buckner M, Tao R, Yilmazel B, Mohr SE, Manning BD, Perrimon N Identification of potential drug targets for tuberous sclerosis complex by synthetic screens combining CRISPR-based knockouts with RNAi. Sci Signal. 2015 Sep 8;8(393):rs9. doi: 10.1126/scisignal.aab3729. (PMID: 26350902)",
      },
      {
        Tools: "CUNE",
        Availability: " Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": "No",
        "Score Oligos": " Yes",
        "Search by": "Coordinate",
        Enzyme: "Custom",
        PAM: "Custom",
        Organisms: "Mus musculus",
        Reference:
          "O'Brien AR, Wilson LOW, Burgio G, Bauer DC Unlocking HDR-mediated nucleotide editing by identifying high-efficiency target sites using machine learning. Sci Rep. 2019 Feb 26;9(1):2788. doi: 10.1038/s41598-019-39142-0. (PMID: 30808944)",
        Comments: "Api available, base editor, nucleotide editor, HDR",
        __EMPTY: "https://gt-scan.csiro.au/cune/Mus_musculus",
      },
      {
        Availability: " ",
        Purpose: "Other",
      },
      {
        Tools: "CrisprZ",
        Availability: " Yes",
        Purpose: "Database",
        Platform: "Web",
        "Off‑targets": "No",
        "Score Oligos": " Yes",
        "Search by": "GeneID",
        Enzyme: "not applicable",
        PAM: "Not Applicable",
        Organisms: "Danio rerio",
        Reference:
          "Varshney GK, Zhang S, Pei W, Adomako-Ankomah A, Fohtung J, Schaffer K, Carrington B, Maskeri A, Slevin C, Wolfsberg T, Ledin J, Sood R, Burgess SM CRISPRz: a database of zebrafish validated sgRNAs. Nucleic Acids Res. 2016 Jan 4;44(D1):D822-6. doi: 10.1093/nar/gkv998. Epub 2015 Oct 4. (PMID: 26438539)",
        Comments:
          "It provides links to the literature related to each CRISPR sequence. It allows users to add new sequences (moderated).",
        __EMPTY: "https://research.nhgri.nih.gov/CRISPRz/",
      },
      {
        Availability: " ",
        "Search by": "Other",
      },
      {
        Tools: "Toronto Knockout Library",
        Availability: " Yes",
        Purpose: "Database",
        Platform: "Web",
        "Off‑targets": "No",
        "Score Oligos": " Yes",
        "Search by": "GeneID",
        Enzyme: "not applicable",
        PAM: "Not Applicable",
        Organisms: "Homo sapiens",
        Reference:
          "Hart T, Tong AHY, Chan K, Van Leeuwen J, Seetharaman A, Aregger M, Chandrashekhar M, Hustedt N, Seth S, Noonan A, Habsid A, Sizova O, Nedyalkova L, Climie R, Tworzyanski L, Lawson K, Sartori MA, Alibeh S, Tieu D, Masud S, Mero P, Weiss A, Brown KR, Usaj M, Billmann M, Rahman M, Constanzo M, Myers CL, Andrews BJ, Boone C, Durocher D, Moffat J Evaluation and Design of Genome-Wide CRISPR/SpCas9 Knockout Screens. G3 (Bethesda). 2017 Aug 7;7(8):2719-2727. doi: 10.1534/g3.117.041277. (PMID: 28655737)",
        Comments:
          "Current version (TKOv3) is accessible through a genomic viewer (https://crispr.ccbr.utoronto.ca/public/library/TKOv3/) . A file containing a previous version (TKOv1) of the database can be downloaded.",
        __EMPTY: "http://tko.ccbr.utoronto.ca/",
      },
      {
        Availability: " ",
      },
      {
        Tools: "CRISPY-yeast",
        Availability: " Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": " Yes",
        "Score Oligos": "No",
        "Search by": "GeneID",
        Enzyme: "SpCas9 (S. pyogenes)",
        PAM: "NGG",
        Organisms: "Saccharomyces cerevisiae S288C",
        Reference:
          "Jakociunas T, Bonde I, Herrgard M, Harrison SJ, Kristensen M, Pedersen LE, Jensen MK, Keasling JD Multiplex metabolic pathway engineering using CRISPR/Cas9 in Saccharomyces cerevisiae. Metab Eng. 2015 Mar;28:213-222. doi: 10.1016/j.ymben.2015.01.008. Epub 2015 Jan 28. (PMID: 25638686)",
        __EMPTY: "http://staff.biosustain.dtu.dk/laeb/crispy_yeast/",
      },
      {
        Availability: " ",
      },
      {
        Tools: "GUIDE Picker",
        Availability: " Yes",
        Purpose: "Database",
        Platform: "Web",
        "Off‑targets": "No",
        "Score Oligos": " Yes",
        "Search by": "GeneID",
        Enzyme: "SpCas9",
        PAM: "NRG",
        Organisms: "Homo sapiens, Mus musculus",
        Reference:
          "Hough SH, Kancleris K, Brody L, Humphryes-Kirilov N, Wolanski J, Dunaway K, Ajetunmobi A, Dillard V Guide Picker is a comprehensive design tool for visualizing and selecting guides for CRISPR experiments. BMC Bioinformatics. 2017 Mar 14;18(1):167. doi: 10.1186/s12859-017-1581-4. (PMID: 28288556)",
      },
      {
        Availability: " ",
      },
      {
        Tools: "CRISPR Surf",
        Availability: " Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": "No",
        "Score Oligos": "No",
        "Search by": "Coordinate",
        Enzyme: "Custom",
        PAM: "Custom",
        Organisms: "Homo Sapiens",
        Reference:
          "Hsu JY, Fulco CP, Cole MA, Canver MC, Pellin D, Sher F, Farouni R, Clement K, Guo JA, Biasco L, Orkin SH, Engreitz JM, Lander ES, Joung JK, Bauer DE, Pinello L CRISPR-SURF: discovering regulatory elements by deconvolution of CRISPR tiling screen data. Nat Methods. 2018 Dec;15(12):992-993. doi: 10.1038/s41592-018-0225-6. (PMID: 30504875)",
        Comments:
          "CRISPR-SURF can be run as a stand-alone\r\ncommand line utility (https://github.com/\r\npinellolab/CRISPR-SURF) or as a web\r\napplication (http://crisprsurf.pinellolab.org/)",
      },
      {
        Availability: " ",
        Purpose: "Post-analysis",
        Platform: "Command-line",
      },
      {
        Tools:
          "GeneArt CRISPR Search and Design Tool (from Thermo Fisher Scientific)",
        Availability: " Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": " Yes",
        "Score Oligos": " Yes",
        "Search by": "GeneID",
        Enzyme: "SpCas9",
        PAM: "NGG",
        Organisms: "Human, mouse, rat, zebrafish, roundworm",
        Comments:
          "Design CRISPRs de novo or search CRISPR targets from pre-designed database of > 600,000 gRNAs. Output include PAM sites, potential binding sites, exon maps with gRNA locations, and recommendations based on potential off-target effects for each CRISPR sequence. The tool will design the correct gRNA format for your preferred CRISPR-Cas9 product format. ",
      },
      {
        Availability: " ",
        Purpose: "Database",
        "Search by": "Sequence",
      },
      {
        Tools: "AsCRISPR",
        Availability: " Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": " Yes",
        "Score Oligos": " Yes",
        "Search by": "GeneID",
        Enzyme:
          "SpCas9\r\nSpCas9-VRER\r\nSpCas9-EQR\r\nSpCas9-V(R)QR\r\nSaCas9-KKH\r\nNmCas9\r\nStCas9\r\nTdCas9\r\nCjCas9\r\nAs(Lb)Cpf1\r\nFnCpf1\r\nAsCpf1-RR\r\nAsCpf1-RVR\r\nAaCas12b\r\nAkCas12b\r\nSpCas9\r\nSpCas9-VRER\r\nSpCas9-EQR\r\nSpCas9-V(R)QR\r\nSaCas9-KKH\r\nNmCas9\r\nStCas9\r\nTdCas9\r\nCjCas9\r\nAs(Lb)Cpf1\r\nFnCpf1\r\nAsCpf1-RR\r\nAsCpf1-RVR\r\nAaCas12b\r\nAkCas12b\r\nBhCas12b\r\nDpb(Plm)CasX",
        PAM: "NGG\r\nNGCG\r\nNGAG\r\nNGA\r\nNNGRRT (R = A or G)\r\nNNNRRT (R = A or G)\r\nNNNNGATT\r\nNNAGAAW (W = A or T)\r\nNAAAAC\r\nNNNNRYAC (R = A or G; Y = C or T )\r\nTTTV (V = A or C or G)\r\nTTV (V = A or C or G)\r\nKYTV (K = G or T; Y = C or T; V = A or C or G)\r\nTYCV (Y = C or T; V = A or C or G)\r\nTATV (V = A or C or G)\r\nTTN\r\nTTTN\r\nATTN\r\nTTCN",
        Organisms:
          "Homo sapiens (hg19/GRCh37)\r\nHomo sapiens (hg38/GRCh38)\r\nMus musculus (mm10/GRCm38)",
        Reference: "https://www.biorxiv.org/content/10.1101/672634v2",
      },
      {
        Availability: " ",
        "Search by": "Sequence",
      },
      {
        "Search by": "Other",
      },
      {
        Tools: "CRISP-view",
        Availability: " Yes",
        Purpose: "Database",
        Platform: "Web",
        "Off‑targets": "No",
        "Score Oligos": " Yes",
        "Search by": "GeneID",
        Enzyme: "Cas9",
        PAM: "Cas9",
        Organisms: "Human, mouse, monkey",
        Reference:
          "Cui Y, Cheng X, Chen Q, Song B, Chiu A, Gao Y, Dawson T, Chao L, Zhang W, Li D, Zeng Z, Yu J, Li Z, Fei T, Peng S, Li W CRISP-view: a database of functional genetic screens spanning multiple phenotypes. Nucleic Acids Res. 2021 Jan 8;49(D1):D848-D854. doi: 10.1093/nar/gkaa809. (PMID: 33010154)",
        Comments:
          "A comprehensive database of CRISPR/Cas9 and RNAi screening datasets that span multiple phenotypes, including in vitro and in vivo cell proliferation and viability, response to cancer immunotherapy, virus response, protein expression, etc.",
        __EMPTY: "http://crispview.weililab.org/",
      },
      {
        Availability: " ",
      },
      {
        Tools: "MHcut",
        Availability: " Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": "No",
        "Score Oligos": " Yes",
        "Search by": "GeneID",
        Enzyme: "Custom",
        PAM: "Custom",
        Organisms: "Homo sapiens",
        Reference:
          "Grajcarek J, Monlong J, Nishinaka-Arai Y, Nakamura M, Nagai M, Matsuo S, Lougheed D, Sakurai H, Saito MK, Bourque G, Woltjen K Genome-wide microhomologies enable precise template-free editing of biologically relevant deletion mutations. Nat Commun. 2019 Oct 24;10(1):4856. doi: 10.1038/s41467-019-12829-8. (PMID: 31649251)",
        Comments:
          "MHcut Browser contains deletion data from dbSNP and ClinVar, and may be searched by gene, location, or rsID. Deletions are scored by inDelphi. Export unique gRNAs to re-create deletion variants. Python script and Docker image available at: https://github.com/WoltjenLab/MHcut",
      },
      {
        Availability: " ",
        Purpose: "Database",
        Platform: "Command-line",
        "Search by": "Coordinate",
      },
      {
        "Search by": "Other",
      },
    ],
    "Post Anlaysis": [
      {
        Tools: "CRISPR Target",
        Availability: " Yes",
        Purpose: "Other",
        Platform: "Web",
        "Off‑targets": " No",
        "Score Oligos": " No",
        "Search by": "Sequence",
        Enzyme: "Not applicable",
        PAM: "Not Applicable",
        Organisms: "Not Applicable",
        Reference:
          "Biswas A, Gagnon JN, Brouns SJ, Fineran PC, Brown CM CRISPRTarget: bioinformatic prediction and analysis of crRNA targets. RNA Biol. 2013 May;10(5):817-27. doi: 10.4161/rna.24046. Epub 2013 Mar 14. (PMID: 23492433)",
        Comments:
          "Finds CRISPR arrays and detect Cas proteins. Tool to detect targets of CRISPR non coding RNAs. ",
        __EMPTY:
          "http://bioanalysis.otago.ac.nz/CRISPRTarget/crispr_analysis.html",
      },
      {
        Availability: " ",
      },
      {
        Tools: "CRISPRfinder",
        Availability: " Yes",
        Purpose: "Other",
        Platform: "Web",
        "Off‑targets": " No",
        "Score Oligos": " No",
        "Search by": "Sequence",
        Enzyme: "Not applicable",
        PAM: "Not Applicable",
        Organisms: "Provided by user",
        Reference:
          "Grissa I, Vergnaud G, Pourcel C CRISPRFinder: a web tool to identify clustered regularly interspaced short palindromic repeats. Nucleic Acids Res. 2007 Jul;35(Web Server issue):W52-7. doi: 10.1093/nar/gkm360. Epub 2007 May 30. (PMID: 17537822)",
        Comments: "Finds CRISPR arrays and detect Cas proteins",
        __EMPTY: "http:// crispr.u-psud.fr/Server/CRISPRfinder.php",
      },
      {
        Availability: " ",
      },
      {
        Tools: "CRISPRCasFinder",
        Availability: " Yes",
        Purpose: "Other",
        Platform: "Web",
        "Off‑targets": " No",
        "Score Oligos": " No",
        "Search by": "Sequence",
        Enzyme: "Not applicable",
        PAM: "Not Applicable",
        Organisms: "Provided by user",
        Reference:
          "Couvin D, Bernheim A, Toffano-Nioche C, Touchon M, Michalik J, Neron B, Rocha EPC, Vergnaud G, Gautheret D, Pourcel C CRISPRCasFinder, an update of CRISRFinder, includes a portable version, enhanced performance and integrates search for Cas proteins. Nucleic Acids Res. 2018 Jul 2;46(W1):W246-W251. doi: 10.1093/nar/gky425. (PMID: 29790974)",
        Comments:
          "An update of CRISPRFinder. Tools to detect CRISPR ad CAS genes in user-submitted sequences.",
        __EMPTY: "https://crisprcas.i2bc.paris-saclay.fr/",
      },
      {
        Availability: " ",
        Platform: "Command-line",
      },
      {
        Tools: "CRISPR-GA",
        Availability: " Yes",
        Purpose: "Post-analysis",
        Platform: "Web",
        "Off‑targets": " No",
        "Score Oligos": " No",
        "Search by": "Other",
        Enzyme: "Not applicable",
        PAM: "Not Applicable",
        Organisms: "Not Applicable",
        Reference:
          "Guell M, Yang L, Church GM Genome editing assessment using CRISPR Genome Analyzer (CRISPR-GA). Bioinformatics. 2014 Oct 15;30(20):2968-70. doi: 10.1093/bioinformatics/btu427. Epub 2014 Jul 1. (PMID: 24990609)",
        Comments:
          "Assess the quality of gene editing using next gen sequencing data",
        __EMPTY: "crispr-ga.net",
      },
      {
        Availability: " ",
      },
      {
        Tools: "Microhomology Predictor",
        Availability: " Yes",
        Purpose: "Other",
        Platform: "Web",
        "Off‑targets": " Yes",
        "Score Oligos": " Yes",
        "Search by": "Sequence",
        Enzyme: "Not applicable",
        PAM: "Not Applicable",
        Organisms: "Not Applicable",
        Reference:
          "Bae S, Kweon J, Kim HS, Kim JS Microhomology-based choice of Cas9 nuclease target sites. Nat Methods. 2014 Jul;11(7):705-6. doi: 10.1038/nmeth.3015. (PMID: 24972169)",
        Comments:
          "For microhomology-based choice of Cas9 nuclease target sites. It predicts mutation patterns caused by MMEJ in an input sequence and estimate frequency of in frame deletions. It forms part of the RGEN suite of tools.",
        __EMPTY: "http://www.rgenome.net/mich-calculator/",
      },
      {
        Availability: " ",
      },
      {
        Tools: "caRpools",
        Availability: " Yes",
        Purpose: "Post-analysis",
        Platform: "Command-line",
        "Off‑targets": " No",
        "Score Oligos": " No",
        "Search by": "Other",
        Enzyme: "Not applicable",
        PAM: "Not Applicable",
        Organisms: "Not Applicable",
        Reference:
          "Winter J, Breinig M, Heigwer F, Brugemann D, Leible S, Pelz O, Zhan T, Boutros M caRpools: an R package for exploratory data analysis and documentation of pooled CRISPR/Cas9 screens. Bioinformatics. 2016 Feb 15;32(4):632-4. doi: 10.1093/bioinformatics/btv617. Epub 2015 Oct 27. (PMID: 26508755)",
        Comments:
          "For exploratory data analysis of pooled CRISPR/Cas9 screens (screening quality, candidate hit calling). Author recommends using an update tool, CRISPRAnalyzeR, instead of this one.",
        __EMPTY: "https://github.com/boutroslab/caRpools",
      },
      {
        Availability: " ",
      },
      {
        Tools: "MAGeCK",
        Availability: " Yes",
        Purpose: "Post-analysis",
        Platform: "Command-line",
        "Off‑targets": " No",
        "Score Oligos": " No",
        "Search by": "Other",
        Enzyme: "Not applicable",
        PAM: "Not Applicable",
        Organisms: "Provided by user",
        Reference:
          "Li W, Xu H, Xiao T, Cong L, Love MI, Zhang F, Irizarry RA, Liu JS, Brown M, Liu XS MAGeCK enables robust identification of essential genes from genome-scale CRISPR/Cas9 knockout screens. Genome Biol. 2014;15(12):554. doi: 10.1186/s13059-014-0554-4. (PMID: 25476604)",
        Comments:
          "For identifying  positively or negatively selected sgRNAs, genes or pathways from two CRISPR/Cas9 knockout screens in two different conditions.",
        __EMPTY: "http://liulab.dfci.harvard.edu/Mageck",
      },
      {
        Availability: " ",
      },
      {
        Tools: "CRISPResso",
        Availability: " No",
        Purpose: "Post-analysis",
        Platform: "Web",
        "Off‑targets": " No",
        "Score Oligos": " No",
        "Search by": "Other",
        Enzyme: "Custom",
        PAM: "Custom",
        Organisms: "Not Applicable",
        Reference:
          "Pinello L, Canver MC, Hoban MD, Orkin SH, Kohn DB, Bauer DE, Yuan GC Analyzing CRISPR genome-editing experiments with CRISPResso. Nat Biotechnol. 2016 Jul 12;34(7):695-7. doi: 10.1038/nbt.3583. (PMID: 27404874)",
        Comments:
          "(Deprecated) Analyze the output of an CRISPR-Cas9 single or paired ends NGS experiment from their FASTQ files: NEHJ events like insertions, deletions or substitutions and their effects",
        __EMPTY: "http://crispresso.rocks/",
      },
      {
        Availability: " ",
        Platform: "Command-line",
      },
      {
        Tools: "CRISPResso2",
        Availability: " Yes",
        Purpose: "Post-analysis",
        Platform: "Web",
        "Off‑targets": " No",
        "Score Oligos": " No",
        "Search by": "Other",
        Enzyme: "Custom",
        PAM: "Custom",
        Organisms: "Not Applicable",
        Reference:
          "Clement K, Rees H, Canver MC, Gehrke JM, Farouni R, Hsu JY, Cole MA, Liu DR, Joung JK, Bauer DE, Pinello L CRISPResso2 provides accurate and rapid genome editing sequence analysis. Nat Biotechnol. 2019 Feb 26. pii: 10.1038/s41587-019-0032-3. doi: 10.1038/s41587-019-0032-3. (PMID: 30809026)",
        Comments:
          "Analysis and visualization of next-generation sequencing data, including analysis of sequencing data from Cas9 and variants, base editors, and prime editors. Several modes allow for the analysis and comparison of different samples (batch mode) different sites (pooled mode) or different sequencing strategies (e.g. WGS mode). Other features include allele-specific quantification of heterozygous or polymorphic references and a biologically informed alignment algorithm.",
        __EMPTY: "http://crispresso.pinellolab.org/submission",
      },
      {
        Availability: " ",
        Platform: "Command-line",
      },
      {
        Tools: "CrispRVariants",
        Availability: " Yes",
        Purpose: "Post-analysis",
        Platform: "Command-line",
        "Off‑targets": " No",
        "Score Oligos": " No",
        "Search by": "Other",
        Enzyme: "Custom",
        PAM: "Custom",
        Organisms: "Provided by user",
        Reference:
          "Lindsay H, Burger A, Biyong B, Felker A, Hess C, Zaugg J, Chiavacci E, Anders C, Jinek M, Mosimann C, Robinson MD CrispRVariants charts the mutation spectrum of genome engineering experiments. Nat Biotechnol. 2016 Jul 12;34(7):701-2. doi: 10.1038/nbt.3628. (PMID: 27404876)",
        Comments:
          "Extract the variations produced in an experiment of CRISPR-Cas9 NGS sequencing. It also calculates the mutation efficiency, get consensus alleles and plot chimeric alignments.",
        __EMPTY: "https://github.com/markrobinsonuzh/CrispRVariants",
      },
      {
        Availability: " ",
      },
      {
        Tools: "CAGE",
        Availability: " Yes",
        Purpose: "Post-analysis",
        Platform: "Command-line",
        "Off‑targets": " No",
        "Score Oligos": " No",
        "Search by": "Other",
        Enzyme: "Custom",
        PAM: "Custom",
        Organisms: "Provided by user",
        Reference:
          "Chuai G, Yang F, Yan J, Chen Y, Ma Q, Zhou C, Zhu C, Gu F, Liu Q Deciphering relationship between microhomology and in-frame mutation occurrence in human CRISPR-based gene knockout. Mol Ther Nucleic Acids. 2016 Jun 14;5(6):e323. doi: 10.1038/mtna.2016.35. (PMID: 27301063)",
        Comments:
          "Analysis of indels and microhomology patterns and enrichment analyis and evaluation of the sgRNA KO efficiency and identification of features based on the CRISPR-Cas9 KO NGS data.",
        __EMPTY: "https://github.com/bm2-lab/cage",
      },
      {
        Availability: " ",
      },
      {
        Tools: "ScreenBEAM",
        Availability: " Yes",
        Purpose: "Post-analysis",
        Platform: "Command-line",
        "Off‑targets": " No",
        "Score Oligos": " No",
        "Search by": "Other",
        Enzyme: "Custom",
        PAM: "Not Applicable",
        Organisms: "Not Applicable",
        Reference:
          "Yu J, Silva J, Califano A ScreenBEAM: a novel meta-analysis algorithm for functional genomics screens via Bayesian hierarchical modeling. Bioinformatics. 2016 Jan 15;32(2):260-7. doi: 10.1093/bioinformatics/btv556. Epub 2015 Sep 28. (PMID: 26415723)",
        Comments:
          "Analysis of differential efficiency of CRISPR-CAs9 NGS experiments between samples in different conditions. Inputs are count tables.",
        __EMPTY: "https://github.com/jyyu/ScreenBEAM",
      },
      {
        Availability: " ",
        Purpose: "Other",
      },
      {
        Tools: "HitSelect",
        Availability: " Yes",
        Purpose: "Post-analysis",
        Platform: "Desktop",
        "Off‑targets": " No",
        "Score Oligos": " No",
        "Search by": "Other",
        Enzyme: "Custom",
        PAM: "Not Applicable",
        Organisms: "Not Applicable",
        Reference:
          "Diaz AA, Qin H, Ramalho-Santos M, Song JS HiTSelect: a comprehensive tool for high-complexity-pooled screen analysis. Nucleic Acids Res. 2015 Feb 18;43(3):e16. doi: 10.1093/nar/gku1197. Epub 2014 Nov 26. (PMID: 25428347)",
        Comments:
          "Performs functional enrichment (using DAVID), pathways identification and gene interaction study from CRISPR-CAs9 NGS experiments of samples in different conditions. Inputs are count tables and differential expression results (p-value and fold change).",
        __EMPTY: "https://sourceforge.net/projects/hitselect/",
      },
      {
        Availability: " ",
      },
      {
        Tools: "COSMID",
        Availability: " Yes",
        Purpose: "Post-analysis",
        Platform: "Web",
        "Off‑targets": " Yes",
        "Score Oligos": " No",
        "Search by": "Sequence",
        Enzyme: "Custom",
        PAM: "Custom",
        Organisms:
          "Homo sapiens GRCh38 (hg38)\r\nHomo sapiens GRCh37 (hg19)\r\nHomo sapiens NCBI36 (hg18)\r\nC elegans (ce10/WS220)\r\nMacaca mulatta Mmul_051212 (rheMac2)\r\nRattus norvegicus (rn6)\r\nDanio rerio Zebrafish (danRer10)\r\nSus scrofa Pig (susScr3)\r\nMus musculus GRCm38 (mm10)\r\nMus musculus NCBI37 (mm9)",
        Reference:
          "Cradick TJ, Qiu P, Lee CM, Fine EJ, Bao G COSMID: A Web-based Tool for Identifying and Validating CRISPR/Cas Off-target Sites. Mol Ther Nucleic Acids. 2014 Dec 2;3:e214. doi: 10.1038/mtna.2014.64. (PMID: 25462530)",
        Comments: "Tool for identifying and validating Off-target sites.",
        __EMPTY: "https://crispr.bme.gatech.edu/",
      },
      {
        Availability: " ",
      },
      {
        Tools: "AGEseq",
        Availability: " Yes",
        Purpose: "Post-analysis",
        Platform: "Command-line",
        "Off‑targets": " No",
        "Score Oligos": " No",
        "Search by": "Sequence",
        Enzyme: "Custom",
        PAM: "Custom",
        Organisms: "Provided by user",
        Reference:
          "Xue LJ, Tsai CJ AGEseq: Analysis of Genome Editing by Sequencing. Mol Plant. 2015 Sep;8(9):1428-30. doi: 10.1016/j.molp.2015.06.001. Epub 2015 Jun 6. (PMID: 26057235)",
        Comments:
          "PERL script available at https://github.com/liangjiaoxue/AGEseq. Galaxy server plugin (it uses port 8085)",
        __EMPTY: "https://www.aspendb.org/databases",
      },
      {
        Availability: " ",
        Platform: "Other",
        "Search by": "Other",
      },
      {
        Tools: "Synthego CRISPR Guide Design",
        Availability: " Yes",
        Purpose: "Oligo Designer",
        Platform: "Web",
        "Off‑targets": " Yes",
        "Score Oligos": " Yes",
        "Search by": "GeneID",
        Enzyme: "SpCas9 (S. pyogenes)",
        PAM: "NGG",
        Organisms: "Provided by user",
        Comments:
          "Commercial kit for the designed sgRNA can be purchased in the web site.",
        __EMPTY:
          "https://www.synthego.com/products/bioinformatics/crispr-design-tool",
      },
      {
        Availability: " ",
        Purpose: "Post-analysis",
      },
      {
        Tools: "GuideScan",
        Availability: " Yes",
        Purpose: "Post-analysis",
        Platform: "Web",
        "Off‑targets": " Yes",
        "Score Oligos": " Yes",
        "Search by": "GeneID",
        Enzyme: "SpCas9\r\nAsCpf1/LbCpf1",
        PAM: "NGG\r\nNAG\r\nTTTN",
        Organisms:
          "Homo sapiens (hg38)\r\nMus musculus (mm10)\r\nDrosphila melanogaster (dm6)\r\nSaccharomyces cerevisiae (SacCerv)\r\nDanio rerio (danRer10)\r\nCaenorhabditis elegans (ce11)",
        Reference:
          "Perez AR, Pritykin Y, Vidigal JA, Chhangawala S, Zamparo L, Leslie CS, Ventura A GuideScan software for improved single and paired CRISPR guide RNA design. Nat Biotechnol. 2017 Apr;35(4):347-349. doi: 10.1038/nbt.3804. Epub 2017 Mar 6. (PMID: 28263296)",
        Comments: "Allows mixed queries of coordinates and gene symbols.",
        __EMPTY: "https://guidescan.com/",
      },
      {
        Availability: " ",
        Platform: "Command-line",
        "Search by": "Coordinate",
      },
      {
        Tools: "MAGeCK-VISPR",
        Availability: " Yes",
        Purpose: "Post-analysis",
        Platform: "Command-line",
        "Off‑targets": "?",
        "Score Oligos": "?",
        "Search by": "Other",
        Enzyme: "not applicable",
        PAM: "Not Applicable",
        Organisms: "Not Applicable",
        Reference:
          "Li W, Koster J, Xu H, Chen CH, Xiao T, Liu JS, Brown M, Liu XS Quality control, modeling, and visualization of CRISPR screens with MAGeCK-VISPR. Genome Biol. 2015 Dec 16;16:281. doi: 10.1186/s13059-015-0843-6. (PMID: 26673418)",
        Comments:
          "Tool for quality control analysis and visualization of CRISPR/Cas9 screens.",
        __EMPTY: "https://bitbucket.org/liulab/mageck-vispr/src/master/",
      },
      {
        Availability: " ",
      },
      {
        Tools: "CAS-Analyzer",
        Availability: " Yes",
        Purpose: "Post-analysis",
        Platform: "Web",
        "Off‑targets": " No",
        "Score Oligos": " No",
        "Search by": "Sequence",
        Enzyme:
          "SpCas9 from Streptococcus pyogenes: 5'-NGG-3'\r\nStCas9 from Streptococcus thermophilus: 5'-NNAGAAW-3' (W = A or T)\r\nNmCas9 from Neisseria meningitidis: 5'-NNNNGMTT-3' (M = A or C)\r\nSaCas9 from Staphylococcus aureus: 5'-NNGRRT-'3 (R=A or G)\r\nCjCas9 from Campylobacter jejuni: 5'-NNNNACAC-3'\r\nAsCpf1 from Acidaminococcus or LbCpf1 from Lachnospiraceae: 5'-TTTN-3'\r\nxCas9 from Streptococcus pyogene 5'-NGT-3'\r\nFnCpf1 from francisella novicida: 5'-TTN-3'\r\nSpy-macCas9 from Streptococcus pyogenes and Streptococcus macacae: 5'-NAAN-3'\r\nZinc finger nucleases (ZFNs)\r\nTale effector nucleases (TALENs)\r\nCas9 Nickases\r\nCRISPR RNA guided FokI nucleases (dCas9-FokI)",
        PAM: "NGG\r\nNNAGAAW\r\nNNNNGMTT\r\nNNGRRT\r\nNNNNACAC\r\nTTTN\r\nNGT",
        Organisms: "Provided by user",
        Reference:
          "Park J, Lim K, Kim JS, Bae S Cas-analyzer: an online tool for assessing genome editing results using NGS data. Bioinformatics. 2017 Jan 15;33(2):286-288. doi: 10.1093/bioinformatics/btw561. Epub 2016 Aug 24. (PMID: 27559154)",
        Comments:
          "Alignment and analysis of NGS reads against a region reference sequence",
        __EMPTY: "http://www.rgenome.net/cas-analyzer/#!",
      },
      {
        Availability: " ",
      },
      {
        Tools: "CRISPR/Cas9 sgRNA Design",
        Availability: " Yes",
        Purpose: "Oligo Designer",
        Platform: "Command-line",
        "Off‑targets": " Yes",
        "Score Oligos": " No",
        "Search by": "GeneID",
        Enzyme: "custom",
        PAM: "Custom",
        Organisms: "Provided by user",
        Reference:
          "Brocal I, White RJ, Dooley CM, Carruthers SN, Clark R, Hall A, Busch-Nentwich EM, Stemple DL, Kettleborough RN Efficient identification of CRISPR/Cas9-induced insertions/deletions by direct germline screening in zebrafish. BMC Genomics. 2016 Mar 24;17:259. doi: 10.1186/s12864-016-2563-z. (PMID: 27009152)",
        Comments:
          "Sites can be filtered based on known SNP data. Allows designing PCR primers for screening and post-analysis of MiSeq sequencing data. Oligos are scored for genomic location: exon-oligos score highest, intron oligos lower, intergenic lowest. Oligo scoring for targeting efficiency is not performed.",
        __EMPTY: "https://github.com/richysix/Crispr",
      },
      {
        Availability: " ",
        Purpose: "Post-analysis",
        "Search by": "Coordinate",
      },
      {
        Tools: "GUIDEseq",
        Availability: " Yes",
        Purpose: "Post-analysis",
        Platform: "Command-line",
        "Off‑targets": " Yes",
        "Score Oligos": " Yes",
        "Search by": "Other",
        Enzyme: "Custom",
        PAM: "Custom",
        Organisms: "Provided by user",
        Reference:
          "Zhu LJ, Lawrence M, Gupta A, Pages H, Kucukural A, Garber M, Wolfe SA GUIDEseq: a bioconductor package to analyze GUIDE-Seq datasets for CRISPR-Cas nucleases. BMC Genomics. 2017 May 15;18(1):379. doi: 10.1186/s12864-017-3746-y. (PMID: 28506212)",
        Comments:
          "R library for processing GUIDE-seq data. Organism availability depends on BSgenome R package.",
        __EMPTY:
          "https://bioconductor.org/packages/release/bioc/html/GUIDEseq.html",
      },
      {
        Availability: " ",
      },
      {
        Tools: "guideseq",
        Availability: " Yes",
        Purpose: "Post-analysis",
        Platform: "Command-line",
        "Off‑targets": " Yes",
        "Score Oligos": "?",
        "Search by": "Sequence",
        PAM: "Custom",
        Organisms: "Provided by user",
        Reference:
          "Tsai SQ, Topkar VV, Joung JK, Aryee MJ Open-source guideseq software for analysis of GUIDE-seq data. Nat Biotechnol. 2016 May 6;34(5):483. doi: 10.1038/nbt.3534. (PMID: 27153277)",
        Comments: "Python library for processing GUIDE-seq data",
        __EMPTY: "https://github.com/aryeelab/guideseq",
      },
      {
        Availability: " ",
      },
      {
        Tools: "OutKnocker",
        Availability: " Yes",
        Purpose: "Post-analysis",
        Platform: "Web",
        "Off‑targets": " No",
        "Score Oligos": " No",
        "Search by": "Other",
        Enzyme: "Not applicable",
        PAM: "Not Applicable",
        Organisms: "Homo sapiens , Mus musculus",
        Reference:
          "Schmid-Burgk JL, Schmidt T, Gaidt MM, Pelka K, Latz E, Ebert TS, Hornung V OutKnocker: a web tool for rapid and simple genotyping of designer nuclease edited cell lines. Genome Res. 2014 Oct;24(10):1719-23. doi: 10.1101/gr.176701.114. Epub 2014 Sep 3. (PMID: 25186908)",
        Comments:
          "Easy-to-follow deep sequencing workflow which allows convenient, reliable and cost-effective identification of knock-out cell lines",
      },
      {
        Availability: "(2022-11-22)",
      },
      {
        Tools: "MaGeCKFlute",
        Availability: " Yes",
        Purpose: "Post-analysis",
        Platform: "Command-line",
        "Off‑targets": " No",
        "Score Oligos": " No",
        "Search by": "Other",
        Enzyme: "Not applicable",
        PAM: "Not Applicable",
        Organisms: "Not Applicable",
        Reference:
          "Wang B, Wang M, Zhang W, Xiao T, Chen CH, Wu A, Wu F, Traugh N, Wang X, Li Z, Mei S, Cui Y, Shi S, Lipp JJ, Hinterndorfer M, Zuber J, Brown M, Li W, Liu XS Integrative analysis of pooled CRISPR genetic screens using MAGeCKFlute. Nat Protoc. 2019 Mar;14(3):756-780. doi: 10.1038/s41596-018-0113-7. Epub 2019 Feb 1. (PMID: 30710114)",
        Comments:
          "R package to perform quality control (QC), normalization, batch effect removal, gene hit identification and downstream functional enrichment analysis for CRISPR screens.",
        __EMPTY: "https://github.com/WubingZhang/MAGeCKFlute",
      },
      {
        Availability: "(2022-11-22)",
      },
      {
        Tools: "predictSGRNA",
        Availability: " Yes",
        Purpose: "analysis of sgRNA",
        Platform: "Command-line",
        "Off‑targets": " No",
        "Score Oligos": " Yes",
        "Search by": "Other",
        Enzyme: "Not Applicable",
        PAM: "Not Applicable",
        Organisms: "Not Applicable",
        Reference:
          "Kuan PF, Powers S, He S, Li K, Zhao X, Huang B A systematic evaluation of nucleotide properties for CRISPR sgRNA design. BMC Bioinformatics. 2017 Jun 6;18(1):297. doi: 10.1186/s12859-017-1697-6. (PMID: 28587596)",
        Comments:
          "Machine learning approach for the prediction of sgRNA design efficiency",
      },
      {
        Availability: " ",
      },
      {
        Tools: "OffScan",
        Availability: " Yes",
        Purpose: "Oligo Designer",
        Platform: "Command-line",
        "Off‑targets": " Yes",
        "Score Oligos": " Yes",
        "Search by": "Other",
        Enzyme: "Not Applicable",
        PAM: "Not Applicable",
        Organisms: "Not Applicable",
        Reference:
          "Cui Y, Liao X, Peng S, Tang T, Huang C, Yang C OffScan: a universal and fast CRISPR off-target sites detection tool. BMC Genomics. 2020 Mar 5;21(Suppl 1):872. doi: 10.1186/s12864-019-6241-9. (PMID: 32138651)",
        Comments:
          "OffScan is not limited by the number of mismatches and allows custom protospacer-adjacent motif (PAM), which is the target site by Cas protein. Besides, OffScan adopts the FM-index, which efficiently improves query speed and reduce memory consumption.",
      },
      {
        Availability: " ",
      },
    ],
  },
];
